---
layout: post
title: FungiDB 29 Released
category: [EuPathDB,VEuPathDB,FungiDB]
---

<p><strong>We are pleased to announce the release of FungiDB 29</strong></p>
    
        <section>
    <h3>FungiDB 29</h3>
           FungiDB 29 represents extensive efforts to improve navigation and data accessibility 
           on record pages, provide consistent verbiage and data categories, and improve presentation of alternative transcripts. 
           All data were refreshed to the latest version and genome analyses were rerun.
           <ul>
             <li>Here is some help information to get you started with our new features.<br>
                 <a href="http://fungidb.org/documents/genePageTutorialGUS4.pdf">Gene Page Improvements</a> <br>
                 <a href="http://fungidb.org/documents/SearchCatsTutorial_3.pdf">Search Category Improvements</a> <br>
                 <a href="http://fungidb.org/documents/ExploreTranscripts.pdf">Explore Transcripts in Search Results</a><br>
                 <!--Workspaces<br>-->
             </li><br>
             <li>Send Us Your Feedback!<br>We can polish our new features based on your valuable feedback. Please use the site 
             and <a href="http://fungidb.org/fungidb/app/contact-us">Contact Us</a> to tell us about your experience. 
              </li><br>
             <li>FungiDB 28 is Still Available - <br><a href="http://r28.fungidb.org/fungidb/">FungiDB 28</a> is available until we finalize our new features.</li><br>
           </ul>
     </section>
      
     <section> 
      <h3>New Features</h3>
         
           <ul>
             <li>Data Categories - <br>Searches, data and GBrowse tracks are now categorized according to the 
             <a href="http://www.ontobee.org/ontology/EDAM?iri=http://edamontology.org/topic_3070">EMBRACE Data & Methods Ontology (EDAM)</a>. 
             This offers a controlled vocabulary and consistent data structure throughout the site. 
             </li><br>
             <li>Record Pages Redesigned - <br>Gene pages, and in-fact all our record pages, have a new structure. 
             Features include a new streamlined and uncluttered look, new navigation tools, and improved 
             tables with search functions for quickly finding and displaying the data you want. 
             </li><br>
             <li>Transcripts Represented on Gene Pages and Search Results - <br>Transcripts are graphically represented on 
             gene pages and listed in gene page tables when data can be 
             unambiguously assigned to a transcript. All search results now 
             contain a Transcript ID column.  The results of searches that query transcript-specific 
             data (e.g. Predicted Signal Peptide) contain an Explore function for investigating transcript subsets.
             </li><br>
             <li>Galaxy Workspace - <br> EuPathDB sites now include a Galaxy-based workspace for large-scale data analyses. 
             Developed in partnership with <a href="https://www.globus.org/genomics">Globus Genomics</a>, 
             workspaces offer a private analysis platform with published workflows and pre-loaded annotated genomes. 
             The workspace is accessed through the "Analyze My Experiment" tab on the home page of any EuPathDB resource 
             and can be used to upload your own data, compose and run custom workflows, retrieve results and share 
             workflows and data analyses with colleagues.
             </li><br>
             <li>Download Search Result Redesigned - <br>The page for downloading search results and associated data has 
             an improved design that reflects the new ontological data categories and offers the ability to download 
             all transcripts per gene when downloading gene results.
             </li><br>
             <li>RNA Sequencing Workflow - We updated our RNA sequencing analysis. The new workflow aligns 
             reads with GSNAP, calculates FPKM/RPKM from HT-Seq counts and determines differential expression with DESeq2.
             </li>
           </ul>
        </section>  
        
      <section>
      <h3>New data in this release</h3>
      <div style="margin-left: 1em;">
        <h4>Updated Phenotype</h4>
        <ul>
          <li>
            <a href="http://fungidb.org/fungidb/app/record/dataset/DS_7cdba4f78a">Manually Curated Phenotypes </a> &mdash; 
            New phenotypes added to Manually Curated Phenotypes for <i>Aspergillus</i> species. See table below for details.  
          </li>
        </ul>  
          <br><br>
       <div style="margin-left: 1em;">         
        <style>
           #Phenotypes_Bld29 table, #Phenotypes_Bld29 td, #Phenotypes_Bld29 th, #Phenotypes_Bld29 tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #Phenotypes_Bld29 {
           margin-left : 5 em;
           }
         </style> 
           <table id="Phenotypes_Bld29"> 
           <tr>
             <th>Species</th>
             <th>Phenotype mappings</th>
             <th>Genes with phenotypes</th>
           </tr>
           <tr>
             <td> <i>Aspergillus fumigatus</i> Af293</td>
             <td> 599</td>
             <td>271</td>
           </tr>
           <tr>
             <td> <i>Aspergillus nidulans</i> FGSC A4</td>
             <td> 2356</td>
             <td> 1008</td>
           </tr>
           <tr>
             <td> <i>Aspergillus niger</i> CBS 513.88</td>
             <td> 135</td>
             <td> 86</td>
           </tr>
           <tr>
             <td> <i>Aspergillus oryzae</i> RIB40</td>
             <td> 96</td>
             <td> 63</td>
           </tr>
           </table>
        <br>
      </section>      
    </div>

    <div style="margin-left: 1em;"> 
      <h4>Annotation is updated for the following genomes:</h4>
         <ul>
          <li><i>Ajellomyces capsulatus</i> G186AR</li>
          <li><i>Ajellomyces capsulatus</i> NAm1</li>
          <li><i>Albugo laibachii</i> Nc14</li>
          <li><i>Allomyces macrogynus</i> ATCC 38327</li>
          <li><i>Allomyces macrogynus</i> ATCC 38327</li>
          <li><i>Aphanomyces astaci</i> strain APO3</li>
          <li><i>Aphanomyces invadans</i> NJM9701</li>
          <li><i>Aspergillus clavatus</i> NRRL 1</li>
          <li><i>Aspergillus flavus</i> NRRL3357</li>
          <li><i>Aspergillus fumigatus</i> Af293</li>
          <li><i>Aspergillus nidulans</i> FGSC A4</li>
          <li><i>Aspergillus niger</i> ATCC 1015</li>
          <li><i>Aspergillus niger</i> CBS 513.88</li>
          <li><i>Aspergillus oryzae</i> RIB40</li>
          <li><i>Aspergillus terreus</i> NIH2624</li>
          <li><i>Botrytis cinerea</i> B05.10</li>
          <li><i>Candida albicans</i> SC5314</li>
          <li><i>Candida glabrata</i> CBS 138</li>
          <li><i>Coprinopsis cinerea</i> okayama7#130</li>
          <li><i>Cryptococcus gattii</i> R265</li>
          <li><i>Cryptococcus gattii</i> WM276</li>
          <li><i>Cryptococcus neoformans</i> var. grubii H99</li>
          <li><i>Cryptococcus neoformans</i> var. neoformans B-3501A</li>
          <li><i>Cryptococcus neoformans</i> var. neoformans JEC21</li>
          <li><i>Fusarium graminearum</i> PH-1</li>
          <li><i>Fusarium oxysporum f. sp. lycopersici</i> 4287</li>
          <li><i>Fusarium verticillioides</i> 7600</li>
          <li><i>Hyaloperonospora arabidopsidis</i> Emoy2</li>
          <li><i>Magnaporthe oryzae</i> 70-15</li>
          <li><i>Malassezia globosa</i> CBS 7966</li>
          <li><i>Melampsora larici-populina</i> 98AG31</li>
          <li><i>Neosartorya fischeri</i> NRRL 181</li>
          <li><i>Neurospora crassa</i> OR74A</li>
          <li><i>Neurospora tetrasperma</i> FGSC 2508</li>
          <li><i>Phanerochaete chrysosporium</i> RP-78</li>
          <li><i>Phytophthora infestans</i> T30-4</li>
          <li><i>Phytophthora parasitica</i> INRA-310</li>
          <li><i>Phytophthora sojae strain</i> P6497</li>
          <li><i>Pneumocystis jirovecii</i> SE8</li>
          <li><i>Puccinia graminis f. sp. tritici</i> CRL 75-36-700-3</li>
          <li><i>Pythium aphanidermatum</i> DAOM BR444</li>
          <li><i>Pythium arrhenomanes</i> ATCC 12531</li>
          <li><i>Pythium irregulare</i> DAOM BR486</li>
          <li><i>Pythium iwayamai</i> DAOM BR242034</li>
          <li><i>Pythium ultimum</i> DAOM BR144</li>
          <li><i>Pythium ultimum</i> var. sporangiiferum BR650</li>
          <li><i>Rhizopus delemar</i> RA 99-880</li>
          <li><i>Saprolegnia diclina</i> VS20</li>
          <li><i>Saprolegnia parasitica</i> CBS 223.65</li>
          <li><i>Schizosaccharomyces japonicus</i> yFS275</li>
          <li><i>Schizosaccharomyces octosporus</i> yFS286</li>
          <li><i>Schizosaccharomyces pombe</i> 972h-</li>
          <li><i>Sclerotinia sclerotiorum</i> 1980 UF-70</li>
          <li><i>Sordaria macrospora</i> k-hell</li>
          <li><i>Spizellomyces punctatus</i> DAOM BR117</li>
          <li><i>Sporisorium reilianum</i> SRZ2</li>
          <li><i>Talaromyces marneffei</i> ATCC 18224</li>
          <li><i>Talaromyces stipitatus </i>ATCC 10500</li>
          <li><i>Tremella mesenterica</i> DSM 1558</li>
          <li><i>Trichoderma reesei</i> QM6a</li>
          <li><i>Yarrowia lipolytica</i> CLIB122</li>
          </ul>
      </div>    
      </div>
      <aside>
        Note: Our bimonthly database releases incorporate new data and correct old data when necessary.
        Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
        When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
        When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
      </aside>

    </section>
