---
layout: post
title: FungiDB 41 Released
category: [FungiDB]
---

<p><strong>We are pleased to announce the release of FungiDB 41</strong></p>
      
      <section>
        <h3><b>New data in FungiDB 41</b></h3>
        <div style="margin-left: 3em;">
        
        <h4>Genome Sequence and Annotation: these annotated genomes appear for the first time in FungiDB.</h4>
           <ul>
             <li><a href="/a/app/record/dataset/DS_d6438671bd"><i>Aspergillus niger</i> N402</a> &ndash;  
                 <a href="https://www.ncbi.nlm.nih.gov/pubmed/3224384">Bos et al. 1988</a>.
             </li>
             
              <li><a href="/a/app/record/dataset/DS_ba79cc3de1"><i>Aspergillus kawachii</i> IFO 4308</a> &ndash;  
                 <a href="https://www.ncbi.nlm.nih.gov/pubmed/22045919">Futagami et al. 2011</a>.
             </li>
             
              <li><a href="/a/app/record/dataset/DS_d1767fcbd3"><i>Candida parapsilosis</i> CDC 317</a> &ndash;  
                 <a href="https://www.ncbi.nlm.nih.gov/pubmed/19465905">Butler et al. 2009</a>.
             </li>
             
             <li><a href="/a/app/record/dataset/DS_8d24f658c0"><i>Cryptococcus gattii</i> NT-10</a> &ndash;  
                 <a href="http://www.ncbi.nlm.nih.gov/bioproject/PRJNA191330">NCBI BioProject PRJNA191330</a>.
             </li>
             
             <li><a href="/a/app/record/dataset/DS_32a5330fef"><i>Fusarium proliferatum</i> ET1</a> &ndash; 
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/28040774">Niehaus et al. 2016</a>.
             </li>
             
               <li><a href="/a/app/record/dataset/DS_a62f2e98ce"><i>Lomentospora prolificans</i> JHH-5317</a> &ndash;  
                 <a href="https://www.ncbi.nlm.nih.gov/pubmed/28963165">Luo et al. 2017</a>.
             </li> 
             
             <li><a href="/a/app/record/dataset/DS_1566006e1e"><i>Scedosporium apiospermum</i> IHEM 14462</a> &ndash;  
                 <a href="https://www.ncbi.nlm.nih.gov/pubmed/25278533">Vandeputte et al. 2014</a>.
             </li>
           
             <li><a href="/a/app/record/dataset/DS_3d23d8f8c0"><i>Sporothrix brasiliensis</i> 5110</a> &ndash;  
                 <a href="https://www.ncbi.nlm.nih.gov/pubmed/25351875">Teixeira et al. 2014</a>.
             </li>
             
            <li><a href="/a/app/record/dataset/DS_6048ff9e23"><i>Yarrowia lipolytica</i> CLIB89 (W29)</a> &ndash;  
                 <a href="https://www.ncbi.nlm.nih.gov/pubmed/27603307">Magnan et al. 2016</a>.
             </li>   
            </ul>
           <br>
           
        <h4>Genome Sequence and Annotation: updated genomes</h4> 
         	<ul>
        	<li><a href="/a/app/record/dataset/DS_f5f72ac794"><i>Aspergillus niger</i> ATCC 1015</a> &ndash;  
                 <a href="https://www.ncbi.nlm.nih.gov/pubmed/21543515">Andersen et al. 2011</a>.
             </li> 
             
             </li>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_f670f03393"><i>Cryptococcus neoformans</i> var. grubii KN99</a> &ndash; 
                 Gene mapping and functional annotation corrections: (1) old_locus_tag was mapped to new_locus_tag; (2) new_locus_tag is available in the <i>Names, 
                 Previous Identifiers, and Aliases</i> section of the gene record page and should be used as the most recent and correct gene identifier; 
                 (3) functional annotation (product description) was updated using new_locus_tag
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/29305388">Friedman et al. 2018</a>.
             </li>  
            </ul>
           <br> 
           
        <h4>Transcriptomics</h4>
           <ul>
             <li><a href="/a/app/record/dataset/DS_cc8da1fe9f">Comparison of gene expression in 23 <i>Cryptococcus gattii</i> isolates</a> &ndash; 
                 23 <i>C. gattii</i> isolates, representing both VIO strains and control strains, were grown for 24 hours in mammalian macrophages. 
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/19651610" target="_blank">Ma et al. 2009</a>.
             </li>
             
             <li><a href="/a/app/record/dataset/DS_56f8fb561f">Transcriptomic response of <i>Aspergillus niger</i> to lignocellulose (willow)</a> &ndash;
             Transcriptomic profiling of <i>Aspergillus niger</i> growing on lignocellulose (AB SOLiD System 3.0). 
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/22912594" target="_blank">Delmas et al. 2012</a>.
             </li>
             
             <li><a href="/a/app/record/dataset/DS_7270577827">Transcriptome of <i>N. crassa</i> strains growing on various carbon sources</a> &ndash;
             Transcriptomic profiling to determine the role of the transcription factors CLR-1, CLR-2, and XLR-1 in plant cell wall degradation
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/26463163" target="_blank">Craig et al. 2015</a>.
             </li>
             
             <li><a href="/a/app/record/dataset/DS_3fbf685d63">Transcriptomics of banana pathogen <i>Fusarium oxysporum</i> f.sp. cubense race 1</a> &ndash;
             Deep sequencing of mRNA from <i>Fusarium oxysporum</i> f. sp. cubense 1 after infecting <i>Musa acuminata</i> at 0h and 48h.
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/24743270" target="_blank">Guo et al. 2014</a>.
             </li>
        	</ul>
        <br>
        	 
        <h4>Proteomics</h4>
           <ul>
             <li><a href="/a/app/record/dataset/DS_9efaab4256">Protein composition of <i>C neoformans</i> yeast and spore forms</i></a> &ndash; 
                 Nanoscale liquid chromatography/mass spectrometry was used to determine protein composition of <i>Cryptococcus</i> vegetative cells (yeast) and spores.
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/26313153" target="_blank">Huang et al. 2015</a>.
             </li>
        	</ul>
        <br>
           
           <h4>Chip-Seq data</h4>
           <ul>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_17f48c08a1">ChiP-Seq of CLR-1, CLR-2, and XLR-1 knockout mutants and wild type strains growing on various carbon sources</a> &ndash;
             The plant cell wall is composed of many complex polymers, and its deconstruction requires an equally complex orchestration of a wide array of enzymes. 
             In <i>Neurospora crassa</i>, <i>clr-1</i>, <i>clr-2</i> and <i>xlr-1</i> have been identified as the key transcription factors involved in cell wall breakdown. In order to define their regulons, we performed ChIPseq upon these three transcription factors.  
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/26463163" target="_blank">Craig et al. 2015</a>.
             </li>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_e030fc9abf">ChiP-Seq of histone modifications in <i>F. graminearum</i> strains growing on media with high or low nitrogen levels</a> &ndash;
             ChIP-seq of histone modifications in WT and kmt6 mutant of <i>Fusarium graminearum</i> grown in high and low nitrogen conditions. 
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/24204317" target="_blank">Connolly et al. 2013</a>.
             </li>
           </ul>  
            <br>
           
        <h4>Whole genome sequence of isolates for SNP analysis</h4>
           <ul>
             <li><a href="/a/app/record/dataset/DS_cc66146b8b">Aligned genomic sequence reads - <i>Cryptococcus neoformans</i> var. neoformans</a> &ndash;
                 A total of 387 isolates from various lineages we sequenced (185 from global VNI lineage, 186 from African VNB, and 16 from VNII lineage corresponding to non-recombining split in VNB (VNBI and VNBII).
                 <a href="https://www.ncbi.nlm.nih.gov/pubmed/28611159"> Desjardins et al. 2017</a>.
             </li>
             
             <li><a href="/a/app/record/dataset/DS_59d1c30302">Aligned genomic sequence reads - <i>Neurospora crassa</i></a> &ndash;
                 Genome-wide association study (GWAS) to identify novel loci underlying trait variation in the wild isolates of the filamentous fungus <i>Neurospora crassa</i> (Louisiana lineage).
                 <a href="https://www.ncbi.nlm.nih.gov/pubmed/26025978"> Zhao et al. 2015</a> and
                 <a href="https://www.ncbi.nlm.nih.gov/pubmed/23935534"> Palma-Guerrero et al. 2013</a>.
             </li>
             
             <li><a href="/a/app/record/dataset/DS_edc6f8f95f">Aligned genomic sequence reads - <i>Paracoccidioides brasiliensis</i></a> &ndash;
                 Genome diversity, recombination, and virulence across the major lineages of <i>Paracoccidioides</i>.
                 <a href="https://www.ncbi.nlm.nih.gov/pubmed/27704050"> Mu&ntildeoz et al. 2016</a> .
             </li>
             
             <li><a href="/a/app/record/dataset/DS_4db802da0d">Aligned genomic sequence reads - <i>Candida albicans</i></a> &ndash;
                 Whole genome sequencing of wild-type and clinical isolates as well as laboratory-evolved strains of <i>Candida albicans</i> to identify and characterize mutations conferring amphotericin resistance.
                 <a href="https://www.ncbi.nlm.nih.gov/pubmed/24204207"> Vincent et al. 2013</a> .
             </li>
           </ul>
            <br>
           
        <h4>Phenotype data: updated</h4>
           <ul>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_c334877c42">Phenotypic analysis of <i>Neurospora crassa</i> knockout mutants</a> &ndash; A phenotypic pipeline was designed for high&#45;throughput analysis of 10 different traits during hyphal growth, asexual sporulation (macroconidiation), and the sexual cycle in knockout mutants from <i>Neurospora crassa</i>. Provider: Katherine Borkovich; dataset provided prior to publication.
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/17352902" target="_blank">Borkovich et al. 2007</a>.
             </li>
             </ul>
           <br>
           
           <h4>Link Outs</h4>
           Link outs to the <i>Cryptococcus neoformans</i> kinase phenome and <i>Cryptococcus neoformans</i> transcription factor databases were 
           created on individual gene record pages for <i>Cryptococcus neoformans</i>. Also, NCBI RefSeq, MEROPS, and CAZy link outs were loaded for 
           <i>Neurospora crassa</i> OR74A , <i>Cryptococcus neoformans</i> JEC21 and H99, and NCBI RefSeq and MEROPS link outs for <i>Coccidioides immitis</i> RS. 
           <br>
           
 <h4>Updated Functional Annotation</h4>
        <div style="margin-left: 3em;">
        
            
        <a href="https://docs.google.com/spreadsheets/d/19AdZL1ZYWTAe3eV_McbXsfl2tVVrRf8NMskRf_knIkg/edit?usp=sharing" target="_blank">Summary of annotation updates</a> which include products, GO terms, EC numbers, gene names, synonyms, publications and notes from curators.
        <br><br>
        Functional annotations have been updated for the following genomes:

        <style>
           #FunctAnnot_Bld41 table, #FunctAnnot_Bld41 td, #FunctAnnot_Bld41 th, #FunctAnnot_Bld41 tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #FunctAnnot_Bld41 {
           margin-left : 5 em;
           }
         </style> 
           <table id="FunctAnnot_Bld41"> 
           <tr>
             <th>FungiDB Genome Version</th>
             <th>Genes Updated</th>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_60b6b264d3"><i>Coccidioides immitis</i> RS </a></td>
             <td>9756 </td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_8f0322af73"><i>Cryptococcus neoformans</i> H99 </a></td>
             <td>6974</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_4d171eb9ea"><i>Cryptococcus neoformans</i> JEC21 </a></td>
             <td>6632</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_62f0e6170f"><i>Neurospora crassa</i> OR74A</a></td>
             <td>9758</td>
           </tr>
           </table>
           <br>
       </div>
 
       <h4>Curated User Comments:</h4>  
        Gene page comments that are entered by users are curated and integrated as functional annotations in FungiDB. 139 user comments have been reviewed and updated in FungiDB 41.
       <br><br>
        
        <div style="margin-left: 3em;">
        
        <style>
           #UComment_Bld41 table, #UComment_Bld41 td, #UComment_Bld41 th, #UComment_Bld41 tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #UComment_Bld41 {
           margin-left : 5 em;
           }
         </style> 
           <table id="UComment_Bld41"> 
           <tr>
             <th>FungiDB Genome Version</th>
             <th>Comments Integrated</th>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_62f0e6170f"><i>Neurospora crassa </i> OR74A</a></td>
             <td>139</td>
           </tr>
          </table>
          <br>
          </div> 
      
      <section>
      <h3><b>New features in FungiDB 41</b></h3>
      <h4>Introducing Clustal Omega </h4><br>
      <div style="margin-left: 3em;">
      EuPathDB sites are now using Clustal Omega for gene page protein alignments in the 'Orthologs and Paralogs within EuPathDB' table and the 
      'Retrieve multiple sequence alignment or multi-FASTA' tool in the 'Orthology and synteny' section of gene pages. The 'Isolate Alignments in this Gene Region' tool 
      still uses CulstalW.  
       </div> 
      <h4>New Gene page tools </h4>
         <ul>
           <li>Interpro  &mdash; <br>
              The 'Protein features and properties' section now includes an option to submit the protein sequence for 
              sequence analysis and classification via <a href="https://www.ebi.ac.uk/interpro/" target="_blank">InterPro</a>. 
          </li>
          <li>GPI anchor prediction  &mdash; <br>
              The 'Protein features and properties' section now includes an option to submit the protein sequence for 
              GPI modification site prediction via the <a href="http://mendel.imp.ac.at/sat/gpi/gpi_server.html" target="_blank">GPI Lipid Anchor Project</a>. 
          </li>
         </ul>
      </section>
       

      
     <section>
     <aside>
           <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary.
           Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
           When search parameters change with a new release, we invalidate (Ø) the search and ask you to rerun it.
           When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
         </aside>
      </section>
