---
layout: post
title: FungiDB 37 Released
category: [FungiDB]
---

<p><strong>We are pleased to announce the release of FungiDB 37</strong></p>
      
      <section>
        <h3><b>New data in FungiDB 37</b></h3>
        <div style="margin-left: 3em;">
        
                
        <h4>Genome Sequence and Annotation: these annotated genomes appear for the first time in FungiDB.</h4>
           <ul>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_339f68fa2f"><i>Aspergillus campestris</i> IBT 28561</a> &ndash; 
                 <a href="http://www.ncbi.nlm.nih.gov/bioproject/PRJNA217206">NCBI BioProject PRJNA217206</a>.
             </li>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_ed02a8e11b"><i>Aspergillus novofumigatus</i> IBT 16806</a> &ndash; 
                 <a href="http://www.ncbi.nlm.nih.gov/bioproject/PRJNA217212">NCBI BioProject PRJNA217212</a>.
             </li>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_93f3ae0e82"><i>Aspergillus steynii</i> IBT 23096</a> &ndash; 
                 <a href="http://www.ncbi.nlm.nih.gov/bioproject/PRJNA217208">NCBI BioProject PRJNA217208</a>.
             </li>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_077dbc942e"><i> Candida auris</i> strain B8441</a> &ndash;  
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/27988485">Lockhart et al.</a>
             </li>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_f670f03393"><i>Cryptococcus neoformans</i> var. grubii KN99</a> &ndash; 
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/29305388">Friedman et al.</a>
             </li>
             
           </ul>
           <br>
           
           <h4>Sequence sites, features and motifs</h4>
           <ul>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_1b1cb2f10a">Small RNA sequence profiles during sexual development in <i>Neurospora crassa</i></a> &ndash; Endogenous small RNA profiles from <i>N. crassa</i> protoperithecia and perithecia at 2 and 4 days postfertilization. 
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/26109355" target="_blank">Wang et al. 2015</a>
             </li>
                          
           </ul>
           <br>
           
        <h4>Transcriptomics</h4>
           <ul>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_31af39c3b7">Comparative transcriptomes of yeast and hyphal forms in <i>Histoplasma capsulatum</i> G217B</a> &ndash; 
                 To compare yeast and hyphal transcript expression patterns cultures were grown at 37C for yeast-phase growth and at room temperature (RT) for hyphal phase growth. Total RNA was then isolated from biological duplicates of yeast or hyphal cultures and strand specific cDNA libraries were created from poly(A)-enriched RNA and sequenced using paired-end deep-sequencing. 
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/26177267" target="_blank">Gilmore et al. 2015</a>
             </li>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_be15e27935">Comparative transcriptomes of yeast and hyphal forms in <i>Histoplasma capsulatum</i> G186AR</a> &ndash; 
                 To compare yeast and hyphal transcript expression patterns cultures were grown at 37C for yeast-phase growth and at room temperature (RT) for hyphal phase growth. Total RNA was then isolated from biological duplicates of yeast or hyphal cultures and strand specific cDNA libraries were created from poly(A)-enriched RNA and sequenced using paired-end deep-sequencing.
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/26177267" target="_blank">Gilmore et al. 2015</a>
             </li>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_0a6c43682e">Comparative transcriptomes of yeast and hyphal forms in <i>Histoplasma capsulatum</i> H88</a> &ndash; 
                 To compare yeast and hyphal transcript expression patterns cultures were grown at 37C for yeast-phase growth and at room temperature (RT) for hyphal phase growth. Total RNA was then isolated from biological duplicates of yeast or hyphal cultures and strand specific cDNA libraries were created from poly(A)-enriched RNA and sequenced using paired-end deep-sequencing.
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/26177267" target="_blank">Gilmore et al. 2015</a>
             </li>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_d2ff7c7826">Multidimensional regulation in transcriptome during sexual development in <i>Neurospora crassa</i></a> &ndash; 
                 Genome-wide analysis of <i>N. crassa</i> gene expression during sexual development. Transcriptomic sequencing was primed by random and multitargeted primers and accessed across eight time points encompassing the morphological changes and tissue and cellular development of maturing perithecia. 
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/24243796" target="_blank">Wang et al. 2014</a>, 
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/22626843" target="_blank">Wang et al. 2012</a>, and 
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/25329823" target="_blank">Lehr et al. 2014</a>
             </li>
             
           </ul>
           <br>
           
       <h4>Phenotype data</h4>
           <ul>
             <li><a href="http://fungidb.org/fungidb/app/record/dataset/DS_f10e32c6b8">Neurospora Genome Project Phenotype Image Collection</a> &ndash; 
                 New images were added to this data set describing the systematic characterisation of colony growth and morphology of <i>Neurospora crassa</i> knockout mutants. 
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/17352902" target="_blank">Dunlap et al. 2007</a>,
		 <a href="http://www.ncbi.nlm.nih.gov/pubmed/16801547" target="_blank">Colot et al. 2006</a>, and
		 <a href="http://www.ncbi.nlm.nih.gov/pubmed/21590422" target="_blank">Turner G.E. 2011</a>
             </li>
           </ul>
           <br>
           
        <h4>Updated Functional Annotation</h4>
        <div style="margin-left: 3em;">
        
            
        <a href="https://docs.google.com/spreadsheets/d/1c_CFbU94QclOLVXS6kZOncOrzDNJIagrW8mwioO67JA/edit?usp=sharing" target="_blank">Summary of annotation updates</a> which include products, GO terms, EC numbers, gene names, synonyms, publications and notes from curators.
        <br><br>
        Functional annotations in FungiDB 37 have been updated for the following genomes:

        <style>
           #FunctAnnot_Bld37 table, #FunctAnnot_Bld37 td, #FunctAnnot_Bld37 th, #FunctAnnot_Bld37 tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #FunctAnnot_Bld37 {
           margin-left : 5 em;
           }
         </style> 
           <table id="FunctAnnot_Bld37"> 
           <tr>
             <th>FungiDB Genome Version</th>
             <th>Genes Updated</th>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_8f0322af73"><i>Cryptococcus neoformans</i> H99</a></td>
             <td>452 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_cd90ec7602"><i>Fusarium graminearum</i> PH-1</a></td>
             <td>4243 genes</td>
           </tr>
           </table>
           <br>
       
       </div>
       

        <br>
       <h4>Curated User Comments:</h4>  
        Gene page comments that are entered by users are curated at GeneDB and regularly integrated as 
        functional annotations in FungiDB.  59 user comments (unique) have been reviewed and updated in FungiDB 37.
       <br><br>
        
        <div style="margin-left: 3em;">
        
        <style>
           #UComment_Bld37 table, #UComment_Bld37 td, #UComment_Bld37 th, #UComment_Bld37 tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #UComment_Bld37 {
           margin-left : 5 em;
           }
         </style> 
           <table id="UComment_Bld37"> 
           <tr>
             <th>FungiDB Genome Version</th>
             <th>Comments Integrated</th>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_8f0322af73"><i>Cryptococcus neoformans</i> H99</a></td>
             <td> 59</td>
           </tr>
          </table>
          <br>
          </div> 
          
          
        <h4>Annotations from MIPS Functional Catalogue Database:</h4>  
            <ul> 
             <li>Gene ontology (GO) annotations for <i>Fusarium graminearum</i> PH-1 from 
             MIPS database were integrated as functional annotations in FungiDB 37.
             </li>
           </ul>  
      </div>    
      </section>
      
      <section>
      <h3><b>New features in this release</b></h3>
      <div style="margin-left: 3em;">
      
         <h4>Beta EuPathDB Workspace</h4>
         <ul>
             <li>Announcing the Beta release of EuPathDB Workspace, a private workspace that can interface 
         with EuPathDB Galaxy and allows for viewing and interrogating private data against data that is already 
         integrated into EuPathDB (public data). The first EuPathDB workspace tool, 'Bigwig files to EuPathDB' 
         exports bigWig files from EuPathDB Galaxy to EuPathDB. 
         The tool creates a EuPathDB Data Set which is managed from the My Data Sets page along with your other private data sets. 
         Each data set also receives a record page which lists all files that the data set contains, offers file management 
         functions and ports files to the genome browser for viewing.<br><br>
         <a href="https://youtu.be/igQZHjRBqV0" target="_blank">Video tutorial: Export bigWig files to EuPathDB</a>
             </li>
         </ul>
         
         <h4>Pathway Improvements</h4>
         <ul>
             <li>
               Semantic zooming of Pathway drawings on Pathway record pages - At low zoom levels, pathway nodes appear as 
               glyphs.  At mid zoom levels, the main compound nodes appear as molecular formulas.  At high zoom levels, enzymes and 
               side nodes appear as formulas, terms or Enzyme commission numbers. Clicking on a node will retrieve details about the 
               node, regardless of zoom level.
            </li>
          </ul>  
      </div>
      </section>
      
     <section>
     <aside>
           Note: Our bimonthly database releases incorporate new data and correct old data when necessary.
           Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
           When search parameters change with a new release, we invalidate (Ø) the search and ask you to rerun it.
           When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
         </aside>
      </section>
