---
layout: post
title: FungiDB 48 Released
category: [EuPathDB,VEuPathDB,FungiDB]
---



<p><strong>We are pleased to announce the release of FungiDB 48.</strong><br><br>  
     
<section>
  <h3><b>Release 48 Webinar</b></h3>
  <div style="margin-left: 3em;">
    Join us on TBD, 2020 at 10am EDT for our Release 47 Highlights webinar where we will demonstrate and discuss our new beta sites.    <a href="https://attendee.gotowebinar.com/register/5450036552813162253" target="_blank">Register here</a>
  </div>
  </section>  
    <section>
      <h3><b>New data in FungiDB 48</b></h3>
      <div style="margin-left: 3em;">         
        <h4>Transcriptomics</h4>
        <ul>
          <li><b><i>Aspergillus fumigatus </i>:</b> 
              <a href="/a/app/record/dataset/DS_a449033cd5">Transcriptomes of WT, &Delta;<i>nctA</i>, and &Delta;<i>nctB</i> mutants in response to itraconazole.</a> &ndash;
                 Genome-wide gene expression profile was investigated in the wild-type and null mutants.
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/31969561" target="_blank">Furukawa et al. 2019</a>.
          </li>
          <li><b><i>Neurospora crassa </i>:</b> 
              <a href="/a/app/record/dataset/DS_61dc8b7a8d">Transcriptome changes in response to genotoxic stress.</a> &ndash;
                 Transcriptomes of WT and H3K9me3-deficient mutants.
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/26578794" target="_blank">Basenko et al. 2015</a>.
          </li>
          <li><b><i>Fusarium graminearum </i>:</b> 
            <a href="/a/app/record/dataset/DS_fc6e396e4a">Transcriptome in response to an active or passive host.</a> &ndash;
            Transcriptomes of cells growing on live and dead wheat heads.
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/27507961" target="_blank">Boedi et al. 2016</a>.
          </li>
            <li><b><i>Cryptococcus neoformans </i>:</b> 
              <a href="/a/app/record/dataset/DS_4784f0a006">Transcriptome and translatome changes in response to host temperature stress.</a> &ndash;
              Total and polysome-associated mRNA profiles of two strains during optimal conditions and one hour following host temperature stress. 
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/31666517" target="_blank">Bloom et al. 2019</a>.
          </li>
        </ul>
        <br>

        <h4>Chip-Seq data</h4>
           <ul>
             <li><a href="/a/app/record/dataset/DS_827c906ace">Genome-wide binding profiles of C-terminally S-tagged NctA protein.</a> &ndash;
                    Genome-wide binding profiles of NctA protein in the absence and the presence of itraconazole.
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/31969561" target="_blank">Furukawa et al. 2019</a>.
             </li>
          </ul>  
        <br>

        <h4>Phenotype data</h4>
           <ul>
             <li><a href="/a/app/record/dataset/DS_43c139682a">Manually curated <i>Cryptococcus</i> Phenotypes</a> &ndash; 
                 The phenotypes for <i>Cryptococcus neoformans</i> JEC21 and H99 and <i>Cryptococcus gatii</i> WM276 are manually curated from published literature describing 
                 phenotypes obtained from mutant studies. We have curated and added phenotypes from <a href="https://www.ncbi.nlm.nih.gov/pubmed/30291809" target="_blank">Bairwa et al. 2019</a>.  
                 publication on kinases to the currently existing phenotypic dataset for <i>C. neoformans</i> H99.
                 We followed an &quot;entity &#43; quality&quot; approach and curated phenotypes with multiple ontologies: 
                 Phenotypic Quality Ontology (PATO) for &quot;quality&quot; terms and Gene Ontology (GO), Fungal Gross Anatomy Ontology (FAO),
                 Chemical Entities of Biological Interest (ChEBI) and Ascomycete Phenotype Ontology (APO) for &quot;entity&quot; terms. 
             </li>
             </ul>
      </div>
    </section>  
       <section>
     <aside>
           <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary. Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.   When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.  When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
         </aside>
      </section>
