---
layout: post
title: FungiDB 51 Released
category: [FungiDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 51 .</h3>

<section>
  <h3>Release 51 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on April 1, 2021 at 10am EDT for our Release 51 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 51.  
    <a href="https://attendee.gotowebinar.com/register/8137280361730606607" target="_blank">Register here</a>  
  </div>
</section>

<section>
  <h3>Organism name changes in FungiDB 51</h3>
  <div style="margin-left: 3em;">
    Organism names were updated based on the NCBI Taxonomy database.
    <br><br>
          <div style="margin-left: 3em;">
                    <style>
                      #organismname_Bld51 table, #organismname_Bld51 td, #organismname_Bld51 th {
                      text-align : center;
                      padding-left: 5px;
                      padding-right: 5px;
                      padding-top: 5px;
                      padding-bottom: 5px;
                      border: 1px solid black;
                      }
                      #organismname_Bld51 {
                      margin-left : 5 em;
                      }
                    </style> 
                      <table id="organismname_Bld51"> 
                      <tr>
                        <th>Previous organism name</th>
                        <th>New organism name</th>
                      </tr>

                      <tr>
                        <td><i>Aspergillus kawachii</i> IFO 4308</td>
                        <td><i>Aspergillus luchuensis</i> IFO 4308</td>
                      </tr>

                      <tr>
                        <td><i>Aspergillus zonatus</i> CBS 506.65</td>
                        <td><i>Penicilliopsis zonata</i> CBS 506.65</td>
                      </tr>

                      <tr>
                        <td><i>Fusarium oxysporum</i> f. sp. <i>cubense</i> tropical race 4 54006</td>
                        <td><i>Fusarium odoratissimum</i> NRRL 54006</td>
                      </tr>

                      <tr>
                        <td><i>Fusarium oxysporum</i> f. sp. <i>cubense</i> race 4</td>
                        <td><i>Fusarium odoratissimum</i> strain race 4</td>
                      </tr>

                      <tr>
                        <td><i>Saccharomyces cerevisiae</i> S288c</td>
                        <td><i>Saccharomyces cerevisiae</i> S288C</td>
                      </tr>

                      <tr>
                        <td><i>Magnaporthe oryzae</i> 70-15</td>
                        <td><i>Pyricularia oryzae</i> 70-15</td>
                      </tr>

                      <tr>
                        <td><i>Magnaporthe oryzae</i> BR32</td>
                        <td><i>Pyricularia oryzae</i> BR32</td>
                      </tr>

                      <tr>
                        <td><i>Fusarium oxysporum</i> FOSC 3-a</td>
                        <td><i>Fusarium oxysporum</i> NRRL 32931</td>
                      </tr>

                      <tr>
                        <td><i>Aspergillus kawachii</i> IFO 4308</td>
                        <td><i>Aspergillus luchuensis</i> IFO 4308</td>
                      </tr>

                      <tr>
                        <td><i>Cryptococcus deuterogattii</i> R265</td>
                        <td><i>Cryptococcus gattii</i> VGII R265</td>
                      </tr>

                      <tr>
                        <td><i>Mucor circinelloides f. lusitanicus</i> CBS 277.49</td>
                        <td><i>Mucor lusitanicus</i> CBS 277.49</td>
                      </tr>

                      <tr>
                        <td><i>Mucor circinelloides f. circinelloides</i> 1006PhL</td>
                        <td><i>Mucor circinelloides</i> 1006PhL</td>
                      </tr>

                      <tr>
                        <td><i>Pythium ultimum</i> var. <i></i> BR650</td>
                        <td><i>Globisporangium ultimum var. sporangiiferum</i> BR650</td>
                      </tr>

                      <tr>
                        <td><i>Pythium ultimum</i> DAOM BR144</td>
                        <td><i>Globisporangium ultimum</i> DAOM BR144</td>
                      </tr>

                      <tr>
                        <td><i>Pythium vexans</i> DAOM BR484</td>
                        <td><i>Phytopythium vexans</i> DAOM BR484</td>
                      </tr>

                      <tr>
                        <td><i>Pythium irregulare</i> DAOM BR486</td>
                        <td><i>Globisporangium irregulare</i> DAOM BR486</td>
                      </tr>

                      <tr>
                        <td><i>Pythium iwayamai</i> DAOM BR242034</td>
                        <td><i>Globisporangium iwayamae</i> DAOM BR242034</td>
                      </tr>
                      </table>
  </div>

    <section>
      <h3>New data in FungiDB 51</h3>
      <div style="margin-left: 3em;">

        <h4>EBI Pipeline</h4>
          {{project}} 51 represents the first release that employs the EBI pipeline for analyzing genomes and RNA-Seq data. (<a href="/a/app/static-content/methods.html">{{project}} Analysis Methods</a>) All genomes were run through the EBI pipeline, updating all analyses such as InterproScan and repeat identification.  While much of the EBI pipeline is the same as our previous in-house pipeline, these differences are noted:
          <ul>
            <li>Expression values calculated from RNA-Seq data are now presented as TPM (Transcripts Per Kilobase Million) instead of FPKM (Fragments Per Kilobase Million) or RPKM (Reads Per Kilobase Million). <a href="https://rna-seqblog.com/rpkm-fpkm-and-tpm-clearly-explained/" target="_blank">Learn more</a></li>
            <li>Signal peptide predictions are now made with <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/" target="_blank">SignalP-4.1</a>, updated from <a href="http://www.cbs.dtu.dk/services/SignalP-3.0/" target="_blank">SignalP-3.0</a>.</li>
            <li>All pseudogenes now include automatically predicted UTRs.</li>
          </ul>

        <h4>Genome Sequence and Annotation: these annotated genomes appear for the first time in FungiDB.</h4>
        <ul>
          
          <li><a href="/a/app/record/dataset/DS_5c9a190da6"><i>Aspergillus lentulus</i> IFM 54703</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/26769945">Kusuya et al. 2016</a>.  
            </li>

          <li><a href="/a/app/record/dataset/DS_ce4c296905"><i>Aspergillus thermomutatus</i> HMR AF 39</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/30637395">Parent-Michaud et al. 2019</a>.  
            </li>

          <li><a href="/a/app/record/dataset/DS_69d0bf573e"><i>Blastomyces dermatitidis</i> ER3</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/26439490">Muñoz et al. 2015</a>.  
            </li>

          <li><a href="/a/app/record/dataset/DS_f370731c99"><i>Blastomyces gilchristii</i> SLH14081</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/26439490">Muñoz et al. 2015</a>.  
            </li>        

          <li><a href="/a/app/record/dataset/DS_1f16fb39d7"><i>Candida auris</i> B11245</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/30559369">Muñoz et al. 2018</a>.  
            </li>
          
          <li><a href="/a/app/record/dataset/DS_cf5299d826"><i>Candida auris</i> B11220</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/30559369">Muñoz et al. 2018</a>.  
            </li>
        
          <li><a href="/a/app/record/dataset/DS_90d3beea9a"><i>Candida auris</i> 6684</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/26346253">Chatterjee et al. 2015</a>.  
            </li>
          
          <li><a href="/a/app/record/dataset/DS_b1f0db92e1"><i>Candida auris</i> B11221</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/30559369">Muñoz et al. 2018</a>.  
            </li>

          <li><a href="/a/app/record/dataset/DS_511cf229a7"><i>Candida auris</i> B11243</a> &ndash;
           <a href="http://www.ncbi.nlm.nih.gov/pubmed/30559369">Muñoz et al. 2018</a>. 
           </li>

          <li><a href="/a/app/record/dataset/DS_69358043a0"><i>Candida pseudohaemulonii</i> B12108</a> &ndash;
           <a href="http://www.ncbi.nlm.nih.gov/pubmed/30559369">Muñoz et al. 2018</a>.  
           </li>

          <li><a href="/a/app/record/dataset/DS_06fdfcb16d"><i>Exophiala dermatitidis</i> NIH/UT8656</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/28348446">Teixeira et al. 2017</a>.  
            </li>

          <li><a href="/a/app/record/dataset/DS_98be9d6af8"><i>Hanseniaspora uvarum</i> AWRI3580</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/16423073">Pramateftaki et al. 2006</a>.  
            </li>

          <li><a href="/a/app/record/dataset/DS_20c3a83941"><i>Malassezia restricta</i> BS 7877</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/27882618">Park et al. 2017</a>.  
            </li>
            
          <li><a href="/a/app/record/dataset/DS_8c9d407527"><i>Puccinia striiformis</i> 93-210</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/23607900">Cantu et al. 2011</a>.  
            </li>
            
          <li><a href="/a/app/record/dataset/DS_45fe0e4ca4"><i>Thermothelomyces thermophilus</i> ATCC 42464</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/21964414">Berka et al. 2011</a>.  
            </li>
        </ul>

        <h4>Genome Sequence: available for the first time in FungiDB:</h4>
        <ul>

         <li><a href="/a/app/record/dataset/DS_c2b6d2aada"><i>Cryptococcus neoformans </i>var. <i>neoformans</i> XL280 </a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/24058295">Ni et al. 2013</a>.  
          </li>    

          <li><a href="/a/app/record/dataset/DS_a3b881f49b"><i>Epichloe festucae</i> Fl1</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/30356280">Winter et al. 2018</a>.  
          </li>

          <li><a href="/a/app/record/dataset/DS_0011425ac1"><i>Epichloe glyceriae</i> E277</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/23468653">Schardl et al. 2013</a>.  
          </li>      

          <li><a href="/a/app/record/dataset/DS_c074d9076f"><i>Fusarium graminearum </i> 233423</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/26125491">Walkowiak et al. 2015</a>.  
            </li>

          <li><a href="/a/app/record/dataset/DS_afdba8a671"><i>Trichoderma atroviride</i> JCM 9410</a> &ndash;
          RIKEN Center for Life Science Technologies.  
          </li>

          <li><a href="/a/app/record/dataset/DS_3182ff461b"><i>Trichoderma reesei</i> QM6a (2017)</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/28690679">Li et al. 2017</a>.  
            </li>

          <li><a href="/a/app/record/dataset/DS_3a4d320efe"><i>Verticillium dahliae </i> JR2</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/16733756">de Jonge et al. 2013</a>.  
          </li>
        </ul>

      <h4>Genome Sequence and Annotation: updated genomes </h4>
      <ul>
        <li><a href="/a/app/record/dataset/DS_8ddc55305a"><i>Aspergillus fischeri</i> NRRL 181</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/18404212">Fedorova et al. 2008</a>.  
          </li> 

        <li><a href="/a/app/record/dataset/DS_96e9022cd3"><i>Sclerotinia sclerotiorum</i> 1980 UF-70</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/28204478">Derbyshire et al. 2017</a>.  
          </li>
 
        <li><a href="/a/app/record/dataset/DS_cbbce274f6"><i>Ustilago maydis</i> 521</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/17080091">Kämperet et al. 2006</a>.  
          </li>
      </ul>


        <h4>Whole genome sequence of isolates for SNP analysis:</h4>
        <ul>
          <li>
            <a href="/a/app/record/dataset/DS_6f53a4a049">SNP calls on 20 <i>Candida albicans</i> clinical isolates</a> &ndash;
               Intra-species variation.
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/25514520">Hirakawa et al. 2015</a>.
          </li>

          <li>
            <a href="/a/app/record/dataset/DS_269cf62506">SNP calls on WGS of <i>Pyricularia oryzae</i> isolated from Bangladesh in 2016 and 2017</a> &ndash;
            Nineteen isolates of collected from barley, wheat and torpedo grass.
           The University of Exeter, UK.
          </li>

        </ul>


        <h4>Transcriptomics</h4>
    <ul>
      <li>
        <a href="/a/app/record/dataset/DS_4fb286d98f">Fresh whole blood infected with <i>Candida albicans</i></a> &mdash;
        RNA-pool of both species extracted at 0min (control), 15, 30, 60, 120 and 240 min.  
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/30333805" target="_blank">Sieber et al. 2018</a>. The host component of this data set is available on HostDB: 
        <a href="https://hostdb.org/hostdb/app/record/dataset/DS_fe4f9a11e3">Fresh human whole blood infected with <i>Candida albicans</i></a>.  
      </li>

      <li>
        <a href="/a/app/record/dataset/DS_51d662f7cb">Fresh whole blood infected with <i>Candida glabrata</i></a> &mdash;
        RNA-pool of both species extracted at 0min (control), 15, 30, 60, 120 and 240 min.  
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/30333805" target="_blank">Sieber et al. 2018</a>. The host component of this data set is available on HostDB: 
        <a href="https://hostdb.org/hostdb/app/record/dataset/DS_9b7a0d35d7">Fresh human whole blood infected with <i>Candida glabrata</i></a>.  
      </li>

      <li>
       <a href="/a/app/record/dataset/DS_b041354581">Fresh whole blood infected with <i>Candida parapsilosis</i></a> &mdash;
       RNA-pool of both species extracted at 0min (control), 15, 30, 60, 120 and 240 min.  
       <a href="http://www.ncbi.nlm.nih.gov/pubmed/30333805" target="_blank">Sieber et al. 2018</a>. The host component of this data set is available on HostDB: 
       <a href="https://hostdb.org/hostdb/app/record/dataset/DS_dae3a6918a">Fresh human whole blood infected with <i>Candida parapsilosis</i></a>.   
      </li>
      <li>
        <a href="/a/app/record/dataset/DS_1d4b3f2f63">Fresh whole blood infected with <i>Candida tropicalis</i></a> &mdash;
        RNA-pool of both species extracted at 0min (control), 15, 30, 60, 120 and 240 min.  
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/30333805" target="_blank">Sieber et al. 2018</a>. The host component of this data set is available on HostDB: 
        <a href="https://hostdb.org/hostdb/app/record/dataset/DS_a98a0d1f97">Fresh human whole blood infected with <i>Candida tropicalis</i></a>.   
      </li>

      <li>
        <a href="/a/app/record/dataset/DS_f9aae660d7">Total RNA profiles of cells grown in glucose or GlcNAc medium</a> &mdash;
        Transcriptomic changes occurring during yeast-filament transition in the pathogen fungus <i>Candida tropicalis</i> MYA-3404.
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/26466925" target="_blank">Zhang et al. 2016</a>.   
      </li>

      <li>
        <a href="/a/app/record/dataset/DS_1018e6c09b">RNA-Seq analyses of fruiting body development</a> &mdash;
        Developmental transcriptome of a multicellular basidiomycete <i>Coprinopsis cinerea</i>.
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/30902897" target="_blank">Krizsán et al. 2019</a>.   
      </li>

      <li>
        <a href="/a/app/record/dataset/DS_5f2f1c9822">Transcriptional landscape of <i>Cryptococcus</i> - host response</a> &mdash;
        Animal infection model using the mouse <i>Mus musculus</i> to study the host response to infection. 
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/31329596" target="_blank">Li et al. 2019</a>.   
      </li>

      <li>
        <a href="/a/app/record/dataset/DS_7ce8a791f1">Gene expression variation in response to biologically relevant stresses</a> &mdash;
        Global transcriptomic changes in the pathogen <i>Crypococcus neoformans</i> occurring in response to host temperature stress.
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/31860441" target="_blank">Yu et al. 2020</a>.   
      </li>

      <li>
        <a href="/a/app/record/dataset/DS_42a83032dd">Mouse macrophages infected with <i>Mucor circinelloides</i></a> &mdash;
        A study of the transcriptomics changes occuring in both host and pathogen upon the interaction of <i>Mucor circinelloides</i> spores with phagocytic cells of a mouse.
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/30723131" target="_blank">Pérez-Arques et al. 2019</a>.   
      </li>

    </ul>   
            
        <h4>Phenotype data</h4>
             
             <ul>
             <li><a href="/a/app/record/dataset/DS_7cdba4f78a">Manually Curated <i>Aspergillus</i> Phenotypes</a> &ndash; 
                 The currently existing phenotypes for <i>Aspergillus</i> species were manually curated by <a href="http://www.aspgd.org" target="_blank">AspGD</a> 
                 from published literature describing phenotypes obtained from mutant studies. These phenotypic annotations were 
                 reformatted at FungiDB in-line with VEuPathDB’s approach for phenotype curation. We have curated and integrated 
                 phenotypes from <a href="https://pubmed.ncbi.nlm.nih.gov/31969561/" target="_blank">Furukawa et al. 2020</a> for 
                 <i>Aspergillus fumigatus</i> Af293 and from <a href="https://pubmed.ncbi.nlm.nih.gov/30410052/" target="_blank">Frawley et al. 2018</a> 
                 for <i>Aspergillus nidulans</i> FGSC A4.    
             </li>
             </ul>
          <br>
          <div style="margin-left: 3em;">
                    <style>
                      #Phenotype_Bld51Asp table, #Phenotype_Bld51Asp td, #Phenotype_Bld51Asp th, #Phenotype_Bld51Asp tr {
                      text-align : center;
                      padding-left: 5px;
                      padding-right: 5px;
                      padding-top: 5px;
                      padding-bottom: 5px;
                      border: 1px solid black;
                      }
                      #Phenotype_Bld51Asp {
                      margin-left : 5 em;
                      }
                    </style> 
                      <table id="Phenotype_Bld51Asp"> 
                      <tr>
                        <th>Species</th>
                        <th>Phenotype mapping</th>
                        <th>Genes with phenotypes</th>
                      </tr>
                      <tr>
                        <td><i>Aspergillus fumigatus</i> Af293</td>
                        <td>51</td>
                        <td>12</td>
                      </tr>
                      <tr>
                        <td><i>Aspergillus nidulans</i> FGSC A4</td>
                        <td>80</td>
                        <td>5</td>
                      </tr>
                      </table>
          </div>
            <br>
                
                  <ul>
                  <li><a href="/a/app/record/dataset/DS_43c139682a">Manually Curated <i>Cryptococcus</i> Phenotypes</a> &ndash; 
                      The phenotypes for <i>Cryptococcus neoformans</i> JEC21 and H99 and <i>Cryptococcus gattii</i> WM276 genomes 
                      are manually curated from published literature describing phenotypes obtained from mutant studies. We have 
                      curated and added phenotypes from additional publications to the currently existing phenotypic dataset for 
                      <i>C. neoformans</i> H99. The following are the eleven publications newly added to this dataset: 
                      <a href="https://pubmed.ncbi.nlm.nih.gov/8007987/" target="_blank">Chang & Kwon-Chung. 1994</a>, 
                      <a href="https://pubmed.ncbi.nlm.nih.gov/10639402/" target="_blank">Cox et al. 2000</a>, 
                      <a href="https://pubmed.ncbi.nlm.nih.gov/11123698/" target="_blank">Cox et al. 2001</a>, 
                      <a href="https://pubmed.ncbi.nlm.nih.gov/25827419/" target="_blank">Ianiri & Idnurm. 2015</a>, 
                      <a href="https://pubmed.ncbi.nlm.nih.gov/26037119/" target="_blank">Lev et al. 2015</a>, 
                      <a href="https://pubmed.ncbi.nlm.nih.gov/27033523/" target="_blank">Li et al. 2015</a>, 
                      <a href="https://pubmed.ncbi.nlm.nih.gov/27212659/" target="_blank">Homer et al. 2016</a>, 
                      <a href="https://pubmed.ncbi.nlm.nih.gov/28376087/" target="_blank">Chow et al. 2017</a>, 
                      <a href="https://pubmed.ncbi.nlm.nih.gov/28467481/" target="_blank">Gyawali et al. 2017</a>, 
                      <a href="https://pubmed.ncbi.nlm.nih.gov/32251295/" target="_blank">Lee et al. 2020</a> and 
                      <a href="https://pubmed.ncbi.nlm.nih.gov/32839469/" target="_blank">Jin et al. 2020</a>.  
                  </li>
                  </ul>
             <br>
             <div style="margin-left: 3em;">
                        <style>
                          #Phenotype_Bld51Cryp table, #Phenotype_Bld51Cryp td, #Phenotype_Bld51Cryp th, #Phenotype_Bld51Cryp tr {
                          text-align : center;
                          padding-left: 5px;
                          padding-right: 5px;
                          padding-top: 5px;
                          padding-bottom: 5px;
                          border: 1px solid black;
                          }
                          #Phenotype_Bld51Cryp {
                          margin-left : 5 em;
                          }
                        </style> 
                          <table id="Phenotype_Bld51Cryp"> 
                          <tr>
                            <th>Species</th>
                            <th>Phenotype mapping</th>
                            <th>Genes with phenotypes</th>
                          </tr>
                          <tr>
                            <td><i>Cryptococcus neoformans</i> var <i>grubii</i> H99</td>
                            <td>4891</td>
                            <td>534</td>
                          </tr>
                          </table>

            </div>
            <br>
          
                    <ul>
                      <li><a href="/a/app/record/dataset/DS_8cb36b3135">Manually Curated <i>Pyricularia</i> Phenotypes</a> &ndash; 
                          The phenotypes for <i>Pyricularia oryzae</i> 70-15 are manually curated from published literature describing 
                          phenotypes obtained from mutant studies. These phenotypes have been manually curated in-line with VEuPathDB's 
                          approach for phenotype annotation and integrated into FungiDB. We followed an &quot;entity &#43; quality&quot; 
                          approach and curated phenotypes with multiple ontologies: Phenotypic Quality Ontology (PATO) for &quot;quality&quot; 
                          terms and Gene Ontology (GO), Fungal Gross Anatomy Ontology (FAO), Chemical Entities of Biological Interest (ChEBI) 
                          and Ascomycete Phenotype Ontology (APO) for &quot;entity&quot; terms. We have curated and integrated phenotypes from
                          <a href="https://pubmed.ncbi.nlm.nih.gov/20615116/" target="_blank">Guo et al. 2010</a>, 
                          <a href="https://pubmed.ncbi.nlm.nih.gov/21383978/" target="_blank">Guo et al. 2011</a>, 
                          <a href="https://pubmed.ncbi.nlm.nih.gov/25299517/" target="_blank">Lu et al. 2014</a> and 
                          <a href="https://pubmed.ncbi.nlm.nih.gov/25314920/" target="_blank">Kong et al. 2015</a> publications for bZIP and 
                          Zn2Cys6 transcription factors. 
                      </li>
                      </ul>
	          <br>
            <div style="margin-left: 3em;">
                            <style>
                              #Phenotype_Bld51Mag table, #Phenotype_Bld51Mag td, #Phenotype_Bld51Mag th, #Phenotype_Bld51Mag tr {
                              text-align : center;
                              padding-left: 5px;
                              padding-right: 5px;
                              padding-top: 5px;
                              padding-bottom: 5px;
                              border: 1px solid black;
                              }
                              #Phenotype_Bld51Mag {
                              margin-left : 5 em;
                              }
                            </style> 
                              <table id="Phenotype_Bld51Mag"> 
                              <tr>
                                <th>Species</th>
                                <th>Phenotype mapping</th>
                                <th>Genes with phenotypes</th>
                              </tr>
                              <tr>
                                <td><i>Pyricularia oryzae</i> 70-15</td>
                                <td>1945</td>
                                <td>129</td>
                              </tr>
                              </table>
              </div>
             
  		<h4>Updated Functional Annotation</h4>
      <ul> 
        <li>      
      <a href="https://docs.google.com/spreadsheets/d/1mqHh7CEIuwgBnfLA2x1ZnyqKCn8YWXG6mGSEiKUNBjU/edit?usp=sharing" target="_blank">Summary of annotation updates</a>
               which include product descriptions, GO terms, gene names, synonyms, publications and notes from curators. These functional annotations are current with GeneDB January 2021 GFF versions. </li>
      </ul>    
      <br>   
            <div style="margin-left: 3em;">   
             <style>
                #FunctAnnot_Bld51 table, #FunctAnnot_Bld51 td, #FunctAnnot_Bld51 th, #FunctAnnot_Bld51 tr {
                text-align : center;
                padding-left: 5px;
                padding-right: 5px;
                padding-top: 5px;
                padding-bottom: 5px;
                border: 1px solid black;
                }
                #FunctAnnot_Bld51 {
                margin-left : 5 em;
                }
              </style> 
                <table id="FunctAnnot_Bld51"> 
                <tr>
                  <th>FungiDB Genome Version</th>
                  <th>Download GeneDB GFF file</th>
                  <th>Genes Updated</th>
                </tr>
                <tr>
                  <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_8f0322af73"><i>Cryptococcus neoformans</i> H99</a></td>
                  <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-01/CneoformansH99/CneoformansH99.gff.gz">GeneDB version January, 2021</a></td>
                  <td>777</td>
                </tr>
                <tr>
                  <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="https://fungidb.org/fungidb/app/record/dataset/DS_4d171eb9ea"><i>Cryptococcus neoformans</i> JEC21</a></td>
                  <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-01/CneoformansneoformansJEC21/CneoformansneoformansJEC21.gff.gz">GeneDB version January, 2021</a></td>
                  <td>1125</td>
                </tr>
                </table>
                <br>
            </div>    
 </div>
</section>
        
<section>
  <h3>New features in {{project}} 51</h3>
  <div style="margin-left: 3em;">
   <ul>
    <li><a href="/a/app/workspace/blast/new" target="_blank">BLAST<i><sup>beta</sup></i> tool</a> &ndash; a new BLAST tool with multi-BLAST capabilities.  Input a single sequence or several FASTA-formatted sequences at once.  Our tool uses NCBI BLAST to find genes, genomic sequences, ESTs, or PopSet Sequences that have BLAST similarity to your input sequence.  <a href="https://youtu.be/mEjt8FhC-W8" target="_blank">BLAST<i><sup>beta</sup></i> video introduction (4 min)  </li>
    <li><a href="https://statis-content.veupathdb.org//documents/VEuPathDB_BRC4_usage_metrics_report.pdf">Usage</a> and <a href="https://statis-content.veupathdb.org//documents/VEuPathDB_BRC4_performance_metrics_report.pdf">performance</a> metrics for VEuPathDB sites are now updated monthly.  </li>
    </ul>
  </div>
</section>



<section>
  <aside>
    <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary. Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.   When search parameters change with a new release, we invalidate (Ø) the search and ask you to rerun it.  When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
