---
layout: post
title: FungiDB 52 Released
category: [FungiDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 52.</h3>

<section>
  <h3>Release 52 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on May 27, 2021 at 10am EDT for our Release 52 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 52.  
    <a href="https://attendee.gotowebinar.com/register/8223585429100223502" target="_blank">Register here</a>  
  </div>
</section>

    <section>
      <h3>New data in FungiDB 52</h3>
      <div style="margin-left: 3em;">

        <h4>Genome Sequence and Annotation: these annotated genomes appear for the first time in FungiDB.</h4>
        <ul>
          <li><a href="/a/app/record/dataset/DS_d1aa1a2f2c"><i>Ascosphaera apis</i> ARSEF 7405</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/27071652">Shang et al. 2016</a>.  
          </li>

          <li><a href="/a/app/record/dataset/DS_74a7480ae4"><i>Blastomyces percursus</i> EI222</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/28176377">Dukik et al. 2017</a>.  
          </li>

          <li><a href="/a/app/record/dataset/DS_2d91dcb480"><i>Bremia lactucae</i> SF5</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31201315">Fletcher et al. 2019</a>.  
          </li>

          <li><a href="/a/app/record/dataset/DS_850d056486"><i>Colletotrichum graminicola</i> M1.001</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/22885923">O'Connell et al. 2012</a>.  
          </li>  

          <li><a href="/a/app/record/dataset/DS_ffbe968d03"><i>Hanseniaspora guilliermondii</i> UTAD222</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/30462193">Seixas et al. 2019</a>.  
          </li>

          <li><a href="/a/app/record/dataset/DS_f0c7b5aa4a"><i>Ophiostoma piceae</i> UAMH 11346</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/23725015">Haridas et al. 2013</a>.  
          </li>

          <li><a href="/a/app/record/dataset/DS_085ea6d33a"><i>Pichia kudriavzevii</i> CBS573</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/30024981">Douglass et al. 2018</a>.  
          </li>

          <li><a href="/a/app/record/dataset/DS_d16c621125"><i>Podospora anserina</i> S mat+</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/12892638">Silar et al. 2003</a>.
          </li>  

          <li><a href="/a/app/record/dataset/DS_2dc538cfff"><i>Pseudogymnoascus destructans</i> 20631-21</a> &ndash;
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/29295979">Chibucos et al. 2013</a>.
          </li>

          <li><a href="/a/app/record/dataset/DS_1394ad1592"><i>Puccinia sorghi</i> RO10H11247</a> &ndash;
          BIA - FCEN - UBA, Bia Argentina.  
          </li>

          <li><a href="/a/app/record/dataset/DS_f20c645c6e"><i>Trichosporon asahii</i> var. <i>asahii</i> CBS 2479</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/23193141">Yang et al. 2012</a>.  
            </li>

          <li><a href="/a/app/record/dataset/DS_2ac7768fb1"><i>Verruconis gallopava</i> CBS 43764</a> &ndash;
            The Broad Institute of Harvard and MIT.  
          </li>
              
          <li><a href="/a/app/record/dataset/DS_abf31f6b25"><i>Zymoseptoria tritici</i> ST99CH_3D1</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/29693722">Meile et al. 2018</a>.  
          </li>
 
        </ul>

        <br>

        <h4>Genome Sequence: available for the first time in FungiDB:</h4>
        <ul>
          <li><a href="/a/app/record/dataset/DS_ab15a8399e"><i>Aspergillus niger</i> LDM3</a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/32009199">Sui et al. 2020</a>.  
            </li>

         <li><a href="/a/app/record/dataset/DS_085ea6d33a"><i>Pichia kudriavzevii</i> CBS5147 </a> &ndash;
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/30024981">Douglass et al. 2018</a>.  
          </li>    
        </ul>
      <br>

        <h4>Whole genome sequence of isolates for SNP analysis:</h4>
        <ul>
          <li><a href="/a/app/record/dataset/DS_fb0ad203b2">SNP calls on WGS of clinical isolates of <i>A. lentulus</i></a> &ndash;
            <i>Aspergillus lentulus</i> IFM54703 is a causative agent of aspergillosis. Six clinical isolates were sequenced.
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/32477406">Dos Santos et al. 2020</a>.
          </li>

          <li>
            <a href="/a/app/record/dataset/DS_5a67784cef">SNP calls on WGS of <i>Candida haemulonii</i> isolates</a> &ndash;
              Isolates were received in the Mycotic Diseases Branch Reference Laboratory at the CDC for routine fungal identification or as part of ongoing fungal disease surveillance.
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/32587603">Gade et al. 2020</a>.
          </li>

          <li>
            <a href="/a/app/record/dataset/DS_91b3e4633f">SNP calls on WGS of <i>Pyricularia</i> field-isolates</a> &ndash;
            Whole-genome analysis of fungal strains from different hosts.
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/29487238">Gladieux et al. 2018</a>.
          </li>
        </ul>
        <br>

        <h4>Transcriptomics</h4>
    <ul>
      <li>
        <a href="/a/app/record/dataset/DS_a3ecdd86a4">Comparative transcriptome of <i>Candida parapsilosis</i> wild type and transcription factor mutants</a> &mdash;
        Transcriptomes collected from WT in planktonic growth conditions and WT and seven transcription factor mutants grown in biofilm conditions.  
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/25233198" target="_blank">Holland et al. 2014</a>. 
      </li>

      <li>
       <a href="/a/app/record/dataset/DS_4703a409e8">Transcriptomics of <i>Mucor circinelloides</i> NCRIP deficient spores phagocytosed by mouse macrophages</a> &mdash;
       Transcriptomics of R3B2 and RDRP1 fungal spores and fungal spore-macrophage coculture.  
       <a href="http://www.ncbi.nlm.nih.gov/pubmed/32658892" target="_blank">Pérez-Arques et al. 2020</a>.  
      </li>
    </ul>   
    <br>
 
            
        <h4>Phenotype data</h4>
             
             <ul>
             <li><a href="/a/app/record/dataset/DS_7cdba4f78a">Manually Curated <i>Aspergillus</i> Phenotypes</a> &ndash; 
                 The currently existing phenotypes for <i>Aspergillus</i> species were manually curated by <a href="http://www.aspgd.org" target="_blank">AspGD</a> 
                 from published literature describing phenotypes obtained from mutant studies. These phenotypic annotations were 
                 reformatted at FungiDB in-line with VEuPathDB’s approach for phenotype curation. We have curated and integrated 
                 phenotypes from <a href="https://pubmed.ncbi.nlm.nih.gov/31068436/" target="_blank">Thammahong et al. 2019</a> for 
                 <i>Aspergillus fumigatus</i> Af293.    
             </li>
             </ul>
          <br>
          <div style="margin-left: 3em;">
                    <style>
                      #Phenotype_Bld52Asp table, #Phenotype_Bld52Asp td, #Phenotype_Bld52Asp th, #Phenotype_Bld52Asp tr {
                      text-align : center;
                      padding-left: 5px;
                      padding-right: 5px;
                      padding-top: 5px;
                      padding-bottom: 5px;
                      border: 1px solid black;
                      }
                      #Phenotype_Bld52Asp {
                      margin-left : 5 em;
                      }
                    </style> 
                      <table id="Phenotype_Bld52Asp"> 
                      <tr>
                        <th>Species</th>
                        <th>Phenotype mapping</th>
                        <th>Genes with phenotypes</th>
                      </tr>
                      <tr>
                        <td><i>Aspergillus fumigatus</i> Af293</td>
                        <td>27</td>
                        <td>1</td>
                      </tr>
                      </table>
          </div>  
      <br>
               
  		<h4>Updated Functional Annotation</h4>
      <ul> 
        <li>
             <a href="https://docs.google.com/spreadsheets/d/1g1wac6A-Jgv0OTCXlvkqVd2d2qT8malWZdio9NnD86g/edit?usp=sharing" target="_blank">Summary of annotation updates</a>
               which include product descriptions and notes from curators. These functional annotations are current with GeneDB April 2021 GFF versions. 
        </li>
      </ul>
        <br>
               
            <div style="margin-left: 3em;">   
             <style>
                #FunctAnnot_Bld52 table, #FunctAnnot_Bld52 td, #FunctAnnot_Bld52 th, #FunctAnnot_Bld52 tr {
                text-align : center;
                padding-left: 5px;
                padding-right: 5px;
                padding-top: 5px;
                padding-bottom: 5px;
                border: 1px solid black;
                }
                #FunctAnnot_Bld52 {
                margin-left : 5 em;
                }
              </style> 
                <table id="FunctAnnot_Bld52"> 
                <tr>
                  <th>FungiDB Genome Version</th>
                  <th>Download GeneDB GFF file</th>
                  <th>Genes Updated</th>
                </tr>
                <tr>
                  <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="https://fungidb.org/fungidb/app/record/dataset/DS_4b35c88aa1"><i>Aspergillus nidulans</i> FGSC A4</a></td>
                  <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-04/AnidulansFGSCA4/AnidulansFGSCA4.gff.gz">GeneDB version April, 2021</a></td>
                  <td>240</td>
                </tr>
                </table>
                <br>
            </div>    
 </div>
</section>

<section>
  <h3>New features in {{project}} 52</h3>
  <div style="margin-left: 3em;">
   <ul>
    <li>The gene pages of <a href="https://en.wikipedia.org/wiki/Pseudogene" target="_blank">pseudogenes</a> are modified to improved clarity: <a href="{{'/assets/images/pseudogene-page.png' | absolute_url}}">(helpful image)</a>
      <ul>
        <li>The Transcripts table in the Gene Models section now indicates that no protein products are produced. </li>
        <li>The exons in JBrowse views of pseudogenes are colored gray.  Blue represents genes on the forward strand and red represents genes on the reverse strand. </li>
        <li>The Orthology and Synteny section indicates that orthology is not determined for pseudogenes.</li>
        <li>Protein sequences are no longer available from the Sequences section.</li>
      </ul>
    </li>
    <li>Reference organisms are indicated in the Organism parameter on search pages.  VEuPathDB chooses a representative genome from each species and aligns all functional genomics data to the reference.  In doing so, all data can be viewed and compared in the genome browser against the reference.  In some cases, data are aligned to the source organism in addition to the reference. (helpful image)</li>
    <li>The algorithm for transferring EC Numbers between orthologs is updated. We previously predicted EC numbers, but now we offer scores and statistics to help users determine the confidence in this prediction. The new algorithm compares a gene's protein sequence to the protein sequences of other genes in the same ortholog group. A gene is inferred to receive a specific EC number based on how well its sequence matches genes in it's ortholog group that have already been assigned this EC number.</li>
    </ul>
  </div>
</section>

<section>
  <aside>
    <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary. Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.   When search parameters change with a new release, we invalidate (Ø) the search and ask you to rerun it.  When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
