---
layout: post
title: FungiDB 51 Released
category: [FungiDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 51 .</h3>

<p>The URL <a href="https://{{project}}.org">https://{{project}}.org</a> directs you to the newest version of our interface.  The site contains all previous data plus some new tools and functions in an updated and streamlined interface that empasizes easy access to help information.   Navigate to <a href="https://legacy.{{project}}.org">https://legacy.{{project}}.org</a> for the previous version of the interface.  The legacy interface will remain active for at least one release.</p>

<section>
  <h3>Release 51 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on March 12, 2021 at 10am EDT for our Release 50 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 50.  
    <a href="https://attendee.gotowebinar.com/register/6484104154635614223" target="_blank">Register here</a>  
  </div>
</section>

<section>
  <h3>Educational resources</h3>
  <div style="margin-left: 3em;">
  The educational resources at <a href="/a/app/static-content/landing.html">Learning about VEuPathDB</a> page have been updated. Updates include new webinar recordings, tutorials, methods descriptions, and an all new <a href="https://www.niaid.nih.gov/research/bioinformatics-resource-centers" target="_blank">Bioinformatics Resource Centers </a>calendar of events.
</div>
</section>

    <section>
      <h3>New data in FungiDB 51</h3>
      <div style="margin-left: 3em;">

        <h4>Whole genome sequence of isolates for SNP analysis:</h4>
        <ul>
          <li><b><i>Coccidioides posadasii</i> : </b> 
            <a href="/a/app/record/dataset/DS_d27c9dd420">SNP calls on WGS of <i>Coccidioides posadasii</i> isolates from regions bordering the Caribbean Sea.</a> &ndash;
               Ten isolates were sequenced using Illumina HiSeq 2500 technology.
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31772050">Teixeira et al. 2019</a>.
          </li>

          <li><b><i>Coccidioides immitis</i> : </b> 
            <a href="/a/app/record/dataset/DS_0e0a6f3045">SNP calls on WGS of <i>Coccidioides immitis</i> isolates from Washington State, USA</a> &ndash;
               Fourteen clinical isolates were sequenced using Illumina HiSeq 2500 technology.
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/30789132">Oltean et al. 2019</a>.
          </li>
        </ul>
	<br>
        <h4>Transcriptomics</h4>
    <ul>
      <li>
        <a href="/a/app/record/dataset/DS_fe4f9a11e3">Fresh whole blood infected with <i>Candida albicans</i></a> &mdash;
        RNA-pool of both species extracted at 0min (control), 15, 30, 60, 120 and 240 min.  
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/30333805" target="_blank">Sieber et al. 2018</a>. The host component of this data set is available on HostDB: 
        <a href="a/app/record/dataset/DS_fe4f9a11e3">Fresh human whole blood infected with Candida albicans</a>.  
      </li>
      <li>
       <a href="/a/app/record/dataset/DS_dae3a6918a">Fresh whole blood infected with <i>Candida parapsilosis</i></a> &mdash;
       RNA-pool of both species extracted at 0min (control), 15, 30, 60, 120 and 240 min.  
       <a href="http://www.ncbi.nlm.nih.gov/pubmed/30333805" target="_blank">Sieber et al. 2018</a>. The fungal component of this data set is available on FungiDB: 
       <a href="a/app/record/dataset/DS_dae3a6918a">Fresh human whole blood infected with Candida parapsilosis</a>.   
      </li>
      <li>
        <a href="/a/app/record/dataset/DS_a98a0d1f97">Fresh whole blood infected with <i>Candida tropicalis</i></a> &mdash;
        RNA-pool of both species extracted at 0min (control), 15, 30, 60, 120 and 240 min.  
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/30333805" target="_blank">Sieber et al. 2018</a>. The fungal component of this data set is available on FungiDB: 
        <a href="a/app/record/dataset/DS_a98a0d1f97">Fresh human whole blood infected with Candida tropicalis</a>.   
      </li>
      <li>
        <a href="/a/app/record/dataset/DS_9b7a0d35d7">Fresh whole blood infected with <i>Candida glabrata</i></a> &mdash;
        RNA-pool of both species extracted at 0min (control), 15, 30, 60, 120 and 240 min.  
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/30333805" target="_blank">Sieber et al. 2018</a>. The fungal component of this data set is available on FungiDB: 
        <a href="a/app/record/dataset/DS_9b7a0d35d7">Fresh human whole blood infected with Candida glabrata</a>.  
      </li>
      <li>
        <a href="/a/app/record/dataset/DS_1401c8b50d">Early RNA sequencing of human monocytes in presence of <i>Candida albicans</i></a> &mdash;
        RNA sequencing was prerformed from <i>Candida albicans</i> co-incubated monoyctes from 3 different donors.  
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/32393780" target="_blank">Halder et al. 2020</a>.   
      </li>
      <li>
        <a href="/a/app/record/dataset/DS_6a3c252e75">Transcriptional landscape of <i>Cryptococcus</i> - host response</a> &mdash;
        Animal infection model using the mouse <i>Mus musculus</i> to study the host response to infection. 
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/31329596" target="_blank">Li al. 2019</a>.   
      </li>
      <li>
        <a href="/a/app/record/dataset/DS_5671b6943c">Mouse macrophages infected with <i>Mucor circinelloides</i></a> &mdash;
        A study of the transcriptomics changes occuring in both host and pathogen upon the interaction of <i>Mucor circinelloides</i> spores with phagocytic cells of a mouse.
        <a href="http://www.ncbi.nlm.nih.gov/pubmed/30723131" target="_blank">Pérez-Arques al. 2019</a>.   
      </li>
    </ul>   
    <br>
 
        <h4>Proteomics</h4>
        <ul>
          <li><b><i>Aspergillus clavatus</i> :</b> <a href="/a/app/record/dataset/DS_cbdcb0ce28">Proteome changes in response to human serum</a> &ndash; 
            Activity-based protein profiling was used to identify serum-responsive proteins over time (24 and 48 hrs).
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/23599423" target="_blank">Wiedner et al. 2013</a>.
          </li>

          <li><b><i>Aspergillus fischeri</i> :</b> <a href="/a/app/record/dataset/DS_acb29292bb">Proteome changes in response to human serum</a> &ndash; 
            Activity-based protein profiling was used to identify serum-responsive proteins over time (24 and 48 hrs).
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/23599423" target="_blank">Wiedner et al. 2013</a>.
            </li>

          <li><b><i>Aspergillus fumigatus</i> :</b> <a href="/a/app/record/dataset/DS_cb8fced45f">Proteome changes in response to human serum</a> &ndash; 
            Activity-based protein profiling was used to identify serum-responsive proteins over time (24 and 48 hrs).
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/23599423" target="_blank">Wiedner et al. 2013</a>.
            </li>
        </ul>
    <br>
        
        <h4>Phenotype data</h4>
             
             <ul>
             <li><a href="/a/app/record/dataset/DS_7cdba4f78a">Manually Curated <i>Aspergillus</i> Phenotypes</a> &ndash; 
                 The currently existing phenotypes for <i>Aspergillus</i> species were manually curated by <a href="http://www.aspgd.org" target="_blank">AspGD</a> 
                 from published literature describing phenotypes obtained from mutant studies. These phenotypic annotations were 
                 reformatted at FungiDB in-line with VEuPathDB’s approach for phenotype curation. We have curated and added 
                 phenotypes from additional publications to the currently existing dataset for <i>Aspergillus fumigatus</i> Af293 
                 and <i>Aspergillus nidulans</i> FGSC A4 in this release.    
             </li>
             </ul>
    <br>
               
        <style>
           #Phenotype_Bld51Asp table, #Phenotype_Bld51Asp td, #Phenotype_Bld51Asp th, #Phenotype_Bld51Asp tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #Phenotype_Bld51Asp {
           margin-left : 5 em;
           }
         </style> 
           <table id="Phenotype_Bld51Asp"> 
           <tr>
             <th>Species</th>
             <th>Phenotype mapping</th>
             <th>Genes with phenotypes</th>
           </tr>
           <tr>
             <td><i>Aspergillus fumigatus</i> Af293</td>
             <td>51</td>
             <td>12</td>
           </tr>
           <tr>
             <td><i>Aspergillus nidulans</i> FGSC A4</td>
             <td>80</td>
             <td>5</td>
           </tr>
          </table>
    <br>
          
        <ul>
             <li><a href="/a/app/record/dataset/DS_43c139682a">Manually Curated <i>Cryptococcus</i> Phenotypes</a> &ndash; 
                 The phenotypes for <i>Cryptococcus neoformans</i> JEC21 and H99 and <i>Cryptococcus gattii</i> WM276 genomes 
                 are manually curated from published literature describing phenotypes obtained from mutant studies. We have 
                 curated and added phenotypes from additional publications to the currently existing phenotypic dataset for 
                 <i>C. neoformans</i> H99 in this release. These curated publications include large-scale knockout collections 
                 from <a href="https://pubmed.ncbi.nlm.nih.gov/32251295/" target="_blank">Lee et al. 2020</a> and 
                 <a href="https://pubmed.ncbi.nlm.nih.gov/32839469/" target="_blank">Jin et al. 2020</a> among several others.  
             </li>
             </ul>
     <br>
        
        <style>
           #Phenotype_Bld51Cryp table, #Phenotype_Bld51Cryp td, #Phenotype_Bld51Cryp th, #Phenotype_Bld51Cryp tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #Phenotype_Bld51Cryp {
           margin-left : 5 em;
           }
         </style> 
           <table id="Phenotype_Bld51Cryp"> 
           <tr>
             <th>Species</th>
             <th>Phenotype mapping</th>
             <th>Genes with phenotypes</th>
           </tr>
           <tr>
             <td><i>Cryptococcus neoformans</i> var grubii H99</td>
             <td>4891</td>
             <td>534</td>
           </tr>
          </table>
    <br>
          
          <ul>
             <li><a href="/a/app/record/dataset/DS_43c139682a">Manually Curated <i>Magnaporthe</i> Phenotypes</a> &ndash; 
                 The phenotypes for <i>Magnaporthe oryzae</i> 70-15 are manually curated from published literature describing 
                 phenotypes obtained from mutant studies. These phenotypes have been manually curated in-line with VEuPathDB's 
                 approach for phenotype annotation and integrated into FungiDB. We followed an &quot;entity &#43; quality&quot; 
                 approach and curated phenotypes with multiple ontologies: Phenotypic Quality Ontology (PATO) for &quot;quality&quot; 
                 terms and Gene Ontology (GO), Fungal Gross Anatomy Ontology (FAO), Chemical Entities of Biological Interest (ChEBI) 
                 and Ascomycete Phenotype Ontology (APO) for &quot;entity&quot; terms. We have curated and integrated phenotypes for 
                 bZIP and Zn2Cys6 transcription factors from four publications in this release. 
             </li>
             </ul>
	<br>
	
        <style>
           #Phenotype_Bld51Mag table, #Phenotype_Bld51Mag td, #Phenotype_Bld51Mag th, #Phenotype_Bld51Mag tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #Phenotype_Bld51Mag {
           margin-left : 5 em;
           }
         </style> 
           <table id="Phenotype_Bld51Mag"> 
           <tr>
             <th>Species</th>
             <th>Phenotype mapping</th>
             <th>Genes with phenotypes</th>
           </tr>
           <tr>
             <td><i>Magnaporthe oryzae</i> 70-15</td>
             <td>1945</td>
             <td>129</td>
           </tr>
          </table>
          <br>
             
  		<h4>Updated Functional Annotation</h4>
             <a href="https://docs.google.com/spreadsheets/d/1mqHh7CEIuwgBnfLA2x1ZnyqKCn8YWXG6mGSEiKUNBjU/edit?usp=sharing" target="_blank">Summary of annotation updates</a>
               which include product descriptions, GO terms, gene names, synonyms, publications and notes from curators. These functional annotations are current with GeneDB January 2021 GFF versions. <br><br>
             
             <style>
                #FunctAnnot_Bld51 table, #FunctAnnot_Bld51 td, #FunctAnnot_Bld51 th, #FunctAnnot_Bld51 tr {
                text-align : center;
                padding-left: 5px;
                padding-right: 5px;
                padding-top: 5px;
                padding-bottom: 5px;
                border: 1px solid black;
                }
                #FunctAnnot_Bld51 {
                margin-left : 5 em;
                }
              </style> 
                <table id="FunctAnnot_Bld51"> 
                <tr>
                  <th>FungiDB Genome Version</th>
                  <th>Download GeneDB GFF file</th>
                  <th>Genes Updated</th>
                </tr>
                <tr>
                  <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="http://fungidb.org/fungidb/app/record/dataset/DS_8f0322af73"><i>Cryptococcus neoformans</i> H99</a></td>
                  <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-01/CneoformansH99/CneoformansH99.gff.gz">GeneDB version January, 2021</a></td>
                  <td>777</td>
                </tr>
                <tr>
                  <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on FungiDB."><a href="https://fungidb.org/fungidb/app/record/dataset/DS_4d171eb9ea"><i>Cryptococcus neoformans</i> JEC21</a></td>
                  <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-01/CneoformansneoformansJEC21/CneoformansneoformansJEC21.gff.gz">GeneDB version January, 2021</a></td>
                  <td>1125</td>
                </tr>
                </table>
                <br>    
 </div>
</section>
        
<section>
  <h3>New features in {{project}} 50</h3>
  <div style="margin-left: 3em;">
   <ul>
    <li><a href="https://orthomcl.org/orthomcl/app/" target="_blank">OrthoMCL DB</a> now shares web architecture with other VEuPathDB sites. The updated and streamlined interface emphasizes easy access to tools and help information.  </li>
      <ul>
        <li>OrthoMCL now includes orthology information for <a href="https://orthomcl.org/orthomcl/app/release-summary" target="_blank">564 species</a>, including all of the VEuPathDB organisms with annotated genomes. </li>
        <li>The redesigned home page includes a new site-wide search for easily finding groups, proteins or documents. A categorized list of Searches appears on the left of the home page and the center section is devoted to tutorials for getting you started with OrthoMCL features.</li>
        <li>Group and protein record pages are redesigned and include a contents section for navigating to the data you are interested in. </li>
        <li>For more information see the OrthoMCL DB <a href="https://orthomcl.org/orthomcl/app/static-content/OrthoMCL/about.html" target="_blank">About</a> page, the <a href="https://orthomcl.org/orthomcl/app/static-content/OrthoMCL/faq.html" target="_blank">FAQ</a> page or complete this <a href="{{'/documents/OrthoMCL_tutorial1.pdf' | absolute_url}}" target="_blank">tutorial</a>. </li>
      </ul>
    </ul>
  </div>
</section>



<section>
  <aside>
    <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary. Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.   When search parameters change with a new release, we invalidate (Ø) the search and ask you to rerun it.  When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
