---
permalink: VEuPathDB/about
title: About EuPathDB
tags: [general]
---
<div class="static-content">

<a id="about_general"></a>
<h3>General Information</h3>
The Eukaryotic Pathogen, Vector and Host Informatics Resource (VEuPathDB) is one of two <a href="https://www.niaid.nih.gov/research/bioinformatics-resource-centers">bioinformatics resource centers (BRCs)</a> funded by the US National Institute of Allergy and Infectious Diseases (NIAID) - see <a href=#about_funding>funding section below</a>. The BRC program was initiated in 2004 to provide public access to computational platforms and analysis tools enabling collection, management, integration and mining of genomic information and other large-scale datasets relevant to infectious disease pathogens including their interaction with mammalian hosts and invertebrate vectors of disease. Two BRCs are currently funded:<br>
<br><ul>
<li>VEuPathDB focuses on eukaryotic pathogens and invertebrate vectors of infectious diseases.</li>
<li><a href="https://patricbrc.org">PATRIC</a> & <a href="https://www.viprbrc.org/">VIPR</a> focus on bacterial and viral pathogens.</li>
</ul><br>


In addition to support from NIAID, VEuPathDB also received funding from the Wellcome Trust (UK) to support informatics efforts focusing on kinetoplastida and fungal organisms with special emphasis on improving functional annotation for select genome sequences and families of genes.  
<br><br>

VEuPathDB provides access to diverse genomic and other large scale datasets related to eukaryotic pathogens and invertebrate vectors of disease (see <a href="/a/app/search/organism/GenomeDataTypes?autoRun">Data Summary</a>).
Organisms supported by this resource include (but are not limited to) the NIAID list of <a href="https://www.niaid.nih.gov/research/emerging-infectious-diseases-pathogens">emerging and re-emerging infectious diseases</a>.
<br><br>
Component web sites are constructed using a common infrastructure and standard data analysis and loading procedures, allowing the use of <a href="http://VEuPathDB.org">VEuPathDB</a> as a single point of entry for each (or all) of these community resources, and the opportunity to leverage orthology for searches across taxa.
<br><br><ul>
<li><a href="/a/processQuestion.do?questionFullName=OrganismQuestions.GeneMetrics">Gene Metrics Table</a> summarizes the number of annotated genes for the organisms currently supported by VEuPathDB.</li>
<li><a href = "/a/processQuestion.do?questionFullName=OrganismQuestions.GenomeDataTypes">Genome Summary</a> provides a list of all organisms available in VEuPathDB.
</li>
<li><a href="/a/app/search/dataset/AllDatasets/result">Data Sets</a> provides a list of all information integrated into VEuPathDB, with relevant references.</li>

</ul>

<a id="about_funding"/></a>
<h3>Funding</h3>
The Eukaryotic Pathogen, Vector and Host Informatics Resources (VEuPathDB) is funded by the National Institute of Allergy and Infectious Diseases (NIH / DHHS) under Contract No. NIH HHS 75N93019C00077. 
<br>
VEuPathDB also receives funding from the Wellcome Trust (UK) to support informatics efforts focusing on kinetoplastida and fungal organisms with special emphasis on improving functional annotation of genomes.  Grant numbers: 212929/Z/18/Z  and 218288/Z/19/Z. 

<a id="about_use"/></a>
<h3>Data Access Policy</h3>
All data on these websites are provided freely for public use, through the contributions of many researchers involved in generating genome sequences, functional genomics datasets, and additional information. These data often derive from ongoing research, and are not guaranteed to be accurate. When using data obtained from VEuPathDB, it is important to cite the original publications and contributors (see above). Please see <a href=#about_citing>Citing VEuPathDB</a>.


<a id="about_citing"/></a>
<h3>Citing VEuPathDB in Publications and Presentations</h3>
If you use a VEuPathDB resource, we invite you to please cite the most relevant publication. This <a href="https://pubmed.ncbi.nlm.nih.gov/?term=(%22eupathdb%22%5BTitle%5D%20OR%20%22plasmodb%22%5BTitle%5D%20OR%20%22toxodb%22%5BTitle%5D%20OR%20%22giardiadb%22%5BTitle%5D%20OR%20%22amoebadb%22%5BTitle%5D%20OR%20%22microsporidiadb%22%5BTitle%5D%20OR%20%22tritrypdb%22%5BTitle%5D%20OR%20%22vectorbase%22%5BTitle%5D%20OR%20%22fungidb%22%5BTitle%5D%20OR%20%22apidb%22%5BTitle%5D%20OR%20%22cryptodb%22%5BTitle%5D%20OR%20%22piroplasmadb%22%5BTitle%5D%20OR%20%22trichdb%22%5BTitle%5D%20OR%20%22orthomcl%22%5BTitle%5D)%20AND%20(Roos%20%5Bauthor%5D%20OR%20kissinger%20%5Bauthor%5D%20OR%20collins%20%5Bauthor%5D)&sort=date">PubMed filter</a> provides a list of the most recent VEuPathDB publications.<br><br>
Please note that much of the data in EuPathDB is provided by independent researchers, please cite them if you use their data. See <a href="/a/app/search/dataset/AllDatasets/result">Data Sets</a> for a list of all all information integrated into VEuPathDB, and related publications.<br><br>
For acknowledgements in presentations, you may wish to use one or more of the following logos (right/control click to copy):


<!--<br><br>
When citing EuPathDB, please use:
<br>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/27903906">EuPathDB: the eukaryotic pathogen genomics database resource.</a>
Cristina Aurrecoechea; Ana Barreto; Evelina Y. Basenko; John Brestelli; Brian P. Brunk; Shon Cade; Kathryn Crouch; Ryan Doherty; Dave Falke; Steve Fischer; Bindu Gajria; Omar S. Harb; Mark Heiges; Christiane Hertz-Fowler; Sufen Hu; John Iodice; Jessica C. Kissinger; Cris Lawrence; Wei Li; Deborah F. Pinney; Jane A. Pulman; David S. Roos; Achchuthan Shanmugasundram; Fatima Silva-Franco; Sascha Steinbiss; Christian J. Stoeckert Jr; Drew Spruill; Haiming Wang; Susanne Warrenfeltz; Jie Zheng. 
Nucleic Acids Research 2017 doi: 10.1093/nar/gkw1105
<br>-->
<br><br>
<table style="width: 100%" border="1">
  <tbody>
    <tr>
      <td style="text-align: center; vertical-align: middle; background-color: white;"><img
          src="/a/images/VEuPathDB/Logos-color-text-web/VEuPathDB.png"
          alt=""
          style="width: 361px; height: 75px;"></td>
      
      <td style="text-align: center; vertical-align: middle; background-color: white;"><img
          src="/a/images/VEuPathDB/Logos-color-text-web/amoebaDB.png"
          alt=""
          style="width: 361px; height: 75px;"></td>
    </tr>
    <tr>
      <td style="text-align: center; vertical-align: middle; background-color: white;"><img
          src="/a/images/VEuPathDB/Logos-color-text-web/cryptoDB.png"
          alt=""
          style="width: 361px; height: 75px;"></td>
      <td style="text-align: center; vertical-align: middle; background-color: white;"><img
          src="/a/images/VEuPathDB/Logos-color-text-web/fungiDB.png"
          alt=""
          style="width: 361px; height: 75px;"></td>
    </tr>
    <tr>
      <td style="text-align: center; vertical-align: middle; background-color: white;"><img
          src="/a/images/VEuPathDB/Logos-color-text-web/giardiaDB.png"
          alt=""
          style="width: 361px; height: 75px;"></td>
      <td style="text-align: center; vertical-align: middle; background-color: white;"><img
          src="/a/images/VEuPathDB/Logos-color-text-web/hostDB.png"
          alt=""
          style="width: 361px; height: 75px;"></td>
    </tr>
    <tr>
      <td style="text-align: center; vertical-align: middle; background-color: white;"><img
          src="/a/images/VEuPathDB/Logos-color-text-web/microsporidiaDB.png"
          alt=""
          style="width: 361px; height: 63px;"></td>
      <td style="text-align: center; vertical-align: middle; background-color: white;"><img
          src="https://orthomcl.org/orthomcl/images/OrthoMCL/title_s.png"
          alt=""
          style="width: 361px; height: 75px;"></td>
    </tr>
    <tr>
      <td style="text-align: center; vertical-align: middle; background-color: white;"><img
          src="/a/images/VEuPathDB/Logos-color-text-web/piroplasmaDB.png"
          alt=""
          style="width: 361px; height: 75px;"></td>
      <td style="text-align: center; vertical-align: middle; background-color: white;"><img
          src="/a/images/VEuPathDB/Logos-color-text-web/plasmoDB.png"
          alt=""
          style="width: 361px; height: 75px;"></td>
    </tr>
    <tr>
      <td style="text-align: center; vertical-align: middle; background-color: white;"><img
          src="/a/images/VEuPathDB/Logos-color-text-web/toxoDB.png"
          alt=""
          style="width: 361px; height: 75px;"></td>
      <td style="text-align: center; vertical-align: middle; background-color: white;"><img
          src="/a/images/VEuPathDB/Logos-color-text-web/trichDB.png"
          alt=""
          style="width: 361px; height: 75px;"></td>
    </tr>
    <tr>
      <td style="text-align: center; vertical-align: middle; background-color: white;"><img
          src="/a/images/VEuPathDB/Logos-color-text-web/tritrypDB.png"
          alt=""
          style="width: 361px; height: 75px;"></td>
      <td style="text-align: center; vertical-align: middle; background-color: white;"><img
          src="/a/images/VEuPathDB/Logos-color-text-web/vectorbase.png"
          alt=""
          style="width: 361px; height: 75px;"></td>
    </tr>
  </tbody>
</table>
<br><br>
Additional resources using the same infrastructure: <a href="https://microbiomedb.org">MicrobiomeDB</a> and <a href="https://clinepidb.org">ClinEpiDB</a>.
  
<!--<a id="about_protocols_methods"/></a>
<h3>Finding Integrated Data</h3>
To find data integrated into VEuPathDB and to learn more about them visit the <a href="/a/app/search/dataset/AllDatasets/result">Data Sets</a> page.
 <h4>EST Assemblies</h4>
EST assemblies are transcript predictions based on genome alignments. 
ESTs are obtained from dbEST and from researchers who generously provide 
additional libraries. ESTs are cleaned prior to use; leading or trailing 
poly A or T sequences are trimmed and vector sequences are masked. ESTs 
greater than 50 bases long are used for BLAT alignment (Kent,W.J, 2002. 
Genome Research. 12:656-664) to same species genome sequences with a 
threshold of 90% minimum sequence identity, minimum score of 30, and the 
maximum intron length adjusted for the species. The ESTs are then 
clustered into disjoint sets based on the proximity and quality of their 
alignments, only the best alignments with a 95% minimum percent identity 
are used for clustering. Clusters of ESTs are then assembled and 
consensus sequences produced using the CAP4 assembly program (X. Huang 
and Madan, A., 1999. Genome Res. 9:868-877). Finally, the predicted 
consensus transcripts are themselves aligned to the genome.
-->
 
<a id="about_privacy"/></a>
<h3>Website Privacy Policy</h3>
We do not use or share any of your personal information for any purpose unrelated to the functionality of the websites; however, we do collect some information to help us understand how our sites are being used and to improve community support. See  
<a href="/a/app/static-content/privacyPolicy.html">Website Privacy Policy</a>.

</div>
