---
layout: post
title: HostDB 42 Released
category: [HostDB]
---

<p><strong>We are pleased to announce the release of HostDB 42</strong></p>
    
    <section>
    <h3><b>Release 42 Webinar</b></h3>
    <div style="margin-left: 3em;">
      Join us on March 20, 2019 10:00-11:00 AM EDT for our Release 42 Highlights webinar where we will discuss and 
      demonstrate new features and data available in release 42 of EuPathDB sites. 
      <a href="https://attendee.gotowebinar.com/register/6755367952348158722">Register here</a>
    </div>
    </section>
      
      <section>
      <h3><b>New data in HostDB 42</b></h3>
      <div style="margin-left: 3em;">
        <h4>Genome Sequence and Annotation</h4>
          <ul>
          <li>
            <a href="https://hostdb.org/hostdb/app/record/dataset/DS_f14fc7e764"><i>Bos taurus</i> breed Hereford</a> &mdash; 
            The genome sequence and annotation for the cow genome appears for the first time in HostDB. 
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/19393038" target="_blank">Zimin et al.</a>  
           </li>
         </ul> 
       <h4>Transcriptomics</h4>
          <ul>
          <li>
            <a href="https://hostdb.org/hostdb/app/record/dataset/DS_c8182bae60">Host cell transcriptome in bovine cells infected with <i>Cryptosporidium parvum</i></a> &mdash; <br>
            Transcriptome of MDBK cells during <i>C. parvum</i> intracellular infection. Reads were mapped to the <i>Bos taurus</i>, breed Hereford reference genome. 
            <a href="http://vetprofiles.tufts.edu/faculty/giovanni-f-widmer" target="_blank">Widmer lab</a> unpublished. 
           </li>
         </ul>   
      </section>
      
      <section>
      <h3><b>New features in HostDB 42</b></h3>
      <div style="margin-left: 3em;">
         <ul>
           <li>
             Updated Alignment Tool on Gene Pages &ndash; <br>
             The Clustal Omega alignment tool in the 'Orthologs and Paralogs within EuPathDB' table now 
             accommodates protein, CDS and genomic sequence alignments. To run Clustal Omega, select genes from the table. Then choose the sequence 
             type and initiate the alignment with the 'Run Clutal Omega for selected genes' button. Output formats include Mismatches highlighted, FASTA, PHYLIP, STOCKHOLM, 
             and VIENNA. This tool replaces the 'Retrieve multiple 
             sequence alignment or multi-FASTA' tool.
           </li>
           <li>
             Beta access to JBrowse &ndash; <br>
             A beta implementation of the 
             <a href="https://jbrowse.org/" target="_blank">JBrowse Genome Browser</a>, is available from gene pages. Use the 
             JBrowse Genome Browser link under the graphic Shortcuts on the gene page. 
             <a href="/assets/JBrowseLink.png">Example Link Location</a>
           </li>
           <li>
             New Gene Page STRING tool &ndash; <br>
             The 'Protein features and properties' section now includes an option to submit the protein sequence 
             to the <a href="https://string-db.org/" target="_blank">STRING</a> functional protein association network. 
             <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5210637/" target="_blank">Szklarczyk et al</a>.
           </li>
           <li>
             Result Table Improvements &ndash; <br>
             Result column configurations and sorting are now saved for each strategy step and will be retained 
             when you come back or when you share a strategy with someone else.
           </li>
           <li>
             New Options for FASTA downloads &ndash; <br>
             The header line of FASTA formatted downloads can be configured to contain only the gene ID.  The sequence can 
             be configured to appear on a signle line or the standard 60 characters per line. 
           </li>
         </ul>
       </div> 
      </section>
      
      <section>
         <aside>
           <br><br>Note: Our bimonthly database releases incorporate new data and correct old data when necessary.
           Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
           When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
           When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
         </aside>
      </section>
