---
layout: post
title: HostDB 52 Released
category: [HostDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 52.</h3>

<section>
  <h3>Release 52 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on May 27, 2021 at 10am EDT for our Release 52 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 52.  
    <a href="https://attendee.gotowebinar.com/register/8223585429100223502" target="_blank">Register here</a>  
  </div>
</section>

<section>
  <h3> New data in HostDB 52</h3>
  <div style="margin-left: 3em;">
    <h4>EBI Pipeline</h4>
    {{project}} 52 represents the first release that employs the EBI pipeline for analyzing genomes and RNA-Seq data. (<a href="/a/app/static-content/methods.html">{{project}} Analysis Methods</a>) All genomes were run through the EBI pipeline, updating all analyses such as InterproScan and repeat identification.  While much of the EBI pipeline is the same as our previous in-house pipeline, these differences are noted:
    <ul>
      <li>Expression values calculated from RNA-Seq data are now presented as TPM (Transcripts Per Kilobase Million) instead of FPKM (Fragments Per Kilobase Million) or RPKM (Reads Per Kilobase Million). <a href="https://rna-seqblog.com/rpkm-fpkm-and-tpm-clearly-explained/" target="_blank">Learn more</a></li>
      <li>Signal peptide predictions are now made with <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/" target="_blank">SignalP-4.1</a>, updated from <a href="http://www.cbs.dtu.dk/services/SignalP-3.0/" target="_blank">SignalP-3.0</a>.</li>
      <li>All pseudogenes now include automatically predicted UTRs.</li>
    </ul>
    <br>
    <h4>Annotated Genome</h4>
      <ul>
        <li><a href="/a/app/record/dataset/DS_033cf9c74d"> <i>Macaca fascicularis</i> REF</a> &ndash; This annotated genome appears for the first time in HostDB. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31329596">Li et al. 2019.</a></li>
      </ul>
    <br>
    <h4>Transcriptomics</h4>
      <ul>
        <li><a href="/a/app/record/dataset/DS_833d16a0c8">Host response to early and mid-stage <i>P. berghei</i> liver infection</a> &ndash; Transcriptome sequencing (RNA-Seq) analysis throughout the <i>Plasmodium berghei</i> liver stage, covering as early as 2 hrs postinfection (hpi) and extending to 48 hpi in HuH7 and HepG2 cells. <a href="http://www.ncbi.nlm.nih.gov/pubmed/32019802"> Toro-Moreno et al. 2020</a>. 
          <ul>
            <li>Associated PlasmoDB data: <a href="https://plasmodb.org/plasmo/app/record/dataset/DS_7dae78ae0b">Transcriptome during early and mid-stage <i>P. berghei</i> liver infection</a></li>.
          </ul>
        </li>
        <li><a href="/a/app/record/dataset/DS_d1fb08f3c2">Transcriptional landscape of <i>Cryptococcus</i>-host response</a> &ndash; Animal infection model using the non-human primate cynomolgus monkey <i>Macaca fascicularis</i> to study host response to infection. <a href="http://www.ncbi.nlm.nih.gov/pubmed/31329596"> Li et al. 2019</a>.  
          <ul>
            <li>Associated HostDB data: <a href="/a/app/record/dataset/DS_a28e7f6c83">Transcriptional landscape of Cryptococcus-host response</a>, <i>Mus musculus</i> C57BL6J</li>.
            <li>Associated FungiDB data: <a href="https://fungidb.org/fungidb/app/record/dataset/DS_5f2f1c9822">Transcriptional landscape of Cryptococcus-host response</a>, <i>Mus musculus</i> and <i>Macaca fascicularis</i></li>.
          </ul>
        </li>
      </ul>
  </div>
</section>

<section>
  <h3>New features in {{project}} 52</h3>
  <div style="margin-left: 3em;">
   <ul>
    <li>The gene pages of <a href="https://en.wikipedia.org/wiki/Pseudogene" target="_blank">pseudogenes</a> are modified to improved clarity: <a href="{{'/assets/images/pseudogene-page.png' | absolute_url}}">(helpful image)</a>
      <ul>
        <li>The Transcripts table in the Gene Models section now indicates that no protein products are produced. </li>
        <li>The exons in JBrowse views of pseudogenes are colored gray.  Blue represents genes on the forward strand and red represents genes on the reverse strand. </li>
        <li>The Orthology and Synteny section indicates that orthology is not determined for pseudogenes.</li>
        <li>Protein sequences are no longer available from the Sequences section.</li>
      </ul>
    </li>
    <li>Reference organisms are indicated in the Organism parameter on search pages.  VEuPathDB chooses a representative genome from each species and aligns all functional genomics data to the reference.  In doing so, all data can be viewed and compared in the genome browser against the reference.  In some cases, data are aligned to the source organism in addition to the reference. (helpful image)</li>
    <li>The algorithm for transferring EC Numbers between orthologs is updated. We previously predicted EC numbers, but now we offer scores and statistics to help users determine the confidence in this prediction. The new algorithm compares a gene's protein sequence to the protein sequences of other genes in the same ortholog group. A gene is inferred to receive a specific EC number based on how well its sequence matches genes in it's ortholog group that have already been assigned this EC number.</li>
    <li>Community annotations made in {{project}} via Apollo are now represented on gene pages in the Gene Models graphic. In addition, links from the gene page to Apollo are offered to facilitate community annotations. <a href="{{'/assets/images/51-apollo-news.png' | absolute_url}}">Example image.</a>  Please see our <a href="https://youtu.be/RMXZDBMh8Fo" target="_blank">Apollo webinar</a> for instruction. </li>
    </ul>
  </div>
</section>

<section>
  <aside>
   <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary.  Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits. When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it. When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
