---
layout: post
title: Beta-release of TriTrypDB 1.1
category: [EuPathDB,VEuPathDB,TriTrypDB]
---

We are pleased to announce the availability of a beta-release version of TriTrypDB 1.1, accessible at <a href="http://beta.TriTrypDB.org">http://beta.TriTrypDB.org</a>.  Feel free to forward this link to others.  <br><br>Highlights include:<br><br>
<ul>
<li>The <i>L. tarentolae</i> genome has been provided by the CIHR Group on host pathogen interactions and is available for similarity searches and browsing for open reading frames and BLASTX alignments against the NR protein database.
<li>A microarray experiment examining the axenic promastigote-to-amastigote differentiation of <i>L. donovani</i> was provided by researchers at the Seattle Biomedical Research Institute and Technion Institute and is available for query and display on record pages.
<li>A new search strategy interface that enables users to more easily construct complex searches is released for the first time in TriTrypDB 1.1.
<li>&#39;User Comments&#39; have been updated to enable submission of supporting evidence including pubmed identifiers and images (or other files).  Comments may be added to any gene or sequence page, enhancing available annotation.  Such comments are visible and searchable by others, and are automatically forwarded to the annotation team for incorporation into the official annotation<br>
<li>Files can now be uploaded and provided to the community via TriTrypDB.
</ul>
<br>
By providing access to TriTrypDB 1.1 prior to its official release, we hope to obtain valuable feedback to facilitate the debugging process, in addition to making new data and functionality available to the community as early as possible.  Please click on the &#39;Contact Us&#39; link at the bottom of each page to indicate anything that might help to make the site better: bugs, help requests, features that are unclear or take too long to run, suggestions for additional tools or datasets that might be useful to incorporate, etc.  Where appropriate, please also add to the &#39;User Comments&#39; on each gene page, noting inaccurate (or accurate) gene models, phenotype information, expression data, clone availability, etc.
