---
layout: post
title: TriTrypDB 27 Released
category: [TriTrypDB]
---

<p><strong>We are pleased to announce the release of TriTrypDB 27</strong></p>

    <section>
      <h3>New data in this release</h3>
      
      <section>
        <h4>New RNA Sequencing data</h4>
        <ul>
          <li>
            <a href="http://tritrypdb.org/tritrypdb/app/record/dataset/DS_fd602b26e6">
                                 mRNA Decay in Bloodstream and Procyclic Forms</a> &mdash;
            The decay of <i>T. brucei</i> TREU 927 mRNA in bloodstream and procyclic forms was determined by
            sequencing RNA that was harvested at various times from cultures incubated in ActinomycinD to inhibit transcription.  
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/25145465">Fadda et al.</a>
          </li>
        </ul>
      </section>
      
      <section>
        <h4>New literature references on gene pages</h4>
        <ul>
          <li>
            TriTrypDB gene pages now include the PMID references available from GeneDB's ongoing curation efforts. See the PubMed table in the Annotation 
            section of gene pages for literature references that are current with the GeneDB October 5, gff versions. 
          </li>
        </ul>
      </section>


     <section>
        <h4>Updated Functional Annotation and PMID references</h4>
        
        <aside>
        Functional annotations and PMID references in TriTrypDB Release 27 are current with GeneDB October 5, 2015 gff versions. 
        </aside>
        
        <a href="https://docs.google.com/spreadsheets/d/13xhl0p2kIKiBOeMinMkNa8rStjpF8z2AfqAK0Yxwz_Q/edit#gid=1363153282">Summary of annotation updates</a> which include products, GO terms, EC numbers, gene names, synonyms and PMIDs.
        <br><br>
        
        <style>
           #FunctAnnot_Bld27 table, #FunctAnnot_Bld27 td, #FunctAnnot_Bld27 th, #FunctAnnot_Bld27 tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #FunctAnnot_Bld27 {
           margin-left : 5 em;
           }
         </style> 
           <table id="FunctAnnot_Bld27"> 
           <tr>
             <th>TriTrypDB Genome Version</th>
             <th>Download GeneDB GFF file</th>
             <th>Genes Updated</th>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on TriTrypDB."><a href="http://tritrypdb.org/a/getDataset.do?datasets=lbraMHOMBR75M2904_primary_genome_RSRC"><i>L. braziliensis</i> MHOM/BR/75/M2904 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/pathogens/gff3/2015-10/Lbraziliensis.noseq.gff3.gz">Lbraziliensis.noseq.gff3.gz</a></td>
             <td>87 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on TriTrypDB."><a href="http://tritrypdb.org/a/getDataset.do?datasets=ldonBPK282A1_primary_genome_RSRC"><i>L. donovani</i> BPK282A1 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/pathogens/gff3/2015-10/Ldonovani_BPK282A1.noseq.gff3.gz">Ldonovani_BPK282A1.noseq.gff3.gz</a></td>
             <td>85 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on TriTrypDB."><a href="http://tritrypdb.org/a/getDataset.do?datasets=linfJPCM5_primary_genome_RSRC"><i>L. infantum</i> JPCM5 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/pathogens/gff3/2015-10/Linfantum.noseq.gff3.gz">Linfantum.noseq.gff3.gz</a></td>
             <td>85 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on TriTrypDB."><a href="http://tritrypdb.org/a/getDataset.do?datasets=lmajFriedlin_primary_genome_RSRC"><i>L. major</i> strain Friedlin </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/pathogens/gff3/2015-10/Lmajor.noseq.gff3.gz">Lmajor.noseq.gff3.gz</a></td>
             <td>114 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on TriTrypDB."><a href="http://tritrypdb.org/a/getDataset.do?datasets=lmexMHOMGT2001U1103_primary_genome_RSRC"><i>L. mexicana</i> MHOM/GT/2001/U1103 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/pathogens/gff3/2015-10/Lmexicana.noseq.gff3.gz">Lmexicana.noseq.gff3.gz</a></td>
             <td>87 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on TriTrypDB."><a href="http://tritrypdb.org/a/getDataset.do?datasets=tbruTREU927_primary_genome_RSRC"><i>Trypanosoma brucei</i> 927 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/pathogens/gff3/2015-10/Tbruceibrucei927.noseq.gff3.gz">Tbruceibrucei927.noseq.gff3.gz</a></td>
             <td>792 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on TriTrypDB."><a href="http://tritrypdb.org/a/getDataset.do?datasets=tbrugambienseDAL972_primary_genome_RSRC"><i>T. brucei gambiense</i> DAL972</a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/pathogens/gff3/2015-10/Tbruceigambiense.noseq.gff3.gz">Tbruceigambiense.noseq.gff3.gz</a></td>
             <td>121 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on TriTrypDB."><a href="http://tritrypdb.org/a/getDataset.do?datasets=tconIL3000_primary_genome_RSRC"><i>Trypanosoma congolense</i> IL3000</a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/pathogens/gff3/2015-10/Tcongolense.noseq.gff3.gz">Tcongolense.noseq.gff3.gz</a></td>
             <td>105 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on TriTrypDB."><a href="http://tritrypdb.org/a/getDataset.do?datasets=tcruCLBrener_primary_genome_RSRC"><i>Trypanosoma cruzi</i> CL Brener</a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/pathogens/gff3/2015-10/Tcruzi.noseq.gff3.gz">Tcruzi.noseq.gff3.gz</a></td>
             <td>196 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on TriTrypDB."><a href="http://tritrypdb.org/a/getDataset.do?datasets=tvivY486_primary_genome_RSRC"><i>Trypanosoma vivax</i> Y486 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/pathogens/gff3/2015-10/Tvivax.noseq.gff3.gz">Tvivax.noseq.gff3.gz</a></td>
             <td>100 genes</td>
           </tr>
           </table>
        <br><br>
        

        
      </section>
      </section>
      
      <section>
     <h3>New Features</h3>
      <section>
       <h4>New Tutorials</h4>
        <ul>
          <li>
            <a href="https://youtu.be/z6c_FhMCdPo">Exploring expression distributions with our Percentile Ranking searches</a> - 
            Percentile ranking is one way that EuPathDB analyzes expression data.  This tutorial 
            describes our percentile ranking of expression data and the searches that we offer based on that analysis.
          </li>
          <li>
            <a href="https://youtu.be/Yk9kkaPAmAE">Finding Differentially expressed genes based on Fold Change with a P-value</a> - 
            One way EuPathDB analyzes expression is to determine statistically significant differential gene expression using the  
            Fisher's exact test. This tutorial describes our Fold Change w/ P-value searches that return differentially 
            expressed genes based on the level of statistical confidence that you choose.
          </li>
         </ul>
      </section>

      
      
      <aside>
        <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary.
        Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
        When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
        When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
      </aside>

    </section>
