---
layout: post
title: TriTrypDB 52 Released
category: [TriTrypDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h4>We are pleased to announce the release of {{project}} 52.</h4>

<section>
  <h3>Release 52 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on May 27, 2021 at 10am EDT for our Release 52 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 52.  
    <a href="https://attendee.gotowebinar.com/register/8223585429100223502" target="_blank">Register here</a>  
  </div>
</section>

<section>
  <h3>New data in {{project}} 52</h3>
  <div style="margin-left: 3em;">
  <h4>EBI Pipeline</h4>
    {{project}} 52 represents the first release that employs the EBI pipeline for analyzing genomes and RNA-Seq data. (<a href="/a/app/static-content/methods.html">{{project}} Analysis Methods</a>) All genomes were run through the EBI pipeline, updating all analyses such as InterproScan and repeat identification.  While much of the EBI pipeline is the same as our previous in-house pipeline, these differences are noted:
    <ul>
      <li>Expression values calculated from RNA-Seq data are now presented as TPM (Transcripts Per Kilobase Million) instead of FPKM (Fragments Per Kilobase Million) or RPKM (Reads Per Kilobase Million). <a href="https://rna-seqblog.com/rpkm-fpkm-and-tpm-clearly-explained/" target="_blank">Learn more</a></li>
      <li>Signal peptide predictions are now made with <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/" target="_blank">SignalP-4.1</a>, updated from <a href="http://www.cbs.dtu.dk/services/SignalP-3.0/" target="_blank">SignalP-3.0</a>.</li>
      <li>All pseudogenes now include automatically predicted UTRs.</li>
    </ul>
    <br>

  <h4>Annotated Genomes</h4>
      <ul>
        <li><a href="/a/app/record/dataset/DS_d67261aa58"><i>Angomonas deanei</i> strain Cavalho ATCC PRA-26</a> &ndash; Nuclear genome based on Oxford Nanopore sequence that resolves into 29 complete or close-to-complete chromosomes <a href="http://www.ncbi.nlm.nih.gov/pubmed/33561222" target="_blank">Davey et al. 2021</a> </li>

        <li><a href="/a/app/record/dataset/DS_6820e060c4"><i>Trypanosoma cruzi</i> strain CL</a> &ndash; Purified DNA was used to prepare shotgun and 8 Kbp mate pair libraries for sequencing on the Roche 454 GS FLX+ platform. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30355302" target="_blank">Bradwell et al. 2018</a> </li>

        <li><a href="/a/app/record/dataset/DS_8f8cc536f5"><i>Trypanosoma cruzi</i> strain G</a> &ndash; Purified DNA was used to prepare shotgun and 3 Kbp mate pair libraries for sequencing on the Roche 454 GS FLX+ platform. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30355302" target="_blank">Bradwell et al. 2018</a> </li>
      </ul>
      <br>

  <h4>Genome sequence assemblies without annotation</h4>
  The following genomes of <i>Trypanosoma cruzi</i> field isolates described in <a href="http://www.ncbi.nlm.nih.gov/pubmed/30424726" target="_blank">Reis-Cunha et al. 2018</a> appear for the first time in TriTrypDB. 
      <ul>
        <li><a href="/a/app/record/dataset/DS_2b427cad62"><i>Trypanosoma cruzi</i> S11</a>  </li>

        <li><a href="/a/app/record/dataset/DS_2988b6c351"><i>Trypanosoma cruzi</i> S15</a>  </li>

        <li><a href="/a/app/record/dataset/DS_ec4c051a85"><i>Trypanosoma cruzi</i> S23b</a>  </li>

        <li><a href="/a/app/record/dataset/DS_535211097a"><i>Trypanosoma cruzi</i> S44a</a>  </li>

        <li><a href="/a/app/record/dataset/DS_12589b3fa3"><i>Trypanosoma cruzi</i> S92a</a>  </li>

        <li><a href="/a/app/record/dataset/DS_7b5ef868e6"><i>Trypanosoma cruzi</i> S154a</a>  </li>

        <li><a href="/a/app/record/dataset/DS_42fa3df153"><i>Trypanosoma cruzi</i> S162a</a>  </li>

        <li><a href="/a/app/record/dataset/DS_8d1444e10c"><i>Trypanosoma cruzi</i> Ycl2</a>  </li>

        <li><a href="/a/app/record/dataset/DS_b7cef49ccd"><i>Trypanosoma cruzi</i> Ycl4</a>  </li>

        <li><a href="/a/app/record/dataset/DS_4b06725810"><i>Trypanosoma cruzi</i> Ycl6</a>  </li>
      </ul>
      <br>

  <h4>Updated Functional Annotation</h4>

        <a href="https://docs.google.com/spreadsheets/d/1HFwfPuIEKZGyUYBWhcJfrw35om_CPQUKB0lNuZqXjt8/edit?usp=sharing" target="_blank">Summary of annotation updates</a> which include products, GO terms, gene names, synonyms, publications, and notes from curators. These functional annotations are current with GeneDB April 2021 GFF versions.
        <br><br>
        Functional annotations in TriTrypDB 52 have been updated for the following genomes:
        <br><br>
        <style>
           #FunctAnnot_Bld52 table, #FunctAnnot_Bld52 td, #FunctAnnot_Bld52 th, #FunctAnnot_Bld52 tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #FunctAnnot_Bld52 {
           margin-left : 5 em;
           }
         </style> 
           <table id="FunctAnnot_Bld52"> 
           <tr>
             <th>TriTrypDB Genome Version</th>
             <th>Download GeneDB GFF file</th>
             <th>Genes Updated</th>
           </tr>
           
           <tr>
            <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on TriTrypDB."><a href="/a/app/record/dataset/DS_839623217a"><i>Leishmania major </i>Friedlin</a></td>
            <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-04/Lmajor/Lmajor.gff.gz" target="_blank">GeneDB version April, 2021</a></td> 
            <td>9 genes</td>
           </tr>
           <tr>
            <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on TriTrypDB."><a href="/a/app/record/dataset/DS_e318592c8e"><i>Leishmania mexicana </i>MHOM/GT/2001/U1103</a></td>
            <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-04/Lmexicana/Lmexicana.gff.gz" target="_blank">GeneDB version April, 2021</a></td> 
            <td>7 genes</td>
           </tr>            
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on TriTrypDB."><a href="/a/app/record/dataset/DS_31282613b1"><i>Trypanosoma brucei </i>927</a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-04/Tbruceibrucei927/Tbruceibrucei927.gff.gz" target="_blank">GeneDB version April, 2021</a></td> 
             <td>275 genes</td>
           </tr>
           <tr>
            <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on TriTrypDB."><a href="/a/app/record/dataset/DS_1a6c8bdaac"><i>Trypanosoma cruzi </i>CL Brener</a></td>
            <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2021-04/Tcruzi/Tcruzi.gff.gz" target="_blank">GeneDB version April, 2021</a></td> 
            <td>206 genes</td>
          </tr>
           </table>
    <br><br>
           
		   
	<h4>Curated User Comments</h4>
       
          Gene page comments that are entered by users are curated at GeneDB and regularly integrated as functional  annotations in TriTrypDB. 103 user comments have been integrated in this release.<br><br>
          
         <style>
           #UComment_Bld52 table, #UComment_Bld52 td, #UComment_Bld52 th, #UComment_Bld52 tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #UComment_Bld52 {
           margin-left : 5 em;
           }
         </style> 
           <table id="UComment_Bld52"> 
           <tr>
             <th>TriTrypDB Genome Version</th>
             <th>Comments Integrated</th>
           </tr>
           <tr>
            <td><a href="/a/app/record/dataset/DS_839623217a"><i>Leishmania major </i>Friedlin</a></td>
            <td>7</td>
           </tr>
           <tr>
             <td><a href="/a/app/record/dataset/DS_31282613b1"><i>Trypanosoma brucei</i> 927</a></td>
             <td>96</td>
           </tr>
          </table>  
  </div>
</section>

<section>
  <h3>New features in {{project}} 52</h3>
  <div style="margin-left: 3em;">
   <ul>
    <li>The gene pages of <a href="https://en.wikipedia.org/wiki/Pseudogene" target="_blank">pseudogenes</a> are modified to improved clarity: <a href="{{'/assets/images/pseudogene-page.png' | absolute_url}}">(helpful image)</a>
      <ul>
        <li>The Transcripts table in the Gene Models section now indicates that no protein products are produced. </li>
        <li>The exons in JBrowse views of pseudogenes are colored gray.  Blue represents genes on the forward strand and red represents genes on the reverse strand. </li>
        <li>The Orthology and Synteny section indicates that orthology is not determined for pseudogenes.</li>
        <li>Protein sequences are no longer available from the Sequences section.</li>
      </ul>
    </li>
    <li>Reference organisms are indicated in the Organism parameter on search pages.  VEuPathDB chooses a representative genome from each species and aligns all functional genomics data to the reference.  In doing so, all data can be viewed and compared in the genome browser against the reference.  In some cases, data are aligned to the source organism in addition to the reference. (helpful image)</li>
    <li>The algorithm for transferring EC Numbers between orthologs is updated. We previously predicted EC numbers, but now we offer scores and statistics to help users determine the confidence in this prediction. The new algorithm compares a gene's protein sequence to the protein sequences of other genes in the same ortholog group. A gene is inferred to receive a specific EC number based on how well its sequence matches genes in it's ortholog group that have already been assigned this EC number.</li>
    <li>Community annotations made in {{project}} via Apollo are now represented on gene pages in the Gene Models graphic. In addition, links from the gene page to Apollo are offered to facilitate community annotations. <a href="{{'/assets/images/51-apollo-news.png' | absolute_url}}">Example image.</a>  Please see our <a href="https://youtu.be/RMXZDBMh8Fo" target="_blank">Apollo webinar</a> for instruction. </li>
    </ul>
  </div>
</section>


<section>
  <aside>
   <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary.  Changes in annotation and new experimental data September slightly alter your search results by increasing or decreasing the number of hits. When search parameters change with a new release, we invalidate (Ø) the search and ask you to rerun it. When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
