---
layout: post
title: PlasmoDB 5.3 is released
category: [EuPathDB,VEuPathDB,PlasmoDB]
---

PlasmoDB 5.3 provides the genome and annotation for the primate malarial parasite <i>Plasmodium knowlesi</i>, provided by the Wellcome Trust Sanger Institute Pathogen Sequencing Unit. This brings the number of Plasmodium genomes available for queries at PlasmoDB up to six. Also provided are expression data from recent papers using gene knockouts to study molecular mechanisms in human (<i>P. falciparum</i>) and rodent (<i>P. berghei</i>) malarial parasites. Queries have been developed for the new expression studies and a summary graphic (P.f.) or table (P.b.) added to the gene pages.
<p>
The PlasmoDB 5.3 web site also has significant improvements to the query and results pages. Many of the queries have been reorganized. For example, the Predicted Functional Interaction query now allows input of a list of genes and a clearer explanation of what to get from the query (e.g., find high confidence predicted interactions between members of the input gene list). The results from this and all queries can be sorted. It is also now possible to add columns of interest (e.g., protein features, GO annotation, even expression characteristics) and sort on them.
<p>
New data and queries in PlasmoDB 5.3:
<ul>
<li>Genome sequence and annotation for <i>Plasmodium knowlesi</i> from the Wellcome Trust Sanger Institute Pathogen Sequencing Unit.  <i>P. knowlesi</i> is included in many of the queries to identify genes, genomics sequences, ESTs, and ORFs by various features.
<li>Microarray studies have been incorporated from a series of 3 papers on gene silencing (Duraisingh et al. Cell 2005) and invasion pathways (Baum et al. PLoS Pathogens 2005; Stubbs et al. Science 2005) in <i>P. falciparum</i>.
<li>A microarray study in <i>P. berghei</i> from a paper on translational repression in gametocytes (Mair et al. Science 2006).
<li>A refined mapping of epitopes to <i>Plasmodium</i> provided by the <a href="http://www.immuneepitope.org/home.do">Immune Epitope Database and Analysis Resource</a>. A query is now provided to identify genes whose encoding protein has an epitope at different levels of confidence.
<li>Updated E.C. number and metabolic pathway assignments, Interpro protein domains and associated GO predictions, MR4 reagent links, homologies to entries in the NRDB protein database and links to structures in PDB.
<li>A new build of ortholog groups from Orthomcl for use in PlasmoDB. The list of genomes represented has increased from 55 to 87  and includes all the organisms covered by the different NIH/NIAID funded Bioinformatics Resource Centers.
</ul>
<p>
New features in PlasmoDB 5.3:
<ul>
<li>Most of the queries have been reorganized to make them more intuitive. Some include a new "Advanced Parameters" section. See for example "Identify Genes based on EST evidence."
<li>The columns displayed in query result pages are improved.  The result pages also offer options for sorting, adding new columns, and moving columns around.
<li>Downloading results is improved. A choice of formats is provided including tab-delimited, text, GFF, and FASTA to meet different needs.
  <ul>
  <li>Tab delimited provides an expanded list of annotations that can be directly exported as an Excel spreadsheet.
   <li>Text provides a detailed summary for each gene including SNPs, Y2H interactions, metabolic pathways, MR4 reagents, protein structures, notes, sequence, and many other choices.
   <li> GFF is a format recognized by many applications such as the genome browser, gbrowse.
  <li> FASTA also provides a common format used for BLAST and other applications. The FASTA option provides the functionality of the SRT (sequence retrieval tool) found in earlier versions of PlasmoDB.
  </ul>
</ul>
