---
layout: post
title: CryptoDB 46 Released
category: [EuPathDB,VEuPathDB,CryptoDB]
---

<p><strong>We are pleased to announce the release of CryptoDB 46</strong></p>
    
    <section>
    <h3><b>Release 46 Webinar</b></h3>
    <div style="margin-left: 3em;">
      Join us on December 3, 2019 at 10:00 am (EST) for our Release 46 Highlights webinar where we will discuss and 
      demonstrate new features and data available in release 46 of EuPathDB sites. 
      <a href="https://attendee.gotowebinar.com/register/1533745455884630285" target="_blank">Register here</a>
    </div>
    </section>
      
      <section>
      <h3><b>New data in CryptoDB 46</b></h3>
      <div style="margin-left: 3em;">
        <h4>Genome Sequence and Annotation</h4>
         <ul>
           <li><a href="/a/app/record/dataset/DS_8bd21e94e1"><i>Cryptosporidium parvum</i> strain IOWA ATCC</a> &mdash; <br>
              Based on PacBio and Illumina sequencing of strain IOWA ATCC by the Wellcome Trust Sanger Center with annotation from the 
			  Kissinger group at University of Georgia.  The sequence and annotation has been submitted to the NCBI GenBank under accession 
			  numbers (Bioproject: PRJNA573722, Biosample: SAMN12822784, WGS: CP044415-CP044422)  The data is provided to CryptoDB pre-publication.
           </li>
         </ul> 
		 <h4>Whole Genome Sequencing of Isolates (Genetic Variation)</h4>
         <ul>
           <li><a href="/a/app/record/dataset/DS_61df67732f"><i>Cryptosporidium parvum</i> WGS of clinical isolates</a> &mdash; <br>
              Whole genome sequence of <i>C. parvum</i> isolates determined by an IMS/Illumina-based method for the WGS of clinical isolates.  
			  <a href="http://www.ncbi.nlm.nih.gov/pubmed/30833731" target="_blank">Nader et al.</a>
           </li>
         </ul> 
         </div>
      </section>
      
      <section>
      <h3><b>New features in CryptoDB 46</b></h3>
      <div style="margin-left: 3em;">
         <ul>
           <li>
             Copy Number Variation Pipeline Bug Fix:  To correct skewed copy number predictions, we changed the method for counting reads in our 
			 CNV pipeline from Cufflinks to HTSeq-Count.  In our new pipeline we calculate FPKM manually.  Please note - saved strategies 
			 that include CNV searches will likely produce different results in Release 46. 
           </li>
		   <li>
             Our pipeline of genomic analyses includes open reading frames (ORFs) prediction in addition to loading gene models from the official annotation.  
			 Because data usage analysis showed low ORF usage, in our next release (47) predicted ORFs will be available as a genome-scale fasta file from 
			 our downloads section and removed  from the searches and genome browser.  Please <a href="https://eupathdb.org/eupathdb/app/contact-us" target ="_blank"> Contact Us</a> 
			 with questions or suggestions. 
           </li>
         </ul>
       </div> 
      </section>
      
      <section>
         <aside>
           Note: Our bimonthly database releases incorporate new data and correct old data when necessary.
           Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
           When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
           When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
         </aside>
      </section>