---
categories: []
title: Fall VectorBase Announcements
category: [VectorBase]
created: 1164772800
---
<table border="0" cellspacing="0" cellpadding="2" width="100%">
<tr>
<td>
Dear Colleagues,<br/><br/>
We are pleased to announce a batch of major data and functional updates at the VectorBase website, including gene expression data and manual annotations for the entire <i>A. gambiae</i> 2L chromosome arm. Please read on for more details.<br/>
<br/>
<span style="font-size:12px;font-weight:bold;">Organism &amp; Data Updates</span><br/>
<hr size="1"/>
<table border="0" cellspacing="2" cellpadding="0" width="100%">
<tr>
<td align="center" valign="top" width="8" rowspan="3" bgcolor="#f8f8f8">
&bull;
</td>
<td style="font-weight:bold;font-decoration:underline;">
New Culex pipiens Homepage
</td>
</tr>
<tr>
<td bgcolor="#e8e8e8" height="1">
</td>
</tr>
<tr>
<td>
<i>Culex pipiens quinquefasciatus</i> has a new homepage. Genome data and assembly updates will be posted on this page as assembly nears completion. 
</td>
</tr>
<tr>
<td height="6">
</td>
</tr>
<tr>
<td align="center" valign="top" width="8" rowspan="3" bgcolor="#f8f8f8">
&bull;
</td>
<td style="font-weight:bold;font-decoration:underline;">
Organism Datasets Updated
</td>
</tr>
<tr>
<td bgcolor="#e8e8e8" height="1">
</td>
</tr>
<tr>
<td>
The first full gene set for <i>Aedes aegypti</i> (AaegL1.1) was released in June 2006. The latest gene set for <i>Anopheles gambiae</i> (AgamP3.3, February 2006) is also available. EST and Trace datasets for all organisms have been updated as of October 30th and are available for download and for use with <a href="/blast">BLAST</a>. All current VectorBase gene sets are also indexed at NCBI.
</td>
</tr>
<tr>
<td height="6">
</td>
</tr>
<tr>
<td align="center" valign="top" width="8" rowspan="3" bgcolor="#f8f8f8">
&bull;
</td>
<td style="font-weight:bold;font-decoration:underline;">
<i>A. gambiae Gene</i> Tool Available
</td>
</tr>
<tr>
<td bgcolor="#e8e8e8" height="1">
</td>
</tr>
<tr>
<td>
The AnoBase Gene Tool is now available for use at VectorBase and is accessible. The Gene Tool provides a full summary of gene meta-data on a single page.
</td>
</tr>
<tr>
<td height="6">
</td>
</tr>
<tr>
<td align="center" valign="top" width="8" rowspan="3" bgcolor="#f8f8f8">
&bull;
</td>
<td style="font-weight:bold;font-decoration:underline;">
<i>A. gambiae</i> BAC Hybridization Images Available
</td>
</tr>
<tr>
<td bgcolor="#e8e8e8" height="1">
</td>
</tr>
<tr>
<td>
Images of BAC hybridizations in <i>Anopheles gambiae</i> are now accessible. Data is searchable by BAC name and Locus.
</td>
</tr>
<tr>
<td height="6">
</td>
</tr>
<tr>
<td align="center" valign="top" width="8" rowspan="3" bgcolor="#f8f8f8">
&bull;
</td>
<td style="font-weight:bold;font-decoration:underline;">
<i>A. gambiae</i> manual 2L Gene Models Available
</td>
</tr>
<tr>
<td bgcolor="#e8e8e8" height="1">
</td>
</tr>
<tr>
<td>
Manual annotation of the 2L chromosome arm in <i>A. gambiae</i> is now 
complete! Pending incorporation into the next <i>A. gambiae</i> gene build, 
all new manually curated gene models have been made temporarily 
available in the genome browser via DAS. To activate, select 
'Manual Annotation' under DAS sources when in the Contig View of the 
Genome Browser.
</td>
</tr>
<tr>
<td height="6">
</td>
</tr>
<tr>
<td align="center" valign="top" width="8" rowspan="3" bgcolor="#f8f8f8">
&bull;
</td>
<td style="font-weight:bold;font-decoration:underline;">
Gene Expression Data Submission Available
</td>
</tr>
<tr>
<td bgcolor="#e8e8e8" height="1">
</td>
</tr>
<tr>
<td>
Several microarray experiments have been added to the VectorBase 
gene expression repository.  To submit your data to ArrayExpress via 
VectorBase, please visit <a href="http://www.vectorbase.org/ExpressionData/">http://www.vectorbase.org/ExpressionData/</a>.  Two experiments
are currently published and publicly available through VectorBase:<br/>
&nbsp;&nbsp;* <i>Anopheles gambiae</i>: Vlachou et al. midgut invasion<br/>
&nbsp;&nbsp;* <i>Aedes aegypti</i>: Sanders et al. SINV infection<br/>
</td>
</tr>
<tr>
<td height="6">
</td>
</tr>
</table>
<br/>
<span style="font-size:12px;font-weight:bold;">Functionality Updates</span><br/>
<hr size="1"/>
VectorBase Genome Browser and Genome Data updated based on Ensembl V.41.Updates include:<br/><br/>
<table border="0" cellspacing="2" cellpadding="0" width="100%">
<tr>
<td align="center" valign="top" width="8" rowspan="3" bgcolor="#f8f8f8">
&bull;
</td>
<td style="font-weight:bold;font-decoration:underline;">
Inter-species comparison data Available
</td>
</tr>
<tr>
<td bgcolor="#e8e8e8" height="1">
</td>
</tr>
<tr>
<td>
<table border="0" cellspacing="1">
<tr>
<td valign="top">
&raquo;
</td>
<td>
Ortholog mapping between <i>Aedes aegypti</i>, <i>Anopheles gambiae</i>, and <i>Drosophila melanogaster</i> genes, plus gene sets from two external organisms: <i>Homo sapiens</i> and <i>C.elegans</i>. 
</td>
</tr>
<tr>
<td valign="top">
&raquo;
</td>
<td>
DNA/DNA comparison (tBLAT) between <i>Aedes aegypti</i>, <i>Anopheles gambiae</i>, and <i>Drosophila melanogaster</i> genomic sequence.
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td height="4">
</td>
</tr>
<tr>
<td align="center" valign="top" width="8" rowspan="3" bgcolor="#f8f8f8">
&bull;
</td>
<td style="font-weight:bold;font-decoration:underline;">
Gene Ontology Associations
</td>
</tr>
<tr>
<td bgcolor="#e8e8e8" height="1">
</td>
</tr>
<tr>
<td>
<table border="0" cellspacing="1">
<tr>
<td valign="top">
&raquo;
</td>
<td>
GO Term mappings from UniProt to organism genes.
</td>
</tr>
<tr>
<td valign="top">
&raquo;
</td>
<td>
GO Term mappings from <i>Drosophila orthologs</i>.
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td height="4">
</td>
</tr>
<tr>
<td align="center" valign="top" width="8" rowspan="3" bgcolor="#f8f8f8">
&bull;
</td>
<td style="font-weight:bold;font-decoration:underline;">
Gene expression (microarray) data integration
</td>
</tr>
<tr>
<td bgcolor="#e8e8e8" height="1">
</td>
</tr>
<tr>
<td>
<table border="0" cellspacing="1">
<tr>
<td valign="top">
&raquo;
</td>
<td>
Genome browser contigview (click on the dark green microarray features):<br/>
<a href="http://www.vectorbase.org/Anopheles_gambiae/Location/View?l=2L:36678503-36682502;r=2L:36678503-36682502">
http://www.vectorbase.org/Anopheles_gambiae/Location/View?l=2L:36678503-36682502;r=2L:36678503-36682502</a>
</td>
</tr>
<tr>
<td valign="top">
&raquo;
</td>
<td>
VectorBase gene page with links to expression data in the "Similarity matches" section:<br/>

<a href="http://www.vectorbase.org/Anopheles_gambiae/Location/View?db=core;g=AGAP000375;r=X:6864349-6866041;t=AGAP000375-RA">
http://www.vectorbase.org/Anopheles_gambiae/Location/View?db=core;g=AGAP000375;r=X:6864349-6866041;t=AGAP000375-RA</a><br/>

<a href="http://www.vectorbase.org/Aedes_aegypti/Gene/Summary?g=AAEL000231;gene=AAEL000231">
http://www.vectorbase.org/Aedes_aegypti/Gene/Summary?g=AAEL000231;gene=AAEL000231</a><br/>
</td>
</tr>
</table>
</td>
</tr>
<tr>
<td height="4">
</td>
</tr>
<tr>
<td align="center" valign="top" width="8" rowspan="3" bgcolor="#f8f8f8">
&bull;
</td>
<td style="font-weight:bold;font-decoration:underline;">
BioMart Initial Testing Release Available
</td>
</tr>
<tr>
<td bgcolor="#e8e8e8" height="1">
</td>
</tr>
<tr>
<td>
An initial version of the <a href="http://biomart.dev.vectorbase.org/biomart/martview">VectorBase BioMart</a> database has been released to allow data mining of genomic data through the VectorBase website. BioMart is a query-building tool designed to make it possible to filter and retrieve data such as genes, exons, and proteins in a variety of formats. 
</td>
</tr>
<tr>
<td height="4">
</td>
</tr>
<tr>
<td align="center" valign="top" width="8" rowspan="3" bgcolor="#f8f8f8">
&bull;
</td>
<td style="font-weight:bold;font-decoration:underline;">
BLAST Functionality Improved
</td>
</tr>
<tr>
<td bgcolor="#e8e8e8" height="1">
</td>
</tr>
<tr>
<td>
Specialized displays of results are now available for specific target databases.
Results of BLASTs against genomic sequence databases can now be viewed directly
in the genome browser. Additionally, sequences hit identified by a BLAST query 
can be downloaded or used as input for the VectorBase ClustalW service. BLAST is available
in  <a href="/blast">here</a>.
</td>
</tr>
<tr>
<td height="4">
</td>
</tr>
<tr>
<td align="center" valign="top" width="8" rowspan="3" bgcolor="#f8f8f8">
&bull;
</td>
<td style="font-weight:bold;font-decoration:underline;">
ClustalW Available
</td>
</tr>
<tr>
<td bgcolor="#e8e8e8" height="1">
</td>
</tr>
<tr>
<td>
The multiple sequence alignment tool ClustalW is now available for use at VectorBase. 
Users may upload their own sequences or pass in input directly from VectorBase BLAST results.
ClustalW is available <a href="/clustalw/">here</a>.
</td>
</tr>
<tr>
<td height="4">
</td>
</tr>
</table>
<br>
As always, feedback is much appreciated! Please don't hesitate to contact us with questions or comments at help@vectorbase.org - we'd love to hear from you.
<br/><br/>
Thanks,<br/>
The VectorBase Team
</td>
</tr>
</table>
