<!-- news.html -->


<!-- main body start -->




   
  

  
  
  

  
  
    <a name="newsItem1"/>
    <a name="PlasmoDB7_19_release"/>
    <table id="news">
      

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 44 Released <i class="date">2 July 2019</i>
                
            </h1>
            <p><strong>We are pleased to announce the release of PlasmoDB 44</strong></p>
    
     <section>
    <h3><b>Release 44 Webinar</b></h3>
    <div style="margin-left: 3em;">
      Join us on Tuesday July 16th at 10:00 am (EST) for our Release 44 Highlights webinar where we will discuss and 
      demonstrate new features and data available in release 44 of EuPathDB sites. 
      <a href="https://attendee.gotowebinar.com/register/7430028289888360460">Register here</a>
    </div>
    </section>
      
      <section>
      <h3><b>New data in PlasmoDB 44</b></h3>
      <div style="margin-left: 3em;">
        <h4>RNA Sequence</h4>
         <ul>
           <li>
             <a href="/a/app/record/dataset/DS_b1c60e998b">Intraerythrocytic development cycle transcriptome (2018)</a> &ndash; 
             Strand specific RNA-seq analysis of 8 time points during the <i>P. falciparum</i> 3D7 intraerythrocytic development cycle.
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/29649445" target="_blank">Toenhake et al</a>.
           </li>
           <li>
             <a href="/a/app/record/dataset/DS_de592be00f">Sporozoite transcriptome in different microenvironments</a> &ndash; 
             <i>Plasmodium vivax</i> transcriptome in 4 microenvironments at 3 time points and 2 temperatures to investigate th e skin-to-liver-stage journey. 
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/30111801" target="_blank">Roth et al</a>.
           </li>
          </ul>
          
        <h4>ChIP-Seq</h4>
         <ul>
           <li>
             <a href="/a/app/record/dataset/DS_11b957c1d4">Regulatory elements during intraerythrocytic development by ATACseq</a> &ndash; 
             Chromatin accessibility defined by directly profiling chromatin accessibility using the assay for transposase accessible chromatin sequencing (ATAC-seq) at eight time points during the intraerythrocytic cycle.
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/29649445" target="_blank">Toenhake et al</a>.
           </li>
         </ul>
      </div>      
      </section>
      
      <section>
      <h3><b>New features in PlasmoDB 44</b></h3>
      <div style="margin-left: 3em;">
         <ul>
           <li>
             Synteny and the Protein Browser are availabe in our beta implementation of the 
             <a href="https://jbrowse.org/" target="_blank">JBrowse Genome Browser</a>. 
           </li>
           <li>
             New EuPathDB RNA Seq Tool. The EuPathDB Galaxy Workspace now offers a tool that converts 
             read counts from HTSeq-count output to FPKM values. 
           </li>
           <li>
             New EuPathDB Workflows. The EuPathDB Galaxy Workspace is updated with several new 
             pre-configured workflows that are designed to allow export of strand specific data 
             to EuPathDB for visualization and searching.
           </li>
         </ul>
       </div> 
      </section>
      
      <section>
         <aside>
           Note: Our bimonthly database releases incorporate new data and correct old data when necessary.
           Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
           When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
           When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
         </aside>
      </section>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem2"/>
    <a name="PlasmoDB4_19_release"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 43 Released <i class="date">25 April 2019</i>
                
            </h1>
            <p><strong>We are pleased to announce the release of PlasmoDB 43</strong></p>
    
     <section>
    <h3><b>Release 43 Webinar</b></h3>
    <div style="margin-left: 3em;">
      Join us on May 15th at 10:00 am (EST) for our Release 43 Highlights webinar where we will discuss and 
      demonstrate new features and data available in release 43 of EuPathDB sites. 
      <a href="https://attendee.gotowebinar.com/register/5760372602565130253">Register here</a>
    </div>
    </section>
      
      <section>
      <h3><b>New data in PlasmoDB 43</b></h3>
        <h4>RNA Sequence</h4>
         <ul>
           <li>
             <a href="/a/app/record/dataset/DS_5d989bb8e4">Strand specific transcriptome of the intraerythrocytic developmental cycle</a> &ndash; 
             Strand specific RNA seq analysis of intraerythrocytic cycle (10, 20, 30, 40 hpi). 
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/24559473" target="_blank">Siegel et al</a>.
           </li>
           <li>
             <a href="/a/app/record/dataset/DS_84fb1047ea">Hypnozoite, schizont and blood stage transcriptomes (laser microdissection)</a> &ndash; 
             <i>Plasmodium cynomolgi</i> hypnozoites and schizonts were isolated by laser capture microdissection from infected monkey hepatocytes. These samples were analyzed by RNA seq along with RNA extracted from frozen, infected <i>M. fascicularis</i> blood samples.
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/28256794" target="_blank">Cubi et al</a>.
           </li>
          </ul>
          
        <h4>ChIP-Seq</h4>
         <ul>
           <li>
             <a href="/a/app/record/dataset/DS_76d32b8164">H3K4me3, H3K9me3, HP1, H4K31me1 & H4K31ac histone post-translational modifications</a> &ndash; 
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/29101771" target="_blank">Sindikubwabo et al</a>.
           </li>
         </ul>
         
        <h4>Updated Functional Annotation</h4>
          <div style="margin-left: 3em;">
        
        <a href="https://docs.google.com/spreadsheets/d/1v0YW1FBOYZ68BYYjqkSuHrIG2oPYhNCtAo51sC4NgMc/edit?usp=sharing" target="_blank">Summary of annotation updates</a>
          which include products, GO terms, gene names, synonyms and PMID. Functional annotations in PlasmoDB 43 are current with the GeneDB GFF versions from 30th of March 2019.<br><br>
        
        <style>
           #FunctAnnot_Bld43 table, #FunctAnnot_Bld43 td, #FunctAnnot_Bld43 th, #FunctAnnot_Bld43 tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #FunctAnnot_Bld43 {
           margin-left : 5 em;
           }
         </style> 
           <table id="FunctAnnot_Bld43"> 
           <tr>
             <th>PlasmoDB Genome Version</th>
             <th>Download GeneDB GFF file</th>
             <th>Genes Updated</th>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_f2d80510c7"><i>P. berghei</i> ANKA </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2019-04/Pberghei/">GeneDB version March, 2019</a></td>
             <td>108 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_5dab6e4a9f"><i>P. chabaudi</i> AS </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2019-04/Pchabaudi/">GeneDB version March, 2019</a></td>
             <td>94 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_1d17c1883c"><i>P. falciparum</i> 3D7 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2019-04/Pfalciparum/">GeneDB version March, 2019</a></td>
             <td>880 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_22b4dadc51"><i>P. gallinaceum</i> 8A </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2019-04/Pgallinaceum/">GeneDB version March, 2019</a></td>
             <td>89 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_8793a259f4"><i>P. knowlesi</i> H </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2019-04/Pknowlesi/">GeneDB version March, 2019</a></td>
             <td>90 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_3986a0b578"><i>P. malariae</i> UG01 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2019-04/Pmalariae/">GeneDB version March, 2019</a></td>
             <td>89 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_a4928ff670"><i>P. ovale curtisi</i> GH01 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2019-04/Povale/">GeneDB version March, 2019</a></td>
             <td>88 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_91fa338f8f"><i>P. reichenowi</i> CDC </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2019-04/Preichenowi/">GeneDB version March, 2019</a></td>
             <td>90 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_c3f289d21a"><i>P. relictum</i> SGS1 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2019-04/Prelictum/">GeneDB version March, 2019</a></td>
             <td>87 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://alpha.plasmodb.org/plasmo.alpha/app/record/dataset/DS_ad07e73d42"><i>P. vivax</i> P01 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2019-04/PvivaxP01/">GeneDB version March, 2019</a></td>
             <td>92 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_b5a76b5939"><i>P. yoelii yoelii</i> 17X </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/genedb/releases/2019-04/Pyoelii/">GeneDB version March, 2019</a></td>
             <td>89 genes</td>
           </tr>
           </table>
           <br><br>    
      </section>
      
      <section>
      <h3><b>New features in PlasmoDB 43</b></h3>
      <div style="margin-left: 3em;">
         <ul>
           <li>
             A beta implementation of the 
             <a href="https://jbrowse.org/" target="_blank">JBrowse Genome Browser</a> is now available from gene 
             record pages on any EuPathDB site.  
           </li>
         </ul>
       </div> 
      </section>
      
      <section>
         <aside>
           Note: Our bimonthly database releases incorporate new data and correct old data when necessary.
           Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
           When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
           When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
         </aside>
      </section>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem3"/>
    <a name="PlasmoDB3_19_release"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 42 Released <i class="date">7 March 2019</i>
                
            </h1>
            <p><strong>We are pleased to announce the release of PlasmoDB 42</strong></p>
    
    <section>
    <h3><b>Release 42 Webinar</b></h3>
    <div style="margin-left: 3em;">
      Join us on March 20, 2019 10:00-11:00 AM EDT for our Release 42 Highlights webinar where we will discuss and 
      demonstrate new features and data available in release 42 of EuPathDB sites. 
      <a href="https://attendee.gotowebinar.com/register/6755367952348158722">Register here</a>
    </div>
    </section>
      
      <section>
      <h3><b>New data in PlasmoDB 42</b></h3>
      <div style="margin-left: 3em;">
      <h4>Genome Sequence and Annotation</h4>
            <ul>
             <li><a href="/a/app/record/dataset/DS_22083c762c"><i>P. vivax-like</i> Pvl01</a> &ndash; 
                 This genome appears for the first time in PlasmoDB and was downloaded from the 
                 <a href="ftp://ftp.sanger.ac.uk/pub/pathogens/Plasmodium/P_vivaxLike/" target="_blank">Wellcome Sanger Institute</a>. 
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/30142149" target="_blank">Gilabert et al</a>.
             </li>
            </ul>
      <h4>Proteomics</h4>
            <ul>
             <li><a href="/a/app/record/dataset/DS_32db942cc7">Apicoplast and ER Proteomes (Quantitative)</a> &ndash; 
                 Apicoplast proteome determined by BioID followed by mass spec of <i>P. falciparum</i> Dd2 parasites during asexual blood-stage growth. 
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/30212465" target="_blank">Boucher et al</a>.
             </li>
            </ul>
      <h4>ChIP Sequence</h4>
            <ul>
             <li><a href="/a/app/record/dataset/DS_e3365ae3b5">Genome-wide HP1 occupancy in <i>P. berghei</i> schizonts</a> &ndash; 
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/29503181" target="_blank">Fraschka et al</a>.
             </li>
             <li><a href="/a/app/record/dataset/DS_99ead1d783">Genome-wide HP1 occupancy in <i>P. chabaudi</i> schizonts</a> &ndash; 
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/29503181" target="_blank">Fraschka et al</a>.
             </li>
             <li><a href="/a/app/record/dataset/DS_364cb4d495">Genome-wide HP1 Occupancy in <i>P. falciparum</i> strains</a> &ndash;
                 Genome-wide distribution of HP1 in <i>P. falciparum</i> 3D7, NF54, NF135, Pf2004 schizonts, as well as Pf3D7 ring, Pf3D7 
                 trophozoites, Pf2004 gameotcyte II/III and Pf2004 gametocyte IV/V.
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/29503181" target="_blank">Fraschka et al</a>.
             </li>
             <li><a href="/a/app/record/dataset/DS_a49c437a4e">Genome-wide HP1 occupancy in <i>P. knowlesi</i></a> &ndash; 
                 Genome-wide occupancy of HP1 in schizonts from <i>P. knowlesi</i> A1-C.1 and A1-H.1. 
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/29503181" target="_blank">Fraschka et al</a>.
             </li>
             <li><a href="/a/app/record/dataset/DS_3779bfbffd">Genome-wide HP1 occupancy in <i>P. vivax</i> clinical isolate pool</a> &ndash; 
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/29503181" target="_blank">Fraschka et al</a>.
             </li>
             <li><a href="/a/app/record/dataset/DS_b16ffaf81b">Genome-wide HP1 occupancy in <i>P. yoelii</i> yoelii YM schizonts</a> &ndash; 
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/29503181" target="_blank">Fraschka et al</a>.
             </li>
            </ul>
       <h4>Whole Genome Sequencing of Isolates</h4>
            <ul>
             <li><a href="/a/app/record/dataset/DS_60f232b2f3">Whole genome sequencing of <i>P. vivax-like</i> isolates</a> &ndash; 
                 A total of 11 <i>P. vivax-like</i> samples were obtained from infected chimpanzee blood samples and an infected mosquito collected in Gabon.
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/30142149" target="_blank">Gilabert et al</a>.
             </li>
             <li><a href="/a/app/record/dataset/DS_7df59d2fa4">African Ape <i>Plasmodium vivax</i> isolates</a> &ndash; 
                 Whole genome sequence data of <i>P. vivax</i> strains infecting six chimpanzees
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/30127015" target="_blank">Loyt et al</a>.
             </li>
            </ul>
       <h4>Protein Microarray</h4>
            <ul>
             <li><a href="/a/app/record/dataset/DS_114ffa1e42">Antibody array of recent exposure to <i>Plasmodium</i> infection</a> &ndash; 
                 This study assessed responses to 856 <i>P. falciparum</i> antigens by protein microarray in 186 Ugandan children, for 
                 whom detailed <i>P. falciparum</i> exposure data were available.
                 <a href="http://www.ncbi.nlm.nih.gov/pubmed/26216993" target="_blank">Helb et al</a>.
             </li>
            </ul>
      </div>
      </section>
      
      <section>
      <h3><b>New features in PlasmoDB 42</b></h3>
      <div style="margin-left: 3em;">
         <ul>
           <li>
             Updated Alignment Tool on Gene Pages &ndash; <br>
             The Clustal Omega alignment tool in the 'Orthologs and Paralogs within EuPathDB' table now 
             accommodates protein, CDS and genomic sequence alignments. To run Clustal Omega, select genes from the table. Then choose the sequence 
             type and initiate the alignment with the 'Run Clutal Omega for selected genes' button. Output formats include Mismatches highlighted, FASTA, PHYLIP, STOCKHOLM, 
             and VIENNA. This tool replaces the 'Retrieve multiple 
             sequence alignment or multi-FASTA' tool.
           </li>
           <li>
             Beta access to JBrowse &ndash; <br>
             A beta implementation of the 
             <a href="https://jbrowse.org/" target="_blank">JBrowse Genome Browser</a>, is available from gene pages. Use the 
             JBrowse Genome Browser link under the graphic Shortcuts on the gene page. 
             <a href="/assets/JBrowseLink.png">Example Link Location</a>
           </li>
           <li>
             New Gene Page STRING tool &ndash; <br>
             The 'Protein features and properties' section now includes an option to submit the protein sequence 
             to the <a href="https://string-db.org/" target="_blank">STRING</a> functional protein association network. 
             <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5210637/" target="_blank">Szklarczyk et al</a>.
           </li>
           <li>
             Result Table Improvements &ndash; <br>
             Result column configurations and sorting are now saved for each strategy step and will be retained 
             when you come back or when you share a strategy with someone else.
           </li>
           <li>
             New Options for FASTA downloads &ndash; <br>
             The header line of FASTA formatted downloads can be configured to contain only the gene ID.  The sequence can 
             be configured to appear on a signle line or the standard 60 characters per line. 
           </li>
         </ul>
       </div> 
      </section>
      
      <section>
         <aside>
           Note: Our bimonthly database releases incorporate new data and correct old data when necessary.
           Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
           When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
           When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
         </aside>
      </section>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem4"/>
    <a name="PlasmoDB12_18_release"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 41 Released <i class="date">13 December 2018</i>
                
            </h1>
            <p><strong>We are pleased to announce the release of PlasmoDB 41</strong></p>
      
      <section>
      <h3><b>New data in PlasmoDB 41</b></h3>
        <div style="margin-left: 3em;">
        <h4>Genome sequence and annotation</h4>
        These annotated genomes analyzed in <a href="http://www.ncbi.nlm.nih.gov/pubmed/29862326" target="_blank">Otto et al.</a> are integrated into PlasmoDB:
           <ul>
             <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_87081c7aae"><i>Plasmodium falciparum</i> 7G8</a> 
             </li>
             <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_1abc3cd4c9"><i>Plasmodium falciparum</i> CD01</a> 
             </li>
             <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_58283fc81a"><i>Plasmodium falciparum</i> Dd2</a> 
             </li>
             <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_1f8272dd26"><i>Plasmodium falciparum</i> GA01</a>
             </li>
             <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_76a6761cc5"><i>Plasmodium falciparum</i> GB4</a>
             </li>
             <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_e93d27ac3f"><i>Plasmodium falciparum</i> GN01</a>
             </li>
             <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_c4279fbeb1"><i>Plasmodium falciparum</i> HB3</a>
             </li>
             <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_d5ed3f9510"><i>Plasmodium falciparum</i> IT</a>
             </li>
             <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_87335f3dd3"><i>Plasmodium falciparum</i> KE01</a>
             </li>
             <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_d3f46b855b"><i>Plasmodium falciparum</i> KH01</a>
             </li>
             <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_87488c4d13"><i>Plasmodium falciparum</i> KH02</a>
             </li>
             <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_2a60a79d5d"><i>Plasmodium falciparum</i> ML01</a>
             </li>
             <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_44bdb781e5"><i>Plasmodium falciparum</i> SD01</a>
             </li>
             </li>
             <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_f307cb349b"><i>Plasmodium falciparum</i> SN01</a>
             </li>
             <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_bcd8da0561"><i>Plasmodium falciparum</i> TG01</a>
             </li>
           </ul> 
        <h4>Whole genome sequence of isolates</h4>
           <ul>
             <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_2d13238b19">Geographically dispersed <i>P. falciparum</i> isolates</a> &mdash;<br>
             Illumina whole genome sequencing of <i>P. falciparum</i> isolates and lab strains. Data were aligned to the Pf3D7 reference genome and used to determine SNPs.
             <a href="http://www.ncbi.nlm.nih.gov/pubmed/29862326" target="_blank">Otto et al.</a>
             </li>
           </ul> 
           
        <h4>Updated Functional Annotation</h4>
          <div style="margin-left: 3em;">
        
        <a href="https://docs.google.com/spreadsheets/d/1Mwh0RCDXiOAi5imBLDnJQy8cTUrO30AfJlaLeFJewGE/edit#gid=1112700546" target="_blank">Summary of annotation updates</a>
          which include products, GO terms, gene names, synonyms and PMID. <br><br>
        
        <style>
           #FunctAnnot_Bld41 table, #FunctAnnot_Bld41 td, #FunctAnnot_Bld41 th, #FunctAnnot_Bld41 tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #FunctAnnot_Bld41 {
           margin-left : 5 em;
           }
         </style> 
           <table id="FunctAnnot_Bld41"> 
           <tr>
             <th>PlasmoDB Genome Version</th>
             <th>Download GeneDB GFF file</th>
             <th>Genes Updated</th>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_f2d80510c7"><i>P. berghei</i> ANKA </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2018-11/Pberghei.gff3.gz">GeneDB version November, 2018</a></td>
             <td>666 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_5dab6e4a9f"><i>P. chabaudi</i> AS </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2018-11/Pchabaudi.gff3.gz">GeneDB version November, 2018</a></td>
             <td>663 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_1d17c1883c"><i>P. falciparum</i> 3D7 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2018-11/Pfalciparum.gff3.gz">GeneDB version November, 2018</a></td>
             <td>1194 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_22b4dadc51"><i>P. gallinaceum</i> 8A </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2018-11/Pgallinaceum.gff3.gz">GeneDB version November, 2018</a></td>
             <td>682 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_8793a259f4"><i>P. knowlesi</i> H </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/ftp.sanger.ac.uk/pub/project/pathogens/gff3/2018-11/Pknowlesi.gff3.gz">GeneDB version November, 2018</a></td>
             <td>655 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_3986a0b578"><i>P. malariae</i> UG01 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2018-11/Pmalariae.gff3.gz">GeneDB version November, 2018</a></td>
             <td>677 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_a4928ff670"><i>P. ovale curtisi</i> GH01 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2018-11/Povale.gff3.gz">GeneDB version November, 2018</a></td>
             <td>671 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_91fa338f8f"><i>P. reichenowi</i> CDC </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2018-11/Preichenowi.gff3.gz">GeneDB version November, 2018</a></td>
             <td>636 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_c3f289d21a"><i>P. relictum</i> SGS1 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2018-11/Prelictum.gff3.gz">GeneDB version November, 2018</a></td>
             <td>683 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://alpha.plasmodb.org/plasmo.alpha/app/record/dataset/DS_ad07e73d42"><i>P. vivax</i> P01 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2018-11/PvivaxP01.gff3.gz">GeneDB version November, 2018</a></td>
             <td>700 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_b5a76b5939"><i>P. yoelii yoelii</i> 17X </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2018-11/Pyoelii.gff3.gz">GeneDB version November, 2018</a></td>
             <td>628 genes</td>
           </tr>
           </table>
           <br><br>    
           </div>
      </section>
      
      <section>
      <h3><b>New features in PlasmoDB 41</b></h3>
      <h4>Introducing Clustal Omega </h4><br>
      <div style="margin-left: 3em;">
      EuPathDB sites are now using Clustal Omega for gene page protein alignments in the 'Orthologs and Paralogs within EuPathDB' table and the 
      'Retrieve multiple sequence alignment or multi-FASTA' tool in the 'Orthology and synteny' section of gene pages. The 'Isolate Alignments in this Gene Region' tool 
      still uses CulstalW.  
       </div> 
      <h4>New Gene page tools </h4>
         <ul>
           <li>Interpro  &mdash; <br>
              The 'Protein features and properties' section now includes an option to submit the protein sequence for 
              sequence analysis and classification via <a href="https://www.ebi.ac.uk/interpro/" target="_blank">InterPro</a>. 
          </li>
          <li>GPI anchor prediction  &mdash; <br>
              The 'Protein features and properties' section now includes an option to submit the protein sequence for 
              GPI modification site prediction via the <a href="http://mendel.imp.ac.at/sat/gpi/gpi_server.html" target="_blank">GPI Lipid Anchor Project</a>. 
          </li>
         </ul>
      </section>
      
      <section>
         <aside>
           Note: Our bimonthly database releases incorporate new data and correct old data when necessary.
           Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
           When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
           When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
         </aside>
      </section>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem5"/>
    <a name="PlasmoDB10_18_release"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 40 Released <i class="date">15 October 2018</i>
                
            </h1>
            <p><strong>We are pleased to announce the release of PlasmoDB 40.<br>
    PlasmoDB 40 represents a software and features release.</strong></p>
      
      <section>
      <h3><b>New features in PlasmoDB 40</b></h3>
        <div style="margin-left: 3em;">
        <h4>New Sense/Antisense RNA seq search</h4>
         <ul>
           <li><a href="/a/showQuestion.do?questionFullName=InternalGeneDatasetQuestions.GenesByRNASeqEvidence">RNA Seq searches</a> &mdash; <br>
           A new Sense/Antisense search for exploring strand specific transcription is available for any strand specific RNA sequence data set.  
           For each gene, the search compares the fold change in sense reads to the fold change in antisense reads and returns genes that 
           have simultaneous changes in sense vs antisense expression between two samples. Versatile search parameters allow for choosing 
           the direction and magnitude of the change in sense and antisense transcripts.  The search result page lists genes and indicates 
           the paired Reference->Comparison samples that meet the criteria, and offers two graphs to help interpret 
           the search results (FPKM vs sample bar graph, antisense FC vs sense FC scatter plot)
           <a href="/assets/SenseAntisenseTutorial.pdf">Sense/Antisense search tutorial</a>.
           </li>
         </ul>  <br>
         
         <h4>New Gene page tools </h4>
         <ul>
           <li>Protein alignment tool &mdash; <br>
              The Orthologs and Paralogs within EuPathDB
              table now includes an option to run protein ClustalW alignments on orthologs.  Use the 'Protein clustalW' column to include orthologs in 
              the aligment and click 'Run clustalW for selected genes' to initiate the alignment.  For more information see this
              <a href="/assets/ClustalWonGenePage.pdf">ClustalW tutorial</a>.
          </li>
          <li>Protein-protein BLAST tool &mdash; <br>
             The Protein features and properties
             data section now includes an option to BLASTP the protein sequence against 8 data bases including Non-redundant protein sequences (nr), 
             Reference proteins (refseq_protein), UniProtKB/Swiss-Prot (swissprot), Model Organisms (landmark), Patented protein sequences(pat), 
             Protein Data Bank proteins(pdb), Metagenomic proteins(env_nr), and Transcriptome Shotgun Assembly proteins (tsa_nr).  For more information see this 
             <a href="/assets/RunningBlastP.pdf">BlastP tutorial</a>.
          </li>
          <li>MitoProt tool &mdash; <br>
             The Protein features and properties 
             data section now includes an option to retrieve mitochondrial prediction data from  
             <a href="https://ihg.gsf.de/ihg/mitoprot.html" target="_blank">MITOPROT: Prediction of mitochondrial targeting sequences</a>. Clicking 
             the gene page MitoProt tool Submit button initiates a query of the protein sequence in MitoProt and opens a new tab 
             populated with the prediction results. For more information see this 
             <a href="/assets/RunningMitoProt.pdf">MitoProt tutorial</a>.
          </li>
          <li>WoLF PSORT tool &mdash; <br>
             The Protein features and properties
             data section now includes an option to retrieve predictions of protein subcellular localization sites from  
             <a href="https://wolfpsort.hgc.jp/aboutWoLF_PSORT.html.en" target="_blank">WoLF PSORT</a> based on amino acid sequences. Choose an 
             Organism type and click Submit to initiate a query of the protein sequence in WoLF PSORT and open a new tab 
             populated with the prediction results. For more information see this 
             <a href="/assets/RunningWoLFPSORT.pdf">WoLF PSORT tutorial</a>.
          </li>
         </ul><br>
      <!--   
         <h4>New Gene Result Analysis tool</h4>
         <ul>
           <li>A new tool to find 'Experiments with Similar Results' is available for any gene result.  Use the blue 'Analysis 
           Result' tab on any gene result page to access the tool.  The tool compares the genes in your current result to 
           gene lists generated from each transcriptomic data set based on GSEA. The tool returns a list of @PROJECT_ID@ data sets whose 
           gene lists are statistically similar and may be worth investigating. For more information, see this 
           <a href="/assets/XXXX">Sense/Experiments with Similar Results tutorial</a>.
           </li>
         </ul><br>
       -->  
         <h4>New Galaxy tool and workflow</h4>
         <ul>
           <li>Map Your Proteins to OrthoMCL Groups workflow &mdash; <br>
           EuPathDB Galaxy now includes a preconfigured workflow for mapping proteins to OrthoMCL groups. The workflow 
           uses BLAST and the OrthoMCL algorithm to assign your set of proteins to OrthoMCL groups.
           Access the  workflow from the EuPathDB Galaxy home page under 
           'Get started with pre-configured workflows'. The workflow employs three tools which can be accessed 
           separately in the EuPathDB Galaxy tools menu under EuPathDB OrthoMCL Tools. This workflow duplicates 
           and replaces the 'Assign your proteins to groups' function in the Tools menu of the 
           <a href="http://orthomcl.org/orthomcl/" target="_blank">OrthoMCL site</a>.
           For more information see this 
           <a href="/assets/MapProteinsOrthoMCL.pdf">OrthoMCL proteome upload tutorial</a>.
           </li>
           <li>RNA-Seq to EuPathDB export tool &mdash; <br>
           EuPathDB Galaxy now includes a tool for exporting the results of your RNA seq analysis to any EuPathDB site. The tool transfers 
           bigwig and FPKM text files to your private My Data Sets page on EuPathDB. From the My Data Sets page the files 
           are available for GBrowse visualization and the Search Strategy system.
           For more information see this 
           <a href="/assets/GalaxyRNASeqExportTool.pdf">RNA Seq to EuPathDB export tool tutorial</a>.
           </li>
         </ul><br>
         
         <br>
       </div> 
      </section>
      
      <section>
      <h3><b>No new data in PlasmoDB 40</b></h3>
      
      </section>
      
      
      
      <section>
         <aside>
           Note: Our bimonthly database releases incorporate new data and correct old data when necessary.
           Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
           When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
           When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
         </aside>
      </section>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem6"/>
    <a name="PlasmoDB08_18_release"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 39 Released <i class="date">30 August 2018</i>
                
            </h1>
            <p><strong>We are pleased to announce the release of PlasmoDB 39</strong></p>
       
      
       <section>
       <h3><b>New data in PlasmoDB 39</b></h3>
       <div style="margin-left: 3em;">
       
       <h4>Whole genome sequencing reads for SNP and CNV analysis</h4>
           <ul>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_b7ec60f469">Aligned genomic sequence reads - strain P01</a>  &ndash;
               <a href="http://www.ncbi.nlm.nih.gov/pubmed/28008421">Auburn et al.</a>
               </li>
           </ul>
       <div>
       </section>

        
      <section>
      <h3><b>New features in PlasmoDB 39</b></h3>
        <div style="margin-left: 3em;">
        <h4>New Phenotype piggyBac display on gene pages</h4>
         <ul>
           <li>The <a href="http://plasmodb.org/plasmo/app/record/dataset/DS_1cc763e9d0">piggyBac insertion mutagenesis</a> display on 
           gene pages now includes a preview of the GBRowse track showing the insertion mutants.  For an example please see the phenotype section of the 
           <a href="http://plasmodb.org/plasmo/app/record/gene/PF3D7_1133400#PhenotypeScoreGraphs">AMA1 gene page</a>
           </li>
         </ul>
        <h4>New Phenotype Searches</h4>
         <ul>
           <li>The <a href="/a/showQuestion.do?questionFullName=InternalGeneDatasetQuestions.GenesByPhenotypeEvidence">Identify Genes based on Phenotype Evidence </a> 
           page now contains two new searches: 1. The Curated Phenotype (CP) search returns genes based on phenotype characteristics. For data sets without Phenotype Qualifier data, 
           the CP search incorporates a filter parameter for quickly choosing the desired phenotype characteristics. The 
           new Phenotype Text (PT) search returns genes whose phenotypic descriptions contain your text term. 
           </li>
         </ul>
          <h4>FPKM Floor parameter in RNA-Seq fold change searches</h4>
         <ul>
           <li><a href="/a/showQuestion.do?questionFullName=InternalGeneDatasetQuestions.GenesByRNASeqEvidence">RNA Seq fold change searches</a> 
           now offer an <b>FPKM Floor</b> parameter for adjusting the FPKM value of low expressing genes. To avoid 
           misleading fold change values generated by dividing with very small numbers, we set very low FPKM values to a default <b>FPKM floor</b> which represents
           10 mapped reads. The new parameter offers a range of floor values for raising or lowering the expression threshold of the search 
           depending on the user's objectives. For more information use the parameter help (blue question mark icon) on the search page or read the 
           <a href="/assets/Fold_Change_Help.pdf">detailed RNA seq search help</a>.
           </li>
         </ul>  
       </div> 
      </section>
       
      <section>
         <aside>
           Note: Our bimonthly database releases incorporate new data and correct old data when necessary.
           Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
           When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
           When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
         </aside>
      </section>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem7"/>
    <a name="PlasmoDB07_18_release"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 38 Released <i class="date">5 July 2018</i>
                
            </h1>
            <p><strong>We are pleased to announce the release of PlasmoDB 38</strong></p>
       
       <section>
       <h3><b>Announcement:</b></h3>
        Starting with PlasmoDB 38, <i>P. vivax</i> P01 replaces <i>P. vivax</i> Sal-1 as reference for all <i>P. vivax</i> data in PlasmoDB. 
        Functional genomics data for all strains of <i>P. vivax</i> are now aligned to <i>P. vivax</i> P01.  PlasmoDB 37, which uses 
        <i>P. vivax</i> Sal-1 as reference is temporarily available here:  
        <a href="http://r37.plasmodb.org/plasmo.b37/">http://r37.plasmodb.org/plasmo.b37/</a>
       </section>
       
       <section>
       <h3><b>New data in PlasmoDB 38</b></h3>
       <div style="margin-left: 3em;">
       
       <h4>RNA Sequence</h4>
           <ul>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_e764c70e8e">Hypnozoite RNAseq</a> &ndash; 
              <i>P. vivax</i> hypnozoite RNAseq
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/29478773">Gural et al.</a> 
              </li>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_8eeacbc46e">Ring, Oocyst and Sporozoite Transcriptomes</a> &ndash; 
              Transcriptomes of oocyst, ring and sporozoite stages in <i>P. falciparum</i> determined by RNA sequence.
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/29539423">Zanghi et al.</a> 
              </li>
           </ul>
           
        <h4>ChIP Sequencing data</h4>
           <ul>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_c930c27891">Epigenome of sporozoites (HP-1, H3K9me3, H3K9ac) and rings (HP-1, H3K9me3)</a> &ndash; 
              Genome-wide distribution of HP-1 and H3K9me3 in <i>P. falciparum</i> rings and sporozoites, as well as H3K9ac in sporozoites.
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/29539423">Zanghi et al.</a> 
              </li>
           </ul>
        </section>
        <div>
      <section>
       <h3><b>No new features in PlasmoDB 38</b></h3>
      </section>
       
      <section>
         <aside>
           Note: Our bimonthly database releases incorporate new data and correct old data when necessary.
           Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
           When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
           When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
         </aside>
      </section>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem8"/>
    <a name="plasmodb04_18_release"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 37 Released <i class="date">25 April 2018</i>
                
            </h1>
            <p><strong>We are pleased to announce the release of PlasmoDB 37</strong></p>
       
       <section>
       <h3><b>New data in PlasmoDB 37</b></h3>
       <div style="margin-left: 3em;">
       
       <h4>RNA Sequence</h4>
           <ul>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_c35d971a20">Transcriptome of <i>P. cynomolgi</i> during 100-day infection in <i>M. mulatta</i></a> &ndash; 
              <i>P. cynomolgi</i> transcriptome of whole blood and bone marrow collected during 100-day infection of <i>M. mulatta</i>.
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/27590312">Joyner et al.</a> This data set has corresponding host data in HostDB called: 
              <a href="http://hostdb.org/hostdb/app/record/dataset/DS_4585d065bf" target="_blank"><i>M. mulatta</i> infected with <i>P. cynomolgi</i> over 100 days</a>.
              </li>
           </ul>
       
       <h4>Updated Functional Annotation</h4>
          <div style="margin-left: 3em;">
        
        <a href="https://docs.google.com/spreadsheets/d/1DGUPB2J9TXNjDgdsgmCLKuYKtzMm8FWwHZRU1655uAY/edit#gid=1112700546">Summary of annotation updates</a>
          which include products, GO terms, gene names, synonyms and PMID. <br><br>
        
        <style>
           #FunctAnnot_Bld37 table, #FunctAnnot_Bld37 td, #FunctAnnot_Bld37 th, #FunctAnnot_Bld37 tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #FunctAnnot_Bld37 {
           margin-left : 5 em;
           }
         </style> 
           <table id="FunctAnnot_Bld37"> 
           <tr>
             <th>PlasmoDB Genome Version</th>
             <th>Download GeneDB GFF file</th>
             <th>Genes Updated</th>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_f2d80510c7"><i>P. berghei</i> ANKA </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2018-03/Pberghei.gff3.gz">GeneDB version March, 2018</a></td>
             <td>109 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_5dab6e4a9f"><i>P. chabaudi</i> AS </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2018-03/Pchabaudi.gff3.gz">GeneDB version March, 2018</a></td>
             <td>125 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_1d17c1883c"><i>P. falciparum</i> 3D7 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2018-03/Pfalciparum.gff3.gz">GeneDB version March, 2018</a></td>
             <td>291 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_22b4dadc51"><i>P. gallinaceum</i> 8A </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2018-03/Pgallinaceum.gff3.gz">GeneDB version March, 2018</a></td>
             <td>94 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_8793a259f4"><i>P. knowlesi</i> H </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2018-03/Pknowlesi.gff3.gz">GeneDB version March, 2018</a></td>
             <td>235 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_3986a0b578"><i>P. malariae</i> UG01 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2018-03/Pmalariae.gff3.gz">GeneDB version March, 2018</a></td>
             <td>345 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_a4928ff670"><i>P. ovale curtisi</i> GH01 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2018-03/Povale.gff3.gz">GeneDB version March, 2018</a></td>
             <td>124 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_91fa338f8f"><i>P. reichenowi</i> CDC </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2018-03/Preichenowi.gff3.gz">GeneDB version March, 2018</a></td>
             <td>210 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_c3f289d21a"><i>P. relictum</i> SGS1 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2018-03/Prelictum.gff3.gz">GeneDB version March, 2018</a></td>
             <td>92 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://alpha.plasmodb.org/plasmo.alpha/app/record/dataset/DS_ad07e73d42"><i>P. vivax</i> P01 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2018-03/PvivaxP01.gff3.gz">GeneDB version March, 2018</a></td>
             <td>306 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_b5a76b5939"><i>P. yoelii yoelii</i> 17X </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2018-03/Pyoelii.gff3.gz">GeneDB version March, 2018</a></td>
             <td>124 genes</td>
           </tr>
           </table>
           <br><br>
      </div>
      </section>
      
      <section>
      <h3><b>New features in this release</b></h3>
      <div style="margin-left: 3em;">
      
         <h4>Beta EuPathDB Workspace</h4>
         <ul>
             <li>Announcing the Beta release of EuPathDB Workspace, a private workspace that can interface 
         with EuPathDB Galaxy and allows for viewing and interrogating private data against data that is already 
         integrated into EuPathDB (public data). The first EuPathDB workspace tool, 'Bigwig files to EuPathDB' 
         exports bigwig files from EuPathDB Galaxy to EuPathDB. 
         The tool creates a EuPathDB Data Set which is managed from the My Data Sets page along with your other private data sets. 
         Each data set also receives a record page which lists all files that the data set contains, offers file management 
         functions and ports files to the genome browser for viewing.<br><br>
         <a href="https://youtu.be/igQZHjRBqV0" target="_blank">Video tutorial: Export Bigwig files to EuPathDB</a>
             </li>
         </ul>
         
         <h4>Pathway Improvements</h4>
         <ul>
             <li>
               Semantic zooming of Pathway drawings on Pathway record pages - At low zoom levels, pathway nodes appear as 
               glyphs.  At mid zoom levels, the main compound nodes appear as molecular formulas.  At high zoom levels, enzymes and 
               side nodes appear as formulas, terms or Enzyme commission numbers. Clicking on a node will retrieve details about the 
               node, regardless of zoom level.
            </li>
          </ul>  
      </div>
      </section>
            
      <section>
         <aside>
           Note: Our bimonthly database releases incorporate new data and correct old data when necessary.
           Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
           When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
           When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
         </aside>
      </section>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem9"/>
    <a name="plasmodb02_18_release"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 36 Released <i class="date">19 February 2018</i>
                
            </h1>
            <p><strong>We are pleased to announce the release of PlasmoDB 36</strong></p>
       
       <section>
       <h3><b>New data in this release</b></h3>
       <div style="margin-left: 3em;">
       
       <h4>Genome Sequence and Annotation</h4> 
       These annotated genomes appear for the first time in PlasmoDB.
           <ul>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_9f2c561c52"><i>Plasmodium adleri </i> strain G01 </a> &ndash; 
              <a href="https://www.biorxiv.org/content/early/2017/08/04/095679" target="_blank">Otto et al.</a>
              </li>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_7bd8ffb906"><i>Plasmodium billcollinsi </i> strain G01 </a> &ndash; 
              <a href="https://www.biorxiv.org/content/early/2017/08/04/095679" target="_blank">Otto et al.</a>
              </li>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_e8c5132b16"><i>Plasmodium blacklocki </i> strain G01 </a> &ndash; 
              <a href="https://www.biorxiv.org/content/early/2017/08/04/095679" target="_blank">Otto et al.</a>
              </li>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_68b476f200"><i>Plasmodium cynomolgi </i> strain M </a> &ndash; 
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/28748222" target="_blank">Pasini et al.</a>
              </li>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_d786be5502"><i>Plasmodium gaboni</i> strain G01 </a> &ndash; 
              <a href="https://www.biorxiv.org/content/early/2017/08/04/095679" target="_blank">Otto et al.</a>
              </li>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_8b3f023c86"><i>Plasmodium praefalciparum</i> strain G01 </a> &ndash; 
              <a href="https://www.biorxiv.org/content/early/2017/08/04/095679" target="_blank">Otto et al.</a>
              </li>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_022384d18a"><i>Plasmodium reichenowi </i> strain G01 </a> &ndash; 
              <a href="https://www.biorxiv.org/content/early/2017/08/04/095679" target="_blank">Otto et al.</a>
              </li>
           </ul>
       
       <h4>Proteomics</h4>
           <ul>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_5cf309fb6e">Salivary gland sporozoite total, N-acetyl-, phospho- and surface proteomes (NF54)</a> &ndash; 
              Data for N-acetylated and phosphorylated total proteome were added to this data set. 
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/28759593">Swearingen et al.</a>
              </li>
           </ul>
           
       <h4>Phenotypes:</h4>
           <ul>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_90eea17ef6">Growth Rate Phenotype</a> &ndash; 
              PlasmoGEM knockout vectors were used to measure growth rate phenotypes in mice for 2,578 <i>P. berghei</i> genes, 
              representing more than half of the protein-coding genome.
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/28708996" target="_blank">Bushell et al.</a>
              </li>
           </ul>
           
      </div>
      </section>
      
      <section>
      <h3><b>New features in this release</b></h3>
        <div style="margin-left: 3em;">
      
      <h4>Search Page Improvements: search pages were redesigned to include an organism filter and provide better access to help information.</h4>
      
          <ul>
            <li>Search parameters are now left justified. The Get Answer button remains center justified below the parameters. Parameter help 
                icons, which offer information concerning the parameters when hovered over, now appear before the parameter name instead of after.
                </li>
            <li>The Organism Parameter, the first parameter on search pages, has a new look and now contains a filter box for quickly finding 
                your organism of interest.  Type any portion of an organism name (at leaset two letters of the genus, species, or strain) 
                in the filter box and the tree collapses to show 
                only those organisms containing your entry.  To return to the full tree, click the X in the filter box.  
                </li>
          </ul>
          
          <h4>New Paralog Count search</h4>
      
          <ul>
            <li>The new 
                <a href="/a/showQuestion.do?questionFullName=GeneQuestions.GenesByParalogCount">Paralog Count search</a>
                returns genes based on the number of assigned paralogs. Find genes that have a number of paralogs within the specified range. 
                Paralogs are defined as genes from the same species that belong to the same OrthoMCL group.
                </li>
            <li>Find the search under the Orthology and Synteny category.
                </li>
          </ul>

    
       
       <h4>Gene Record Page Improvements:</h4>
 
          <ul>
            <li>A new GO Slim table now appears above the GO Term table on the record pages of genes with GO associated data. 
                The GO Slim table maps GO Terms form the GO Term table to their GO Slim terms based on the 
                <a href="http://www.geneontology.org/page/go-slim-and-subset-guide" target="_blank">Generic GO slim developed by GO Consortium</a>.
                </li>
            <li>Column headers in gene page tables now show a help icon (
                <img src="/a/images/HelpIcon.png"  height="15px" width="16px">)
                when additional information or definitions are available 
                for the column. Hover over the help icons to reveal additional information.  
                </li>
            <li>Metabolic pathway and reaction tables on gene pages now include reaction compound names, with CHEBI IDs on mouse over, 
                and a column that indicates if the EC number associated with a gene is an exact match to the EC number in metabolic pathway.   
                </li>
          </ul>

        
        <h4>Search Strategy Improvement: new option for combining steps in a search strategy. </h4>
       
          <ul>
            <li>Search strategies now offer an option to 'ignore' a step when editing a multi-step strategy. To remove a search 
                from the strategy without deleting the step, click Edit for the combine graphic of the step you are interested in ignoring. 
                Then choose 'IGNORE step #' from the Revise Operation popup. For more information see this 
                <a href="/a/images/ignoreStep.jpg">Ignore Step help document</a>.
                </li>
          </ul>
        </div>
      </section>
      
      <section>
      <h3><b>Introducing ClinEpiDB, a new member of the EuPathDB family of databases:</b></h3>
      <div style="margin-left: 3em;">
      
          <ul>
            <li>We are excited to announce the launch of 
                <a href="http://www.clinepidb.org/" target="_blank">ClinEpiDB</a>, 
                which hosts clinical and epidemiological data for the infectious disease research community. The current release contains data from the 
                <a href="https://www.niaid.nih.gov/research/east-africa-icemr" target="_blank">PRISM</a> 
                study on malaria in Uganda. Check out this 
                <a href="https://www.youtube.com/watch?v=535PcFrBH8M&feature=youtu.be" target="_blank">video</a> 
                to learn more about our newest EuPathDB resource!
                </li>
          </ul>
       </div>
      </section>  
            
      <section>
         <aside>
           Note: Our bimonthly database releases incorporate new data and correct old data when necessary.
           Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
           When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
           When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
         </aside>
      </section>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem10"/>
    <a name="plasmodb12_17_release"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 35 Released <i class="date">1 December 2017</i>
                
            </h1>
            <p><strong>We are pleased to announce the release of PlasmoDB 35</strong></p>
       
       <section>
       <h3><b>New data in this release</b></h3>
       
       <h4>Genome Sequence and Annotation</h4>
           <ul>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_3f2eadf75b"><i>Plasmodium knowlesi</i> strain Malayan Strain Pk1 A </a> &ndash; 
              PacBio assembly of a <i>Plasmodium knowlesi</i> genome sequence with Hi-C correction and some manual annotation. First release of this genome in PlasmoDB.  
              <a href="https://www.ncbi.nlm.nih.gov/pubmed/28720171">Lapp et al.</a>
              </li>
           </ul>
       
       <h4>RNA Sequence</h4>
           <ul>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_6301d619de">Female and male gametocyte transcriptomes</a> &ndash; 
              RNAseq data comparing transcript abundance for FACS sorted male and female <i>P. berghei</i> gametocytes 
              compared these to a mixed asexual population.
              <a href="https://www.ncbi.nlm.nih.gov/pubmed/28923023">Yeoh et al.</a>
              </li>
           </ul>
       
       <h4>Proteomics</h4>
           <ul>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_c351e9a043">Salivary gland sporozoite total and surface proteome (VK210, VK247)</a> &ndash; 
              Proteins identified from <i>P. vivax</i> salivary gland sporozoites, including putatively surface-exposed proteins identified by surface labeling with a membrane-impermeable biotin tag.  
              <a href="https://www.ncbi.nlm.nih.gov/pubmed/28759593">Swearingen et al.</a>
              </li>
           </ul>
           
       <h4>Protein Array</h4>
           <ul>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_4267c95a1c">India ICEMR Antibody Array</a> &ndash; 
              Serological profiling of immune responses at three sites in India by probing protein microarrays consisting of 
              515 <i>Plasmodium vivax</i> and 500 <i>Plasmodium falciparum</i> proteins with 353 plasma samples. A total of 
              236 malaria-positive (symptomatic and asymptomatic) plasma samples and 117 malaria-negative samples were 
              collected at three field sites in Raurkela, Nadiad, and Chennai. Indian samples showed significant 
              seroreactivity to 265 <i>P. vivax</i> and 373 <i>P. falciparum</i> antigens, but overall seroreactivity 
              to <i>P. vivax</i> antigens was lower compared to <i>P. falciparum</i> antigens.  
              <a href="https://www.ncbi.nlm.nih.gov/pubmed/28118367">Uplekar et al.</a>
              </li>
           </ul>
           
            <h4>Updated Functional Annotation</h4>
          <div style="margin-left: 3em;">
        
        <a href="https://docs.google.com/spreadsheets/d/1Ti3G-OfdTOcwsfbx5ep6-QUkKFvaS7wTm-Kg_ll1xJI/edit?usp=sharing">Summary of annotation updates</a>
          which include products, GO terms, gene names, synonyms and PMID. <br><br>
        
        <style>
           #FunctAnnot_Bld35 table, #FunctAnnot_Bld35 td, #FunctAnnot_Bld35 th, #FunctAnnot_Bld35 tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #FunctAnnot_Bld35 {
           margin-left : 5 em;
           }
         </style> 
           <table id="FunctAnnot_Bld35"> 
           <tr>
             <th>PlasmoDB Genome Version</th>
             <th>Download GeneDB GFF file</th>
             <th>Genes Updated</th>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_f2d80510c7"><i>P. berghei</i> ANKA </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2017-09/Pberghei.gff3.gz">GeneDB version September, 2017</a></td>
             <td>264 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_5dab6e4a9f"><i>P. chabaudi</i> AS </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2017-09/Pchabaudi.gff3.gz">GeneDB version September, 2017</a></td>
             <td>237 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_1d17c1883c"><i>P. falciparum</i> 3D7 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2017-09/Pfalciparum.gff3.gz">GeneDB version September, 2017</a></td>
             <td>412 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_22b4dadc51"><i>P. gallinaceum</i> 8A </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="http://www.genedb.org/Homepage/Pgallinaceum">GeneDB version September, 2017</a></td>
             <td>248 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_8793a259f4"><i>P. knowlesi</i> H </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2017-09/Pknowlesi.noseq.gff3.gz">GeneDB version September, 2017</a></td>
             <td>236 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_3986a0b578"><i>P. malariae</i> UG01 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2017-09/Pmalariae.noseq.gff3.gz">GeneDB version September, 2017</a></td>
             <td>242 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_a4928ff670"><i>P. ovale curtisi</i> GH01 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2017-09/Povale.noseq.gff3.gz">GeneDB version September, 2017</a></td>
             <td>243 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_91fa338f8f"><i>P. reichenowi</i> CDC </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2017-09/Preichenowi.noseq.gff3.gz">GeneDB version September, 2017</a></td>
             <td>235 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_c3f289d21a"><i>P. relictum</i> SGS1 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="http://www.genedb.org/Homepage/Prelictum">GeneDB version September, 2017</a></td>
             <td>246 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://alpha.plasmodb.org/plasmo.alpha/app/record/dataset/DS_ad07e73d42"><i>P. vivax</i> P01 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2017-09/PvivaxP01.noseq.gff3.gz">GeneDB version September, 2017</a></td>
             <td>249 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_b5a76b5939"><i>P. yoelii yoelii</i> 17X </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2017-09/Pyoelii.gff3.gz">GeneDB version September, 2017</a></td>
             <td>141 genes</td>
           </tr>
           </table>
           <br><br>
        </div>
      </section>
      
      <section>
      <h3><b>New features in this release</b></h3>
      
      <h4>Dedicated search for Genes with protein products that have post-translational modifications</h4>
       <div style="margin-left: 3em;">
       
          PlasmoDB offers a new search 
          <a href="/a/showQuestion.do?questionFullName=GeneQuestions.GenesByPTM">'Identify Genes based on Post-Translational Modifications</a>, 
          which returns genes whose protein products were experimentally determined to be post-translationally modified. This search is categorized under Search for Genes, Proteomics, Post-Translational Modification.
       </div>
      
      <h4>GO Slim Implementation<h4>
      <div style="margin-left: 3em;">
          
          GO Slim terms are now indicated in EuPathDB sites in the following ways:<br>
          <ul>
            <li>The search,
                <a href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=GeneQuestions.GenesByGoTerm">Identify Genes based on GO Terms</a>, 
                now includes an GO Subset option for limiting the search to only GO Slim generic terms.
                </li>
            <li>GO Terms tables on gene pages now contain a 'GO Subset' column which identifies GO Slim generic terms.
                </li>
            <li>The GO enrichment analysis has an option for limiting the analysis to GO Slim terms. This will limit both 
                the background and the gene list of interest in the Fisher's exact test that determines enrichment. 
                </li>
          </ul>
       </div>
       
       <h4>Revigo and Word Cloud access from GO Enrichment results</h4>
       <div style="margin-left: 3em;">
       
          PlasmoDB offers a link between our GO Enrichment results and a Word Cloud visualization as well as 
          Revigo, a tool that provides GO term summarization and visualization. The links to Revigo and the 
          Word Cloud now appear as buttons that are right justified above the GO enrichment result table. 
       
      </div>
      </section>
      
      
      <section>
         <aside>
           Note: Our bimonthly database releases incorporate new data and correct old data when necessary.
           Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
           When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
           When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
         </aside>
      </section>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem11"/>
    <a name="plasmodb08_17_release"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 34 Released <i class="date">7 September 2017</i>
                
            </h1>
            <p><strong>We are pleased to announce the release of PlasmoDB 34</strong></p>
      
     <section>
        <h3><b>New data in this release</b></h3> 

        <h4>Microarray:</h4>
           <ul>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_9ec3204249">Intraerythrocytic cycle expression profile: in vitro and ex vivo (Pkno PK1(A+))</a> &ndash; 
              Microarray expression profiles of <i>Plasmodium knowlesi</i> Pk1(A+) intraerythrocytic cycle: in vitro and ex vivo.  
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/25880967">Lapp et al.</a>.
              </li>
           </ul>

        </section>
      
      <section>
      <h3><b>New features in this release</b></h3>
      <h4>REVIGO direct link <h4>
      <div style="margin-left: 3em;">
          
          EuPathDB now offers a direct link from gene ontology (GO) enrichment results to 
          <a href="http://revigo.irb.hr/">REVIGO</a>, a tool that provides GO term summarization and visualization.  
          To access this feature first run a GO term enrichment using the 'Analyze Results' option from any gene search result. Then click on 
          the 'Start Revigo Analysis' link (<a href="/plasmodb/images/Revigo.png">upper right hand side of enrichment results</a>) 
          to populate the REVIGO submission page with your GO enrichment results from EuPathDB.  Follow instructions 
          on the REVIGO page to complete your analysis.
          
      </div>
      </section>
      
      <section>
         <aside>
           Note: Our bimonthly database releases incorporate new data and correct old data when necessary.
           Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
           When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
           When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
         </aside>
      </section>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem12"/>
    <a name="info08_17_release"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                Interpreting RNA Seq expression values <i class="date">20 August 2017</i>
                
            </h1>
            <section>
        
        <div style="margin-left: 3em;">
        
        In the transcriptomics section of gene pages, expression graphs display FPKM or RPKM values determined from uniquely aligned 
        reads (those that align to one location only). The coverage section shows plots of unique reads aligned to the genome.  
        This may underrepresent abundance for genes that are duplicated, such as paralogs. Non-unique reads are only available 
        as GBrowse subtracks. To view non-unique reads from a gene page, open the Coverage section and click 'View in 
        genome browser' above the coverage plots. 
        (<a href="/plasmodb/images/FromCoverageNonuniqueReads.pdf">Tutorial document about viewing non-unique reads from a gene page</a>)
        
        <br><br>
        To view non-unique reads without navigating to the gene page, open GBrowse from the home page.  Use the Select Tracks 
        tab to turn on the data track of interest and the subtracks configuration tool to turn on the non-unique reads subtracks. 
        (<a href="/plasmodb/images/FromHomeNonuniqueReads.pdf">Tutorial document for viewing non-unique reads from the home page</a>)
        
        <br><br>
        In release 34, gene pages contain a message about non-unique reads, an accurate description of the y-axis and links to 
        tutorials on how to display non-unique reads.  Future releases will offer searches based on the non-unique reads and 
        their contribution to expression. 
        
        <br><br>
        For future releases, we are working to implement a graphic visualization that provides information about both unique and a non-unique reads. 
     </section>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem13"/>
    <a name="plasmodb06_17_release"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 33 Released <i class="date">30 June 2017</i>
                
            </h1>
            <p><strong>We are pleased to announce the release of PlasmoDB 33</strong></p>
      
     <section>
        <h3><b>New data in this release</b></h3> 
        
        <h4>Chromatin immunoprecipitation and sequencing (ChIP-seq):</h4>
           <ul>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_793e7c4f61">Ap2-G2 ChIP-seq</a> &ndash; 
              AP2-G2 target genes predicted by ChIP-sequencing  
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/26417110">Yuda et al.</a>.
              </li>
           </ul>
           
        <h4>Microarray:</h4>
           <ul>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_d4f8765d13">AP2-G2 knock out and WT expression profiles</a> &ndash; 
              Microarray of wild type and AP2-G2 knock out <i> P. berghei</i> ANKA parasites.  
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/26417110">Yuda et al.</a>.
              </li>
           </ul>
        
        <h4>Proteomics:</h4>
           <ul>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_7e2bd6d15f">Lysine acetylation proteomics</a> &ndash; 
              Peptide fractions from <i>P. falciparum</i> synchronous cultures (trophozoite stage) were enriched for acetylated lysine and analyzed by mass spectrometry.  
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/26813983">Cobbold et al.</a>.
              </li>
           </ul>
           
         <h4>RmGM phenotypes:</h4>
           <ul>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_3bcc2ee874">Phenotypes from RMgmDB</a> - Rodent Malaria genetically modified Parasites &ndash; 
              Phenotypes from double mutants were added in this release. The Rodent Malaria genetic modifications (RMgmDB) table on gene pages now contains a 
              'Double Knockout' that displays the ID of the second knock out and links to its gene page. 
            </li>
           </ul>
         
        </section>
     
     <section>
        <h3><b>New features in this release</b></h3> 

           <ul>
            <li>Transcriptomics data are available for display and download from a strategy result.  For any strategy that returns genes, 
                use the Add Columns tool to display graphs and normalized expression values.  Use the Download tool to obtain expression values as a file.<br><br> 
            </li>
            <li>Improved image resolution, legend and axis text, as well as additional mouseover information for graphs on gene pages and in result columns.<br><br> 
            </li>
            <li>Gene Ontology and Metabolic Pathway enrichment analysis results now include a word cloud of enriched terms.  
                The size and color depth of GO terms or metabolic pathway names in the word cloud are based on the p-values of the enriched terms.<br><br> 
            </li>
            <li>Gene Ontology and Metabolic Pathway enrichment analysis results now include a column with for viewing the genes that contribute to the enrichment. 
                Click the number in the column, 'Genes in you result with this term/pathway', to populate the Gene ID search with the IDs of genes contributing to the 
                enrichment.  Click Get Answer to initiate the search and retrieve the list of genes.<br><br>
            </li>
            <li>Galaxy workflows for variant calling are now available from the EuPathDB Galaxy site which can be reached by clicking 'Analyze My Experiment' from the 
                gray tool bar under the header. 
            </li>
           </ul>
     </section>
        
      <section>
         <aside>
           Note: Our bimonthly database releases incorporate new data and correct old data when necessary.
           Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
           When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
           When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
         </aside>
      </section>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem14"/>
    <a name="plasmodb04_17_release"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 32 Released <i class="date">19 April 2017</i>
                
            </h1>
            <p><strong>We are pleased to announce the release of PlasmoDB 32</strong></p>
     
      <section>
        <h3>New Data in Release 32</h3> 
        
          <h4>Genome Sequence and Annotation:</h4>

           <ul>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_22b4dadc51"><i>P. gallinaceum</i> 8A</a> &ndash; 
              first release of this genome in PlasmoDB (version from 9th of January 2017). Provided by the 
              <a href="http://www.sanger.ac.uk/science/groups/berriman-group">Parasite Genomics Group</a> at the  
              <a href="http://www.sanger.ac.uk/">Wellcome Trust Sanger Institute</a>.
              </li>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_c3f289d21a"><i>Plasmodium relictum </i> SGS1</a> &ndash; 
              first release of this genome in PlasmoDB (version from 9th of January 2017). Provided by the 
              <a href="http://www.sanger.ac.uk/science/groups/berriman-group">Parasite Genomics Group</a> at the  
              <a href="http://www.sanger.ac.uk/">Wellcome Trust Sanger Institute</a>.
              </li>
           </ul>
           
        <h4>Updated Functional Annotation</h4>
          <div style="margin-left: 3em;">
        
        <a href="https://docs.google.com/spreadsheets/d/1Yu9ymLEpdQKyPnRlebdovKea4aF7d42siqO_mMWcjIA/edit?usp=sharing">Summary of annotation updates</a>
          which include products, GO terms, gene names, synonyms and PMID. <br><br>
        
        <style>
           #FunctAnnot_Bld32 table, #FunctAnnot_Bld32 td, #FunctAnnot_Bld32 th, #FunctAnnot_Bld32 tr {
           text-align : center;
           padding-left: 5px;
           padding-right: 5px;
           padding-top: 5px;
           padding-bottom: 5px;
           border: 1px solid black;
           }
           #FunctAnnot_Bld32 {
           margin-left : 5 em;
           }
         </style> 
           <table id="FunctAnnot_Bld32"> 
           <tr>
             <th>PlasmoDB Genome Version</th>
             <th>Download GeneDB GFF file</th>
             <th>Genes Updated</th>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_f2d80510c7"><i>P. berghei</i> ANKA </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2017-03/Pberghei.noseq.gff3.gz">GeneDB version March, 2017</a></td>
             <td>179 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_1d17c1883c"><i>P. falciparum</i> 3D7 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2017-03/Pfalciparum.noseq.gff3.gz">GeneDB version March, 2017</a></td>
             <td>355 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_5dab6e4a9f"><i>P. chabaudi</i> AS </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2017-03/Pchabaudi.noseq.gff3.gz">GeneDB version March, 2017</a></td>
             <td>165 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_8793a259f4"><i>P. knowlesi</i> H </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2017-03/Pknowlesi.noseq.gff3.gz">GeneDB version March, 2017</a></td>
             <td>251 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_3986a0b578"><i>P. malariae</i> UG01 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2017-03/Pmalariae.noseq.gff3.gz">GeneDB version March, 2017</a></td>
             <td>180 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_a4928ff670"><i>P. ovale curtisi</i> GH01 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2017-03/Povale.noseq.gff3.gz">GeneDB version March, 2017</a></td>
             <td>187 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_91fa338f8f"><i>P. reichenowi</i> CDC </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2017-03/Preichenowi.noseq.gff3.gz">GeneDB version March, 2017</a></td>
             <td>221 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_b5a76b5939"><i>P. yoelii yoelii</i> 17X </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2017-03/Pyoelii.noseq.gff3.gz">GeneDB version March, 2017</a></td>
             <td>151 genes</td>
           </tr>
           <tr>
             <td title="Click here to view this genome's full description and a list of associated searches and GBrowse tracks on PlasmoDB."><a href="http://alpha.plasmodb.org/plasmo.alpha/app/record/dataset/DS_ad07e73d42"><i>P. vivax</i> P01 </a></td>
             <td title="Click here to download GeneDB's GFF file for this genome."><a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2017-03/PvivaxP01.noseq.gff3.gz">GeneDB version March, 2017</a></td>
             <td>277 genes</td>
           </tr>
           </table>
           <br><br>
        </div>
        
      <h4>Proteomics</h4>
          <ul>
           <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_aaa667e298">Salivary gland sporozoite total and surface proteome (17XNL)</a> &ndash; 
               Added samples from the PlasmoSEP surface proteomes (2016) to the <i>P yoelii</i> 17XNL sporozoite total and surface proteomes (2013).
               <a href="https://www.ncbi.nlm.nih.gov/pubmed/27298255">El-Manzalawy et al.</a>
           </li>
         </ul>
         <br>  
     </section>    
         
     <section>
         
         <aside>
           Note: Our bimonthly database releases incorporate new data and correct old data when necessary.
           Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
           When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
           When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
         </aside>
      </section>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem15"/>
    <a name="plasmodb03_17_release"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 31 Released <i class="date">8 March 2017</i>
                
            </h1>
            <p><strong>We are pleased to announce the release of PlasmoDB 31</strong></p>
      
     <section>
        <h3>No New data in this release</h3> 
         <aside>
           Note: Our bimonthly database releases incorporate new data and correct old data when necessary.
           Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
           When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
           When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
         </aside>
      </section>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem16"/>
    <a name="job_2_2017"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                EuPathDB is Hiring <i class="date">21 February 2017</i>
                
            </h1>
            <html>
  <head>
    <meta content="text/html; charset=UTF-8" http-equiv="content-type">
    <title></title>
  </head>
  <body>
    <p>
      <!--StartFragment-->
    </p>
    <p class="MsoNormal" style="text-align:justify"><b style="mso-bidi-font-weight:
normal"><span style="font-size:11.0pt;font-family:Palatino;mso-fareast-font-family:
Arial;color:#C00000">The Eukaryotic Pathogen Database resource </span></b><span
        style="font-size:11.0pt;font-family:Palatino;mso-fareast-font-family:Arial">(</span><a
        href="http://eupathdb.org/"><span
          style="font-size:11.0pt;font-family:Palatino;
mso-fareast-font-family:Arial">EuPathDB.org</span></a><span style="font-size:
11.0pt;font-family:Palatino;mso-fareast-font-family:Arial">)<span style="color:black;mso-themecolor:text1"><o:p></o:p></span></span></p>
    <p class="MsoNormal" style="text-align:justify"><span style="font-size:11.0pt;
font-family:Palatino;mso-fareast-font-family:Arial">is a large,
        multi-institution project that develops scientific data-mining websites
        used by
        researchers worldwide to study selected microbial pathogens, including
        those responsible
        for malaria and enteric diseases.<span style="mso-spacerun:yes">&nbsp; </span><span
          style="color:black;mso-themecolor:text1">At
          the request of the Bill &amp; Melinda
          Gates Foundation, we are expanding this effort, with the goal of
          leveraging our
          extensive experience in the integration and analysis of pathogen
          genomics and
          functional datasets, so as to accommodate diverse clinical,
          epidemiological,
          socioeconomic, and geospatial information.<span style="mso-spacerun:yes">&nbsp;
          </span>This new initiative focuses on the interplay between early
          childhood nutrition
          and the enteric infectious disease burden (<i style="mso-bidi-font-style:normal">cf.
          </i></span></span><a href="http://mal-ed.fnih.org/"><span style="font-size:
11.0pt;font-family:Palatino;mso-fareast-font-family:Arial">Mal-ED.fnih.org</span></a><span
        style="font-size:11.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
color:black;mso-themecolor:text1">).<o:p></o:p></span></p>
    <p class="MsoNormal" style="text-align:justify"><span style="font-size:4.0pt;
mso-bidi-font-size:6.0pt;font-family:Palatino;mso-fareast-font-family:Arial;color:black;mso-themecolor:text1"><o:p>&nbsp;</o:p></span></p>
    <p class="MsoNormal" style="text-align:justify"><b style="mso-bidi-font-weight:
normal"><i style="mso-bidi-font-style:normal"><span style="font-size:11.0pt;
font-family:Palatino;mso-fareast-font-family:Arial;color:#C00000">We anticipate
            hiring in the following areas:<o:p></o:p></span></i></b></p>
    <p class="MsoNormal" style="text-align:justify"><i style="mso-bidi-font-style:
normal"><span style="font-size:11.0pt;font-family:Palatino;mso-fareast-font-family:
Arial;color:black;mso-themecolor:text1">Note that because EuPathDB staff work
          at several sites, hires could potentially be based at any of these
          institutions<o:p></o:p></span></i></p>
    <p class="MsoNormal"><span style="font-size:4.0pt;mso-bidi-font-size:6.0pt;
font-family:Palatino;mso-fareast-font-family:Arial;color:black;mso-themecolor:text1"><o:p>&nbsp;</o:p></span></p>
    <table class="MsoTable15List7ColorfulAccent1" style="border-collapse: collapse; width: 736px; height: 654px;"
      border="0"
      cellpadding="0"
      cellspacing="0">
      <tbody>
        <tr style="mso-yfti-irow:-1;mso-yfti-firstrow:yes;mso-yfti-lastfirstrow:yes">
          
          <td style="width:13.5pt;border:none;border-bottom:solid #4472C4 1.0pt;
  mso-border-bottom-themecolor:accent1;mso-border-bottom-alt:solid #4472C4 .5pt;  mso-border-bottom-themecolor:accent1;background:white;mso-background-themecolor:
  background1;padding:0in 5.4pt 0in 5.4pt" valign="top" width="14">
            <p class="MsoNormal" style="text-align:right;mso-hyphenate:none;
  mso-yfti-cnfc:517" align="right"><b style="mso-bidi-font-weight:normal"><i><span
                    style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1"><o:p>&nbsp;</o:p></span></i></b></p>
          </td>
          <td style="width:85.5pt;border:none;border-bottom:solid #4472C4 1.0pt;
  mso-border-bottom-themecolor:accent1;mso-border-bottom-alt:solid #4472C4 .5pt;  mso-border-bottom-themecolor:accent1;background:white;mso-background-themecolor:
  background1;padding:0in 5.4pt 0in 5.4pt" valign="top" width="86">
            <p class="MsoNormal" style="mso-hyphenate:none;mso-yfti-cnfc:1"><b style="mso-bidi-font-weight:normal"><i><span
                    style="font-size:10.0pt;
  font-family:Palatino;mso-fareast-font-family:Arial;color:black;mso-themecolor:  text1"><o:p>&nbsp;</o:p></span></i></b></p>
          </td>
          <td style="width:121.5pt;border:none;border-bottom:solid #4472C4 1.0pt;
  mso-border-bottom-themecolor:accent1;mso-border-bottom-alt:solid #4472C4 .5pt;  mso-border-bottom-themecolor:accent1;background:white;mso-background-themecolor:
  background1;padding:0in 5.4pt 0in 5.4pt" valign="top" width="122">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none;
  mso-yfti-cnfc:1" align="center"><b style="mso-bidi-font-weight:normal"><i><span
                    style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1">Univ Pennsylvania (US)<o:p></o:p></span></i></b></p>
          </td>
          <td style="width:135.0pt;border:none;border-bottom:solid #4472C4 1.0pt;
  mso-border-bottom-themecolor:accent1;mso-border-bottom-alt:solid #4472C4 .5pt;  mso-border-bottom-themecolor:accent1;background:white;mso-background-themecolor:
  background1;padding:0in 5.4pt 0in 5.4pt" valign="top" width="135">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none;
  mso-yfti-cnfc:1" align="center"><b style="mso-bidi-font-weight:normal"><i><span
                    style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1">Univ Georgia (US)<o:p></o:p></span></i></b></p>
          </td>
          <td style="width:135.0pt;border:none;border-bottom:solid #4472C4 1.0pt;
  mso-border-bottom-themecolor:accent1;mso-border-bottom-alt:solid #4472C4 .5pt;  mso-border-bottom-themecolor:accent1;background:white;mso-background-themecolor:
  background1;padding:0in 5.4pt 0in 5.4pt" valign="top" width="135">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none;
  mso-yfti-cnfc:1" align="center"><b style="mso-bidi-font-weight:normal"><i><span
                    style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1">Univ Liverpool (UK)<o:p></o:p></span></i></b></p>
          </td>
        </tr>
        <tr style="mso-yfti-irow:0">
          <td style="width:13.5pt;border:none;border-right:solid #4472C4 1.0pt;
  mso-border-right-themecolor:accent1;mso-border-right-alt:solid #4472C4 .5pt;  mso-border-right-themecolor:accent1;background:white;mso-background-themecolor:
  background1;padding:0in 5.4pt 0in 5.4pt" valign="top" width="14">
            <p class="MsoNormal" style="text-align:right;mso-hyphenate:none;
  mso-yfti-cnfc:68" align="right"><i><span style="font-size:10.0pt;font-family:Palatino;
  mso-fareast-font-family:&quot;Times New Roman&quot;;mso-fareast-theme-font:major-fareast;  color:#2F5496;mso-themecolor:accent1;mso-themeshade:191"><o:p>&nbsp;</o:p></span></i></p>
          </td>
          <td style="width:85.5pt;background:#D9E2F3;mso-background-themecolor:
  accent1;mso-background-themetint:51;padding:0in 5.4pt 0in 5.4pt" valign="top"
            width="86">
            <p class="MsoNormal" style="mso-hyphenate:none;mso-yfti-cnfc:64"><span
                class="MsoHyperlink"><i
                  style="mso-bidi-font-style:normal"><span
                    style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial"><o:p><span
                        style="text-decoration:none">&nbsp;</span></o:p></span></i></span></p>
          </td>
          <td style="width:121.5pt;background:#D9E2F3;mso-background-themecolor:
  accent1;mso-background-themetint:51;padding:0in 5.4pt 0in 5.4pt" valign="top"
            width="122">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none;
  mso-yfti-cnfc:64" align="center"><i style="mso-bidi-font-style:normal"><span style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:#2F5496;mso-themecolor:accent1;mso-themeshade:191"><a href="https://jobs.hr.upenn.edu/">https://jobs.hr.upenn.edu</a></span></i><span
                class="MsoHyperlink"><i
                  style="mso-bidi-font-style:normal"><span
                    style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial"><o:p></o:p></span></i></span></p>
          </td>
          <td style="width:135.0pt;background:#D9E2F3;mso-background-themecolor:
  accent1;mso-background-themetint:51;padding:0in 5.4pt 0in 5.4pt" valign="top"
            width="135">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none;
  mso-yfti-cnfc:64" align="center"><span class="MsoHyperlink"><i style="mso-bidi-font-style:
  normal"><span style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:
  Arial"><a href="https://www.ugajobsearch.com">https://www.ugajobsearch.com</a><o:p></o:p></span></i></span></p>
          </td>
          <td style="width:135.0pt;background:#D9E2F3;mso-background-themecolor:
  accent1;mso-background-themetint:51;padding:0in 5.4pt 0in 5.4pt" valign="top"
            width="135">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none;
  mso-yfti-cnfc:64" align="center"><span style="font-size:10.0pt;color:#2F5496;mso-themecolor:
  accent1;mso-themeshade:191"><a href="https://recruit.liverpool.ac.uk/"><i style="mso-bidi-font-style:normal"><span
                      style="font-family:Palatino;
  mso-fareast-font-family:Arial">https://recruit.liverpool.ac.uk</span></i></a></span><span
                class="MsoHyperlink"><i
                  style="mso-bidi-font-style:normal"><span
                    style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial"><o:p></o:p></span></i></span></p>
          </td>
        </tr>
        <tr style="mso-yfti-irow:1">
          <td style="width:13.5pt;border:none;border-right:solid #4472C4 1.0pt;
  mso-border-right-themecolor:accent1;mso-border-right-alt:solid #4472C4 .5pt;  mso-border-right-themecolor:accent1;background:white;mso-background-themecolor:
  background1;padding:0in 5.4pt 0in 5.4pt" valign="top" width="14">
            <p class="MsoNormal" style="text-align:right;mso-hyphenate:none;
  mso-yfti-cnfc:4" align="right"><b style="mso-bidi-font-weight:normal"><i><span
                    style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:#C00000">Â» </span></i></b><b style="mso-bidi-font-weight:normal"><i><span
                    style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1"><o:p></o:p></span></i></b></p>
          </td>
          <td style="width:85.5pt;padding:0in 5.4pt 0in 5.4pt" valign="top" width="86">
            <p class="MsoNormal" style="mso-hyphenate:none"><b style="mso-bidi-font-weight:
  normal"><span style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:
  Arial;color:black;mso-themecolor:text1">Senior Project Coordinator</span></b><span
                style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1"><o:p></o:p></span></p>
          </td>
          <td style="width:121.5pt;padding:0in 5.4pt 0in 5.4pt" valign="top" width="122">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none" align="center"><span
                style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1">02-24983<o:p></o:p></span></p>
          </td>
          <td style="width:135.0pt;padding:0in 5.4pt 0in 5.4pt" valign="top" width="135">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none" align="center"><span
                style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1">20170049<o:p></o:p></span></p>
          </td>
          <td style="width:135.0pt;padding:0in 5.4pt 0in 5.4pt" valign="top" width="135">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none" align="center"><i
                style="mso-bidi-font-style:normal"><span
                  style="font-size:10.0pt;font-family:
  Palatino;mso-fareast-font-family:Arial;color:black;mso-themecolor:text1"><o:p>NA
                    <br></o:p></span></i></p>
          </td>
        </tr>
        <tr style="mso-yfti-irow:2">
          <td style="width:13.5pt;border:none;border-right:solid #4472C4 1.0pt;
  mso-border-right-themecolor:accent1;mso-border-right-alt:solid #4472C4 .5pt;  mso-border-right-themecolor:accent1;background:white;mso-background-themecolor:
  background1;padding:0in 5.4pt 0in 5.4pt" valign="top" width="14">
            <p class="MsoNormal" style="text-align:right;mso-hyphenate:none;
  mso-yfti-cnfc:68" align="right"><b style="mso-bidi-font-weight:normal"><i><span
                    style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:#C00000">Â»</span></i></b><b style="mso-bidi-font-weight:normal"><i><span
                    style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1"><o:p></o:p></span></i></b></p>
          </td>
          <td style="width:85.5pt;background:#D9E2F3;mso-background-themecolor:
  accent1;mso-background-themetint:51;padding:0in 5.4pt 0in 5.4pt" valign="top"
            width="86">
            <p class="MsoNormal" style="mso-hyphenate:none;mso-yfti-cnfc:64"><b
                style="mso-bidi-font-weight:normal"><span
                  style="font-size:10.0pt;font-family:
  Palatino;mso-fareast-font-family:Arial;color:black;mso-themecolor:text1">Outreach
                  Specialist</span></b><span style="font-size:10.0pt;font-family:Palatino;
  mso-fareast-font-family:Arial;color:black;mso-themecolor:text1"><o:p></o:p></span></p>
          </td>
          <td style="width:121.5pt;background:#D9E2F3;mso-background-themecolor:
  accent1;mso-background-themetint:51;padding:0in 5.4pt 0in 5.4pt" valign="top"
            width="122">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none;
  mso-yfti-cnfc:64" align="center"><span style="font-size:10.0pt;font-family:Palatino;
  mso-fareast-font-family:Arial;color:black;mso-themecolor:text1">02-24758<o:p></o:p></span></p>
          </td>
          <td style="width:135.0pt;background:#D9E2F3;mso-background-themecolor:
  accent1;mso-background-themetint:51;padding:0in 5.4pt 0in 5.4pt" valign="top"
            width="135">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none;
  mso-yfti-cnfc:64" align="center"><span style="font-size:10.0pt;font-family:Palatino;
  mso-fareast-font-family:Arial;color:black;mso-themecolor:text1">20170045<o:p></o:p></span></p>
          </td>
          <td style="width:135.0pt;background:#D9E2F3;mso-background-themecolor:
  accent1;mso-background-themetint:51;padding:0in 5.4pt 0in 5.4pt" valign="top"
            width="135">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none;
  mso-yfti-cnfc:64" align="center"><i style="mso-bidi-font-style:normal"><span style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1">006825</span></i><span style="font-size:10.0pt;font-family:Palatino;color:black"><o:p></o:p></span></p>
          </td>
        </tr>
        <tr style="mso-yfti-irow:3">
          <td style="width:13.5pt;border:none;border-right:solid #4472C4 1.0pt;
  mso-border-right-themecolor:accent1;mso-border-right-alt:solid #4472C4 .5pt;  mso-border-right-themecolor:accent1;background:white;mso-background-themecolor:
  background1;padding:0in 5.4pt 0in 5.4pt" valign="top" width="14">
            <p class="MsoNormal" style="text-align:right;mso-hyphenate:none;
  mso-yfti-cnfc:4" align="right"><b style="mso-bidi-font-weight:normal"><i><span
                    style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:#C00000">Â»</span></i></b><b style="mso-bidi-font-weight:normal"><i><span
                    style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1"><o:p></o:p></span></i></b></p>
          </td>
          <td style="width:85.5pt;padding:0in 5.4pt 0in 5.4pt" valign="top" width="86">
            <p class="MsoNormal" style="mso-hyphenate:none"><b style="mso-bidi-font-weight:
  normal"><span style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:
  Arial;color:black;mso-themecolor:text1">Biomedical OntoloÂ­gist</span></b><span
                style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1"><o:p></o:p></span></p>
          </td>
          <td style="width:121.5pt;padding:0in 5.4pt 0in 5.4pt" valign="top" width="122">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none" align="center"><span
                style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1">02-24756<o:p></o:p></span></p>
          </td>
          <td style="width:135.0pt;padding:0in 5.4pt 0in 5.4pt" valign="top" width="135">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none" align="center"><span
                style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1">20170044<o:p></o:p></span></p>
          </td>
          <td style="width:135.0pt;padding:0in 5.4pt 0in 5.4pt" valign="top" width="135">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none" align="center"><i
                style="mso-bidi-font-style:normal"><span
                  style="font-size:10.0pt;font-family:
  Palatino;mso-fareast-font-family:Arial;color:black;mso-themecolor:text1">006824</span></i><span style="font-size:10.0pt;font-family:Palatino;
  mso-fareast-font-family:Arial;color:black;mso-themecolor:text1"><o:p></o:p></span></p>
          </td>
        </tr>
        <tr style="mso-yfti-irow:4">
          <td style="width:13.5pt;border:none;border-right:solid #4472C4 1.0pt;
  mso-border-right-themecolor:accent1;mso-border-right-alt:solid #4472C4 .5pt;  mso-border-right-themecolor:accent1;background:white;mso-background-themecolor:
  background1;padding:0in 5.4pt 0in 5.4pt" valign="top" width="14">
            <p class="MsoNormal" style="text-align:right;mso-hyphenate:none;
  mso-yfti-cnfc:68" align="right"><b style="mso-bidi-font-weight:normal"><i><span
                    style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:#C00000">Â»</span></i></b><b style="mso-bidi-font-weight:normal"><i><span
                    style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1"><o:p></o:p></span></i></b></p>
          </td>
          <td style="width:85.5pt;background:#D9E2F3;mso-background-themecolor:
  accent1;mso-background-themetint:51;padding:0in 5.4pt 0in 5.4pt" valign="top"
            width="86">
            <p class="MsoNormal" style="mso-hyphenate:none;mso-yfti-cnfc:64"><b
                style="mso-bidi-font-weight:normal"><span
                  style="font-size:10.0pt;font-family:
  Palatino;mso-fareast-font-family:Arial;color:black;mso-themecolor:text1">Data
                  Loader</span></b><span style="font-size:10.0pt;font-family:Palatino;
  mso-fareast-font-family:Arial;color:black;mso-themecolor:text1"><o:p></o:p></span></p>
          </td>
          <td style="width:121.5pt;background:#D9E2F3;mso-background-themecolor:
  accent1;mso-background-themetint:51;padding:0in 5.4pt 0in 5.4pt" valign="top"
            width="122">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none;
  mso-yfti-cnfc:64" align="center"><span style="font-size:10.0pt;font-family:Palatino;
  mso-fareast-font-family:Arial;color:black;mso-themecolor:text1">02-24762<o:p></o:p></span></p>
          </td>
          <td style="width:135.0pt;background:#D9E2F3;mso-background-themecolor:
  accent1;mso-background-themetint:51;padding:0in 5.4pt 0in 5.4pt" valign="top"
            width="135">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none;
  mso-yfti-cnfc:64" align="center"><span style="font-size:10.0pt;font-family:Palatino;
  mso-fareast-font-family:Arial;color:black;mso-themecolor:text1">20170043<o:p></o:p></span></p>
          </td>
          <td style="width:135.0pt;background:#D9E2F3;mso-background-themecolor:
  accent1;mso-background-themetint:51;padding:0in 5.4pt 0in 5.4pt" valign="top"
            width="135">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none;
  mso-yfti-cnfc:64" align="center"><i style="mso-bidi-font-style:normal"><span style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1">not yet posted</span></i><span style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1"><o:p></o:p></span></p>
          </td>
        </tr>
        <tr style="mso-yfti-irow:5">
          <td style="width:13.5pt;border:none;border-right:solid #4472C4 1.0pt;
  mso-border-right-themecolor:accent1;mso-border-right-alt:solid #4472C4 .5pt;  mso-border-right-themecolor:accent1;background:white;mso-background-themecolor:
  background1;padding:0in 5.4pt 0in 5.4pt" valign="top" width="14">
            <p class="MsoNormal" style="text-align:right;mso-hyphenate:none;
  mso-yfti-cnfc:4" align="right"><b style="mso-bidi-font-weight:normal"><i><span
                    style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:#C00000">Â»</span></i></b><b style="mso-bidi-font-weight:normal"><i><span
                    style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1"><o:p></o:p></span></i></b></p>
          </td>
          <td style="width:85.5pt;padding:0in 5.4pt 0in 5.4pt" valign="top" width="86">
            <p class="MsoNormal" style="mso-hyphenate:none"><b style="mso-bidi-font-weight:
  normal"><span style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:
  Arial;color:black;mso-themecolor:text1">SoftÂ­ware Developer</span></b><span style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1"><o:p></o:p></span></p>
          </td>
          <td style="width:121.5pt;padding:0in 5.4pt 0in 5.4pt" valign="top" width="122">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none" align="center"><span
                style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1">02-24765<o:p></o:p></span></p>
          </td>
          <td style="width:135.0pt;padding:0in 5.4pt 0in 5.4pt" valign="top" width="135">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none" align="center"><span
                style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1">20170112<o:p></o:p></span></p>
          </td>
          <td style="width:135.0pt;padding:0in 5.4pt 0in 5.4pt" valign="top" width="135">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none" align="center"><em><span
                  style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1"><o:p>&nbsp;NA</o:p></span></em></p>
          </td>
        </tr>
        <tr style="mso-yfti-irow:6">
          <td style="width:13.5pt;border:none;border-right:solid #4472C4 1.0pt;
  mso-border-right-themecolor:accent1;mso-border-right-alt:solid #4472C4 .5pt;  mso-border-right-themecolor:accent1;background:white;mso-background-themecolor:
  background1;padding:0in 5.4pt 0in 5.4pt" valign="top" width="14">
            <p class="MsoNormal" style="text-align:right;mso-hyphenate:none;
  mso-yfti-cnfc:68" align="right"><b style="mso-bidi-font-weight:normal"><i><span
                    style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:#C00000">Â»<span style="mso-spacerun:yes">&nbsp; </span></span></i></b><i><span
                  style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1"><o:p></o:p></span></i></p>
          </td>
          <td style="width:85.5pt;background:#D9E2F3;mso-background-themecolor:
  accent1;mso-background-themetint:51;padding:0in 5.4pt 0in 5.4pt" valign="top"
            width="86">
            <p class="MsoNormal" style="mso-hyphenate:none;mso-yfti-cnfc:64"><b
                style="mso-bidi-font-weight:normal"><span
                  style="font-size:10.0pt;font-family:
  Palatino;mso-fareast-font-family:Arial;color:black;mso-themecolor:text1">Application
                  Developer I</span></b><span style="font-size:10.0pt;font-family:Palatino;
  mso-fareast-font-family:Arial;color:black;mso-themecolor:text1"><o:p></o:p></span></p>
          </td>
          <td style="width:121.5pt;background:#D9E2F3;mso-background-themecolor:
  accent1;mso-background-themetint:51;padding:0in 5.4pt 0in 5.4pt" valign="top"
            width="122">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none;
  mso-yfti-cnfc:64" align="center"><span style="font-size:10.0pt;font-family:Palatino;
  mso-fareast-font-family:Arial;color:black;mso-themecolor:text1">02-24757<o:p></o:p></span></p>
          </td>
          <td style="width:135.0pt;background:#D9E2F3;mso-background-themecolor:
  accent1;mso-background-themetint:51;padding:0in 5.4pt 0in 5.4pt" valign="top"
            width="135">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none;
  mso-yfti-cnfc:64" align="center"><span style="font-size:10.0pt;font-family:Palatino;
  mso-fareast-font-family:Arial;color:black;mso-themecolor:text1">20170029<o:p></o:p></span></p>
          </td>
          <td style="width:135.0pt;background:#D9E2F3;mso-background-themecolor:
  accent1;mso-background-themetint:51;padding:0in 5.4pt 0in 5.4pt" valign="top"
            width="135">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none;
  mso-yfti-cnfc:64" align="center"><i style="mso-bidi-font-style:normal"><span style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1">not yet posted</span></i><span style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1"><o:p></o:p></span></p>
          </td>
        </tr>
        <tr style="mso-yfti-irow:7;mso-yfti-lastrow:yes">
          <td style="width:13.5pt;border:none;border-right:solid #4472C4 1.0pt;
  mso-border-right-themecolor:accent1;mso-border-right-alt:solid #4472C4 .5pt;  mso-border-right-themecolor:accent1;background:white;mso-background-themecolor:
  background1;padding:0in 5.4pt 0in 5.4pt" valign="top" width="14">
            <p class="MsoNormal" style="text-align:right;mso-hyphenate:none;
  mso-yfti-cnfc:4" align="right"><b style="mso-bidi-font-weight:normal"><i><span
                    style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:#C00000">Â»<span style="mso-spacerun:yes">&nbsp; </span></span></i></b><b
                style="mso-bidi-font-weight:normal"><i><span
                    style="font-size:10.0pt;
  font-family:Palatino;mso-fareast-font-family:Arial;color:black;mso-themecolor:  text1"><o:p></o:p></span></i></b></p>
          </td>
          <td style="width:85.5pt;padding:0in 5.4pt 0in 5.4pt" valign="top" width="86">
            <p class="MsoNormal" style="mso-hyphenate:none"><b style="mso-bidi-font-weight:
  normal"><span style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:
  Arial;color:black;mso-themecolor:text1">Application Developer II</span></b><span
                style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1"><o:p></o:p></span></p>
          </td>
          <td style="width:121.5pt;padding:0in 5.4pt 0in 5.4pt" valign="top" width="122">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none" align="center"><span
                style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1">02-24755<o:p></o:p></span></p>
          </td>
          <td style="width:135.0pt;padding:0in 5.4pt 0in 5.4pt" valign="top" width="135">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none" align="center"><span
                style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1">20170105<o:p></o:p></span></p>
          </td>
          <td style="width:135.0pt;padding:0in 5.4pt 0in 5.4pt" valign="top" width="135">
            <p class="MsoNormal" style="text-align:center;mso-hyphenate:none" align="center"><em><span
                  style="font-size:10.0pt;font-family:Palatino;mso-fareast-font-family:Arial;
  color:black;mso-themecolor:text1"><o:p>&nbsp;NA</o:p></span></em></p>
          </td>
        </tr>
      </tbody>
    </table>
    <p class="MsoNormal"><span style="font-size:4.0pt;mso-bidi-font-size:11.0pt;
font-family:Palatino;mso-fareast-font-family:Arial;color:black;mso-themecolor:text1"><o:p>&nbsp;</o:p></span></p>
    <p class="MsoNormal" style="text-align:justify"><i style="mso-bidi-font-style:
normal"><span style="font-size:11.0pt;font-family:Palatino;mso-fareast-font-family:
Arial">Join our international team of 30+ software developers and biologists,
          combining the advantages of an academic environment with the rigor of
          developing
          a production system.<span style="mso-spacerun:yes">&nbsp; </span><b style="mso-bidi-font-weight:normal">Make
            a difference to global public health!<o:p></o:p></b></span></i></p>
    <p><!--EndFragment-->
    </p>
  </body>
</html>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem17"/>
    <a name="plasmodb02_17_release"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 30 Released <i class="date">3 February 2017</i>
                
            </h1>
            <p><strong>We are pleased to announce the release of PlasmoDB 30</strong></p>
      
      <section>
        <h3>New Data in Release 30</h3> 
        
          <h4>Genome Sequence and Annotation:</h4>

           <ul>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_597478d531"><i>Plasmodium coatneyi</i> strain Hackeri</a> &ndash; 
              Updated to the re-sequenced and re-annotated <a href="https://www.ncbi.nlm.nih.gov/bioproject/315987">GenBank version</a> from the 
              <a href="http://www.systemsbiology.emory.edu/index.html">MaHPIC Consortium</a>.  
              <a href="https://www.ncbi.nlm.nih.gov/pubmed/27587810">Chien et al.</a>
              </li>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_18cd64a91b"><i>Plasmodium fragile</i> strain nilgiri</a> &ndash; 
              New to PlasmoDB and integrated as the <a href="https://www.ncbi.nlm.nih.gov/assembly/GCA_000956335.1">Genbank version</a> from the 
              <a href="https://olive.broadinstitute.org/collections/plasmodium_100_genomes.5/strains">Broad Institute Plasmodium 100 Genomes</a> initiative. 
              (unpublished). 
              </li>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_a9b5b294ad"><i>Plasmodium inui</i> strain San Antonio 1</a> &ndash; 
              Sequenced at the <a href="https://www.broadinstitute.org/genomics">Broad Institute</a>. 
              New to PlasmoDB and integrated as the 
              <a href="https://www.ncbi.nlm.nih.gov/bioproject/176387">Genbank version Jan 31, 2014</a>.
              </li>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_3986a0b578"><i>Plasmodium malariae</i> strain UG01</a> &ndash; 
              First release of this genome in PlasmoDB (version from 19th of September 2016). Provided by the 
              <a href="http://www.sanger.ac.uk/science/groups/berriman-group">Parasite Genomics Group</a> at the 
              <a href="http://www.sanger.ac.uk/">Wellcome Trust Sanger Institute</a>.
              </li>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_a4928ff670"><i>Plasmodium ovale</i> strain curtisi GH01</a> &ndash; 
              First release of this genome in PlasmoDB (version from 19th of September 2016). Provided by the 
              <a href="http://www.sanger.ac.uk/science/groups/berriman-group">Parasite Genomics Group</a> at the 
              <a href="http://www.sanger.ac.uk/">Wellcome Trust Sanger Institute</a>.
              </li>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_4c2903c205"><i>Plasmodium vinckei petteri</i> strain CR</a> &ndash; 
               Sequenced at the <a href="https://www.broadinstitute.org/genomics">Broad Institute</a>. 
               New to PlasmoDB and integrated as the 
               <a href="https://www.ncbi.nlm.nih.gov/bioproject/163125">Genbank version 27 Jan 2014</a>.
               </li>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_19b83aa2be"><i>Plasmodium vinckei vinckei</i> strain vinckei</a> &ndash; 
               Sequenced at the <a href="https://www.broadinstitute.org/genomics">Broad Institute</a>. 
               New to PlasmoDB and integrated as the 
               <a href="https://www.ncbi.nlm.nih.gov/bioproject/163123">Genbank version 17 June 2014</a>.
               </li>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_b5a76b5939"><i>Plasmodium yoelii yoelii</i> 17X</a> &ndash;
               The genome was updated to the reassembled and reannotated 
               <a href="https://www.ncbi.nlm.nih.gov/bioproject/256942">GeneDB version 28 Oct 2016</a>. Provided by the 
              <a href="http://www.sanger.ac.uk/science/groups/berriman-group">Parasite Genomics Group</a> at the 
              <a href="http://www.sanger.ac.uk/">Wellcome Trust Sanger Institute</a>. This is a completed genome (version 3), 
              in 14 pieces with no gaps. 
              </li>
           </ul>
           
          <h4>Proteomics</h4>
          <ul>
           <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_6c7d1a3134">Male and Female Gametocyte Proteomes</a> &ndash; 
               Proteomics data for male, female and mixed <i>P. falciparum</i> 3D7 gametocytes.  This transcriptomics data set has a corresponding proteomics data set called 
               <a href="http://plasmodb.org/plasmo/app/record/dataset/DS_f223d2037c">Female and Male Gametocyte Transcriptomes</a>. 
               <a href="https://www.ncbi.nlm.nih.gov/pubmed/27298255">Lasonder et al.</a>
           </li>
           <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_6c7d1a3134">Sporozoite surface proteome</a> &ndash; 
               Proteome of <i>P. falciparum</i> NF54 salivary gland sporozoite surface proteins isolated by 
               affinity enrichment of surface-exposed proteins labeled with a biotin tag. 
               <a href="https://www.ncbi.nlm.nih.gov/pubmed/27128092">Swearingen et al.</a>
           </li>
          </ul>
          
          <h4>RNA Sequencing</h4>
          <ul>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_f223d2037c">Female and Male Gametocyte Transcriptomes</a> &ndash; 
                Directional RNA-seq data from <i>P. falciparum</i>.  This transcriptomics data set has a corresponding proteomics data set called 
                <a href ="http://plasmodb.org/plasmo/app/record/dataset/DS_6c7d1a3134">Male and Female Gametocyte Proteomes</a>. 
                <a href="https://www.ncbi.nlm.nih.gov/pubmed/27298255">Lasonder et al.</a> 
             </li>
             <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_532da4de76">Transcription profile of intraerythrocytic cycle</a> &ndash; 
                Transcriptomes of the <i>P. vivax</i> intraerythrocytic cycle from two clinical isolates collected from Northwestern Thailand. 
                <a href="https://www.ncbi.nlm.nih.gov/pubmed/26858037">Zhu et al.</a>
             </li>
          </ul>
          
          <h4>Whole genome sequencing of isolates</h4>
           <ul>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_9c34b33d10">Aligned genomic sequence reads - Field and monkey adapted isolates</a>  &ndash;
               Whole genome sequencing of five <i>P. vivax</i> isolates obtained from Malagasy and Cambodian patients, and of the monkey-adapted Belem strain. 
               <a href="https://www.ncbi.nlm.nih.gov/pubmed/22970335">Chan et al.</a>
               </li>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_1d84ebd027">Aligned genomic sequence reads - strain SY57</a>  &ndash;
               Whole genome sequencing of <i>Plasmodium reichenowi</i> SY57. The data were used to call SNPs against the <i>Plasmodium reichenowi</i> CDC reference genome. 
               <a href="https://www.ncbi.nlm.nih.gov/pubmed/27002652">Sundararaman et al.</a>
               </li>
           </ul>
           
           <h4>Phenotypes</h4>
          <ul>
            <li><a href="http://plasmodb.org/plasmo/app/record/dataset/DS_3bcc2ee874">RMgmDB - Rodent Malaria genetically modified Parasites</a> &ndash; 
                This data set is updated to the <a href="http://www.pberghei.eu/index.php">RMgm</a> version from 07 Nov 2016.  Mutant phenotypes are described in the Phenotype table of corresponding gene pages 
                for <i>P. berhei</i> ANKA, <i>P. falciparum</i> 3D7, and <i>P. yoelii</i> YM.  
                 
             </li>
          </ul>
          
          
     </section>
     
     <section>     
         <aside>
           Note: Our bimonthly database releases incorporate new data and correct old data when necessary.
           Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
           When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
           When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
         </aside>
      </section>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem18"/>
    <a name="plasmodb10_16_release"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 29 Released <i class="date">13 October 2016</i>
                
            </h1>
            <p><strong>We are pleased to announce the release of PlasmoDB 29</strong></p>

    <section>
    <h3>PlasmoDB 29</h3>
           PlasmoDB 29 represents extensive efforts to improve navigation and data accessibility 
           on record pages, provide consistent verbiage and data categories, and improve presentation of alternative transcripts. 
           All data were refreshed to the latest version and genome analyses were rerun.
           <ul>
             <li>Here is some help information to get you started with our new features.<br>
                 <a href="http://plasmodb.org/documents/genePageTutorialGUS4.pdf">Gene Page Improvements</a> <br>
                 <a href="http://plasmodb.org/documents/SearchCatsTutorial_3.pdf">Search Category Improvements</a> <br>
                 <a href="http://plasmodb.org/documents/ExploreTranscripts.pdf">Explore Transcripts in Search Results</a><br>
                 <!--Workspaces<br>-->
             </li><br>
             <li>Send Us Your Feedback!<br>We can polish our new features based on your valuable feedback. Please use the site 
             and <a href="http://plasmodb.org/plasmo/app/contact-us">Contact Us</a> to tell us about your experience. 
              </li><br>
             <li>Release 28 is Still Available - <br><a href="http://r28.plasmodb.org/plasmo/">PlasmoDB 28</a> is available until we finalize our new features.</li><br>
            </ul> 
     </section>
      
     <section> 
      <h3>New Features</h3>
         
           <ul>
             <li>Data Categories - <br>Searches, data and GBrowse tracks are now categorized according to the 
             <a href="http://www.ontobee.org/ontology/EDAM?iri=http://edamontology.org/topic_3070">EMBRACE Data & Methods Ontology (EDAM)</a>. 
             This offers a controlled vocabulary and consistent data structure throughout the site. 
             </li><br>
             <li>Record Pages Redesigned - <br>Gene pages, and in-fact all our record pages, have a new structure. 
             Features include a new streamlined and uncluttered look, new navigation tools, and improved 
             tables with search functions for quickly finding and displaying the data you want. 
             </li><br>
             <li>Transcripts Represented on Gene Pages and Search Results - <br>Transcripts are graphically represented on 
             gene pages and listed in gene page tables when data can be 
             unambiguously assigned to a transcript. All search results now 
             contain a Transcript ID column.  The results of searches that query transcript-specific 
             data (e.g. Predicted Signal Peptide) contain an Explore function for investigating transcript subsets.
             </li><br>
             <li>Galaxy Workspace - <br> EuPathDB sites now include a Galaxy-based workspace for large-scale data analyses. 
             Developed in partnership with <a href="https://www.globus.org/genomics">Globus Genomics</a>, 
             workspaces offer a private analysis platform with published workflows and pre-loaded annotated genomes. 
             The workspace is accessed through the "Analyze My Experiment" tab on the home page of any EuPathDB resource 
             and can be used to upload your own data, compose and run custom workflows, retrieve results and share 
             workflows and data analyses with colleagues.
             </li><br>
             <li>Download Search Result Redesigned - <br>The page for downloading search results and associated data has 
             an improved design that reflects the new ontological data categories and offers the ability to download 
             all transcripts per gene when downloading gene results.
             </li><br>
             <li>RNA Sequencing Workflow - We updated our RNA sequencing analysis. The new workflow aligns 
             reads with GSNAP, calculates FPKM/RPKM from HT-Seq counts and determines differential expression with DESeq2.
             </li>
           </ul>
        </section>   


    <section>
      <h3>New data in this release</h3>


     <section>
        <h4>Transcriptomics</h4>
        <ul>
          <li>
            <a href="http://plasmodb.org/plasmo/app/record/dataset/DS_94a7108bd5">Trophozoite transcriptomes after mosquito transmission or direct injection into mice</a> &mdash; 
            Illumina mRNA sequencing of <i>P. chabaudi chabaudi</i> AS trophozoites isolated from mice that were infected with serial blood passaged 
            parasites via mosquito bite or direct injection of parasitized erythrocytes.
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/23719378">Spence et al.</a>.
          </li>
        </ul>
      </section>
      
      <section>
        <h4>Genome Sequence and Annotation</h4>
        <ul>
          <li>
          Genomes in this release were updated to the latest versions.
          </li>  
          
          <li>
            <a href="http://plasmodb.org/plasmo/app/record/dataset/DS_ad07e73d42"><i>Plasmodium vivax</i> P01</a> &mdash; New to PlasmoDB and integrated as  
            <a href="http://www.genedb.org/Homepage/PvivaxP01">GeneDB version 18 June 2015</a> 
            from the 
            <a href="https://www.sanger.ac.uk/research/projects/parasitegenomics/">Parasite Genomics Group</a> at the Wellcome Trust Sanger Institute. 
            <a href="http://www.sanger.ac.uk/datasharing/">Data Use Policy</a>.
          </li>
        </ul>
      </section>
      
      <section>
        <h4>Updated Functional Annotation</h4>
          Note: Only a simplified version of the GO term annotation is available in PlasmoDB 29. It is possible to 
        <a href="ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/CURRENT/">download the full GO annotation files from GeneDB (GAF files).</a><br><br>
        <ul>
          <li>
            <a href="https://docs.google.com/spreadsheets/d/1ecq3kc3hz2ObAjODHVpb6R72Au3TrLOWFSUlGbYC1mo/edit#gid=933942639">Summary of annotation updates </a>which include products, GO terms, EC numbers, gene names, synonyms and PMIDs.
          </li>
          <li>
            <a href="http://plasmodb.org/plasmo/app/record/dataset/DS_1d17c1883c"><i>P. falciparum</i> 3D7 </a> &mdash; (GeneDB version August 2016) 621 genes with updated annotations.
          </li>
          <li>
            <a href="http://plasmodb.org/plasmo/app/record/dataset/DS_91fa338f8f"><i>P. reichenowi</i> CDC </a> &mdash; (GeneDB version August 2016) 447 genes with updated annotations.
          </li>
          <li>
            <a href="http://plasmodb.org/plasmo/app/record/dataset/DS_f2d80510c7"><i>P. berghei</i> ANKA </a> &mdash; (GeneDB version August 2016) 454 genes with updated annotations.
          </li>
          <li>
            <a href="http://plasmodb.org/plasmo/app/record/dataset/DS_5dab6e4a9f"><i>P. chabaudi</i> chabaudi </a> &mdash; (GeneDB version August 2016) 467 genes with updated annotations.
          </li>
          <li>
            <a href="http://plasmodb.org/plasmo/app/record/dataset/DS_b9f1ee69f5"><i>P. yoelii</i> yoelii </a> &mdash; (GeneDB version August 2016) 458 genes with updated annotations.
          </li>
          <li>
            <a href="http://plasmodb.org/plasmo/app/record/dataset/DS_ad07e73d42"><i>P. vivax</i> P01 </a> &mdash; (GeneDB version August 2016) 710 genes with updated annotations.
          </li>
          <li>
            <a href="http://plasmodb.org/plasmo/app/record/dataset/DS_8793a259f4"><i>P. knowlesi</i> H </a> &mdash; (GeneDB version August 2016) 482 genes with updated annotations.
          </li>
        </ul>
      </section>

      
      </section>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem19"/>
    <a name="plasmodb03_16_release"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 28 Released <i class="date">30 March 2016</i>
                
            </h1>
            <p><strong>We are pleased to announce the release of PlasmoDB 28</strong></p>

    <section>
    
     <h3>New data in this release</h3>

      <aside>
        The data set title below links to a full description and list of associated searches and GBrowse tracks.
      </aside>
     
      <section>
       <h4>Transcriptomics:</h4>
        <ul>
          <li>
          <a href="http://plasmodb.org/plasmo/app/record/dataset/DS_4f365680c8">
                               Sexually vs asexually committed schizont trascriptional profiles</a> &mdash;
            Transcriptional profile during schizont sexual committment to gametocytes:  Gene expression (microarray) of 
            sexually vs asexually committed schizonts in the P2G12 gametocyte-producing 
            derivative of <i>P. falciparum</i> 3D7.
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/25722744">Pelle et al.</a>
          </li>

         </ul>
         </section>
         

      <aside>
        Note: Our bimonthly database releases incorporate new data and correct old data when necessary.
        Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
        When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
        When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
      </aside>

    </section>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem20"/>
    <a name="plasmodb02_16_release"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 27 Released <i class="date">19 February 2016</i>
                
            </h1>
            <p><strong>We are pleased to announce the release of PlasmoDB 27</strong></p>

    <section>
    
     <h3>New data in this release</h3>

      <aside>
        The data set title below links to a full description and list of associated searches and GBrowse tracks.
      </aside>
     
      <section>
       <h4>Transcriptomics:</h4>
        <ul>
          <li>
          <a href="http://plasmodb.org/plasmo/app/record/dataset/DS_0fa4237b4b">
                               Asexual life cycle time course with biosynthetic pyrimidine labeling</a> &mdash;
            Microarray analysis of 48-hour timecourse during the asexual development of P. falciparum 3D7. Biosynthetic 
            pyrimidine labeling was used to provide an estimate of newly synthesized vs steady state transcripts at each time point.
          </li>
          <li>
          <a href="http://plasmodb.org/plasmo/app/record/dataset/DS_5bb820a874">
                   Ribosome profiling and mRNA sequencing of asexual cell cycle stages</a> &mdash;
                   RNA sequencing of total and polysome-associated mRNA during 5 stages of the <i>P. falciparum</i> 
                   blood phase developmental cycle. RNA from W2 strain asexual stages were aligned to 
                   <i>P. falciparum</i> 3D7.
                   <a href="http://www.ncbi.nlm.nih.gov/pubmed/25493618">Caro et al.</a>
          </li>
         </ul>
         </section>
         
        <section>
       <h4>Protein Array Data:</h4>
       Malaria-specific antigens in patient serum &mdash; A collaboration between the 
       <a href="https://www.niaid.nih.gov/labsandresources/resources/icemr/Pages/programOverview.aspx">International Centers for Excellence in Malaria Research (ICEMR)</a>  
       encompassing protein array analysis of antibodies in patient serum as well as structured sample and 
       clinical data from the 6 ICEMR groups listed below.
       
       <ul>
          <li>
          <a href="http://plasmodb.org/plasmo/app/record/dataset/DS_4735ffc000">Amazonia ICEMR Malaria-specific human serum antigens</a>
          </li> 
          <li>
          <a href="http://plasmodb.org/plasmo/app/record/dataset/DS_ebd57ee1b5">Malawi ICEMR Malaria-specific human serum antigens </a>
          </li> 
          <li>
          <a href="http://plasmodb.org/plasmo/app/record/dataset/DS_fb75148bb6">South Africa ICEMR Malaria-specific human serum antigens</a>
          </li>  
          <li>
          <a href="http://plasmodb.org/plasmo/app/record/dataset/DS_3bd78c7fec">South Pacific ICEMR Malaria-specific human serum antigens</a>
          </li> 
          <li>
          <a href="http://plasmodb.org/plasmo/app/record/dataset/DS_fe710231ed">Southeast Asia ICEMR Malaria-specific human serum antigens</a>
          </li> 
          <li>
          <a href="http://plasmodb.org/plasmo/app/record/dataset/DS_16d3d360de">Uganda ICEMR Malaria-specific human serum antigens</a>
          </li> 
         </ul>
       
       Use this PlasmoDB search - 
       <a href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=InternalGeneDatasetQuestions.GenesByHostResponse#GeneQuestions.GenesByICEMRHostResponse">Identify Genes based on ICEMR Serum Antibody Levels </a> - 
       to find the <i>Plasmodium</i> genes whose protein products served as antigens during host-immune response. 
       The search employs a filter parameter interface that facilitates searching across data from all ICEMRs. <br><br>
       
       
        
        </section> 
     
     <h3>New Features</h3>
      
      <section>
       <h4>New Tutorials</h4>
        <ul>
          <li>
            <a href="https://youtu.be/z6c_FhMCdPo">Exploring expression distributions with our Percentile Ranking searches</a> - 
            Percentile ranking is one way that EuPathDB analyzes expression data.  This tutorial 
            describes our percentile ranking of expression data and the searches that we offer based on that analysis.
          </li>
          <li>
            <a href="https://youtu.be/Yk9kkaPAmAE">Finding Differentially expressed genes based on Fold Change with a P-value</a> - 
            One way EuPathDB analyzes expression is to determine statistically significant differential gene expression using the  
            Fisher's exact test. This tutorial describes our Fold Change w/ P-value searches that return differentially 
            expressed genes based on the level of statistical confidence that you choose.
          </li>
         </ul>
      </section>
      
    <h3>Request for Feedback from GeneDB about Gene Family Nomenclature</h3>
    
      <section>
       <h4>  </h4>
        <ul>
          <li>
            GeneDB proposes a possible new naming convention for <i>Plasmodium</i> gene families since the current 
            naming convention can be ambiguous. For example, currently several gene families are called fam-a, though they are not related. 
            Please take a look at GeneDB's
            <a href="http://www.genedb.org/artemis/Gene_family_nomenclature_proposal.pdf">gene family nomenclature proposal</a>
            and 
            <a href="mailto:genedb-help@sanger.ac.uk?subject=Gene%20Family%20Nomenclature%20in%20Plasmodium%20Genomes">send GeneDB</a> your comments.
          </li>
        </ul>      
      </section>
 
      <aside>
        Note: Our bimonthly database releases incorporate new data and correct old data when necessary.
        Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
        When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
        When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
      </aside>

    </section>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem21"/>
    <a name="plasmodb10_15_release"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 26 Released <i class="date">15 October 2015</i>
                
            </h1>
            <p><strong>We are pleased to announce the release of PlasmoDB 26</strong></p>

    <section>
      <h3>New data in this release</h3>

      <aside>
        The data set title below links to a full description and list of associated searches and GBrowse tracks.
      </aside>

      <section>
        <h4>Genome Sequence and Annotation: the following genomes were updated to the current GeneDB versions</h4>
        <ul>
          <li>
            <a href="http://PlasmoDB.org/a/getDataset.do?datasets=pberANKA_primary_genome_RSRC"><i>Plasmodium berghei</i> ANKA</a> &mdash;
            updated and improved assembly and annotation
            <a href="http://www.genedb.org/Homepage/Pberghei">GeneDB version 6 July 2015</a>. 
            From the 
            <a href="https://www.sanger.ac.uk/research/projects/parasitegenomics/">Parasite Genomics Group</a> at the Wellcome Trust Sanger Institute. 
            <a href="http://www.sanger.ac.uk/datasharing/">Data Use Policy</a>.
          </li>
          <li>
            <a href="http://PlasmoDB.org/a/getDataset.do?datasets=pchachabaudi_primary_genome_RSRC"><i>Plasmodium chabaudi</i> chabaudi</a> &mdash;
            updated and improved assembly and annotation 
            <a href="http://www.genedb.org/Homepage/Pchabaudi">GeneDB version 18 June 2015</a>. 
            From the 
            <a href="https://www.sanger.ac.uk/research/projects/parasitegenomics/">Parasite Genomics Group</a> at the Wellcome Trust Sanger Institute. 
            <a href="http://www.sanger.ac.uk/datasharing/">Data Use Policy</a>.
          </li>
          <li>
            <a href="http://PlasmoDB.org/a/getDataset.do?datasets=pknoH_primary_genome_RSRC"><i>Plasmodium knowlesi</i> strain H </a> &mdash;
            the genome was updated to the reassembled and reannotated 
            <a href="http://www.genedb.org/Homepage/Pknowlesi">GeneDB version 18 June 2015</a>. 
            From the 
            <a href="https://www.sanger.ac.uk/research/projects/parasitegenomics/">Parasite Genomics Group</a> at the Wellcome Trust Sanger Institute. 
            <a href="http://www.sanger.ac.uk/datasharing/">Data Use Policy</a>.
          </li>
        </ul>
      </section>
      
      <section>
        <h4>Functional Annotation Updated</h4>
        
        <aside>
          Annotation updated: products, GO terms, EC numbers, gene names and synonyms
        </aside>
        
        <ul>
          <li>
            <a href="http://PlasmoDB.org/a/getDataset.do?datasets=pberANKA_primary_genome_RSRC"><i>Plasmodium berghei</i> ANKA</a> &mdash;
            updated 
            <a href="ftp://ftp.sanger.ac.uk/pub/pathogens/gff3/2015-08/">GeneDB version August 2015</a>. 
            101 genes with 
            <a href="http://www.genedb.org/rss/Pberghei/since/4/months">updated annotations</a>.
          </li>
          <li>
            <a href="http://PlasmoDB.org/a/getDataset.do?datasets=pchachabaudi_primary_genome_RSRC"><i>Plasmodium chabaudi chabaudi</i> AS</a> &mdash;
            updated 
            <a href="ftp://ftp.sanger.ac.uk/pub/pathogens/gff3/2015-08/">GeneDB version August 2015</a>. 
            115 genes with 
            <a href="http://www.genedb.org/rss/Pchabaudi/since/4/months">updated annotations</a>.
          </li>
          <li>
            <a href="http://PlasmoDB.org/a/getDataset.do?datasets=pfal3D7_primary_genome_RSRC"><i>Plasmodium falciparum</i> 3D7</a> &mdash;
            updated 
            <a href="ftp://ftp.sanger.ac.uk/pub/pathogens/gff3/2015-08/">GeneDB version August 2015</a>. 
            506 genes with 
            <a href="http://www.genedb.org/rss/Pfalciparum/since/12/months">updated annotations</a>.
          </li>
          <li>
            <a href="http://PlasmoDB.org/a/getDataset.do?datasets=pknoH_primary_genome_RSRC"><i>Plasmodium knowlesi</i> H</a> &mdash;
            updated 
            <a href="ftp://ftp.sanger.ac.uk/pub/pathogens/gff3/2015-08/">GeneDB version August 2015</a>. 
            129 genes with 
            <a href="http://www.genedb.org/rss/Pknowlesi/since/12/months">updated annotations</a>.
          </li>
          <li>
            <a href="http://PlasmoDB.org/a/getDataset.do?datasets=preiCDC_primary_genome_RSRC"><i>Plasmodium rechenowi</i> CDC</a> &mdash;
            updated
            <a href="ftp://ftp.sanger.ac.uk/pub/pathogens/gff3/2015-08/">GeneDB version August 2015</a>. 
            302 genes with 
            <a href="http://www.genedb.org/rss/Pyoelii17X/since/12/months">updated annotations</a>.
          </li>
          <li>
            <a href="http://PlasmoDB.org/a/getDataset.do?datasets=pyoeyoelii17X_primary_genome_RSRC"><i>Plasmodium yoelii yoelii</i> 17X </a> &mdash;
            updated 
            <a href="ftp://ftp.sanger.ac.uk/pub/pathogens/gff3/2015-08/">GeneDB version August 2015</a>. 
            312 genes with 
            <a href="http://www.genedb.org/rss/Pyoelii17X/since/12/months">updated annotations</a>.
          </li>
        </ul>
      </section>
      
      <section>
      <h4>Protein Array:</h4>
      <ul>
        <li>
          <a href="http://PlasmoDB.org/a/getDataset.do?datasets=pfal3D7_microarrayAntibody_Kazura_Reinfection_RSRC">
                                  Treatment-time to reinfection cohort from Kisumu area, Kenya collected in 2003 </a> &mdash;
            Protein microarrays were probed with plasma from residents of a community in Kenya monitored for 12 weeks 
            for reinfection and symptomatic malaria after administration of antimalarial drugs.
          <a href="http://www.ncbi.nlm.nih.gov/pubmed/25883384">Dent et al.</a>.
        </li>
      </section>
      
      <section>
      <h4>Transcription Start Sites:</h4>
      <ul>
        <li>
          <a href="http://PlasmoDB.org/a/getDataset.do?datasets=pfal3D7_Chabbert_TSS_GFF_RSRC">
                                  TSS from 5'cap-selected mRNA </a> &mdash;
            Transcription Start Site locations for P.falciparum 3D7. Genome-wide transcription start sites mapping based on 
            high-throughput sequencing of 5' cap-selected mRNA molecules isolated from 6 time points during <i>P. falciparum</i> 
            3D7 intra-erythrocytic asexual cycle. Lars Steinmetz (EMBL Heidelberg)
        </li>
      </section>
      
     <h3>New Features</h3>
      
      <section>
       <h4>GBrowse Track Search </h4>
        <ul>
          <li>
            The Select Tracks page in the Genome Browser now includes a search function.  Search for specific tracks 
            based on a word or phrase from the track title or category.   
            <a href="http://luffa.gacrc.uga.edu/swfeltz/images/GBrowse_news_item.jpg">image</a>
          </li>
        </ul>
      </section>
      
      <section>
       <h4>New Tutorials</h4>
        <ul>
          <li>
            <a href="https://youtu.be/qGLkAEzQpqU">Find genes with an enriched GO term</a>
          </li>
          <li>
            <a href="https://youtu.be/0fkkJJ9f2cw">Downloading untranslated regions of genes</a>
          </li>
          <li>
            <a href="https://youtu.be/dfx7CCr1_uo">Download sequence for genes in a search result</a>
          </li>
          <li>
            <a href="https://youtu.be/ByBAYywzSWU">Downloading sequence from a gene page</a>
          </li>
          <li>
            <a href="https://youtu.be/3DtOwQM9Cts">Download the sequence of all genes for an organism</a>
          </li>
          <li>
            <a href="https://youtu.be/XNUgT6HEjGY">Download the Sequence (fasta) and Annotation (gff) files for a genome</a>
          </li>
          <li>
            <a href="https://youtu.be/1-onu5raD6M">Downloading coding sequence</a>
          </li>
          <li>
            <a href="https://youtu.be/b2hgT65H4CQ">Downloading protein sequence</a>
          </li>
        </ul>
      </section>
      
      <aside>
        Note: Our bimonthly database releases incorporate new data and correct old data when necessary.
        Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
        When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
        When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
      </aside>

    </section>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem22"/>
    <a name="plasmodb7_15_release"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 25 Released <i class="date">23 July 2015</i>
                
            </h1>
            <p><strong>We are pleased to announce the release of PlasmoDB 25</strong></p>

    <section>
      <h3>New data in this release</h3>

      <aside>
        The data set title below links to a full description and list of associated searches and GBrowse tracks.
      </aside>

      <section>
        <h4>Proteomics</h4>
        <ul>
          <li>
            <a href="http://PlasmoDB.org/a/getDataset.do?datasets=pvivSal1_massSpec_Patarroyo_Intraerythrocytic_Proteome_RSRC">
            Intraerythrocytic life cycle of <i>Plasmodium vivax</i> (VCG-1)</a> &mdash; 
            Proteome during the <i>Plasmodium vivax</i> VCG-1 intraerythrocytic life cycle. Manuel Alfonso Patarroyo, FundaciÃ³n Instituto de InmunologÃ­a de Colombia (FIDIC) 
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/25316051">Moreno-Perez et al.</a>
          </li>
        </ul>
      </section>
      
   <!--
      <section>
        <h4>Proteomics</h4>
        <ul>
          <li>
            <a href="http://PlasmoDB.org/a/getDataset.do?datasets=pfal3D7_SNPChip_SNP_Chip_Artesunate_Resistance_RSRC">
            Isolate genotypes of artemisinin resistance.</a> &mdash; 
            Genotype calls for ~8,000 SNPs generated from Affymetric SNP array analysis on 
            samples collected during clinical trials of artesunate and ACTs in Southeast Asia. Christopher V. Plowe, University of Maryland
            <a href="www.ncbi.nlm.nih.gov/pubmed/23248304">Takala-Harrison et al.</a>.
          </li>
        </ul>
      </section>
    -->
    
      <aside>
        Note: Our bimonthly database releases incorporate new data and correct old data when necessary.
        Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
        When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
        When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
      </aside>

    </section>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem23"/>
    <a name="plasmodb4_15_release"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 24 Released <i class="date">13 April 2015</i>
                
            </h1>
            <!DOCTYPE html>
<html>
  <head>
    <meta content="text/html; charset=windows-1252" http-equiv="content-type">
    <title>PlasmoDB_4_15_news</title>
  </head>
  <body>
    <p> </p>
    <p> </p>
    <p><span style="font-family: Helvetica,Arial,sans-serif;"> </span> </p>
    <div style="margin-left: 40px;"><span style="font-family: Helvetica,Arial,sans-serif;"><strong>We
          are pleased to announce the release of PlasmoDB 24</strong><span style="font-weight: bold;"></span><br>
        <br>
      </span> <span style="font-style: italic;"><span style="font-family: Helvetica,Arial,sans-serif;">All
          EuPathDB release numbers have been synchronized to help us more
          efficiently manage our expanding number of databases.&nbsp; As of this
          release all EuPathDB resource release numbers have been set to release
          24.&nbsp; Subsequent version numbers will increase by one with each
          release.&nbsp; For information about individual dataset version
          numbers see the Data Sets page. </span></span><br>
      <div style="text-align: center;">
      </div>
      <table style="width: 365px; height: 72px; text-align: left; margin-left: auto; margin-right: auto;"
        border="0">
        <tbody>
          <tr>
            <td style="margin-left: 185px; width: 188px; height: 23px; background-color: #cccccc; text-align: center;"><span
                style="font-size:11.0pt;line-height:107%;font-family:Calibri;mso-ascii-theme-font:minor-latin;mso-fareast-font-family:Calibri;mso-fareast-theme-font:minor-latin;mso-hansi-theme-font:minor-latin;mso-bidi-font-family:
&quot;Times New Roman&quot;;mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA">EuPathDB<br>
                Resource</span></td>
            <td style="width: 188px; height: 23px; background-color: #cccccc; text-align: center;"><span
                style="font-size:11.0pt;line-height:107%;font-family:Calibri;mso-ascii-theme-font:minor-latin;mso-fareast-font-family:Calibri;mso-fareast-theme-font:minor-latin;mso-hansi-theme-font:minor-latin;mso-bidi-font-family:
&quot;Times New Roman&quot;;mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA"><span
                  style="font-size:11.0pt;line-height:107%;font-family:Calibri;mso-ascii-theme-font:minor-latin;mso-fareast-font-family:Calibri;mso-fareast-theme-font:minor-latin;mso-hansi-theme-font:minor-latin;mso-bidi-font-family:&quot;Times New Roman&quot;;mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA">Jan
                  2015<br>
                </span>Version #<br>
              </span></td>
            <td style="width: 188px; height: 23px; background-color: #cccccc; text-align: center;">
              <span style="font-size:11.0pt;line-height:107%;font-family:
Calibri;mso-ascii-theme-font:minor-latin;mso-fareast-font-family:Calibri;mso-fareast-theme-font:minor-latin;mso-hansi-theme-font:minor-latin;mso-bidi-font-family:&quot;Times New Roman&quot;;mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA"><span
                  style="font-size:11.0pt;line-height:107%;font-family:Calibri;mso-ascii-theme-font:minor-latin;mso-fareast-font-family:Calibri;mso-fareast-theme-font:minor-latin;mso-hansi-theme-font:minor-latin;mso-bidi-font-family:
&quot;Times New Roman&quot;;mso-bidi-theme-font:minor-bidi;mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA">April
                  2015<br>
                </span>Version #</span>
            </td>
          </tr>
          <tr>
            <td style="width: 188px; height: 24px; background-color: #cccccc; text-align: center;">PlasmoDB</td>
            <td style="width: 188px; height: 24px; background-color: #cccccc; text-align: center;">13.0</td>
            <td style="width: 188px; height: 24px; background-color: #cccccc; text-align: center;">24</td>
          </tr>
        </tbody>
      </table>
      <div style="text-align: left;"><b style="mso-bidi-font-weight:normal">New
          data in this release:</b></div>
      <b style="mso-bidi-font-weight:normal"><o:p></o:p></b><em><br>
        The data set title below links to a full description and list of
        associated searches and genome browser tracks.</em><o:p></o:p><o:p></o:p><b
        style="mso-bidi-font-weight:
normal"><br>
        <br>
        Whole Genome Re-sequencing:<o:p></o:p></b> <br>
      <br>
      <a href="http://plasmodb.org/a/getDataset.do?datasets=pfal3D7_HTS_SNP_Sanger_PreichenowiCDC_QuerySRA_RSRC,pfal3D7_HTS_SNP_Sanger_PreichenowiCDC_QuerySRA_RSRC"><i
          style="mso-bidi-font-style:normal">Plasmodium
          reichenowi</i> CDC</a> - Whole
      genome sequencing (Illumina) of <i style="mso-bidi-font-style:normal">P.reichenowi</i>
      CDC. The data were used to call SNPs against the <i style="mso-bidi-font-style:
normal">Plasmodium falciparum 3D7</i> and<span style="text-decoration: underline;"><span
          style="font-style: italic;">
        </span></span><i style="mso-bidi-font-style:normal">Plasmodium
        reichenowi</i> CDC reference genomes. <a href="http://www.ncbi.nlm.nih.gov/sra?linkname=bioproject_sra_all&amp;from_uid=233970">Genbank
(Jul
        21, 2014)</a><o:p></o:p><b style="mso-bidi-font-weight:
normal"><br>
        <br>
        Proteomics:<o:p></o:p></b><br>
      <br>
      <a href="http://PlasmoDB.org/a/getDataset.do?datasets=pfal3D7_massSpec_Lasonder_Merozoite_Phosphoproteome_RSRC">Merozoite
phosphoproteome
        (3D7)</a><span style="mso-spacerun:yes">&nbsp; </span>Phosphoproteomic
      analysis of <i style="mso-bidi-font-style:normal">Plasmodium falciparum</i>
      3D7
      merozoites (Lasonder <em>et al.</em>)<o:p></o:p>
      <!--EndFragment--><o:p><strong><br>
          <br>
          New Features:</strong></o:p><strong> </strong><span style="font-family: Helvetica,Arial,sans-serif;"></span><span
        style="font-family: Helvetica,Arial,sans-serif;"><em></em></span><span
        style="font-family: Helvetica,Arial,sans-serif;"><br>
        <br>
        <em style="text-decoration: underline;">Help us improve our Data Set
          page</em>: The dataset page includes information about all datasets
        loaded into our resources including, release and date loaded, name of
        the provider, links to searches and data visualization, etc... Take a
        look at the beta version of our dataset page (access it from the data
        summary menu item at the top of our web pages) and let us know what you
        think.<br>
      </span> </div>
    <div style="text-align: justify;"> </div>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;"><br>
      </span> <span style="font-family: Helvetica,Arial,sans-serif;"><span style="font-style: italic;"><span
            style="font-style: italic;"><span
              style="text-decoration: underline;">Note:</span>
            Our bimonthly database releases incorporate new data and correct old
            data when necessary.&nbsp; Changes in annotation and new
            experimental data may slightly alter your search results by
            increasing or decreasing the number of hits. When search parameters
            change with a new release, we invalidate (<span style="color: red;">Ã˜</span>)
            the search and ask you to rerun it. When IDs are updated or removed,
            we map the old IDs to the new ones, remove the old IDs from your
            Basket, and leave your Favorites page alone.</span></span><br>
      </span> <span style="font-family: Helvetica,Arial,sans-serif;"><span style="font-weight: bold;"><span
            style="font-style: italic;"></span></span></span>
      <p><span style="font-family: Helvetica,Arial,sans-serif;"><em><strong>Follow</strong></em>
          <span style="font-weight: bold;"><span style="font-style: italic;">EuPathDB
              on <a href="https://www.facebook.com/pages/EuPathDB/133123003429972">FaceBook</a></span></span><span
            style="font-weight: bold; font-style: italic;">,
            <a href="http://twitter.com/eupathdb">Twitter</a>, and <a href="http://www.youtube.com/user/EuPathDB">YouTube</a></span></span></p>
    </div>
    <span style="font-family: Helvetica,Arial,sans-serif;"><span style="font-weight: bold; font-style: italic;"></span></span>
    <span style="font-family: Helvetica,Arial,sans-serif;"> </span>
    <div style="text-align: justify;"> </div>
    <span style="font-family: Helvetica,Arial,sans-serif;"> </span>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;"><span
          style="font-weight: bold; font-style: italic;"></span><span
          style="font-weight: bold; font-style: italic;"></span></span></div>
    <span style="font-family: Helvetica,Arial,sans-serif;"> </span>
  </body>
</html>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem24"/>
    <a name="comm_lett"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                Letter to the EuPathDB Community <i class="date">19 February 2015</i>
                
            </h1>
            <body>
    <p>Dear colleagues -<br>
      <br>
      We are writing to provide an update on the Eukaryotic Pathogen Genomics
      Resource (<a href="http://EuPathDB.org">EuPathDB.org</a>), which many
      hundreds of laboratories wrote to support in connection with our
      application for renewal as an NIH Bioinformatics Resource Center for
      Infectious Diseases.&nbsp; We are pleased to report that the EuPathDB
      contract has now been renewed.<br>
      <br>
      In addition to continuing to integrate the ever-increasing volume of
      Omic-scale datasets available for microbial eukaryotes, new
      functionalities anticipated over the coming months include:</p>
    <ul>
      <li>full integration of FungiDB into the EuPathDB family of databases</li>
      <li>reorganization of EuPathDB record pages, so as to improve
        navigability, better accommodate curated annotation (when available),
        represent alternative transcripts, summarize functional genomic results,
        etc</li>
      <li>better representation of experimental metadata, including phenotyping
        and clinical/field results</li>
      <li>enhanced functionality of the host response database (<a href="http://HostDB.org">HostDB.org</a>)</li>
      <li>improved handling of metabolic pathways, including incorporation of
        metabolomic datasets</li>
      <li>better support for orthology-based queries and cross-species inference</li>
      <li>implementation of workspaces enabling users to upload and analyze
        their own data (e.g. RNA-seq results)</li>
    </ul>
    <p><br>
      The scope of this renewal extends beyond our previous mandate, which
      officially covered only biosecurity threats (Cryptosporidium, Entamoeba,
      Giardia, Toxoplasma, etc) and reemerging pathogens (e.g.
      Plasmodium).&nbsp; This makes it easier for us to leverage EuPathDB
      infrastructure in support of the kinetoplastida, oomycetes, fungi, and
      other microbial eukaryotes, including both pathogenic and non-pathogenic
      species.<br>
      <br>
      The scale of the EuPathDB award from NIH does not provide adequate funding
      for full support of the TriTrypDB or FungiDB projects (unfortunately) ...
      but the Wellcome Trust has agreed to consider an application covering
      these initiatives, augmenting our existing award for kinetoplastid
      resources.&nbsp; We will be contacting users of these databases shortly
      for their support.<br>
      <br>
      As always, we look forward to feedback from database users, via the
      'Contact Us' link on all EuPathDB web sites, face-to-face discussions at
      major meetings, etc.<br>
      <br>
      Brian P. Brunk -- EuPathDB Senior Project Manager<br>
      Christiane Hertz-Fowler -- Kinetoplastid Resources PI (on behalf of UK
      co-PIs)<br>
      Omar S. Harb -- EuPathDB Scientific Outreach Manager<br>
      Jessica C. Kissinger -- EuPathDB joint-PI<br>
      David S. Roos -- EuPathDB joint-PI, Kinetoplastid Resources co-PI, FungiDB
      co-PI<br>
      Jason E. Stajich -- FungiDB co-PI<br>
      Christian J. Stoeckert -- EuPathDB co-PI</p>
  </body>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem25"/>
    <a name="plasmo12_14_news"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 13.0 Released <i class="date">22 December 2010</i>
                
            </h1>
            <body>
    <div style="text-align: justify;"><strong>We are pleased to announce the
        release of PlasmoDB 13.0</strong><strong></strong><br>
      <br>
      <br>
      <strong>New data in this release</strong>:</div>
    <br>
    
    <p class="MsoNoSpacing">The data set title below links to a full description
      and list of associated searches and GBrowse tracks.</p>
    <p class="MsoNoSpacing"><o:p>&nbsp;</o:p></p>
    <p class="MsoNoSpacing" style="margin-left:.25in"><b style="mso-bidi-font-weight:
normal">Genomes:<o:p></o:p></b></p>
    <p class="MsoNoSpacing" style="margin-left:.5in;text-indent:-.25in"><a href="/a/getDataset.do?datasets=pcoaHackeri_primary_genome_RSRC"><i
          style="mso-bidi-font-style:normal">Plasmodium
          <span class="SpellE">coatneyi</span></i> <span class="SpellE">Hackeri</span></a>
      - first release of this unannotated genome in PlasmoDB (AB Prasad Lab; <span
        class="SpellE">Genbank</span>
      version Jul 21, 2014)</p>
    <p class="MsoNoSpacing" style="margin-left:.5in;text-indent:-.25in"><a href="/a/getDataset.do?datasets=preiCDC_primary_genome_RSRC"><i
          style="mso-bidi-font-style:normal">Plasmodium
          <span class="SpellE">reichenowi</span></i> CDC</a> - Newly-annotated.
      Strain name changed from Dennis to CDC (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25203297">Otto
        et al</a><span class="GramE">.;</span> <span class="SpellE">GeneDB</span>
      version Sept 3, 2014)</p>
    <p class="MsoNoSpacing" style="margin-left:.25in"><o:p>&nbsp;</o:p></p>
    <p class="MsoNoSpacing" style="margin-left:.25in">Proteomics:</p>
    <p class="MsoNoSpacing" style="margin-left:.5in;text-indent:-.25in"><a href="/a/getDataset.do?datasets=pfal3D7_massSpec_Lasonder_Schizont_Phosphoproteome_RSRC">Schizont
        <span class="SpellE">Phosphoproteome</span> (3D7)(2012)(Lasonder et al.)</a>
      - <span class="SpellE">Phosphoproteomic</span> analysis of <i style="mso-bidi-font-style:normal">Plasmodium
        falciparum</i> 3D7 schizont stage</p>
    <p class="MsoNoSpacing" style="margin-left:.5in;text-indent:-.25in"><o:p>&nbsp;</o:p></p>
    <p class="MsoNoSpacing" style="margin-left:.5in;text-indent:-.25in"><b style="mso-bidi-font-weight:normal">RNA
        Sequencing:<o:p></o:p></b></p>
    <p class="MsoNoSpacing" style="margin-left:.5in;text-indent:-.25in"><a href="/a/getDataset.do?datasets=pfal3D7_Bartfai_time_series_rnaSeq_RSRC"><span
          class="SpellE">Intraerythrocytic</span>
        cycle <span class="SpellE">transcriptome</span> (3D7)(<span class="SpellE">Hoeijmakers</span>
        et al.)</a> - RNA sequencing analysis of 8 time points during the <i style="mso-bidi-font-style:normal">P.
        falciparum</i> 3D7 <span class="SpellE">intraerythrocytic</span> cycle.
      This dataset forms a pair with the previously integrated dataset "<em>P.
        falciparum
      </em> 3D7 <span class="SpellE">MNase-Seq</span> Nucleosome Occupancy (<span
        class="SpellE">Barfai</span>
      et al)", which was collected from the exact same batch of synchronized
      parasites.</p>
    <p class="MsoNoSpacing" style="margin-left:.5in;text-indent:-.25in"><o:p>&nbsp;</o:p></p>
    <p class="MsoNoSpacing" style="margin-left:.5in;text-indent:-.25in"><b style="mso-bidi-font-weight:normal">Whole
        genome sequencing of isolates for SNP analysis:<o:p></o:p></b></p>
    <p class="MsoNoSpacing" style="margin-left:.5in;text-indent:-.25in"><a href="http://plasmodb.org/a/getDataset.do?datasets=pvivSal1_HTS_SNP_Broad_whitePaper_hybridSelection_RSRC"><i
          style="mso-bidi-font-style:normal">Plasmodium
          <span class="SpellE">vivax</span></i> aligned genomic sequence reads -
        Hybrid Selection Project (<span class="SpellE">Kidgell</span> et al.)</a>
      - <span class="SpellE">Illumina</span> whole genome sequencing of
      collections of clinical <i style="mso-bidi-font-style:normal">Plasmodium
        <span class="SpellE">vivax</span></i> samples (175 isolates) from
      diverse geographic regions represented by the NIAID-funded International
      Centers of Excellence for Malaria Research (ICEMRs), as well as non-ICEMR
      locations.</p>
    <p class="MsoNoSpacing" style="margin-left:.5in;text-indent:-.25in"><a href="/a/getDataset.do?datasets=pyoeyoelii17X_HTS_SNP_Broad_100_genomes_RSRC,pfal3D7_HTS_SNP_Broad_100_genomes_RSRC"><em>Plasmodium
          <span class="SpellE">yoelii</span></em> and Plasmodium falciparum
        aligned genomic sequence reads - 100 genomes project (<span class="SpellE">Neafsey</span>
        et al.) </a><span style="mso-spacerun:yes">&nbsp;</span><span class="GramE"> - <span
          style="mso-spacerun:yes">&nbsp;
          </span><span class="SpellE">Illumina</span></span> whole genome
      shotgun sequencing of genomic DNA paired-end libraries of 55 <i style="mso-bidi-font-style:normal">P.
        falciparum</i> isolates and <i style="mso-bidi-font-style:normal">P. <span
          class="SpellE">yoelii</span></i>
      17X. </p>
    <p class="MsoNoSpacing" style="margin-left:.5in;text-indent:-.25in"><a href="http://plasmodb.org/a/getDataset.do?datasets=pfal3D7_HTS_SNP_Plowe_symptomatic_malaria_analysis_RSRC"><em>Plasmodium
          falciparum</em> aligned genomic sequence reads - symptomatic malaria
        isolates (<span class="SpellE">Plowe</span> et el.)</a> - Whole genome
      sequencing of isolates collected from symptomatic malaria patients.</p>
    <p class="MsoNoSpacing" style="margin-left:.5in;text-indent:-.25in"><br>
    </p>
    <p class="MsoPlainText"><b style="mso-bidi-font-weight:normal">&nbsp;&nbsp;&nbsp;
        Improvements and updates:<o:p></o:p></b></p>
    <p class="MsoPlainText" style="margin-left:.5in;text-indent:-.25in">BLAST
      was updated to the latest NCBI version 2.2.30. </p>
    <p class="MsoPlainText" style="margin-left:.5in;text-indent:-.25in">GBrowse
      performance is improved following EuPathDB infrastructure changes.</p>
    <div style="text-align: justify;"><br>
      <span style="font-style: italic;"><span style="font-style: italic;"><span
            style="text-decoration: underline;">Note:</span>
          Our bimonthly database releases incorporate new data and correct old
          data when necessary.&nbsp; Changes in annotation and new experimental
          data may slightly alter your search results by increasing or
          decreasing the number of hits. When search parameters change with a
          new release, we invalidate (<span style="color: red;">Ã˜</span>) the
          search and ask you to rerun it. When IDs are updated or removed, we
          map the old IDs to the new ones, remove the old IDs from your Basket,
          and leave your Favorites page alone.</span></span><br>
      <span style="font-weight: bold;"><span style="font-style: italic;"></span></span>
      <p><em><strong>Follow</strong></em> <span style="font-weight: bold;"><span
            style="font-style: italic;">EuPathDB
            on <a href="https://www.facebook.com/pages/EuPathDB/133123003429972">FaceBook</a></span></span><span
          style="font-weight: bold; font-style: italic;">,
          <a href="http://twitter.com/eupathdb">Twitter</a>, and <a href="http://www.youtube.com/user/EuPathDB">YouTube</a></span></p>
      <span style="font-weight: bold; font-style: italic;"></span></div>
    <div style="text-align: justify;"><span style="font-weight: bold; font-style: italic;"></span><span
        style="font-weight: bold; font-style: italic;"></span></div>
  </body>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem26"/>
    <a name="plasmo08_20_news"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 12.0 Released <i class="date">10 September 2014</i>
                
            </h1>
            <body><span style="font-family: Helvetica,Arial,sans-serif;"> </span>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;"><strong>We
          are pleased to announce the release of PlasmoDB 12.0</strong><strong></strong><br>
      </span> <span style="font-family: Helvetica,Arial,sans-serif;"><br>
      </span> <span style="font-family: Helvetica,Arial,sans-serif;"><br>
      </span> <span style="font-family: Helvetica,Arial,sans-serif;"><strong>New
          data in this release</strong>:</span></div>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;">
      </span><br>
      <span style="font-family: Helvetica,Arial,sans-serif;"> </span></div>
    <ul style="text-align: justify;">
      <li><span style="font-family: Helvetica,Arial,sans-serif;"><strong><span style="color: #666666;"><em>P.
                falciparum</em> phosphoproteomic data:</span></strong> <span style="font-weight: normal;"></span></span><span
          style="font-family: Helvetica,Arial,sans-serif;"><span
            style="font-weight: normal;">Data
            was generated from tightly synchronized ring, trophozoites, and
            schizonts (<strong><a href="http://www.ncbi.nlm.nih.gov/pubmed/23914800">Pease
                et al.</a></strong>). View experimental details and links to
            searches in the <strong><a href="http://plasmodb.org/plasmo/getDataset.do?datasets=pfal3D7_massSpec_Chakrabarti_Phosphoproteome_RSRC">dataset
                section</a></strong>.&nbsp;</span></span> </li>
    </ul>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;">
      </span></div>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;"><br>
      </span> </div>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;">
      </span></div>
    <ul style="text-align: justify;">
    </ul>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;">
      </span></div>
    <ul style="text-align: justify;">
      <li><span style="font-family: Helvetica,Arial,sans-serif;"><strong style="color: #666666;"><em>P.
              falciparum</em> Polysomal and steady-state asexual stage
            transcriptomes:</strong>&nbsp;<span style="font-weight: normal;"> </span>Analysis
          of the levels of steady-state mRNA and polysome-associated mRNA at the
          ring, trophozoite, and schizont stages of the intra-erythrocytic cycle
          of <i>P. falciparum</i> 3D7 using Illumina sequencing (<strong><a href="http://www.ncbi.nlm.nih.gov/pubmed/24267660">Bunnik
              et al.</a></strong>). </span><span style="font-family: Helvetica,Arial,sans-serif;"><span
            style="font-weight: normal;">View
            experimental details and links to searches in the <strong><a href="http://plasmodb.org/plasmo/getDataset.do?datasets=pfal3D7_Bunnik_Asexual_Cell_Cycle_rnaSeq_RSRC">dataset
                section</a></strong>.&nbsp;</span></span></li>
    </ul>
    <span style=" font-family: Helvetica,Arial,sans-serif;"><span style=" font-weight: normal;"><br>
      </span></span>
    <ul style="text-align: justify;">
      <li><span style="font-family: Helvetica,Arial,sans-serif;"><strong><span style="color: #666666;"><em>P.
                berghei</em> transcript expression:</span></strong> <span style="font-weight: normal;"></span></span><span
          style="font-family: Helvetica,Arial,sans-serif;"><span
            style="font-weight: normal;">Illumina-based
            sequencing of RNA from five different stages of <em>P. berghei</em>
            ANKA development (4h Ring, 16h Trophozoite, 22h Schizont,
            Gametocytes, 24h Ookinete) (Hoeijmakers et al. unpublished). View
            experimental details and links to searches in the <strong><a href="http://plasmodb.org/plasmo/getDataset.do?datasets=pberANKA_Waters_Stunnenberg_five_stages_rnaSeq_RSRC">dataset
                section</a></strong>.</span></span></li>
    </ul>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;">
      </span></div>
    <ul style="text-align: justify;">
    </ul>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;">
      </span></div>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;"><br>
      </span> <span style="font-family: Helvetica,Arial,sans-serif;"><strong>New
          features:</strong><br>
      </span> <span style="font-family: Helvetica,Arial,sans-serif;"><br>
      </span>
      <blockquote> </blockquote>
      <ul>
        <li><span style="font-family: Helvetica,Arial,sans-serif;"><strong style="color: #666666;">Results
              analysis tools:&nbsp; </strong><span style="color: #020302;">results
              of any gene search can now be analyzed for statistical enrichment or
              Gene Ontology (GO) terms, metabolic pathways or gene product text
              terms. <strong><a href="/a/analysisTools.jsp">A detailed
                  description of this new functionality and a video tutorial are
                  also available</a></strong>.</span></span></li>
      </ul>
      <span style=" font-family: Helvetica,Arial,sans-serif;"><span style=" color: #020302;"><br>
        </span></span>
      <ul>
      </ul>
      <blockquote> </blockquote>
      <span style="font-family: Helvetica,Arial,sans-serif;"><br>
      </span> <span style="font-family: Helvetica,Arial,sans-serif;"><span style="font-weight: bold;">New
          Tutorials:</span><br>
      </span> <span style="font-family: Helvetica,Arial,sans-serif;"><br>
      </span> <a href="http://youtube.com/eupathdb"><span style="font-family: Helvetica,Arial,sans-serif;"><br>
        </span> </a>
      <div style="margin-left: 40px;"><span style="font-family: Helvetica,Arial,sans-serif;"><strong><a
              href="http://youtu.be/npgkkychkrI">Learn how
                  to use the Results Analysis tool</span></span></a>.&nbsp; </strong><br>
        </span> </div>
      <div style="margin-left: 40px;"> <span style="font-family: Helvetica,Arial,sans-serif;"><br>
        </span><span style="font-family: Helvetica,Arial,sans-serif;"><em></em></span><span
          style="font-family: Helvetica,Arial,sans-serif;"><br>
        </span> </div>
    </div>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;">
      </span></div>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;"><br>
      </span> <span style="font-family: Helvetica,Arial,sans-serif;"><span style="font-style: italic;"><span
            style="font-style: italic;"><span
              style="text-decoration: underline;">Note:</span>
            Our bimonthly database releases incorporate new data and correct old
            data when necessary.&nbsp; Changes in annotation and new
            experimental data may slightly alter your search results by
            increasing or decreasing the number of hits. When search parameters
            change with a new release, we invalidate (<span style="color: red;">Ã˜</span>)
            the search and ask you to rerun it. When IDs are updated or removed,
            we map the old IDs to the new ones, remove the old IDs from your
            Basket, and leave your Favorites page alone.</span></span><br>
      </span> <span style="font-family: Helvetica,Arial,sans-serif;"><span style="font-weight: bold;"><span
            style="font-style: italic;"></span></span></span>
      <p><span style="font-family: Helvetica,Arial,sans-serif;"><em><strong>Follow</strong></em>
          <span style="font-weight: bold;"><span style="font-style: italic;">EuPathDB
              on <a href="https://www.facebook.com/pages/EuPathDB/133123003429972">FaceBook</a></span></span><span
            style="font-weight: bold; font-style: italic;">,
            <a href="http://twitter.com/eupathdb">Twitter</a>, and <a href="http://www.youtube.com/user/EuPathDB">YouTube</a></span></span></p>
      <span style="font-family: Helvetica,Arial,sans-serif;"><span style="font-weight: bold; font-style: italic;"></span></span></div>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;">
      </span></div>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;"><span
          style="font-weight: bold; font-style: italic;"></span><span
          style="font-weight: bold; font-style: italic;"></span></span></div>
    <span style="font-family: Helvetica,Arial,sans-serif;"> </span>
  </body>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem27"/>
    <a name="plasmo05_14_news"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 11.1 Released <i class="date">6 May 2014</i>
                
            </h1>
            <body><span style="font-family: Helvetica,Arial,sans-serif;"> </span>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;"><strong>We
          are pleased to announce the release of PlasmoDB 11.1</strong><strong></strong><br>
      </span> <span style="font-family: Helvetica,Arial,sans-serif;"><br>
      </span> <span style="font-family: Helvetica,Arial,sans-serif;"><br>
      </span> <span style="font-family: Helvetica,Arial,sans-serif;"><strong>New
          data in this release</strong>:</span></div>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;">
      </span><br>
      <span style="font-family: Helvetica,Arial,sans-serif;"> </span></div>
    <ul style="text-align: justify;">
      <li><span style="font-family: Helvetica,Arial,sans-serif;"><strong><span style="color: #666666;">ChIP-seq
              data from a EuPathDB-funded project:</span></strong> <span style="font-weight: normal;">Genomic
            position of nucleosome binding in</span> <span style="font-weight: normal;"><em>P.
              falciparum</em> 3D7 was assayed over the intraerythrocytic cycle
            using ChIP-seq.&nbsp; This data is represented as coverage plots in
            the genome browser.&nbsp; View experimental details in the <a href="http://plasmodb.org/plasmo/getDataset.do?datasets=pfal3D7_Stunnenberg_DBP_chipSeq_RSRC">dataset
              section</a>.&nbsp; </span></span> </li>
    </ul>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;">
      </span></div>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;"><br>
      </span> </div>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;">
      </span></div>
    <ul style="text-align: justify;">
    </ul>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;">
      </span></div>
    <ul style="text-align: justify;">
      <li><span style="font-family: Helvetica,Arial,sans-serif;"><strong style="color: #666666;"><em>P.
              falciparum</em> quantitative proteomic data:</strong> <span style="font-weight: normal;">Data
            was generated from tightly synchronized ring, trophozoites, and
            schizonts (<a href="http://www.ncbi.nlm.nih.gov/pubmed/23914800">Pease
              et al.</a>). View experimental details and links to searches in
            the <a href="http://plasmodb.org/plasmo/getDataset.do?datasets=pfal3D7_quantitativeMassSpec_Chakrabarti_Phosphoproteome_RSRC">dataset
              section</a>.&nbsp; </span><br>
        </span> </li>
    </ul>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;">
      </span></div>
    <ul style="text-align: justify;">
    </ul>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;">
      </span></div>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;"><br>
      </span> <span style="font-family: Helvetica,Arial,sans-serif;"><strong>New
          features:</strong><br>
      </span> <span style="font-family: Helvetica,Arial,sans-serif;"><br>
      </span>
      <blockquote> </blockquote>
      <ul>
        <li><span style="font-family: Helvetica,Arial,sans-serif;"><strong style="color: #666666;">Customizable
              metabolic pathways:</strong> <span style="font-weight: normal;">This
              new feature in PlasmoDB remains in beta as we continue to hear
              from you about what is needed and how to improve it.&nbsp; In this
              version the pathways have been integrated using Cytoscape making
              it possible to layer functional data such as transcriptomic data
              or orthology on metabolic pathways.</span></span></li>
      </ul>
      <span style="font-family: Helvetica,Arial,sans-serif;"><span style=" font-weight: normal;"><br>
        </span></span>
      <ul>
        <li><span style="font-family: Helvetica,Arial,sans-serif;"><strong style="color: #666666;">Search
              genes based on polymorphism data:</strong> <span style="font-weight: normal;">The
              high throughput sequencing SNP searches have been upgraded to
              increase their speed and provide additional search functionality.
              Meta data associated with samples is now&nbsp; more intuitively
              organized. Search for genes based on <a href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=GeneQuestions.GenesByNgsSnps">SNP
                characteristics</a> or identify SNPs based on multiple criteria
              including within <a href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=SnpQuestions.NgsSnpsByIsolateGroup">one
                group of isolates</a> or between <a href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=SnpQuestions.NgsSnpsByTwoIsolateGroups">two
                groups of isolates</a>.</span></span></li>
      </ul>
      <span style="font-family: Helvetica,Arial,sans-serif;"><span style=" font-weight: normal;"><br>
        </span></span>
      <ul>
        <li><span style="font-family: Helvetica,Arial,sans-serif;"><strong style="color: #666666;">Filter
              samples based on metadata:</strong> <span style="font-weight: normal;">It
              is</span> <span style="font-weight: normal;">easier now to tailor
              your searches (such as SNP searches) by defining samples based on
              their metadata characteristics</span>.</span></li>
      </ul>
      <blockquote> </blockquote>
      <span style="font-family: Helvetica,Arial,sans-serif;"><br>
      </span> <span style="font-family: Helvetica,Arial,sans-serif;"><span style="font-weight: bold;">New
          Tutorials:</span><br>
      </span> <span style="font-family: Helvetica,Arial,sans-serif;"><br>
      </span> <span style="font-family: Helvetica,Arial,sans-serif;"><br>
      </span>
      <div style="margin-left: 40px;"><span style="font-family: Helvetica,Arial,sans-serif;"><a
            href="http://eupathdb.org/tutorials/GBrowse-UploadTrack-Help.pdf">Learn
            how to upload custom tracks in the genome browser</a>.&nbsp; <br>
        </span> </div>
      <div style="margin-left: 40px;"> <span style="font-family: Helvetica,Arial,sans-serif;"><br>
        </span> <span style="font-family: Helvetica,Arial,sans-serif;"><em>Our
            YouTube channel is updated regularly with new tutorials.&nbsp; The
            latest tutorials include:</em></span></div>
      <span style="font-family: Helvetica,Arial,sans-serif;"><em> </em><br>
      </span>
      <ul style="margin-left: 40px;">
        <li><span style="font-family: Helvetica,Arial,sans-serif;"><a href="http://youtu.be/SzRu9XBe_VM">Transform
              by Orthology Across Taxon Groups</a>&nbsp;</span></li>
      </ul>
      <div style="margin-left: 40px;"> <span style="font-family: Helvetica,Arial,sans-serif;"><br>
        </span> </div>
      <ul style="margin-left: 40px;">
        <li><span style="font-family: Helvetica,Arial,sans-serif;"><a href="https://www.youtube.com/watch?v=nYGS8LbbZ1w">Transform
              by Orthology within a Taxon Group</a></span></li>
      </ul>
      <div style="margin-left: 40px;"> <span style="font-family: Helvetica,Arial,sans-serif;"><br>
        </span> </div>
      <ul style="margin-left: 40px;">
        <li><span style="font-family: Helvetica,Arial,sans-serif;"><a href="https://www.youtube.com/watch?v=Q3wiUvqA82o">Orthology
              information on gene pages</a></span></li>
      </ul>
      <div style="margin-left: 40px;"> <span style="font-family: Helvetica,Arial,sans-serif;"><br>
        </span> </div>
      <ul style="margin-left: 40px;">
        <li><span style="font-family: Helvetica,Arial,sans-serif;"><a href="http://youtu.be/yKIKD8LEU-M">Inferring
              UTR length using RNA sequencing data</a></span></li>
      </ul>
      <div style="margin-left: 40px;"><span style="font-family: Helvetica,Arial,sans-serif;"><br>
        </span> </div>
      <ul style="margin-left: 40px;">
        <li><span style="font-family: Helvetica,Arial,sans-serif;"><a href="https://www.youtube.com/watch?v=dKhRaYINX9M">Decreasing
              Gene Page Load Time</a></span></li>
      </ul>
      <span style="font-family: Helvetica,Arial,sans-serif;"><br>
      </span> </div>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;">
      </span></div>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;"><br>
      </span> <span style="font-family: Helvetica,Arial,sans-serif;"><span style="font-style: italic;"><span
            style="font-style: italic;"><span
              style="text-decoration: underline;">Note:</span>
            Our bimonthly database releases incorporate new data and correct old
            data when necessary.&nbsp; Changes in annotation and new
            experimental data may slightly alter your search results by
            increasing or decreasing the number of hits. When search parameters
            change with a new release, we invalidate (<span style="color: red;">Ã˜</span>)
            the search and ask you to rerun it. When IDs are updated or removed,
            we map the old IDs to the new ones, remove the old IDs from your
            Basket, and leave your Favorites page alone.</span></span><br>
      </span> <span style="font-family: Helvetica,Arial,sans-serif;"><span style="font-weight: bold;"><span
            style="font-style: italic;"></span></span></span>
      <p><span style="font-family: Helvetica,Arial,sans-serif;"><em><strong>Follow</strong></em>
          <span style="font-weight: bold;"><span style="font-style: italic;">EuPathDB
              on <a href="https://www.facebook.com/pages/EuPathDB/133123003429972">FaceBook</a></span></span><span
            style="font-weight: bold; font-style: italic;">,
            <a href="http://twitter.com/eupathdb">Twitter</a>, and <a href="http://www.youtube.com/user/EuPathDB">YouTube</a></span></span></p>
      <span style="font-family: Helvetica,Arial,sans-serif;"><span style="font-weight: bold; font-style: italic;"></span></span></div>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;">
      </span></div>
    <div style="text-align: justify;"><span style="font-family: Helvetica,Arial,sans-serif;"><span
          style="font-weight: bold; font-style: italic;"></span><span
          style="font-weight: bold; font-style: italic;"></span></span></div>
    <span style="font-family: Helvetica,Arial,sans-serif;"> </span>
  </body>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem28"/>
    <a name="plasmo01_14_news"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 11.0 Released <i class="date">30 January 2014</i>
                
            </h1>
            <body>
    <div style="text-align: justify;"><strong>We are pleased to announce the
        release of PlasmoDB 11.0</strong><strong></strong><br>
      <br>
      <br>
      <strong>New data in this release</strong>:</div><br>
    <ul style="text-align: justify;">
      <li>Serum antibody levels determined by protein microarray (<strong><a
            href="http://www.ncbi.nlm.nih.gov/pubmed/20351286">Crompton
            <em>et al.</em></a></strong>).&nbsp; For full attribution and
        available searches please see the <strong><a href="http://plasmodb.org/plasmo/getDataset.do?datasets=pfal3D7_microarrayAntibody_Crompton_Mali_RSRC,pfal3D7_microarrayAntibody_Crompton_Mali_RSRC,pfal3D7_microarrayAntibody_Crompton_Mali_RSRC">dataset
            details</a></strong>.&nbsp; This data can be found under a new
        search category called "Host Response" that has been added to PlasmoDB under
        the heading "Identify genes By" on the home page.&nbsp; The search
        against this dataset offers versatile sample grouping based on metadata
        (ie. age, sex, disease state, etc...) and returns genes whose protein
        products were the target of antibodies found in the serum of malaria
        patients.</li>
    </ul>
    <div style="text-align: justify;"><br>
    </div>
    <ul style="text-align: justify;">
    </ul>
    <ul style="text-align: justify;">
      <li>The genome sequence and annotation of <em>Plasmodium yoelii yoelii </em>17X
        (a non-lethal 17X clone).&nbsp; Additional information about all genomes
        may be viewed in the <strong><a href="/a/getDataset.do?datasets=pyoeyoelii17X_primary_genome_RSRC">dataset
            section</a></strong> of PlasmoDB.&nbsp; Note that this genome was
        graciously provided by GeneDB pre-publication.&nbsp; Please consult with
        the Principal Investigator before undertaking large scale analyses of
        the annotation or underlying sequence.&nbsp; This genome sequence and
        annotation will replace the previous 17XNL sequence and annotation as
        the reference 17X genome. The genome sequence and annotation of the
        previous 17XNL has been temporarily removed but access is still
        available through the <strong><a href="http://v10-0.plasmodb.org/">previous
            version of PlasmoDB</a></strong>.&nbsp; </li>
    </ul>
    <ul style="text-align: justify;">
    </ul>
    <div style="text-align: justify;"><br>
      <strong>New features:</strong><br>
      <br>
      <span style="text-decoration: underline;">Public Strategies</span>:&nbsp;
      Share your search strategies with the community by making them
      public.&nbsp; Making one of your strategies public is simple; go to the
      "All" tab in your "My Strategies" section and click on a strategy's public
      checkbox.&nbsp; <br>
      <br>
      <span style="text-decoration: underline;">Enhanced BLAST results</span>:&nbsp;
      The gene results now include a column for score and percent identity
      enabling sorting of results.<br>
      <br>
      <span style="text-decoration: underline;">Enhanced GO Term and InterPro
        Domain searches</span>:&nbsp; Now this search includes an advanced
      type-ahead function that allows searching for multiple GO Terms or domains
      simultaneously.&nbsp; <br>
      <br>
      <span style="text-decoration: underline;">Email address update</span>:&nbsp;
      Registered users can now update their email addresses from their profiles.<br>
      <br>
      <span style="text-decoration: underline;">Updated organism filter tables</span>:&nbsp;
      The organism filter table has been unified across EuPathDB sites.<br>
      <br>
      <span style="text-decoration: underline;">Improved histogram</span>:&nbsp;
      Numerical columns in a result can now be analyzed with a dynamic histogram
      that enables binning and plotting values in normal or log scales.<br>
      <br>
      <span style="text-decoration: underline;">Fixed</span>: The orthology
      phyletic pattern search now remembers your selected parameters making it
      easier to revise searches.<br>
      <br>
      <br>
      <br>
      <strong>New database:&nbsp; </strong></div><br>
    <ul style="text-align: justify;">
      <li>EuPathDB is pleased to announce the release of a beta version of
        <a href="http://hostdb.org"><strong>HostDB</strong></a> - a resource
        dedicated to integrating functional host response data.&nbsp; The beta
        release includes the human and mouse genomes and two datasets (RNA-Seq
        and quantitative proteomics) examining the host response to Toxoplasma
        infection.&nbsp; We look forward to expanding this database in data
        content and functionality.&nbsp; We encourage you to explore <strong><a
            href="http://hostdb.org">HostDB</a></strong>
        and provide us with your <a href="mailto:help@hostdb.org"><strong>feedback</strong></a>.&nbsp;</li>
    </ul>
    <div style="text-align: justify;"><br>
    </div>
    <div style="text-align: justify;"><br>
      <span style="font-style: italic;"><span style="font-style: italic;"><span
            style="text-decoration: underline;">Note:</span>
          Our bimonthly database releases incorporate new data and correct old
          data when necessary.&nbsp; Changes in annotation and new experimental
          data may slightly alter your search results by increasing or
          decreasing the number of hits. When search parameters change with a
          new release, we invalidate (<span style="color: red;">Ã˜</span>) the
          search and ask you to rerun it. When IDs are updated or removed, we
          map the old IDs to the new ones, remove the old IDs from your Basket,
          and leave your Favorites page alone.</span></span><br>
      <span style="font-weight: bold;"><span style="font-style: italic;"></span></span><br>
      <span style="font-weight: bold;"><span style="font-style: italic;">Follow EuPathDB on <a href="https://www.facebook.com/pages/EuPathDB/133123003429972">FaceBook</a></span></span><span
        style="font-weight: bold; font-style: italic;">,
        <a href="http://twitter.com/eupathdb">Twitter</a>, and <a href="http://www.youtube.com/user/EuPathDB">YouTube</a></span></div>
    <div style="text-align: justify;"><span style="font-weight: bold; font-style: italic;"></span><span
        style="font-weight: bold; font-style: italic;"></span></div>
  </body>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem29"/>
    <a name="eupathworkshop2014"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                EuPathDB Annual Workshop <i class="date">14 January 2014</i>
                
            </h1>
            <body>
    <p>We are pleased to announce the 9th annual EuPathDB workshop to be held in
      Athens, Georgia, USA from June 15 to Wednesday, June 18, 2014.&nbsp;&nbsp;
      <br>
      <br>
      Please note that the workshop expenses, in addition to room and board
      costs are covered by EuPathDB (see cost and fees section below).&nbsp; <br>
      <br>
      Come learn how to effectively use EuPathDB resources and mine available
      data to help guide your research.<br>
      <br>
      For more information and to apply, please visit:
      http://workshop.eupathdb.org/current/<br>
      <br>
      The workshop will cover all of our component databases (EuPathDB,
      AmoebaDB, CryptoDB, GiardiaDB, MicrosporidiaDB, PlasmoDB, PiroplasmaDB,
      ToxoDB, TrichDB, TriTrypDB and OrthoMCL), and will run from 6:00PM (18:00)
      on Sunday, June 15 to 5:00PM (17:00) on Wednesday, June 18, 2014.<br>
      <br>
      The application deadline is February 28, 2014 and the course is limited to
      30 participants.<br>
      <br>
      <em><strong>Who should apply:</strong></em>&nbsp; This workshop is
      intended to be a 3 day thorough and intensive introduction to database
      resources that are part of the EuPathDB.org Bioinformatics Resource Center
      (EuPathDB, AmoebaDB, CryptoDB, GiardiaDB, MicrosporidiaDB, PlasmoDB,
      PiroplasmaDB, ToxoDB, TrichDB, TriTrypDB and OrthoMCL). The ideal
      candidate would have a need to become an expert user of one or more of
      these database resources and presumably has a bioinformatics research
      problem that they might work on as part of their training while attending
      the course. Preference will be given to applicants that are in a position
      to benefit most from intensive training and that are in a position to
      return to their home institution and transfer their newly acquired
      knowledge to others. Evaluation of the applications will focus heavily on
      your responses to the questions asked on the application page and on
      letters of recommendation.<br>
      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <br>
      <strong><em>Costs and Fees:</em> </strong>The course is free. Housing
      (double occupancy), all training materials and lunches are covered.
      Attendees are expected to provide their own transportation to the course
      and pay for a few of their meals. Letters of acceptance will be mailed
      early to facilitate the solicitation of outside travel funds.<br>
      <br>
      Hotel room charges for a shared room will be paid by EuPathDB for the
      nights of June 15 through June 18. Attendees who would like to have a
      private room will need to pay the difference of $47 per night plus
      tax.&nbsp;&nbsp;&nbsp; <br>
      <br>
      For more information and to apply, please visit:
      http://workshop.eupathdb.org/current/<br>
      <br>
      If you have any questions or concerns please email help@eupathdb.org<br>
      <br>
      &nbsp;&nbsp;&nbsp; <br>
      <br>
      &nbsp; </p>
  </body>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem30"/>
    <a name="plasmo09_13_news"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 10.0 Released <i class="date">25 September 2013</i>
                
            </h1>
            <body> <strong>We are pleased to announce the release of PlasmoDB 10.0</strong><strong></strong><br>
    <br>
    <br>
    <strong>New data in this release</strong>:<br>
    <br>
    <ul>
      <li>
        The following genomes have been updated.&nbsp; Additional information
        about all genomes may be viewed in the <strong><a href="http://plasmodb.org/plasmo/getDataset.do?display=detail#Genomes%20and%20Annotation">Dataset
            section</a></strong> of PlasmoDB.&nbsp; Note that many of these
        genomes have been graciously provided prepublication to the scientific
        community - please consult with the Principal Investigator before
        undertaking large scale analyses of the annotation or underlying
        sequence:</li>
    </ul>
    <br>
    <ul>
      <ul>
        <li>Plasmodium berghei ANKA (GeneDB version 03-01-2013)</li>
        <li>Plasmodium chabaudi chabaudi (GeneDB version 03-01-2013)</li>
        <li>Plasmodium cynomolgi B (GenBank version 09-19-2012)</li>
        <li>Plasmodium falciparum 3D7 (GeneDB version 03-01-2013)</li>
        <li>Plasmodium falciparum IT (GeneDB version 03-01-2013)</li>
        <li>Plasmodium knowlesi H (GeneDB version 03-01-2013)</li>
        <li>Plasmodium vivax SAL1 (GeneDB version 05-01-2013)</li>
        <li>Plasmodium yoelii yoeliiYM (GeneDB version 03-01-2013)</li>
      </ul>
    </ul>
    <br>
    <ul>
      <li><em>P. yoelii </em>salivary gland RNA sequence data from wild type
        and Puf2 knockout <em>P. yoelii</em> 17XNL parasites (<a href="http://www.ncbi.nlm.nih.gov/pubmed/23325771"><strong>Lindner
            et al.</strong></a>).&nbsp;</li>
    </ul>
    <br>
    <ul>
      <ul>
        <li>Genes may be identified based on their fold change difference
          between wild-type and knockout parasites or based on their percentile
          expression in each sample using the <a href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=InternalGeneDatasetQuestions.GenesByRNASeqEvidence"><strong>RNA
              Seq Evidence search page</strong></a> in PlasmoDB.&nbsp; </li>
      </ul>
    </ul>
    <br>
    <ul>
      <ul>
        <li>Graphical representation of this data is available in the expression
          section of <a href="http://plasmodb.org/gene/PY07121"><span style="font-weight: bold;">gene
              pages</span></a>.&nbsp;&nbsp;</li>
      </ul>
    </ul>
    <br>
    <ul>
      <ul>
        <li>Data from this experiment is available as coverage plots in the
          genome browser and evidence of intron spanning reads is indicated in
          the "Splice Site Junctions" track.&nbsp; In <strong><a href="http://plasmodb.org/cgi-bin/gbrowse/plasmodb/?start=10400;stop=13699;ref=AABL01002538;width=800;version=100;flip=0;grid=1;id=ca3e4ae045df12e094d286ad0b6823a0;l=Gene%1Epyoeyoelii17XNL_Kappe_rnaSeq_RSRCJunctions%1Epyoeyoelii17XNL_Kappe_rnaSeq_RSRCCoverage;h_feat=py07121%40yellow">this
              example</a></strong> a genome browser region shows differential
          expression of a gene and evidence of an alternative gene model that
          differs from the official annotation. </li>
      </ul>
    </ul>
    <br>
    <ul>
      <li>Total and putative surface proteomes of <em>P. falciparum</em> N54
        and <em>P. yoelii</em> 17XNL Salivary gland sporozoites (<a href="http://www.ncbi.nlm.nih.gov/pubmed/23325771"><strong>Lindner
            et al.</strong></a>).&nbsp; Peptides from <em>P. falciparum</em>
        N54&nbsp; samples were mapped to<em> </em>the<em> P. falciparum </em>3D7genome
        and peptides from <em>P. yoelii</em> 17XNL were mapped both to the
        17XNL and <em>P. berghei</em> genomes.&nbsp; </li>
    </ul>
    <br>
    <ul>
      <ul>
        <li>Genes with mapped peptides may be identified using the <a href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=GeneQuestions.GenesByMassSpec"><strong>Mass
              Spec evidence search page</strong></a> in PlasmoDB.&nbsp;&nbsp;</li>
      </ul>
    </ul>
    <br>
    <ul>
      <ul>
        <li>Peptide evidence is also displayed in the protein section of gene
          pages and in the genome browser by turning on the track - "Plasmodium
          MS/MS Peptides Salivary gland sporozoite total and surface proteomes
          (Lindner and Swearingen et al.)".</li>
      </ul>
    </ul>
    <br>
    <!--EndFragment--> <span class="tracktitle"> </span>
    <ul>
      <o:p></o:p>
    </ul>
    <ul>
      <ul>
      </ul>
    </ul>
    <strong>New features:</strong><br>
    <br>
    <p class="MsoNoSpacing"><u>GBrowse tracks remain open between sessions:</u></p>
    <p class="MsoNoSpacing"><br>
      <u><o:p></o:p></u></p>
    <ul>
      <li>EuPathDB now stores GBrowse track configurations between sessions for
        registered users who log into any EuPathDB site. The EuPathDB login is
        now integrated with the GBrowse login so that your previous session's
        GBrowse tracks are displayed each time you return to a EuPathDB site. </li>
    </ul>
    <br>
    <p class="MsoNoSpacing"><u>New streamlined look for choosing expression data
        sets (microarray, RNA-Seq):</u></p>
    <p class="MsoNoSpacing"><br>
      <u><o:p></o:p></u></p>
    <ul>
      <li>A new interactive table for choosing expression data sets lists
        datasets, searches for each dataset and the organism that the data is
        aligned to. Use the table's filter function to limit the data set list
        based on author name, parasite life stages, etc. Click on an icon under
        "Choose a search" to collapse the table and display the data set's
        search page with its parameter list below the collapsed table. </li>
    </ul>
    <br>
    <p class="MsoNoSpacing"><u>On-page help for expression search pages:</u></p>
    <p class="MsoNoSpacing"><br>
      <u><o:p></o:p></u></p>
    <ul>
      <li>Search parameters appear on the left and are arranged in a sentence
        that helps you choose parameter values. </li>
    </ul>
    <br>
    <ul>
      <li>A help graphic and text appear on the right. To help you interpret
        your parameters choices, the graphic changes to reflect your parameter
        settings.</li>
    </ul>
    <br>
    <p class="MsoNoSpacing"><u>Improved Strategy Panel for viewing and editing
        steps:</u></p>
    <p class="MsoNoSpacing"><br>
      <u><o:p></o:p></u></p>
    <ul>
      <li>The graphic interface of the strategy panel has been updated.<span style="mso-spacerun:yes">&nbsp;
          </span>View the results of a strategy step by clicking anywhere inside
        the box representing that step. To edit the parameter values for that
        step, click on the EDIT icon that appears in the corner of each strategy
        step box when hovering over the strategy panel.<o:p></o:p></li>
    </ul>
    <br>
    <span style="font-style: italic;"><span style="font-style: italic;"><span style="text-decoration: underline;">Note:</span>
        Our bimonthly database releases incorporate new data and correct old
        data when necessary.&nbsp; Changes in annotation and new experimental
        data may slightly alter your search results by increasing or decreasing
        the number of hits. When search parameters change with a new release, we
        invalidate (<span style="color: red;">Ã˜</span>) the search and ask you
        to rerun it. When IDs are updated or removed, we map the old IDs to the
        new ones, remove the old IDs from your Basket, and leave your Favorites
        page alone.</span></span><br>
    <span style="font-weight: bold;"><span style="font-style: italic;"></span></span><br>
    <span style="font-weight: bold;"><span style="font-style: italic;">EuPathDB
        is now on <a href="https://www.facebook.com/pages/EuPathDB/133123003429972">FaceBook</a></span></span><span
      style="font-weight: bold; font-style: italic;">,
      <a href="http://twitter.com/eupathdb">Twitter</a>, and <a href="http://www.youtube.com/EuPathDB/">YouTube</a></span><span
      style="font-weight: bold; font-style: italic;"></span>
  </body>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem31"/>
    <a name="plasmo03_13_news"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 9.3 Released <i class="date">10 March 2013</i>
                
            </h1>
            <body>
    <p style="text-align: justify;"><b style="mso-bidi-font-weight:normal">We
        are pleased to announce the release of PlasmoDB 9.3<o:p></o:p></b> </p>
    <p style="text-align: justify;" class="MsoNormal"><b style="mso-bidi-font-weight:normal">New
        data in this release: </b></p>
    <ul style="text-align: justify;">
      <li>Transcriptional variation across subclonal strains derived from <em>P.
          falciparum</em> strains 3D7, 7G8, D10 and HB3 <span style="font-weight: bold;">(<a
            href="http://www.ncbi.nlm.nih.gov/pubmed/22415456">Rovira-Graells
            <em>et al.</em></a>)</span>.&nbsp; This data may be used to search
        for genes based on fold change, percentile expression and copy number
        variation (<a href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=InternalQuestions.GenesByMicroarrayEvidence"><span
            style="font-weight: bold;">available
            in the microarray query section under the heading "Transcriptional
            variation of <em>P. falciparum</em> strains"</span></a>).&nbsp; In
        addition, expression graphs are available on individual gene pages in
        the transcript expression section under the heading "<a href="http://plasmodb.org/gene/PF3D7_0220800"><strong>Transcriptional
            Variation of P. fal parasite lines</strong></a>".&nbsp; Note that
        graphs representing lineage-specific expression over the
        intraerythrocytic life cycle may be displayed by selecting the
        appropriate check box to the right of the graphs.</li>
    </ul>
    <br>
    <ul style="text-align: justify;">
    </ul>
    <ul style="text-align: justify;">
      <li>High throughput sequencing (HTS) of 65 clinical isolates from Gambia (<a
          href="http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1002992"><strong>Amambua-Ngwa
            </strong><strong><em>et al.</em></strong></a>).<u><o:p></o:p></u>&nbsp;
        This data may be used to identify single nucleotide polymorphisms (SNPs)
        between the sequenced isolates:</li>
    </ul>
    <br>
    <ul style="text-align: justify;">
    </ul>
    <ul style="text-align: justify;">
      <ul>
        <li><o:p></o:p> SNPs between all or a subset of strains from this study
          can be defined based on a comparison with each other or with strains
          from other studies.&nbsp; This <a href="http://plasmodb.org/plasmo/im.do?s=01512139d0389c89"><strong>search
              strategy</strong></a> defines all SNPs present in strains from
          this study.&nbsp; </li>
      </ul>
    </ul>
    <br>
    <ul style="text-align: justify;">
      <ul>
      </ul>
    </ul>
    <ul style="text-align: justify;">
      <ul>
        <li>Genes may defined based on their relation to the above defined SNPs
          using the genomic colocation query, for example:
          <ul>
            <li><a href="http://plasmodb.org/plasmo/im.do?s=e5ec7e2dde7949f8"><strong>Gene
                  that contain any of the SNPs</strong></a></li>
            <li><a href="http://plasmodb.org/plasmo/im.do?s=bb44a66099c67fc1"><strong>Genes
                  that are lo</strong><strong>cated downstream of a SNP (in this
                  example 200 nucleotides downstream)</strong></a></li>
          </ul>
          <br>
          <ul>
          </ul>
        </li>
      </ul>
    </ul>
    <ul>
      <ul>
        <li>View and download SNP records generated from pairwise comparisons of
          isolate sequences.</li>
      </ul>
    </ul>
    <br>
    <ul>
      <ul>
      </ul>
    </ul>
    <ul>
      <ul>
        <li>Search for <strong><a href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=InternalQuestions.GenesBySnps">genes
based
              on the characteristics of SNPs contained within the gene</a></strong>.
          This dataset can be queried by clicking on "HTS SNP Characteristics".</li>
      </ul>
    </ul>
    <br>
    <ul>
      <ul>
      </ul>
    </ul>
    <ul>
      <ul>
        <li>Search for individual SNP records based on criteria such as SNP ID,
          strain, genomic location, etc. </li>
      </ul>
    </ul>
    <br>
    <ul>
      <ul>
      </ul>
    </ul>
    <ul>
      <ul>
        <li>View sequencing read pileups, reads aligned to the <i>P. falciparum 3D7</i>
          genome, and read density plots in GBrowse using the options under the
          Select Tracks heading: <a href="http://plasmodb.org/cgi-bin/gbrowse/plasmodb/?name=Pf3D7_11_v3:1278856..1310724;h_feat=pf3d7_1133400@yellow"><strong>HTS
SNPs,
              pfal3D7_Conway_HTS_SNP</strong>.</a></li>
      </ul>
    </ul>
    <br>
    <ul>
      <ul>
      </ul>
    </ul>
    <ul>
      <ul>
        <li>View SNP information for individual genes on record pages.<span style="mso-spacerun:yes">&nbsp;
            </span></li>
      </ul>
    </ul>
    <span style=" mso-spacerun:yes"><br>
    </span>
    <ul>
      <ul>
      </ul>
    </ul>
    <span style=" mso-spacerun:yes"></span>
    <ul>
      <ul>
        <li>View individual <strong><a href="http://qa.plasmodb.org/cgi-bin/gbrowse/plasmodb/?name=Pf3D7_11_v3:1278856..1310724;h_feat=pf3d7_1133400@yellow">HTS
SNPs
              aligned to the genome</a></strong><strong> </strong>with the
          GBrowse track: HTS SNPs by coding potential.</li>
      </ul>
    </ul>
    <br>
    <ul>
      <ul>
      </ul>
    </ul>
    <ul>
      <li>Metabolite data generated from <em>P. falciparum</em> parasites and
        infected and uninfected red blood cells.&nbsp; <a href="http://plasmodb.org/plasmo/getDataset.do?question=CompoundQuestions.CompoundsByFoldChange&amp;display=detail#30"><span
            style="font-weight: bold;">Experimental
            details</span></a> are available in the data set section of
        PlasmoDB.&nbsp; This data was generated as part of the EuPathDB-funded
        driving biological project and made available pre-publication (Llinas,
        Baska and Lewis).&nbsp; Data from this experiment may be searched based
        on <a href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=CompoundQuestions.CompoundsByFoldChange"><span
            style="font-weight: bold;">metabolite
            levels</span></a> and viewed graphically on compound pages.</li>
    </ul>
    <br>
    <ul>
    </ul>
    <ul style="text-align: justify;">
      <li>Two new data types with searches have been added in this release:
        compounds and metabolic pathways.&nbsp; This beta-release of Compound
        and Metabolic Pathways data sets incorporates KEGG pathways and a subset
        of PubChem records. Future releases will include compounds for
        additional metabolites and small molecules, and pathways from other
        sources such as MPMP, MetaCyc, etc. Please explore the site and contact
        us with your feedback. </li>
    </ul>
    <br>
    <ul style="text-align: justify;">
    </ul>
    <b style=" mso-bidi-font-weight:normal"></b>
    <ul style="text-align: justify;">
      <ul>
        <li><span style=" mso-bidi-font-weight:normal">Metabolic Pathways may be
            searched based on </span><b style=" mso-bidi-font-weight:normal"><a
              href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=PathwayQuestions.PathwaysByPathwayID">pathway
              Name/ID</a></b><span style=" mso-bidi-font-weight:normal">, </span><b
            style=" mso-bidi-font-weight:normal"><a
              href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=PathwayQuestions.PathwaysByGeneList">genes</a></b><span
            style=" mso-bidi-font-weight:normal">,
            or</span><b style=" mso-bidi-font-weight:normal"> <a href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=PathwayQuestions.PathwaysByCompounds">compounds</a></b><span
            style=" mso-bidi-font-weight:normal">.</span></li>
      </ul>
    </ul>
    <span style="  mso-bidi-font-weight:normal"><br>
    </span>
    <ul style="text-align: justify;">
      <ul>
      </ul>
    </ul>
    <span style="  mso-bidi-font-weight:normal"></span>
    <ul style="text-align: justify;">
      <ul>
        <li><span style="  mso-bidi-font-weight:normal">Compounds may be
            searched based on </span><a href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=CompoundQuestions.CompoundsByCompoundID"><b
              style="  mso-bidi-font-weight:normal">compound
              ID</b></a><span style="  mso-bidi-font-weight:normal">, </span><a
            href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=CompoundQuestions.CompoundsByTextSearch"><b
              style="  mso-bidi-font-weight:normal">text
              (synonym, InChI, etc.)</b></a><span style="  mso-bidi-font-weight:normal">,</span><b
            style="  mso-bidi-font-weight:normal">
            <a href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=CompoundQuestions.CompoundsByEcReaction">enzymes</a></b><span
            style="  mso-bidi-font-weight:normal">,</span><b
            style="  mso-bidi-font-weight:normal">
            <a href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=CompoundQuestions.CompoundsByPathway">metabolic
              pathway</a></b><span style="  mso-bidi-font-weight:normal">,</span><b
            style="  mso-bidi-font-weight:normal">
            <a href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=CompoundQuestions.CompoundsByMolecularFormula">molecular
              formula</a></b><span style="  mso-bidi-font-weight:normal">,</span><b
            style="  mso-bidi-font-weight:normal">
            <a href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=CompoundQuestions.CompoundsByMolecularWeight">molecular
              weight</a> </b><span style="  mso-bidi-font-weight:normal">or</span><b
            style="  mso-bidi-font-weight:normal">
            <a href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=CompoundQuestions.CompoundsByFoldChange">metabolite
              levels</a></b><span style="  mso-bidi-font-weight:normal">.</span></li>
      </ul>
    </ul>
    <p style="text-align: justify;" class="MsoNormal"><b style="mso-bidi-font-weight:normal"><br>
      </b></p>
    <p style="text-align: justify;" class="MsoNormal"><b style="mso-bidi-font-weight:normal">New
        Functionality:<o:p></o:p></b></p>
    <p style="text-align: justify;" class="MsoNoSpacing"><u>PCR Primer Design
        tool added to GBrowse. <o:p></o:p></u></p>
    <p style="text-align: justify;" class="MsoNormal">Design PCR primers to any
      region of the genome using this interactive, graphic interface. To reach
      this tool, <strong><a href="http://plasmodb.org/cgi-bin/gbrowse/plasmodb/?name=Pf3D7_11_v3:1278856..1310724&amp;h_feat=pf3d7_1133400@yellow&amp;auth_tkt=MTM0YWNkZGRhODEwYjJkZjkwODIyOTdhOGYyZWIyOGY1MTZjOWQxOWFwaWRiIWFwaWRiITEzNjYwNzI2MDE6MC4wLjAuMA==">open
the
          PlasmoDB Genome Browser</a></strong> at your genomic region of
      interest, choose "Design PCR primers" from the dropdown menu
      and click "GO".<span style="mso-spacerun:yes">&nbsp; </span>To design
      primers, focus on a PCR product size range by highlighting the area on the
      scale, then enter your preferred PCR parameters in the text boxes and
      click Design Primers. <o:p></o:p></p>
    <p style="text-align: justify;" class="MsoNormal"><o:p>&nbsp;</o:p></p>
  </body>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem32"/>
    <a name="eupathworkshop"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                EuPathDB Annual Workshop <i class="date">20 January 2013</i>
                
            </h1>
            <body> 
    <p align="justify">We are pleased to announce the 8th annual EuPathDB
      workshop to be held in Athens, Georgia, USA from June 2-5, 2013.   <br />
      <br />
      Please note that the workshop expenses, in addition to room and board
      costs are covered by EuPathDB (see cost and fees section below).  <br />
      <br />
      Come learn how to effectively use EuPathDB resources and mine available
      data to help guide your research.<br />
      <br />
      <b><a href="http://workshop.eupathdb.org/current/">For more information
          and to apply, please visit: http://workshop.eupathdb.org/current/</a></b><br />
      <br />
      The workshop will cover all of our component databases (EuPathDB,
      AmoebaDB, CryptoDB, GiardiaDB, MicrosporidiaDB, PlasmoDB, PiroplasmaDB,
      ToxoDB, TrichDB, TriTrypDB and OrthoMCL), and will run from 6:00PM (18:00)
      on Sunday, June 2 to 5:00PM (17:00) on Wednesday, June 5, 2013.<br />
      <br />
      The application deadline is February 18, 2013 and the course is limited to
      30 participants.<br />
      <br />
      Who should apply:  This workshop is intended to be a 3 day thorough and
      intensive introduction to database resources that are part of the
      EuPathDB.org Bioinformatics Resource Center (EuPathDB, AmoebaDB, CryptoDB,
      GiardiaDB, MicrosporidiaDB, PlasmoDB, PiroplasmaDB, ToxoDB, TrichDB,
      TriTrypDB and OrthoMCL). The ideal candidate would have a need to become
      an expert user of one or more of these database resources and presumably
      has a bioinformatics research problem that they might work on as part of
      their training while attending the course. Preference will be given to
      applicants that are in a position to benefit most from intensive training
      and that are in a position to return to their home institution and
      transfer their newly acquired knowledge to others. Evaluation of the
      applications will focus heavily on your responses to the questions asked
      on the application page and on letters of recommendation.<br />
      <br />
      Costs and Fees: The course is free. Housing, all training materials and
      most meals are covered. Attendees are expected to provide their own
      transportation to the course and pay for a few of their meals. No travel
      funds are available. Letters of acceptance will be mailed early to
      facilitate the solicitation of outside travel funds. Hotel room charges
      will be paid by EuPathDB from June 2 through 11:00am June 6 (Note: course
      ends at 5PM on June 5). Participants traveling from abroad may be able to
      arrive a day early or stay a day late as needed to accommodate travel
      arrangements.<br />
      <br />
      For more information and to apply, please visit: <b><a href="http://workshop.eupathdb.org/current/">http://workshop.eupathdb.org/current/</a></b><br />
      <br />
      If you have any questions or concerns please email <strong><a href="mailto:help@eupathdb.org">help@eupathdb.org</a></strong><br />
    </p>
  </body>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem33"/>
    <a name="plasmo_icmer_news"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                DEVELOPMENT OF A MOLECULAR BARCODE FOR PLASMODIUM VIVAX <i class="date">2 November 2012</i>
                
            </h1>
            <?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta http-equiv="Content-Type" content="text/html;charset=UTF-8" />
    <title></title>
  </head>
  <body>
    <p>
      <!--StartFragment-->
    </p>
    <p align="justify" style="margin-bottom:6.0pt;mso-pagination:none;mso-layout-grid-align:
none;text-autospace:none" class="MsoNormal"><span style="font-family:Optima;mso-bidi-font-family:
Helvetica">The Harvard School of Public Health and Broad Institute including
        Professors Sarah Volkman, Manoj Duraisingh, Daniel Hartl, Daniel
        Neafsey,
        Pardis Sabeti, and Dyann Wirth with funding support from the Bill and
        Melinda
        Gates Foundation (BMGF) are in the process of developing a molecular
        barcode
        tool for <i style="mso-bidi-font-style:normal">Plasmodium vivax</i>
        with the
        collaboration of the malaria community.<span style="mso-spacerun:yes"> 
        </span>We invite members of this community who are interested in either
        collaboration or potential application of this tool to participate in
        this
        endeavor. <br />
      </span></p>
    <p align="justify" style="margin-bottom:6.0pt;mso-pagination:none;mso-layout-grid-align:
none;text-autospace:none" class="MsoNormal"><span style="font-family:Optima;mso-bidi-font-family:
Helvetica"><br />
      </span></p>
    <p align="justify" style="margin-bottom:6.0pt;mso-pagination:none;mso-layout-grid-align:
none;text-autospace:none" class="MsoNormal"><span style="font-family:Optima;mso-bidi-font-family:
Helvetica">Additional information is available in <strong><a href="http://plasmodb.org/plasmo/communityDownload.do?fname=Pvivax_MolecularBarcode_ConceptPaper_MalariaCommunity.pdf">this
            downloadable document</a></strong>.<br />
      </span></p>
    <span style="font-family:Optima;mso-bidi-font-family:
Helvetica"><span style="mso-spacerun:yes"></span></span><!--EndFragment-->
  </body>
</html>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem34"/>
    <a name="plasmo08_12_news"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 9.1 Released <i class="date">31 August 2012</i>
                
            </h1>
            <html>
<head>
<meta content="text/html; charset=ISO-8859-1"
http-equiv="content-type">
<title>plasmo08_12</title>
</head>
<body>
<div style="font-family: Arial; text-align: justify;">
<span style="font-weight: bold;">We are pleased to
announce the release of PlasmoDB 9.1</span><br>
<span style="font-weight: bold;">
</span><br>
<br style="font-weight: bold;">
<span style="font-weight: bold;">New Data in this release:</span><br>
<br>
</div>
<div style="text-align: justify; font-family: Arial;">
</div>
<div style="font-family: Arial; text-align: justify;"><span
style="font-style: italic;"></span>
</div>
<div style="text-align: justify; font-family: Arial;"><br>
</div>
<ul style="text-align: justify; font-family: Arial;">
<li><span style="font-style: italic;">P. falciparum</span> schizont
phosphoproteomic data (<a style="font-weight: bold;"
href="http://www.ncbi.nlm.nih.gov/pubmed/22127061">Solyakov et al.</a>).&nbsp;
This data can be searched by choosing "Schizont Phospho-proteomic analysis
(Solyakov et al.)" in the <span style="font-style: italic;">P.
falciparum</span> section of the <a
href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=GeneQuestions.GenesByMassSpec"><span
style="font-weight: bold;">Mass spec. evidence search page</span></a>.
Peptides from this dataset are displayed on gene pages in the <a
href="http://plasmodb.org/gene/PF3D7_1149000"><span
style="font-weight: bold;">Protein section</span></a> and in the
Genome Browser by turning on the <a style="font-weight: bold;"
href="http://plasmodb.org/cgi-bin/gbrowse/plasmodb/?start=1838341;stop=2038340;ref=Pf3D7_11_v3;width=800;version=100;flip=0;grid=1;id=5cc2e495ddd82740efe816ccb900a3d8;l=Gene%1ETobinPhosphoPeptides;h_feat=pf3d7_1149000%40yellow">data
track: Pf. Mass Spec - Schizont Pohspho-proteomic analysis (Solyakov et
al.)</a>. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <br>
</li>
</ul>
<br>
<div style="margin-left: 40px; font-family: Arial;">
<div style="margin-left: 40px;">
<ul style="list-style-type: circle;">
<li>Click <a style="font-weight: bold;"
href="http://plasmodb.org/plasmo/im.do?s=610ffb3f192cddbd">here</a> to
access a
search strategy that identifies all <span style="font-style: italic;">P.
falciparum</span> genes with evidence of phosphorylation from this
study and from a previously incorporated phosphoproteomic study (<a
href="http://www.ncbi.nlm.nih.gov/pubmed/22018241"><span
style="font-weight: bold;">Treeck &amp; Sanders <span
style="font-style: italic;">et. al.</span></span></a>).</li>
</ul>
</div>
<br>
<ul style="text-align: justify; font-family: Arial;">
<li>Single nucleotide polymorphism calls based on high throughput
sequencing data of:</li>
</ul>
<br style="font-family: Arial;">
<ul style="text-align: justify; font-family: Arial;">
<ul>
<li><span style="font-style: italic;">P. falciparum</span>&nbsp;
isolates: 7G8, CS2, GB4, T9_94, V1_S generated at the the Sanger
institute (<a style="font-weight: bold;"
href="Single%20nucleotide%20polymorphism%20calls%20based%20on%20high%20throughput%20sequencing%20data%20of:">additional
details available here</a>).</li>
</ul>
</ul>
<br>
<ul style="text-align: justify; font-family: Arial;">
<ul>
<li><span style="font-style: italic;">P. vivax </span>isolates:
Brazil I, Mauritania I, North Korea and India- VII generated at the
the Broad Institute (<a style="font-weight: bold;"
href="Single%20nucleotide%20polymorphism%20calls%20based%20on%20high%20throughput%20sequencing%20data%20of:">additional
details available here</a>).</li>
</ul>
</ul>
<br>
<ul>
<ul>
<ul>
<li><font face="Arial">To identify genes based on their SNP
characteristics, click on "<a style="font-weight: bold;"
href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=GeneQuestions.GenesByHtsSnps">SNP
Characteristics</a>" under the "Population Biology" heading in the
"Identify Genes By" section of the home page. &nbsp;&nbsp; You can
select whether you to identify SNPs in <span
style="font-style: italic;"><span style="font-style: italic;"><span
style="font-style: italic;">P. vivax </span></span></span>or <span
style="font-style: italic;">P. falciparum </span>using the organism
drop-down menu.</font></li>
</ul>
</ul>
</ul>
<font face="Arial"></font>
<ul>
<ul>
<ul>
<li><font face="Arial">SNP searches may be combined with other
searches, for example, <a style="font-weight: bold;"
href="http://plasmodb.org/plasmo/im.do?s=0bb92940672255a8">this
search strategy</a> identifies secreted (predicted by SignalP) <span
style="font-style: italic;">P. vivax</span> genes that contain at
least 10 non-synonymous SNPs between the Sal1 and India-VII isolates.</font></li>
</ul>
</ul>
</ul>
<font face="Arial"></font>
<ul>
<ul>
<ul>
<li><font face="Arial">SNPs may also be visualized in tabular
format on gene pages or as a <a style="font-weight: bold;"
href="http://plasmodb.org/cgi-bin/gbrowse/plasmodb/?start=3013885;stop=3023884;ref=PviS_CM000455;width=800;version=100;flip=0;grid=1;id=92cab3ccf37d8dcd44a48a8db9fa3bb5;l=HtsAlignmentSNPsByClass%1EGene;h_feat=pvx_101555%40yellow">Genome
Browser track</a>. In the Genome Browser select the track called "HTS
SNPs by coding potential".&nbsp; Individual diamonds in the genome
browser track represent SNPs.&nbsp; Mousing over these graphics reveals
additional information such as the type of SNP, the position of the SNP
and the nucleotide and amino acid changes.</font></li>
</ul>
</ul>
</ul>
<font face="Arial"></font>
<ul>
<ul>
<ul>
<li><font face="Arial"><span style="font-weight: bold;"><a
href="http://plasmodb.org/cgi-bin/gbrowse/plasmodb/?start=1294740;stop=1294839;ref=Pf3D7_11_v3;width=800;version=100;flip=0;grid=1;id=92cab3ccf37d8dcd44a48a8db9fa3bb5;l=Gene%1Epfal3D7_Sanger_HTS_Isolates_7G8_Alignment;h_feat=pf3d7_1133400%40yellow">Alignment
of high throughput sequence reads</a> </span>are also available via
the Genome Browser.</font></li>
</ul>
</ul>
</ul>
<font face="Arial"></font>
<ul>
<ul>
<ul>
<li><font face="Arial"><a style="font-weight: bold;"
href="http://plasmodb.org/plasmo/showRecord.do?name=SnpRecordClasses.SnpRecordClass&amp;source_id=NGS_SNP.Pf3D7_08_v3.1820&amp;project_id=PlasmoDB">Individual
SNP records may also be visualized</a>. SNP record pages include
information such as the type of the SNP, strain information and links
to the read alignments supporting the SNP call.</font></li>
</ul>
</ul>
</ul>
<br>
</div>
<div style="font-family: Arial; text-align: justify;">&nbsp;<br>
<span style="font-style: italic;"><span style="font-style: italic;"><span
style="text-decoration: underline;">Note:</span>
Our bimonthly database releases incorporate new data and correct old
data when necessary.&nbsp; Changes in annotation and new experimental
data
may slightly alter your search results by increasing or decreasing the
number of hits. When search parameters change with a new release, we
invalidate (<span style="color: red;">Ã˜</span>) the search and ask you
to rerun it. When IDs are updated or removed, we map the old IDs to the
new ones, remove the old IDs from your Basket, and leave your Favorites
page alone.</span></span><br>
<span style="font-weight: bold;"><span style="font-style: italic;"></span></span><br>
<br>
<span style="font-weight: bold;"><span style="font-style: italic;"></span></span><br>
<span style="font-weight: bold;"><span style="font-style: italic;">EuPathDB
is now on <a
href="https://www.facebook.com/pages/EuPathDB/133123003429972">FaceBook</a></span></span><span
style="font-weight: bold; font-style: italic;"> and <a
href="http://twitter.com/eupathdb">Twitter</a></span></div>
</body>
</html>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem35"/>
    <a name="plasmo05_12_news"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 9.0 Released <i class="date">23 May 2012</i>
                
            </h1>
            <html>
<head>
<meta content="text/html; charset=ISO-8859-1"
http-equiv="content-type">
<title>plasmo05_12</title>
</head>
<body>
<div style="font-family: Arial; text-align: justify;">
<span style="font-weight: bold;">We are pleased to
announce the release of PlasmoDB 9.0</span><span
style="font-weight: bold;"></span><br>
<span style="font-weight: bold;">
</span><br>
This is a major release of PlasmoDB which constitutes a complete
rebuild of the database.&nbsp; This means that all genomes, data and
analysis were redone and improved. In addition, many of the
expression searches were updated to enable you to perform more accurate
searches.&nbsp; We encourage you to explore these searches and as usual
we look forward to hearing from you about how to further improve
them.&nbsp; <br>
<br>
Since this is a major update, we are keeping the old version (<a
style="font-weight: bold;" href="http://old.plasmodb.org">PlasmoDB 8.2</a>)
available so you can access your old queries and previous analysis if
needed. <br>
<br style="font-weight: bold;">
<span style="font-weight: bold;">New Data:</span><br>
<br>
</div>
<div style="font-family: Arial; text-align: justify;"><span
style="font-style: italic;"></span>
</div>
<ul style="text-align: justify; font-family: Arial;">
<li>The following genomes and annotation have been updated from
GeneDB:</li>
</ul>
<br>
<ul style="text-align: justify; font-family: Arial;">
<ul>
<li><span style="font-style: italic;">P. falciparum</span> 3D7
(February
2012 GeneDB version) <span style="font-style: italic;">Note: The new
P. falciparum gene IDs have
been fully integrated in this release.&nbsp; Old gene IDs are still
displayed on gene pages and searches can still be performed with old
gene IDs.</span> </li>
<li><span style="font-style: italic;">P. falciparum </span>IT
(February
2012 GeneDB version)<span style="font-style: italic;"></span></li>
<li><span style="font-style: italic;">P. berghei</span>&nbsp; ANKA
(March 2012 GeneDB version)<br>
</li>
<li><span style="font-style: italic;">P. chabaudi</span> AS (March
2011 GeneDB version) <span style="font-style: italic;">Note: A newer
version of this genome and annotation will be included in the next
release of PlasmoDB.</span></li>
</ul>
</ul>
<br>
<div style="margin-left: 80px;"><span style="font-style: italic;">Note:
A</span><a style="font-weight: bold; font-style: italic;"
href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=GeneQuestions.GenesByOldAnnotation">
query is available</a><span style="font-style: italic;"> to identify
genes that have changed between releases 8.2 and 9.0.&nbsp; In
addition, you can view gene models from the previous release as a </span><a
style="font-weight: bold; font-style: italic;"
href="http://plasmodb.org/cgi-bin/gbrowse/plasmodb/?start=886494;stop=917853;ref=Pf3D7_02_v3;width=800;version=100;flip=0;grid=1;id=f8185909a53487c6f854ad46c720c922;l=Gene%1EVersion82Genes;h_feat=pf3d7_0222900%40yellow">genome
browser track</a><span style="font-style: italic;">. <br>
<br>
</span> </div>
<ul style="text-align: justify; font-family: Arial;">
<li>Strand-specific, amplification-free transcriptome sequence (<a
style="font-weight: bold;"
href="http://www.ncbi.nlm.nih.gov/pubmed/20081834">FRT-Seq</a>) from <span
style="font-style: italic;">P. falciparum</span> 3D7 parasites (Pooled
red blood cell stage).&nbsp; This data was graciously provided
prepublication by the Berriman and Newbold labs and may be visualized
as a track in the genome browser.&nbsp; <a style="font-weight: bold;"
href="http://plasmodb.org/cgi-bin/gbrowse/plasmodb/?start=1190700;stop=1240699;ref=Pf3D7_06_v3;width=800;version=100;flip=0;grid=1;id=d84b8c628a88e60af27b19119a69f911;l=RnaSeqOttoCont%1EGene">Here
is an example genome browser view of chromosome 6 with the "Pf RNA-Seq
- Sanger FRT (log2) [Newbold / Berriman labs]" turned on</a>.&nbsp;
Note that histogram colors correspond to transcript direction (blue =
forward strand, red = reverse strand).</li>
</ul>
<br>
<ul style="text-align: justify; font-family: Arial;">
<li>Updated linkouts to the <a style="font-weight: bold;"
href="http://sites.huji.ac.il/malaria/">Malaria Parasite Metabolic
Pathways
(MPMP) database</a>.&nbsp; Numerous pathways have been added and
updated by
Dr. Hagai Ginsburg.&nbsp;&nbsp; Genes included in the MPMP database can
be searched based on
metabolic pathway in PlasmoDB <a style="font-weight: bold;"
href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=GeneQuestions.GenesByMetabolicPathway">here</a>.</li>
</ul>
<br>
<div style="font-family: Arial; text-align: justify;"><span
style="font-weight: bold;"></span><span style="font-weight: bold;">New
functionality:</span><br>
<br>
<ul>
<li>We are rolling out a new feature that allows you to view a
graphical representation of your gene results mapped to genomic
sequences.&nbsp; To use this feature, run any gene search then click on
the "Genome View" tab.&nbsp; This view allows you to interactively
visualize the genomic distribution of your results -- you can zoom
in/out, sort results or visit specific gene pages.&nbsp; This is being
released as a beta feature which we hope to improve in the coming
months based on your valuable feedback.</li>
</ul>
<br>
<ul>
<li>With many of the expression graphs on gene pages you now have the
option to view the data on a log2 scale, as percentile or as raw
expression values.<br>
</li>
</ul>
&nbsp;<br>
<span style="font-style: italic;"><span style="font-style: italic;"><span
style="text-decoration: underline;">Note:</span>
Our bimonthly database releases incorporate new data and correct old
data when necessary.&nbsp; Changes in annotation and new experimental
data
may slightly alter your search results by increasing or decreasing the
number of hits. When search parameters change with a new release, we
invalidate (<span style="color: red;">Ã˜</span>) the search and ask you
to rerun it. When IDs are updated or removed, we map the old IDs to the
new ones, remove the old IDs from your Basket, and leave your Favorites
page alone.</span></span><br>
<span style="font-weight: bold;"><span style="font-style: italic;"></span></span><br>
<br>
<span style="font-weight: bold;"><span style="font-style: italic;"></span></span><br>
<span style="font-weight: bold;"><span style="font-style: italic;">EuPathDB
is now on <a
href="https://www.facebook.com/pages/EuPathDB/133123003429972">FaceBook</a></span></span><span
style="font-weight: bold; font-style: italic;"> and <a
href="http://twitter.com/eupathdb">Twitter</a></span></div>
</body>
</html>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem36"/>
    <a name="plasmo12_11_news"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 8.2 Released <i class="date">11 January 2012</i>
                
            </h1>
            <body>
<div style="text-align: justify; font-family: Arial;">
<span style="font-weight: bold;">We are pleased to
announce the release of PlasmoDB 8.2</span><br>
<span style="font-weight: bold;">
</span><br>
<span style="font-weight: bold;">
</span><br>
<span style="font-weight: bold;">
New data in release 8.1 and 8.2:</span><br>
<br>
</div>
<div style="text-align: justify; font-family: Arial;">
<span style="font-style: italic;"></span></div>
<ul style="text-align: justify; font-family: Arial;">
<li><span style="font-style: italic;">P. vivax</span> Schizont stage
proteome from human blood samples (<a
href="http://www.ncbi.nlm.nih.gov/pubmed/21515433"><span
style="font-weight: bold;">Roobsoong </span><span
style="font-style: italic; font-weight: bold;">et. al.</span></a>).&nbsp;
This data can be searched by choosing "Schizont stage proteome from
human blood sample (Roobsoong et al.)" in the <span
style="font-style: italic;">P. vivax</span> section of the <a
href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=GeneQuestions.GenesByMassSpec"><span
style="font-weight: bold;">Mass Spec. Evidence search page</span></a>.
In
addition, peptide evidence is displayed on gene pages in the <a
style="font-weight: bold;" href="http://plasmodb.org/gene/PVX_083220">Protein
Section</a>
and in the <a
href="/cgi-bin/gbrowse/plasmodb/?start=294208;stop=314207;ref=CM000453;width=800;version=100;flip=0;grid=1;id=6c3b4af1919866034614b141fae61097;l=CuiMassSpecPeptides%1EGene;h_feat=pvx_083220%40yellow"><span
style="font-weight: bold;">genome browser</span></a> by turning on
the track specific for this data.&nbsp; <a style="font-weight: bold;"
href="/plasmo/im.do?s=fdf774c4ffc8e23a">This
search strategy</a> identifies all <span style="font-style: italic;">P.
vivax</span> genes with mass spec
evidence from this study.</li>
</ul>
<div style="text-align: justify; font-family: Arial;">
<br>
</div>
<ul style="text-align: justify; font-family: Arial;">
<li><span style="font-style: italic;">P. falciparum</span> nuclear
and cytosolic fractions from rings, trophozoites, and schizonts
graciously provided prepublication by Till Voss and colleagues.&nbsp;
This
data can be searched by choosing "<span style="font-style: italic;">P.
falciparum</span>
cytoplasmic and nuclear fractions from rings, trophozoites and
schizonts (3D7) (Oehring and Woodcroft et al. - unpublished)" in the <span
style="font-style: italic;">P. falciparum</span> section of the <a
href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=GeneQuestions.GenesByMassSpec"><span
style="font-weight: bold;">Mass spec. evidence search page</span></a>.
In
addition, peptide evidence is displayed on gene pages in the <a
style="font-weight: bold;" href="http://plasmodb.org/gene/PF11_0506">Protein
Section</a>
and in the <a
href="/cgi-bin/gbrowse/plasmodb/?start=1954465;stop=1964464;ref=Pf3D7_11;width=800;version=100;flip=0;grid=1;id=6c3b4af1919866034614b141fae61097;l=VossMassSpecPeptides%1EGene;h_feat=pf11_0506%40yellow"><span
style="font-weight: bold;">genome browser</span></a> by turning on
the track specific for this data.&nbsp; <a style="font-weight: bold;"
href="/plasmo/im.do?s=5c64c3e69b25f955">This
search strategy</a> identifies all <span style="font-style: italic;">P.
falciparum</span> genes with mass spec
evidence from this study.&nbsp;</li>
</ul>
<br style="font-family: Arial;">
<ul style="text-align: justify; font-family: Arial;">
<li><span style="font-style: italic;">P. falciparum</span> blood
stage phosphoproteome (<a
href="http://www.ncbi.nlm.nih.gov/pubmed/22018241"><span
style="font-weight: bold;">Treeck &amp; Sanders <span
style="font-style: italic;">et. al.</span></span></a>).&nbsp; This
data can be searched by choosing "Blood stage phosphoproteome and total
proteome (3D7) (Treeck &amp; Sanders et al.)" in the <span
style="font-style: italic;">P. falciparum</span> section of the <a
href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=GeneQuestions.GenesByMassSpec"><span
style="font-weight: bold;">Mass spec. evidence search page</span></a>.
In
addition, peptide evidence is displayed on gene pages in the <a
style="font-weight: bold;" href="http://plasmodb.org/gene/MAL8P1.82">Protein
Section</a>
and in the <a
href="/cgi-bin/gbrowse/plasmodb/?start=737052;stop=742051;ref=Pf3D7_08;width=800;version=100;flip=0;grid=1;id=5cf2afba4e5155b9ebe2784bf8347957;l=BoothroydPhosphoPeptides%1EGene;h_feat=mal8p1.82%40yellow"><span
style="font-weight: bold;">genome browser</span></a> by turning on
the track specific for this data.&nbsp; <a style="font-weight: bold;"
href="/plasmo/im.do?s=1f596b3d78e0780a">This
search strategy</a> identifies all <span style="font-style: italic;">P.
falciparum</span> genes with phospho mass spec
evidence from this study. </li>
</ul>
<div style="text-align: justify; font-family: Arial;">
<br>
</div>
<ul style="text-align: justify; font-family: Arial;">
<li><span style="font-style: italic;">P. falciparum</span>
ChIP-sequence data from the histone variant H2Bv (three time points)
and&nbsp; CenH3 (schizonts) represented as <a
href="/cgi-bin/gbrowse/plasmodb/?start=1975000;stop=2014999;ref=Pf3D7_11;width=800;version=100;flip=0;grid=1;id=6c3b4af1919866034614b141fae61097;l=ChipSeq_H2Bv_Stunnenberg%1EChipSeq_CenH3_40h_Stunnenberg%1EGene;h_feat=pf11_0506%40yellow"><span
style="font-weight: bold;">coverage plots in the genome browser</span></a>.&nbsp;
This data was graciously provided prepublication by the Stunnenberg
group.</li>
</ul>
<br style="font-family: Arial;">
<ul style="text-align: justify; font-family: Arial;">
<li>Genome-wide expression level polymorphisms (ELPs) from a genetic
cross between phenotypically distinct parasite clones of <span
style="font-style: italic;">P. falciparum</span> (HB3 and Dd2) (<a
href="http://www.ncbi.nlm.nih.gov/pubmed/18828674"><span
style="font-weight: bold;">Gonzales </span><span
style="font-style: italic; font-weight: bold;">et. al.</span></a>).
This data may be searched and visualized in many ways:</li>
</ul>
<br style="font-family: Arial;">
<ul style="font-family: Arial;">
<ul>
<li>Genes may be identified based on their association to genomic
segments, expression profile similarity or similarity of genetic
association.&nbsp; These searches are based on the microarray
expression profiles generated in each of the progeny and can be
accessed in the <a
href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=InternalQuestions.GenesByMicroarrayEvidence"><span
style="font-weight: bold;">microarray search page</span></a> under the
heading "eQTL studies on Hb3 X Dd2 progeny (Ferdig)".&nbsp;</li>
</ul>
</ul>
<span style="font-family: Arial;">&nbsp;</span>
<ul style="font-family: Arial;">
<ul>
<li>Genomic segments can be identified based on their association
to genes.&nbsp; This search is available by clicking on "<a
href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=SpanQuestions.DynSpansByEQTLtoGenes"><span
style="font-weight: bold;">P.f. eQTL HB3-Dd2 cross (segments by
association to genes)</span></a>" under the heading "Genomic Segments"
in the "Identify other data types" section of the PlasmoDB home page. <br>
</li>
</ul>
</ul>
<br style="font-family: Arial;">
<ul style="font-family: Arial;">
<ul>
<li>Regions/spans that are associated by eQTL are displayed in a
table on gene pages (<a href="http://plasmodb.org/gene/PF13_0019"><span
style="font-weight: bold;">example gene page</span></a>). The table
"Regions/Spans associated by eQTL experiment on HB3 x DD2 progeny"
includes a column for haplotype blocks associated the gene, the
coordinates of the haplotype block, LOD scores, and links to searches
for genes contained <a style="font-weight: bold;"
href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=GeneQuestions.GenesByLocation&amp;value%28sequenceId%29=Pf3D7_05&amp;value%28organism%29=Plasmodium+falciparum&amp;value%28end_point%29=225880&amp;value%28start_point%29=86612&amp;weight=10">within
the region</a> or <a
href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=GeneQuestions.GenesByEQTL_Segments&amp;value%28lod_score%29=1.5&amp;value%28end_point_segment%29=225880&amp;value%28sequence_id%29=Pf3D7_05&amp;value%28liberal_conservative%29=Liberal+Locations&amp;value%28start_point%29=86612&amp;weight=10"><span
style="font-weight: bold;">associated with the region</span></a>.</li>
</ul>
</ul>
<br style="font-family: Arial;">
<ul style="font-family: Arial;">
<ul>
<li>Both microsatellites and haplotype blocks are available <a
href="/cgi-bin/gbrowse/plasmodb/?start=155076;stop=190991;ref=Pf3D7_13;width=800;version=100;flip=0;grid=1;id=5cf2afba4e5155b9ebe2784bf8347957;l=Microsatellite%1EHaploBlock%1EGene;h_feat=mal8p1.82%40yellow"><span
style="font-weight: bold;">as tracks in the genome browser</span></a>.&nbsp;
<br>
</li>
</ul>
</ul>
<div style="text-align: justify; font-family: Arial;">
<br>
</div>
<ul style="text-align: justify; font-family: Arial;">
<li>Improved linking to the <a href="http://pberghei.eu/"><span
style="font-weight: bold;">Rodent Malaria Genetically Modified (RMgm)</span></a>
parasites database.&nbsp; A dedicated "Phenotype" table is now
available on gene pages with information about the genetic modification
and linkouts to PubMed and the RMgm database.&nbsp; As an example, you
can scroll down and expand the table under the heading "Phenotype" on
the&nbsp; gene page for a <a
href="http://plasmodb.org/gene/PBANKA_091030"><span
style="font-weight: bold;">putative </span><span
style="font-style: italic; font-weight: bold;">P. berghei</span><span
style="font-weight: bold;"> guanylyl cyclase</span></a>.<br>
</li>
</ul>
<div style="text-align: justify; font-family: Arial;">
&nbsp;&nbsp; <br>
</div>
<ul style="text-align: justify; font-family: Arial;">
<li>Updated linkouts to the <a style="font-weight: bold;"
href="http://sites.huji.ac.il/malaria/">Malaria Parasite Metabolic
Pathways
(MPMP) database</a>.&nbsp; Numerous pathways have been added and
updated by
Dr. Hagai Ginsburg.&nbsp;&nbsp; Genes included in the MPMP database can
be searched based on
metabolic pathway in PlasmoDB <a style="font-weight: bold;"
href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=GeneQuestions.GenesByMetabolicPathway">here</a>.</li>
</ul>
<br style="font-family: Arial;">
<ul style="text-align: justify; font-family: Arial;">
<li><span style="font-weight: bold; text-decoration: underline;">New
Gene IDs</span> -- Please read the <a style="font-weight: bold;"
href="/plasmo/showXmlDataContent.do?name=XmlQuestions.News#genedb_newPFgenome">news
item from GeneDB</a> regarding assignment of new <span
style="font-style: italic;">P. falciparum </span>gene IDs. We will be
updating to these new IDs in the next release. Rest assured that both
PlasmoDB and GeneDB will proper mapping of old IDs to the new ones (in
other words, you will always be able to search with the old or new
IDs). Currently in PlasmoDB we are displaying the new Gene ID next to
the old one at the top of gene pages (<a style="font-weight: bold;"
href="http://plasmodb.org/gene/PF11_0344">old ID/new ID</a>). &nbsp;
&nbsp;&nbsp;</li>
</ul>
<br style="font-family: Arial;">
<div style="text-align: justify; font-family: Arial;">
</div>
<div style="text-align: justify; font-family: Arial;">
<span style="font-weight: bold;"></span><br style="font-weight: bold;">
<span style="font-weight: bold;">New functionality:</span><br>
<br>
<span style="font-weight: bold;">
</span>
<ul>
<li>The genome browser has been updated.&nbsp; This newer version
includes several improvements and new functionality.&nbsp; <a
href="http://www.youtube.com/watch?v=jxA6VMN97Y8"><span
style="font-weight: bold;">A tutorial </span></a>highlighting some of
the new features is also available.&nbsp;&nbsp;</li>
</ul>
<br>
<ul>
<li>A new column analysis tool is now available that allows you to
generate a word cloud or histogram based on the data in result
columns.&nbsp; To access this feature, run any search that returns a
list of results then click on the icon next to the product description
column name.&nbsp; This will reveal a popup with the option to select
"Word Cloud".&nbsp; Look for this analysis feature to be activated on
additional columns in future releases.</li>
</ul>
<br>
<ul>
<li>Several new tutorials have been generated:</li>
</ul>
<br>
<ul>
<ul style="font-weight: bold;">
<li><a href="http://www.youtube.com/watch?v=jxA6VMN97Y8">The genome
browser.</a></li>
</ul>
<ul>
<li style="font-weight: bold;"><a href="http://youtu.be/0IP1d9raiMs">Customizing
restriction sites in the genome browser.</a></li>
<li style="font-weight: bold;"><a href="http://youtu.be/_It1Nghu-NI">Identifying
isolates based on geographic location.</a></li>
<li style="font-weight: bold;"><a href="http://youtu.be/8pP1rhSrjqo">Dealing
with invalid IDs and search strategies.</a></li>
<li style="font-weight: bold;"><a href="http://youtu.be/P7ptqGnECJ4">Using
the orthology phylogenetic profile search.</a></li>
</ul>
</ul>
<br>
<span style="font-style: italic;"><span style="font-style: italic;"><span
style="text-decoration: underline;">Note:</span>
Our bimonthly database releases incorporate new data and correct old
data when necessary.&nbsp; Changes in annotation and new experimental
data
may slightly alter your search results by increasing or decreasing the
number of hits. When search parameters change with a new release, we
invalidate (<span style="color: red;">Ã˜</span>) the search and ask you
to rerun it. When IDs are updated or removed, we map the old IDs to the
new ones, remove the old IDs from your Basket, and leave your Favorites
page alone.</span></span><br>
<span style="font-weight: bold;"><span style="font-style: italic;"></span></span><br>
<br>
<span style="font-weight: bold;"><span style="font-style: italic;"></span></span><br>
<span style="font-weight: bold;"><span style="font-style: italic;">EuPathDB
is now on <a
href="https://www.facebook.com/pages/EuPathDB/133123003429972">FaceBook</a></span></span><span
style="font-weight: bold; font-style: italic;"> and <a
href="http://twitter.com/eupathdb">Twitter</a></span></div>
</body>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem37"/>
    <a name="genedb_newPY_YMgenome"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                Version 1 of P. yoelii yoelii YM is now available on GeneDB! <i class="date">4 January 2012</i>
                
            </h1>
            <div align="justify">
A first-pass assembly and annotation of <i>P. yoelii yoelii</i> YM is now publicly available on GeneDB. The assembly was made 
using short-fragment Illumina reads and scaffolded with 454 paired-reads from a 3kb library. The assembly was 
computationally improved using PAGIT (http://www.sanger.ac.uk/resources/software/pagit/) and annotated by transfer 
from the <i>P. berghei</i> ANKA reference. Genes in subtelomeric regions were predicted <i>ab initio</i>. The annotation has been 
manually re-inspected genome-wide.<br><br>
<i>P. yoelii yoelii</i> YM can be viewed and searched on GeneDB.  EMBL and GFF format files can be downloaded from the 
following FTP site:<br><br>
<b>ftp://ftp.sanger.ac.uk/pub/pathogens/Plasmodium/yoeliiYM/version_1/January_2012</b><br><br>
A publication describing the new sequence is in preparation and permission should therefore be obtained before undertaking 
genome-scale or whole chromosome wide analyses.<br><br>
This work was funded by the Wellcome Trust Sanger Institute and the European Union's 7th Framework Programme (FP7/2007-2013) 
under grant agreement no 242095. The DNA was provided by Tony Holder (Division of Parasitology, MRC National Institute for Medical 
Research, Mill Hill, London). 
</div>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem38"/>
    <a name="genedb_newPFITgenome"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                Verson 2 of P. falciparum IT is now available on GeneDB! <i class="date">17 November 2011</i>
                
            </h1>
            <div align="justify">
A new assembly of the P. falciparum IT genome is now available,
replacing the version 1 sequence that has been available via PlasmoDB
(since release 8.0) and the Sanger FTP site. The version 1 sequence was
produced by "correcting" (or morphing) the reference 3D7 sequence,
using aligned Illumina reads from IT.&nbsp;&nbsp; However, in version
2, the sequence data have been assembled de novo. A key aim for this
genome project is to fully assemble the subtelomeric regions. Although
we were able to correctly assemble some subtelomeric sequences onto the
?core? of the genome (i.e. the interstitial or housekeeping region),
many subtelomeric sequences remain problematic and are being actively
refined. The annotation is automated and no manual curation has taken
place.<br>
<br>
The new version contains new systematic identifiers compliant with
those now in use for 3D7.&nbsp; However, old identifiers remain in the
database as fully searchable synonyms.&nbsp; The new version can be
viewed and searched on GeneDB and will be loaded into PlasmoDB in early
2012.&nbsp; EMBL and GFF format files can be downloaded from the
following FTP site:<br>
<br>
<span style="font-weight: bold;">ftp://ftp.sanger.ac.uk/pub/pathogens/Plasmodium/falciparum/IT_strain/version_2</span><br>
<br>
This work was funded by the Wellcome Trust [grant number
WT085775/Z/08/Z] and the European Union's 7th Framework Programme
(FP7/2007-2013) under grant agreement no 242095.<br>
<br>
</div>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem39"/>
    <a name="genedb_newPFgenome"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                A new sequence version of P. falciparum 3D7 is now available on GeneDB! <i class="date">24 September 2011</i>
                
            </h1>
            <div align="justify">
<br>Since the publication of the genome in 2002, work has continued at the Sanger Institute to produce a single base-perfect <i>P. falciparum</i> 
reference genome.  This new sequence version is an upgrade from version 2.1.4 to version 3 and includes some substantial changes. Barring some highly 
AT-rich tracts that remain to be fully verified, we expect this to be the last major sequence update. 
<br><br>Differences include the correction of major mis-assemblies on chromosome 7 and 8 &#8211; one end of chromosome 7 now replaces one end of 
chromosome 8 (and vice versa), the replacement of all ambiguous bases (N&#8217;s) in the genome with the correct sequence and the correction of 
hundreds of sequencing errors. The original <i>P. falciparum</i> 3D7 genome project was started by three different genome centers &#8211; Sanger, 
TIGR and Stanford &#8211; all of whom used different naming schemes (e.g MAL7P1.10, PF11_0344, PFE0010c).  For the submission in 2002 all three naming 
schemes were used.  In October 2007, the systematic re-annotation of <i>P. falciparum</i> 3D7 started in earnest with a community annotation workshop 
and work has continued at Sanger ever since. During that time, new genes have been created and other genes have been split.  When a new ID needed 
to be assigned but a logical numerical identifier was not available, a suffix of &#8216;a&#8217; and &#8216;b&#8217; was added to distinguish from the 
neighbouring gene.  In addition, now that part of chromosome 7 is located on chr8, and vice versa, it would be more confusing to have MAL7 identifiers 
on chr8 and MAL8 identifiers on chr7.  <br><br>
We have therefore decided to adopt a consistent new scheme to identify genes.  The old identifiers will be 
maintained as fully searchable synonyms but will be made significantly less prominent in PlasmoDB and GeneDB.  An important additional change is that 
both protein-coding and non-protein coding genes will now use the same consistent nomenclature. The new naming scheme complies with the standards 
set by the International Nucleotide Sequence Database Collaboration (Genbank, EMBL & DDBJ).  All new identifiers now start with a two-part prefix: 
PF3D7_ to identify the genome and two numerical digits to identify the chromosome, e.g. the first gene on chr1 is now called PF3D7_0100100.  The new 
naming scheme increments intervals of 100 to allow significant numbers of genes to be added (although we believe that this is unlikely).  This new 
naming scheme will also be adopted to other genomes that are being worked on at the Sanger Institute, e.g. <i>P. falciparum</i> IT genes will start with 
PFIT_.  The new sequence version can be viewed and searched on GeneDB and will be loaded into PlasmoDB in early 2012.  EMBL and GFF format files, 
together with a conversion table of old:new identifiers, can be downloaded from the following FTP site:<br><br>
ftp://ftp.sanger.ac.uk/pub/pathogens/Plasmodium/falciparum/3D7/3D7.latest_version/version3/October_2011/
</div>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem40"/>
    <a name="plasmo7_11_news"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 8.0 Released <i class="date">21 July 2011</i>
                
            </h1>
            <body>
<div style="text-align: justify;">
<span style="font-weight: bold;">We are pleased to
announce the release of PlasmoDB 8.0</span><br>
<span style="font-weight: bold;">
</span><br>
<span style="font-weight: bold;">
</span><br>
<span style="font-weight: bold;">
New data:</span><br>
<br>
</div>
<span style="font-style: italic;">
</span>
<ul>
<li>Sequence and annotation of the<span style="font-style: italic;">
P. falciparum </span>IT strain<span style="font-style: italic;"> </span>produced
by the Parasite Genomics Group at the Wellcome Trust Sanger Institute
with funding from the EVIMalaR Consortium (a European Commission Funded
Network of Excellence).<span style="font-style: italic;">&nbsp; </span>While
this data has been graciously provided prepublication to the community
its usage for large scale genomic analysis is bound by a <a
style="font-weight: bold;"
href="http://plasmodb.org/common/downloads/release-8.0/PfalciparumIT/fasta/">data
usage
policy</a>.<span style="font-style: italic;"></span></li>
</ul>
<br>
<ul>
<li>Updated linkouts to the <a style="font-weight: bold;"
href="http://sites.huji.ac.il/malaria/">Malaria Parasite Metabolic
Pathways
(MPMP) database</a>.&nbsp; Numerous pathways have been added and
updated by
Dr. Hagai Ginsburg.&nbsp;&nbsp; A full description of new pathways is
available <a style="font-weight: bold;"
href="http://plasmodb.org/plasmo/communityDownload.do?fname=New%20maps%204.7.2011.pdf">here</a>
and genes included in the MPMP database can be searched based on
metabolic pathway in PlasmoDB <a style="font-weight: bold;"
href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=GeneQuestions.GenesByMetabolicPathway">here</a>.</li>
</ul>
<div style="text-align: justify;">
<span style="font-weight: bold;"></span><br style="font-weight: bold;">
<span style="font-weight: bold;">New functionality:</span><br>
<br>
<span style="font-weight: bold;">
</span></div>
<ul style="text-align: justify;">
<li>You can now view intron/exon boundaries based on available RNA
sequence data.&nbsp; boundary determination was performed using the <a
style="font-weight: bold;"
href="http://www.ncbi.nlm.nih.gov/pubmed/21775302">RNA-Seq Unified
Mapper (RUM)</a>. To view intron/exon boundary calls, turn on the
"RNA-seq alignments RUM Intron" track in the genome browser.&nbsp; <a
style="font-weight: bold;"
href="http://plasmodb.org/cgi-bin/gbrowse/plasmodb/?start=1430000;stop=1439999;ref=Pf3D7_11;width=800;version=100;cache=0;show_tooltips=on;grid=on;id=5f6c1beeb1b94b24ea6ed5148ba1d93c;label=Gene-RUMIntronDuffy-RUMIntronNewbold-RUMIntronStunnenberg">Here
is a link to a gene</a>
that is predicted to have multiple splice variants based on the
annotation provided by GeneDB with the RUM intron tracks turned
on.&nbsp; To get additional information mouse over the tracks and/or
the track names.</li>
</ul>
<br><br>
<span style="font-weight: bold;"><span style="font-style: italic;">EuPathDB
is now on <a
href="https://www.facebook.com/pages/EuPathDB/133123003429972">FaceBook</a></span></span><span
style="font-weight: bold; font-style: italic;"> and <a
href="http://twitter.com/eupathdb">Twitter</a></span><br>
</body>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem41"/>
    <a name="plasmo5_11_news"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 7.2 Released <i class="date">3 May 2011</i>
                
            </h1>
            <body>
<div style="text-align: justify;">
<span style="font-weight: bold;">We are pleased to
announce the release of PlasmoDB 7.2</span><br>
<span style="font-weight: bold;">
</span><br>
<span style="font-weight: bold;">
</span><br>
<span style="font-weight: bold;">
New data:</span><br>
<br>
</div>
<ul>
<li><span style="font-style: italic;">P. falciparum</span> mRNA
half-life data intraerythrocytic developmental cycle (<a
style="font-weight: bold;"
href="http://www.ncbi.nlm.nih.gov/pubmed/17612404">Shock <span
style="font-style: italic;">et. al.</span></a>)
is now available as mRNA expression and half-life graphs on gene
pages.&nbsp; To view this data visit any gene page and scroll down to
the section called "mRNA half-lives during intraerythrocytic
development - 3D7 post treatment with actD" in the Expression section
of any gene page.</li>
</ul>
<br>
<ul>
<li><span style="font-style: italic;">P. falciparum</span> <a
style="font-weight: bold;"
href="http://plasmodb.org/cgi-bin/gbrowse/plasmodb/?start=1016930;stop=1516929;ref=Pf3D7_10;width=800;version=100;cache=on;show_tooltips=on;grid=on;id=3773730293eb07c2332b85e249bf2271;label=Gene-JeffaresSNPs-WTSI_pJazz_clones;h_feat=pf10_0308@yellow">genetic
diversity estimate graph is provided in the genome browser</a> based on
<a style="font-weight: bold;"
href="http://www.ncbi.nlm.nih.gov/pubmed/20838588">Nygaard <span
style="font-style: italic;">et. al.</span></a></li>
</ul>
<span style="font-style: italic;"><br>
</span>
<ul>
<li>Updated linkouts to the <a style="font-weight: bold;"
href="http://sites.huji.ac.il/malaria/">Malaria Parasite Metabolic
Pathways
(MPMP) database</a>.&nbsp; Numerous pathways have been added and
updated by
Dr. Hagai Ginsburg.&nbsp;&nbsp; A full description of new pathways is
available <a style="font-weight: bold;"
href="http://plasmodb.org/plasmo/communityDownload.do?fname=New%20maps%20on%20Malaria%20Parasite%20Metabolic%20pathway1.pdf">here</a>
and genes included in the MPMP database can be searched based on
metabolic pathway in PlasmoDB <a style="font-weight: bold;"
href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=GeneQuestions.GenesByMetabolicPathway">here</a>.</li>
</ul>
<br>
<div style="text-align: justify;">
<span style="font-weight: bold;"></span><br style="font-weight: bold;">
<span style="font-weight: bold;">New functionality:</span><br>
<br>
<span style="font-weight: bold;">
</span></div>
<ul style="text-align: justify;">
<li>You can now search for DNA segments defined by their chromosomal
location or their nucleotide sequence (DNA motif pattern).&nbsp; This
new search is available under "Identify Other Data Types"; click on
"Genomic Segments (DNA motif)" then select either <a
style="font-weight: bold;"
href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=SpanQuestions.DynSpansByMotifSearch">DNA
motif pattern</a> or <a style="font-weight: bold;"
href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=SpanQuestions.DynSpansBySourceId">Genomic
Location</a>.</li>
</ul>
<div style="text-align: justify;"><br>
</div>
<ul style="text-align: justify;">
<li>We introduced a new combine operation that allows users to
combine
steps based on relative locations on the genome.&nbsp; This enables
you, for
example, to return all genes that contain a specific DNA motif within
1000 nucleotides upstream of the annotated start site. Or you could
return all genes that have a SNP between two strains within 500
nucleotides of the start of the gene (or return all SNPs that are
located in this region relative to genes).&nbsp;&nbsp;</li>
</ul>
<br>
<ul style="text-align: justify;">
<ul>
<li>Here is an <a style="font-weight: bold;"
href="http://plasmodb.org/plasmo/im.do?s=0ffa670cc2b0a579">example
search strategy</a> that identifies protein coding genes with an AP-2
like motif within 1000 bp of their 5' end.</li>
</ul>
</ul>
<br>
<ul style="text-align: justify;">
<ul>
<li>Tutorials highlighting the <a style="font-weight: bold;"
href="http://eupathdb.org/tutorials/DNA_motif/dna_motif_viewlet_swf.html">DNA
motif</a> search and the <a style="font-weight: bold;"
href="http://eupathdb.org/tutorials/colocate/colocate_viewlet_swf.html">locational
relationship functionality</a> are available.</li>
</ul>
</ul>
<br>
<div style="text-align: justify;">
In addition, several new tutorials have been developed:<br>
<br>
</div>
<ul style="text-align: justify;">
<li><a style="font-weight: bold;"
href="http://eupathdb.org/tutorials/Registration/register_final_viewlet_swf.html">How
and why should you register for an account</a><br>
</li>
<li style="font-weight: bold;"><a
href="http://eupathdb.org/tutorials/TextSearch/Text_Search_viewlet_swf.html">How
to run a text search</a></li>
<li><a style="font-weight: bold;"
href="http://eupathdb.org/tutorials/Favorites/favorites_viewlet_swf.html">How
to use Favorites</a></li>
<li style="font-weight: bold;"><a
href="http://eupathdb.org/tutorials/DNA_motif/dna_motif_viewlet_swf.html">How
to find a DNA motif</a></li>
<li><a style="font-weight: bold;"
href="http://eupathdb.org/tutorials/colocate/colocate_viewlet_swf.html">How
to use genomic co-location</a></li>
</ul>
<small><a href="http://twitter.com/eupathdb"><span
style="font-weight: bold; font-style: italic;">Follow EuPathDB on
Twitter</span></a></small>
</body>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem42"/>
    <a name="plasmo3_11_news"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                Plasmodium genome sequencing project update <i class="date">25 March 2011</i>
                
            </h1>
            <body>
<div style="text-align: justify;">Following extensive discussion with
the malaria research community, the list of Plasmodium genome
sequencing projects supported by the US National Institutes of Allergy
and Infectious Diseases (NIAID) has been updated, replacing entries
that have been difficult to source with additional isolates and species
of interest.&nbsp; <br>
<br>
The initial white paper is available <a style="font-weight: bold;"
href="http://plasmodb.org/plasmo/communityDownload.do?fname=Plasmodium_Sequencing_White_Paper.pdf">here</a>;
click <a style="font-weight: bold;"
href="http://plasmodb.org/plasmo/communityDownload.do?fname=Table2_021208_SampleStatus_032111%20SampleStatus.pdf">here</a>
for the revised list of approved projects. &nbsp;The Plasmodium
sequencing contract has been awarded to the <a
style="font-weight: bold;"
href="http://www.broadinstitute.org/science/projects/gscid/genomic-sequencing-center-infectious-diseases">Broad
Institute</a>, and data is made publicly available to the broader
research community via deposition in GenBank and through relevant <a
style="font-weight: bold;" href="http://rnaseq.pathogenportal.net/">Bioinformatics
Resource Centers</a>, including <a style="font-weight: bold;"
href="http://PlasmoDB.org">PlasmoDB</a>. &nbsp;Further information on
the 'white paper' process by which key pathogen species are approved
for sequencing is available <a style="font-weight: bold;"
href="http://www.niaid.nih.gov/LabsAndResources/resources/gsc/Pages/app.aspx">here</a>.<br>
</div>
</body>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem43"/>
    <a name="plasmo11_10_news"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 7.1 Released <i class="date">18 November 2010</i>
                
            </h1>
            <body>
<span style="font-weight: bold;">We are pleased to
announce the release of PlasmoDB 7.1<br>
<br>
<br>
This release of PlasmoDB contains the following new data:</span><span
style="font-weight: bold;">&nbsp;<br>
</span><br>
<ul>
<li><span style="font-style: italic;">P. falciparum </span>blood
stage proteomics data (trophozoites and early and mature gametocytes)
from <a href="http://www.ncbi.nlm.nih.gov/pubmed/20332084"><span
style="font-style: italic;">Silvestrini et. al.</span></a>&nbsp; This
data can be searched from the Mass spec. evidence search page
under the heading "Protein Expression" -- listed under <span
style="font-style: italic;">Plasmodium falciparum </span>and
selecting "Blood Stage Proteome - Trophozoites, Early Gametocytes (gct
I/II) and/or Mature Gametocytes (gct V)".&nbsp; In
addition, peptide evidence is displayed on gene pages in the <a
style="font-weight: bold;" href="http://plasmodb.org/gene/PFL2405c">Protein
Section</a>
and in the <a
href="http://plasmodb.org/cgi-bin/gbrowse/plasmodb/?start=1080000;stop=1270000;ref=Pf3D7_12;width=800;version=100;cache=on;show_tooltips=on;grid=on;id=6436e0c6f1efb6e68613f9388b25e99f;label=Gene-LasonderMassSpecPeptidesBloodStage;h_feat=pfl2415w%40yellow"><span
style="font-weight: bold;">genome browser</span></a> by turning on
the track specific for this data.&nbsp; An example search strategy that
identifies genes that have mass spec evidence in the early gametocytes
but not in trophozoites or mature gametocytes can be accessed and
modified <a href="http://plasmodb.org/plasmo/im.do?s=55d0c2d79d37e20d">here</a>.<br>
</li>
</ul>
<br>
<span style="font-weight: bold;">New Feature in PlasmoDB</span><span
style="font-weight: bold;">:</span> <br>
<br>
<ul>
<li>You can now add publications to your user comments using their <a
href="http://www.doi.org/index.html">Digital Object Identifier</a>
(DOI) numbers.&nbsp; To view a list of all genes that have a user
comment in PlasmoDB, click <a
href="http://plasmodb.org/plasmo/showSummary.do?questionFullName=GeneQuestions.GenesByTextSearch&amp;array%28text_search_organism%29=Plasmodium%20berghei,Plasmodium%20chabaudi,Plasmodium%20falciparum,Plasmodium%20knowlesi,Plasmodium%20vivax,Plasmodium%20yoelii&amp;array%28text_fields%29=User%20comments&amp;array%28whole_words%29=no&amp;value%28max_pvalue%29=-30&amp;value%28text_expression%29=*&amp;value%28timestamp%29=Nov%2017,%202010%209:39:32%20PM">here</a>.</li>
</ul>
<ul>
<li>You can view microarray oligo mapping for all arrays with data in
PlasmoDB.&nbsp; Oligo mapping can be viewed in the <a
href="http://plasmodb.org/cgi-bin/gbrowse/plasmodb/?start=1243200;stop=1248199;ref=Pf3D7_11;width=800;version=100;cache=on;show_tooltips=on;grid=on;id=f1cec45354b677a542dade3988db24e1;label=Gene-MicroArrayProbesPfSanger-MicroArrayProbesPfDerisi">Genome
Browser</a> by turning on tracks under the heading "Expression
Microarray Probes".<br>
</li>
</ul>
<br>
<span style="font-style: italic;">Note: We have corrected errors
inadvertently introduced into the last build of
PlasmoDB (7.0, released on Sept. 21, 2010) that resulted in inaccurate
representation of P. yoelii
liver stage expression profiling data.</span>&nbsp; <span
style="font-style: italic;">Please do not hesitate to <a
href="mailto:help@plasmodb.org">contact us</a> if you have any
questions.</span><br>
<br>
<span style="font-weight: bold;"></span><span style="font-weight: bold;"></span>
</body>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem44"/>
    <a name="dbp_award_10202010"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                Driving Biological Projects awarded <i class="date">20 October 2010</i>
                
            </h1>
            <b>We are pleased to report that the following Driving Biological Projects (DBPs) have been awarded in connection with the EuPathDB Bioinformatics Resource Center (BRC):</b><br><br>
<ul>
<li><b>The Toxoplasma oocyst transcriptome and proteome</b><br>
   John Boothroyd, Matt Bogyo (Stanford) & Pat Conrad (UC Davis)<br>
   This project is expected to yield expression data from an important, but inaccessible, life cycle stage (1 yr). <br><br></li>
<li><b>A comprehensive catalog of the T. gondii & N. caninum parasite & infected host cell transcriptome & proteome</b><br>
   Brian Gregory (Univ. Pennsylvania) & Jonathan Wastling (Univ. Liverpool)<br>
   This project will produce an immense volume of sequence data, driving database improvements for handling 'next-gen' sequencing, non-coding RNAs, and interrogating host & pathogen datasets in parallel (2 yr).<br><br></li>

<li><b>Dissection of Plasmodium falciparum genetic crosses by mass spectrometry-based metabolite QTL</b><br>
   Manuel Llin&aacute;s (Princeton) & Michael Ferdig (Notre Dame)<br>
   This project is expected to improve handling of genetic mapping data, incorporation of metabolomics results (integrated with pathway maps), and linking of specific genetic loci with metabolite abundance (2 yr).<br><br></li>
</ul>

A biennial competition for DBP awards is mandated by the BRC contract from NIH/NIAID, and supported by a total of $1.2M in total costs per award cycle.  The goal of this program is to produce high value genomic-scale datasets for the EuPathDB user community, and drive the further development of bioinformatics resources for data analysis and mining.<br><br>

Letters of intent (LOI) were solicited in February (see Feb 4 News item), and reviewed by the EuPathDB Scientific Working Group (SWG; see News, March 31).  12 full applications were solicited from among the ~50 LOIs received, and awards were based on further scientific review by the SWG, and in consultation with NIAID program staff.  Cost negotiations and 'in kind' contributions from other sources made it possible to support 3 applications, from 6 investigators (in some cases, more than one LOI was merged into a single award).<br><br>

We received far more high quality proposals than could be supported, testifying to the need for such mechanisms for data production bioinformatics analysis.  The EuPathDB team looks forward to working with members of our user community who are seeking funding for such projects from external sources.  Inquiries, including requests for support letters, should be directed to the EuPathDB staff (click on the 'Contact Us' link).<br><br>

<b>A new solicitation for DBPs is anticipated in 2011, and will be announced on the EuPathDB web site.</b>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem45"/>
    <a name="plasmo09_10_news"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 7.0 Released <i class="date">21 September 2010</i>
                
            </h1>
            <body>
<span style="font-weight: bold;">We are pleased to
announce the release of PlasmoDB 7.0<br>
<br>
<br>
This release of PlasmoDB contains the following new datasets and
features:</span><span style="font-weight: bold;">&nbsp;<br>
<br>
New Data:</span><br>
<ul>
<li><span style="font-style: italic;"></span>Updated sequence and
annotation for <span style="font-style: italic;">Plasmodium</span> <span
style="font-style: italic;">falciparum</span>, <span
style="font-style: italic;">P. chabaudi</span>, <span
style="font-style: italic;">P. knowlesi </span>and <span
style="font-style: italic;">P. berghei </span>was obtained from the
Pathogen Sequencing Unit at the Wellcome Trust Sanger Institute.&nbsp;
Additional details may be viewed <a
href="http://plasmodb.org/plasmo/getDataset.do?display=detail&amp;datasets=P.berghei_wholeGenomeShotgunSequence,P.chabaudi_chromosomes,P.falciparum_chromosomes,P.gallinaceum.contigs,P.knowlesi_chromosomes,P.reichenowi.contigs,P.vivax_contigsGB,PvivaxChromosomes,P.vivax_mitochondrial,P.yoelii_Annotation&amp;title=Query#P.berghei_wholeGenomeShotgunSequence">here</a>.</li>
</ul>
<ul>
<li>Updated linkouts to the Malaria Parasite Metabolic Pathways
(MPMP) database.&nbsp; Numerous pathways have been added and updated by
Dr. Hagai Ginsburg.&nbsp;&nbsp; A full description of new pathways is
available <a
href="http://plasmodb.org/plasmo/communityDownload.do?fname=New_maps_on_Malaria_Parasite_Metabolic_pathways.pdf">here</a>
and genes included in the MPMP database can be searched based on
metabolic pathway in PlasmoDB <a
href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=GeneQuestions.GenesByMetabolicPathway">here</a>.<br>
</li>
</ul>
<ul>
<li><span style="font-style: italic;">Plasmodium falciparum, P.
vivax, P. yoelii, P. berghei </span>and<span
style="font-style: italic;"> P. chabaudi</span> predicted ApiAP2
transcription-factor (TF) binding site data.&nbsp; This data was
graciously provided prepublication by Dr. Manuel Llinas.&nbsp; Gene
with predicted binding sites to each ApiAP2 TF may be identified using
the <a
href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=GeneQuestions.GenesByBindingSiteFeature">TF
binding site search</a> available via the Transcript Expression section
of PlasmoDB.&nbsp; Search parameter such as distance from the 5' end,
number of TF binding sites and probability score can be adjusted as
needed.&nbsp; In addition, TF binding sites can be viewed in the genome
browser with additional information available when mousing over TF
binding site graphics.&nbsp; An example genome browser page may be
viewed <a
href="http://plasmodb.org/cgi-bin/gbrowse/plasmodb/?start=883339;stop=917361;ref=Pf3D7_06;width=800;version=100;flip=;cache=1;drag_and_drop=0;show_tooltips=1;grid=1;id=5e34fc82bb97015347ac950c4404cf72;label=Gene-BindingSites;h_feat=pff1065c@thistle">here</a>.</li>
</ul>
<ul>
<li><span style="font-style: italic;">P. falciparum</span>
RNA-sequence data from the intraerythrocytic life cycle.&nbsp; This
data was graciously provided prepublication to the community via
PlasmoDB by Dr. RichÃ¡rd BÃ¡rtfai, Wieteke A.M. Hoeijmakers and Dr.
Hendrik G. Stunnenberg.&nbsp; Searches against this data may be
accessed from the <a
href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=InternalQuestions.GenesByRNASeqEvidence">RNA-seq
evidence section</a> in the Transcript Expression section
of PlasmoDB.&nbsp; RNA-seq data may also be viewed on individual gene
pages where a graphical representation of the gene's expression profile
throughout the time points of this study is presented.&nbsp;&nbsp; In
addition,&nbsp; RNA-seq coverage graphs can be viewed through the
PlasmoDB <a
href="http://plasmodb.org/cgi-bin/gbrowse/plasmodb/?start=840000;stop=910000;ref=Pf3D7_06;width=800;version=100;cache=on;show_tooltips=on;grid=on;id=5e34fc82bb97015347ac950c4404cf72;label=Gene-RnaSeqStunnenberg5Cont-RnaSeqStunnenberg10Cont-RnaSeqStunnenberg15Cont-RnaSeqStunnenberg20Cont-RnaSeqStunnenberg25Cont-RnaSeqStunnenberg30Cont-RnaSeqStunnenberg35Cont-RnaSeqStunnenberg40Cont;h_feat=pff1065c%40thistle">genome
browser</a>.<br>
</li>
</ul>
<ul>
<li><span style="font-style: italic;">P. falciparum</span>
ChIP-sequence data from the intraerythrocytic life cycle.&nbsp; ChIP
was performed using antibodies to H2A.Z, H3K9ac &amp; H3K4me3. This
data was graciously provided prepublication to the community via
PlasmoDB by Dr. RichÃ¡rd BÃ¡rtfai, Wieteke A.M. Hoeijmakers and Dr.
Hendrik G. Stunnenberg.&nbsp; This data can be viewed through the
PlasmoDB <a
href="http://plasmodb.org/cgi-bin/gbrowse/plasmodb/?start=840000;stop=910000;ref=Pf3D7_06;width=800;version=100;cache=on;show_tooltips=on;grid=on;id=5e34fc82bb97015347ac950c4404cf72;label=Gene-ChipSeqH2A10Stunnenberg-ChipSeqH2A20Stunnenberg-ChipSeqH2A30Stunnenberg-ChipSeqH2A40Stunnenberg-ChipSeqH3K10Stunnenberg-ChipSeqH3K20Stunnenberg-ChipSeqH3K30Stunnenberg-ChipSeqH3K40Stunnenberg-ChipSeqH3K9ac10Stunnenberg-ChipSeqH3K9ac20Stunnenberg-ChipSeqH3K9ac30Stunnenberg-ChipSeqH3K9ac40Stunnenberg;h_feat=pff1065c@thistle">genome
browser</a>.</li>
</ul>
<ul>
<li><span style="font-style: italic;">P. falciparum</span>
RNA-sequence data of lab-adapted field parasites from malaria-infected
pregnant women and children.&nbsp; This data was graciously provided
prepublication to the community via PlasmoDB by Drs. Marissa Vignali
and Patrick E. Duffy.&nbsp; Searches against this data may be accessed
from the <a
href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=InternalQuestions.GenesByRNASeqEvidence">RNA-seq
evidence section</a> in the Transcript
Expression section
of PlasmoDB.&nbsp; RNA-seq data may also be viewed on individual gene
pages
where a graphical representation of the gene's expression profile
throughout the time points of this study is presented.&nbsp;&nbsp; In
addition,&nbsp;
RNA-seq coverage graphs can be viewed through the PlasmoDB <a
href="http://plasmodb.org/cgi-bin/gbrowse/plasmodb/?start=2100000;stop=2270000;ref=Pf3D7_12;width=800;version=100;cache=on;show_tooltips=on;grid=on;id=5e34fc82bb97015347ac950c4404cf72;label=Gene-RnaSeqDuffCont-RnaSeqDuffPL01-RnaSeqDuffPL02;h_feat=pfl0010c@yellow">genome
browser</a>.</li>
</ul>
<span style="font-weight: bold;">New features in this release include:</span><br>
<ul>
<li>Improved organization of the <a
href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=InternalQuestions.GenesByMicroarrayEvidence">microarray
search page</a>.</li>
<li>The search strategy window is now resizable.</li>
</ul>
<br>
<span style="font-weight: bold;"></span>
</body>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem46"/>
    <a name="plasmo07_15_news"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 6.5 released <i class="date">15 July 2010</i>
                
            </h1>
            <span style="font-weight: bold;">We are pleased to
announce the release of PlasmoDB 6.5<br>
<br>
This release of PlasmoDB contains the following new datasets:</span><br>
<br>
<ul>
<li>A proteomics dataset for <i>P. falciparum</i> from 42 and 48 hrs post infection kindly provided prepublication by Paul Bowyer and Matthew Bogyo, Stanford University School of Medicine.</li>
<li><i>P. falciparum</i> SNP calls for 186 culture adapted strains provided by Jianbing Mu and Xinzhuan Su, NIH.<br>  <i>Plasmodium falciparum</i> genome-wide scans for positive selection, recombination hot spots and resistance to antimalarial drugs.  <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=20101240">Nat. Genet. 2010 Mar;42(3):268-71 Mu et al.</a></li>
</ul>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem47"/>
    <a name="plasmo05_10_news"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 6.4 released <i class="date">13 May 2010</i>
                
            </h1>
            <body>
<span style="font-weight: bold;">We are pleased to
announce the release of PlasmoDB 6.4<br>
<br>
<br>
This release of PlasmoDB contains the following new datasets and
features:</span><span style="font-weight: bold;">&nbsp;<br>
<br>
New Data:</span><br>
<br>
<ul>
  <li><span style="font-style: italic;">PiggyBac</span>
insertion sites in&nbsp;<span style="font-style: italic;">P.
falciparum</span> (NF54) from <a
 href="http://www.ncbi.nlm.nih.gov/pubmed/19422698">Balu <span
 style="font-style: italic;">et. al.</span></a>
are available via the genome browser. &nbsp;To view insertion sites
turn on the track called "P. falciparum piggyBac Transposable Elements"
under the heading "Insertions". &nbsp;Here is a quick link to the
genome browser with a region with insertions: <a
 href="http://plasmodb.org/cgi-bin/gbrowse/plasmodb/?start=1;stop=2038337;ref=Pf3D7_11;width=800;version=100;cache=on;show_tooltips=on;grid=on;id=8f78c428af13ed4911cb7a52c6fb380d;label=Gene-TransposableElements"><span
 style="font-style: italic;">piggyBac </span>insertions
in chromosome 11</a>.&nbsp;</li>
  <br>
  <li>ChIP-chip data using histone H4 antibodies from <a
 href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2801526/">Westenberger<span
 style="font-style: italic;"> et. al.</span></a>
are available via the genome browser under the ChIP on chip heading.
You may view tracks for different RBC life stages as in this <a
 href="http://plasmodb.org/cgi-bin/gbrowse/plasmodb/?start=850000;stop=1190000;ref=Pf3D7_11;width=800;version=100;cache=on;show_tooltips=on;grid=on;id=8f78c428af13ed4911cb7a52c6fb380d;label=Gene-ChIP_Nucleosome_Ring_H4-ChIP_Nucleosome_Troph_H4-ChIP_Nucleosome_Schizont_H4">example</a>.</li>
  <br>
  <li>Nucleosome positioning data from <a
 href="http://www.ncbi.nlm.nih.gov/pubmed/20054063">Ponts <span
 style="font-style: italic;">et. al.</span> </a>&nbsp;are
available via the genome browser under the ChIP-seq heading.
&nbsp;You
may view tracks for different time points during the RBC life cycle as
in this <a
 href="http://plasmodb.org/cgi-bin/gbrowse/plasmodb/?start=850000;stop=1190000;ref=Pf3D7_11;width=800;version=100;cache=on;show_tooltips=on;grid=on;id=8f78c428af13ed4911cb7a52c6fb380d;label=Gene-ChipSeq0CombNorm-ChipSeq6CombNorm-ChipSeq12CombNorm-ChipSeq18CombNorm-ChipSeq24CombNorm-ChipSeq30CombNorm-ChipSeq36CombNorm">example</a>.</li>
</ul>
<span style="font-weight: bold;"><br>
Features introduced this year:<br>
</span>
<ol>
  <li><span style="font-weight: bold; font-style: italic;">"My
Basket"</span> allows one to cherry pick genes and add them to
the basket for subsequent conversion into a step in a strategy.
&nbsp;The basket is accessible from the gray menu bar and the
number of items in your basket is indicated.</li>
  <br>
  <li><span style="font-weight: bold; font-style: italic;">Weighted
searches</span>&nbsp;allows you to assign an arbitrary weight
to the searches (steps) in
your search strategy making it possible for you to view a ranked list
of your results. &nbsp;</li>
  <div style="margin-left: 40px;">Here is an example of
the
same strategy looking for putative vaccine candidates. &nbsp;Notice
that you
have a larger number of results with weighting and those can be sorted
based on the sum of the weights, giving a ranked list of results.<br>
  <br>
  <a href="http://plasmodb.org/plasmo/im.do?s=0fdc12df42cc3c61">Putative
vaccine candidates (unweighted)</a><br>
  <br>
  <a href="http://plasmodb.org/plasmo/im.do?s=4d4df392a0952327">Putative&nbsp;</a><a
 href="http://plasmodb.org/plasmo/im.do?s=4d4df392a0952327">vaccine
candidates</a><a
 href="http://plasmodb.org/plasmo/im.do?s=4d4df392a0952327">
(Weighted)</a></div>
  <br>
  <li><span style="font-weight: bold; font-style: italic;">"My
Favorites"</span> (new in this release) allows you to bookmark
and organize genes you
access frequently. &nbsp;A tutorial on how to use this feature is
available <a
 href="http://eupathdb.org/tutorials/Favorites/favorites_viewlet_swf.html">here</a>.&nbsp;</li>
  <br>
  <li>We now provide prominent linkouts to GeneDB from PlasmoDB
gene pages when there is an update in GeneDB not yet available in
PlasmoDB. &nbsp;This feature enables PlasmoDB users to have easy
access to the most up-to-date information. &nbsp;It is important to
highlight that your comments on gene pages in PlasmoDB constitute one
of the tools made available to the annotators at GeneDB to improve and
enhance the genome annotation.</li>
</ol>
<span style="font-weight: bold;"></span><span
 style="font-weight: bold;">Feature evaluators still needed:<br>
<br>
</span>
<div style="margin-left: 40px;">We welcome you to *sign
up* to be an early evaluator of new features on the EuPathDB sites.
This will be an opportunity for you to help us refine and improve
features early in the development process.&nbsp;In fact, the "My
Favorites" feature was the subject of our first survey and feature
evaluator's comments and suggestions contributed to enhancing this
feature. <br>
<br>
Signing on is easy and will
require minimal effort on your part in the form of answering questions
or rendering your opinion. &nbsp;<br>
<br>
To become&nbsp;a feature evaluator, <a
 href="https://mail.pcbi.upenn.edu/mailman/listinfo/eupathdb_testers">click
here</a>.</div>
<br>
<br>
<span style="font-weight: bold;"></span>
</body>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem48"/>
    <a name="swg_announce_2010"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                EuPathDB Scientific Working Group <i class="date">31 March 2010</i>
                
            </h1>
            EuPathDB is pleased to announce the members of the scientific working group
(SWG). The SWG provides EuPathDB with scientific oversight and 
guidance.  Members of the SWG also review the driving biological projects initiative.<br><br>

<table>
<tr>
<td><a href="http://www.fnih.org/about">Michael Gottlieb</a>, Chair<br>
Foundation for National Institutes of Health (FNIH)<br>
Broad expertise in parasitology, large integrative projects (chair)<br>
</td>
<td>
<a href="http://www.sanger.ac.uk/research/faculty/mberriman/">Matt Berriman</a><br>
Wellcome Trust Sanger Institute<br>
Pathogen genomics & functional annotation<br>
</td>
</tr><tr>
<td>
<a href="http://med.stanford.edu/profiles/John_Boothroyd/">John Boothroyd</a><br>
Stanford University<br>
Toxoplasma biology & host-pathogen interactions<br>
</td>
<td>
<a href="http://www.vcu.edu/lifesci/research/res_fac_int_buck.html">Greg Buck</a><br>
Virginia Commonwealth University<br>
Pathogenomics of apicomplexan and kinetoplastid parasites<br>
</td>
</tr><tr>
<td>
<a href="http://www3.niaid.nih.gov/labs/aboutlabs/lmiv/default.htm">Patrick Duffy</a><br>
National Institutes of Health (NIAID)<br>
Laboratory & clinical aspects of malaria biology<br>
</td>
<td>
<a href="http://www.cgm.northwestern.edu/">Pascale Gaudet</a><br>
Northwestern University<br>
Biocuration, ontologies, DIctyostelium<br>
</td>
</tr><tr>
<td>
<a href="http://www.paras.uzh.ch/research/microbiology.html">Adrian Hehl</a><br>
University of  Zurich<br>
Cellular and molecular biology of Giardia<br>
</td>
<td>
<a href="http://www.botany.ubc.ca/keeling/">Patrick Keeling</a><br>
University of British Columbia<br>
Comparative genomics<br>
</td>
</tr><tr>
<td>
<a href="http://www.medicine.virginia.edu/clinical/departments/pathology/faculty/petri-page">Bill Petri</a><br>
University of Virginia<br>
Laboratory & clinical aspects of Entamoeba and diarrheal diseases<br>
</td>
<td>
<a href="http://www.oicr.on.ca/Research/stein.htm">Lincoln Stein</a><br>
Ontario Cancer Center<br>
Bioinformatics tools & analysis<br>
</td>
</tr><tr>
<td>
<a href="http://www.genome.gov/26525388">Lu Wang</a><br>
National Institutes of Health (NHGRI)<br>
Large-scale sequencing<br>
</td>
<td>
<a href="http://www.liv.ac.uk/infection-and-global-health/staff/jonathan-wastling/">Jonathan Wastling</a><br>
University of Liverpool<br>
Proteomics of apicomplexan parasites<br>
</td>
</tr><tr>
<td>
<a href="http://www.einstein.yu.edu/home/faculty/profile.asp?id=4218">Louis Weiss</a><br>
Albert Einstein College of Medicine<br>
Microsporidia, Toxoplasma, and clinical microbiology<br>
</td>
<td>
<a href="http://www.hsph.harvard.edu/faculty/dyann-wirth/">Dyann Wirth</a><br>
Harvard School of Public Health<br>
Plasmodium population biology, drug discovery<br>
</td></table>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem49"/>
    <a name="plasmo_newFeatures_0310"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB new features <i class="date">15 March 2010</i>
                
            </h1>
            <span style="font-weight: bold;">We are pleased to 
announce new features in PlasmoDB 6.3<br><br> 
</span><span style="font-weight: bold;">New Features:</span><br><br>
<ul>
 <li>The gray tool bar at the top of all PlasmoDB web pages has 
been enhanced to include a link to the "My Basket" feature including an 
indication of how may items are in the basket.</li> 
 <li>The genome browser has been updated to version 1.7, which 
offers enhanced features and stability.&nbsp;</li> 
 <li>Weighted searches are now available. &nbsp;This feature 
allows you to assign an arbitrary weight to the searches (steps) in 
your search strategy making it possible for you to view a ranked list 
of your results.&nbsp;</li> 
</ul> 
<br>
<span style="font-weight: bold;">***<a href="https://mail.pcbi.upenn.edu/mailman/listinfo/eupathdb_testers">Sign up to be a EuPathDB feature evaluator:</a
>***<br><br>
</span> 
<div style="margin-left: 20px;">
We welcome you to <a href="https://mail.pcbi.upenn.edu/mailman/listinfo/eupathdb_testers">*sign up* to be an early evaluator</a>
 of new features on the EuPathDB sites.  This will be an opportunity for you to help us refine and improve features early in the development process.  
Signing up is easy.  Evaluating will require minimal effort on your part, 
typically in the form of answering specific questions, using the feature(s) and rendering your opinion.
</div>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem50"/>
    <a name="eupath_plas44"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 4.4 is no longer available <i class="date">15 February 2010</i>
                
            </h1>
            PlasmoDB 4.4 has been discontinued.  If there are tools or other functionality that you used on PlasmoDB 4.4 that you need, please <b><a href="http://plasmodb.org/plasmo/app/contact-us" class="new-window" data-name="contact_us">contact us</a></b> to let us know and we will help you regain access to the missing functionality.
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem51"/>
    <a name="eupath_dbp_2010"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                EuPathDB Driving Biological Projects <i class="date">4 February 2010</i>
                
            </h1>
            The NIAID-funded Eukaryotic Pathogen Genome Database
(<a href="http://eupathdb.org">EuPathDB.org</a>) invites proposals for Driving Biological
Projects (DBPs) focused on one or more of the eukaryotic
pathogens supported by this Bioinformatics Resource Center
(BRC). We anticipate making two awards of up to $600K each
by July 1, 2010 (a second round of solicitations is expected
in 2012).<i> A two page Letter of Intent (LOI) is due Feb 28,
following which full applications will be solicited by
invitation only, with a deadline of March 31.</i>  
<a href="http://eupathdb.org/common/EuPathDB_DBP_announcement.pdf">
Click here to
download the RFP</a>.
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem52"/>
    <a name="Plasmo1209news"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 6.3 released <i class="date">22 December 2009</i>
                
            </h1>
            <span style="font-weight: bold;">We are pleased to
announce the release of PlasmoDB 6.3<br>
<br>
<br>
</span>This release of PlasmoDB contains the following new
datasets and features:<br>
<br>
New Data:<br>
<ul>
  <li><span style="font-style: italic;">P.
falciparum </span>transcriptome sequencing (RNA-seq) profiles.
&nbsp;This data was graciously provided to PlasmoDB prepublication
and will soon appear in press: New insights into the blood stage
transcriptome of Plasmodium falciparum using RNA-Seq. &nbsp;Thomas
D. Otto, Daniel Wilinski, Sammy Assefa, Thomas M. Keane, Louis R.
Sarry, Ulrike Bohme, Jacob Lemieux, Bart Barrell, Arnab Pain, Matthew
Berriman, Chris Newbold, Manuel Llinas. &nbsp;Molecular
Microbiology (in press). &nbsp;You can search for genes based on
this data by going to the "<a
 href="http://plasmodb.org/plasmo6.3/showQuestion.do?questionFullName=GeneQuestions.GenesByRNASeqExpressionTiming&amp;target=GENE"><span
 style="font-style: italic;">P. falciparum </span>transcriptome
sequencing (RNA-seq) profiles</a>" search under "Transcript
Expression" in PlasmoDB.</li>
</ul>
<ul>
  <li><span style="font-style: italic;">P. falciparum</span>
transcription profile of wild-type and Sir2a and Sir2b knock-out
parasites. &nbsp;Using this search you can find genes that are
differentially expressed between wild-type and knock-out parasites
during the intraerythrocytic life-cycle. &nbsp;You can search for
genes based on this data by going to the "<a
 href="http://qa.plasmodb.org/plasmo6.3/showQuestion.do?questionFullName=InternalQuestions.GenesByMicroarrayEvidence&amp;target=GENE">Microarray
Evidence</a>" search under "Transcript Expression" in PlasmoDB.
&nbsp; Details of this study can be found here: &nbsp;<a
 href="http://www.ncbi.nlm.nih.gov/pubmed/19402747">Tonkin
CJ, Carret CK, Duraisingh MT, Voss TS, Ralph SA, Hommel M, Duffy MF,
Silva LM, Scherf A, Ivens A, Speed TP, Beeson JG, Cowman AF. Sir2
paralogues cooperate to regulate virulence genes and antigenic
variation in Plasmodium falciparum. PLoS Biol. 2009 Apr 14;7(4):e84.</a></li>
</ul>
<ul>
  <li>Transcription profiling of <span
 style="font-style: italic;">Plasmodium falciparum</span>
after exposing drug-selected mutants to short term CQ treatment.
&nbsp;Identify genes differentially expressed during chloroquine
treatment. &nbsp;You can search for genes based on this data by
going to the "<a
 href="http://plasmodb.org/plasmo6.3/showQuestion.do?questionFullName=InternalQuestions.GenesByMicroarrayEvidence&amp;target=GENE">Microarray
Evidence</a><a
 href="http://qa.plasmodb.org/plasmo6.3/showQuestion.do?questionFullName=GeneQuestions.GenesByChIPchipPlasmo&amp;target=GENE"><span
 style="text-decoration: underline;"></span></a><a
 href="http://qa.plasmodb.org/plasmo6.3/showQuestion.do?questionFullName=GeneQuestions.GenesByChIPchipPlasmo&amp;target=GENE"></a>"
search under "Transcript Expression" in PlasmoDB. Details of this study
can be found here:&nbsp;<a
 href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=Pubmed&amp;term=18575593%5BUID%5D">Jiang
H, Patel JJ, Yi M, Mu J, Ding J, Stephens R, Cooper RA, Ferdig MT, Su
XZ. Genome-wide compensatory changes accompany drug- selected mutations
in the Plasmodium falciparum crt gene. PLoS One 2008 Jun 25;3(6):e2484.</a></li>
</ul>
<ul>
  <li><span style="font-style: italic;">P.
falciparum </span>HP1 chromatin immunoprecipitation data.
&nbsp;You can search for genes based on this data by going to the "<a
 href="http://plasmodb.org/plasmo6.3/showQuestion.do?questionFullName=GeneQuestions.GenesByChIPchipPlasmo&amp;target=GENE"><span
 style="text-decoration: underline;">P.f ChIP on chip</span>
evidence</a>" search under "Transcript Expression" in PlasmoDB.
Details of this study can be found here: <a
 href="http://www.ncbi.nlm.nih.gov/pubmed/19730695">Flueck C,
Bartfai R, Volz J, Niederwieser I, Salcedo-Amaya AM, Alako BT, Ehlgen
F, Ralph SA, Cowman AF, Bozdech Z, Stunnenberg HG, Voss TS.&nbsp;
Plasmodium falciparum heterochromatin protein 1 marks genomic loci
linked to phenotypic variation of exported virulence factors. PLoS
Pathog. 2009 Sep;5(9):e1000569. Epub 2009 Sep 4.</a></li>
</ul>
<ul>
  <li>Chromatin immunoprecipitation to identify histone
modifications (H3K9me3, H3K9Ac, H3K4me3, H4K20me3) in <span
 style="font-style: italic;">P. falciparum </span>wild
type and Sir2 knock-outs. You can search for genes based on this data
by going to the "<a
 href="http://plasmodb.org/plasmo6.3/showQuestion.do?questionFullName=GeneQuestions.GenesByChIPchipPlasmo&amp;target=GENE"><span
 style="text-decoration: underline;">P.f ChIP on chip</span>
evidence</a>" search under "Transcript Expression" in PlasmoDB.
Details of this study can be found here: &nbsp;<a
 href="http://www.ncbi.nlm.nih.gov/pubmed/19218088">Lopez-Rubio
JJ, Mancio-Silva L, Scherf A. Genome-wide analysis of heterochromatin
associates clonally variant gene regulation with perinuclear repressive
centers in malaria parasites. Cell Host Microbe. 2009 Feb
19;5(2):179-90.</a></li>
</ul>
<br>
New Features:<br>
<br>
This release features the addition of a "My
Basket" tool. &nbsp;This feature allows you to select various genes
from
your searches and add them to your basket (much like a shopping cart).
&nbsp;To add genes to your basket simply click on the basket icon
in a
result list or on a gene page. &nbsp;A green basket icon indicates
that a
genes is already in your basket. &nbsp;You can easily convert your
basket to a strategy to save it and perform additional searches or
combine it with one of your already existing strategies.
&nbsp;Please note that using the basket feature requires you to
login first.<br>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem53"/>
    <a name="HinxtonWorkShop"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                <b>Working with Parasite Database Resources Workshop (April 17-21, 2010)</b> <i class="date">20 November 2009</i>
                
            </h1>
            <big><br>
We are pleased to announce a workshop taught as part of a collaboration between the Pathogen Genomics Group 
(Wellcome Trust Sanger Institute, UK), the Eukaryotic Pathogen Bioinformatics Resource Center 
(University of Georgia and University of Pennsylvania, US), and the Seattle Biomedical Research Institute (US). 
This residential workshop aims to provide experimental biologists with hands-on experience in 
genomic-scale data analysis, including genome browsers and comparison tools, methods for 
data integration, and resources for sophisticated data mining.
Many examples and exercises will be drawn from the kinetoplastida, including <i>Leishmania spp.</i>, 
<i>Trypanosoma brucei</i>, and <i>Trypanosoma cruzi</i>, although applicants working on other protozoan 
parasites will also benefit.<br><br>

<span style="font-weight: bold;"><span
 style="color: red;">!!!!!</span>Application Deadline
is December 18th 2009<span style="color: red;">!!!!!!</span></span>
<br><br>

<b>For registration and additional information (including scholarships) please click <a href="http://www.wellcome.ac.uk/Education-resources/Courses-and-conferences/Advanced-Courses/Courses/WTX057366.htm">here</a> (Note: you will be diverted to and external site).
</b><br><br>
As usual, if you have any questions please send us an <a href="mailto:help@eupathdb.org">Email</a>.
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem54"/>
    <a name="Plasmo1109news"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 6.2 released <i class="date">13 November 2009</i>
                
            </h1>
            <span style="font-weight: bold;">We are pleased toannounce
the release of PlasmoDB 6.2<br>
<br>
</span><span style="text-decoration: underline;">New
Data:</span><br>
<ul>
  <li><span style="font-style: italic;">P.
&nbsp;falciparum</span> and <span
 style="font-style: italic;">P. vivax</span> proteomics
from clinical samples graciously provided prior to publication by Dr.
Utpal Tatu (Department of Biochemistry, Indian Institute of Science,
Bangalore, India). This data may be accessed using the "<a
 href="http://plasmodb.org/plasmo/showQuestion.do?questionFullName=GeneQuestions.GenesByMassSpec&amp;target=GENE">Identify
Genes based on Mass Spec. Evidence</a>" query and combined with
other searches in PlasmoDB.</li>
</ul>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem55"/>
    <a name="PlasmoMicroError"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                Important announcement regarding microarray data in PlasmoDB <i class="date">26 October 2009</i>
                
            </h1>
            Dear all,<br>
<br>
The PlasmoDB team would like to alert you to two important issues
concerning the presentation of <span style="font-style: italic;">P.
falciparum</span> microarray data in PlasmoDB.&nbsp; <br>
<br>
1.&nbsp; Beginning with PlasmoDB 6.0 all profiles using the
Winzeler Affymetrix scrMalaria array were updated to reflect the latest
3D7 genome annotation. Previously, probes were mapped to the P.
falciparum 3D7 genome annotation released in July 2001.&nbsp; The
array was originally designed with a tiling of probes, so the new
annotation has resulted in many genes gaining new probes which did not
previously map to the gene, and dropping old probes which no longer map
(Note: The new mapping allowed up to two mismatches per probe, and only
probes which mapped uniquely, up to two mismatches, were
retained).&nbsp; Previous gene expression profiles were generated
using the MOID algorithm with the 2001 annotation, while the current
profiles were generated using RMA with the new annotation.&nbsp;
Many of the expression profiles exhibit minor differences and a
few&nbsp;changed significantly.&nbsp; Such differences are
expected and should result in a more accurate representation of the
data.<br>
<br>
2.&nbsp; An internal error with one the files concerning the P.
falciparum 3D7 gametocyte expression data was discovered.&nbsp;
This error appeared with the release of PlasmoDB 6.0 in July of 2009
and was fixed on October 20th, 2009.&nbsp; Also, please note that
probe remapping and MOID vs. RMA analysis differences described above
also apply to this data set.&nbsp; <br>
<br>
We regret any disruption this may cause you.&nbsp; Please do not
hesitate to contact us with any questions or concerns.<br>
<br>
Sincerely,<br>
The PlasmoDB Team
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem56"/>
    <a name="Plasmo909news"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 6.1 released <i class="date">25 September 2009</i>
                
            </h1>
            <b>We are pleased to
announce the release of PlasmoDB 6.1</b><br>
<br>
<u>New Data:</u><br>
<ul>
  <li><i>P.
&nbsp;falciparum</i> intraerythrocytic expression profiles
from&nbsp;infected patients (<a
 href="http://www.ncbi.nlm.nih.gov/pubmed/19376968">Lemieux <i>et. al.</i></a>)
-- data available on gene pages, for example, click <a
 href="http://plasmodb.org/plasmo/app/record/gene/PF11_0344">here</a>
and scroll down to the expression section.</li>
</ul>
<u>
New/Updated Features:</u><br>
<ul>
  <li>Now after you perform a BlastN against genomic sequence you
can
link from the blast output to the correct coordinates in the genome
browser.&nbsp; This allows you to explore annotation and other
functional
data in relation to your BLAST hits.</li>
  <li>Linkouts to Gene Ontology annotation predictions at <a
 href="http://www.atgc-montpellier.fr/PlasmoDraft/">Plasmodraft</a>
have been updated. (<a
 href="http://www.ncbi.nlm.nih.gov/pubmed/18925948">BrÃ©hÃ©lin
et. al.</a>).</li>
  <li>Linkouts to the <a
 href="http://sites.huji.ac.il/malaria/">Malaria Parasite
Metabolic Pathways</a> resource (Hagai Ginsburg).</li>
  <li>The type ahead feature has been implemented for GO term, EC
number and Interpro domain searches.</li>
  <li>You can now edit or delete your user comments.</li>
  <li>You can now modify your user profile, change your password
or your email address.</li>
</ul>
Please also note that the parent portal site <a
 href="http://eupathdb.org">EuPathDB</a> has just been
updated, offering you the ability to explore results across organisms.<br>
<br>
Sincerely,<br>
<br>
The EuPathDB team.
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem57"/>
    <a name="Release6"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 6.0 released <i class="date">14 July 2009</i>
                
            </h1>
            <p>
    The EuPathDB team is happy to announce the release of PlasmoDB 6.0.  
    <p>
This release includes several new and updated data sets (many prepublication), a new user interface with added functionality and a graphical strategy system for building complex searches.
<p>
The new user interface contains many new features including:
<ul>
<li>  A new search strategy interface that enables users to more easily construct complex searches.
<li>  'User Comments' have been updated to enable submission of supporting evidence including pubmed identifiers and images (or other files). Comments may be added to any gene or sequence page, enhancing available annotation. 
<li>  Files can now be uploaded and provided to the community via the file repository section.
</ul>
<p>
New and updated Data:
<ul>
<li>  <i>P. falciparum</i> reannotation integrated as the official annotation in PlasmoDB (See news item dated 1 February 2008 for details on the reannotation effort).
<li>  <i>P. falciparum</i> High density genotyping array (HD Array) from the Broad Institute (Unpublished).
<li>  <i>P. vivax</i> intraerythrocytic microarray life-cycle (<a href="http://www.ncbi.nlm.nih.gov/pubmed/18852452">Bozdech et. al.</a>).
<li>  <i>P. vivax</i> microarray from patient samples (Westenberger and Winzeler, unpublished).
<li>  <i>P. chabaudi</i> new 8X assembly and reannotation (Pathogen Sequencing Unit at the Wellcome Trust Sanger Institute, unpublished).
<li>  <i>P. yoelii</i> published gene calls appear as provisional genes in the genome browser (<a href="http://www.ncbi.nlm.nih.gov/pubmed/18586738">Vaughan et. al.</a>).
<li>  <i>P. yoelii</i> oocyst and salivary gland microarray data (<a href="http://www.ncbi.nlm.nih.gov/pubmed/18270564">Zhou et. al.</a>).
<li>  Isolate data from GenBank is now integrated and searchable.
</ul>
<p>
Additional features:
<ul>
<li>  Linkouts are now provided from genes with mutant information in the <a href="http://pberghei.eu">Genetically Modified Rodent Malaria Parasites database</a>.
<li>  Linkouts are provided to Gene Ontology annotation predictions at <a href="http://www.atgc-montpellier.fr/PlasmoDraft/">Plasmodraft</a>. (<a href="http://www.ncbi.nlm.nih.gov/pubmed/18925948">Br&#233;h&#233;lin et. al.</a>).
<p>
The previous version of PlasmoDB may still be accessed <a href='http://old.plasmodb.org'>here</a>.</ul>


<u>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem58"/>
    <a name="1December2008GatesFoundation"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                EuPathDB to take on the kinetoplastida <i class="date">1 December 2008</i>
                
            </h1>
            We are pleased to report
an award from the <a href="http://www.gatesfoundation.org">Bill & Melinda Gates Foundation</a> in support of a
TrypDB component of the <a href="http://eupathdb.org/">EuPathDB</a> family of databases,
encompassing <i>Leishmania</i> species, <i>Trypanosoma brucei</i>, and
<i>Trypanosoma cruzi</i>.  A coordinated award from the<a href="http://www.wellcome.ac.uk/"> Wellcome Trust</a>
will permit enhanced annotation of kinetoplastid parasites. 
These projects represent a collaborative effort between the
GeneDB team (at the Wellcome Trust Sanger Institute), colleagues
at the <a href="http://www.sbri.org/">Seattle Biomedical Research Institute</a>, and the EuPathDB
team -- an NIAID-supported Bioinformatics Resource Center. Look
for the first release of TrypDB in Jan 2009!
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem59"/>
    <a name="reannotation5"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                Fifth updated snapshot of the reannotation of the <i>P. falciparum</i> 3D7 genome <i class="date">18 November 2008</i>
                
            </h1>
            We are pleased to announce the release of the fifth snapshot of the reannotation of the <i>P. falciparum</i> 3D7 genome.
<p>
The effort to systematically update the annotation for <i>P. falciparum</i> 3D7 genome started in October 2007 with a weeklong workshop co-organized by staff from the Wellcome Trust Sanger Institute (WTSI) and the EuPathDB team. The Wellcome Trust and Burroughs Wellcome Fund-sponsored workshop was held in Hinxton, UK and was attended by nearly 30 Plasmodium researchers contributing expertise in various aspects of Plasmodium biology, plus more than 10 bioinformatics support staff from the Artemis, GeneDB and EuPathDB teams.  Ongoing annotation and error checking is being carried out by the GeneDB group from WTSI.
<p>
For further information please refer to the February 1st, 2008 news item (below).
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem60"/>
    <a name="NAR2008"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB publication in Nucleic Acid Research <i class="date">4 November 2008</i>
                
            </h1>
            We are pleased to announce the publication of a paper in Nucleic Acid Research describing PlasmoDB.<br>
Please cite this manuscript when referencing the PlasmoDB resource in your publications.
<p>
<a href="http://www.ncbi.nlm.nih.gov/sites/entrez?db=pubmed&cmd=search&term=18957442">
PlasmoDB: a functional genomic database for malaria parasites.</a> Nucleic Acids Res. 2008 Oct 31. Aurrecoechea C, et al.
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem61"/>
    <a name="currentRelease"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 5.5 released <i class="date">16 September 2008</i>
                
            </h1>
            <p>
    The EuPathDB team is happy to announce the release of PlasmoDB 5.5.  
    <p>
    
    <u>New Data:</u>
    
    <ul>
    <li>Proteomics:</li>
      <ul>
      <li>P. falciparum purified merozoite proteomics data provided prepublication by the Leiden Malaria Group.</li>
      <li>P. yoelii liver stage proteomics (Tarun et al. <a href="http://www.ncbi.nlm.nih.gov/pubmed/18172196?dopt=Abstract">Proc. Natl. Acad. Sci. U.S.A. (2008) vol. 105 (1) pp. 305-10</a>).</li>
      </ul>
    </ul>
    
    <ul>
    <li>Microarray:</li>
      <ul>
      <li>P. yoelii liver, mosquito, and red cell stage expression data (Tarun et al.<a href="http://www.ncbi.nlm.nih.gov/pubmed/18172196?dopt=Abstract"> Proc. Natl. Acad. Sci. U.S.A. (2008) vol. 105 (1) pp. 305-10</a>).</li>
      <li>P. falciparum expression data from infected patients (Daily et al.<a href="http://www.ncbi.nlm.nih.gov/pubmed/18046333?dopt=Abstract"> Nature 2007 Dec 13;450 (7172):1091-5</a>).</li>
      </ul>
    </ul>
    
    <ul>
    <li>Population Biology:</li>
      <ul>
      <li>Isolate data from P. falciparum (3k chip and 24bp barcode) (Volkman et al. <a href="http://www.ncbi.nlm.nih.gov/pubmed/17159979?dopt=Abstract">Nat Genet. 2007 Jan;39(1):113-9</a>).</li>
      <li>Microsatellite data for P. falciparum available through Gbrowse with link-outs to GenBank records (Su et al. <a href="http://www.ncbi.nlm.nih.gov/pubmed/10558988?dopt=Abstract">Science, 1999 vol. 286 (5443) pp. 1351-3</a>).</li>
      </ul>
    </ul>
    
    <ul>
    <li>Promoter Prediction:</li>
      <ul>
      <li>Prediction of transcription start sites in P. falciparum provided prepublication and available on gbrowse (Brick K, Watanabe J, and Pizzi E).  </li>
      </ul>
    </ul>
    
    <u>Improvements:</u>
    
    <ul>
    <li>Documentation related to data types has been improved.</li>
    <li>Gbrowse tracks have been organized and help associated with gbrowse items have been improved.</li>
    </ul>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem62"/>
    <a name="microarray_listing"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                Plasmodium microarray resources <i class="date">22 July 2008</i>
                
            </h1>
            A listing of microarrays that are available to Plasmodium researchers has been posted at 
<a href="http://plasmodb.org/common/microarrays/">http://plasmodb.org/common/microarrays/</a>. 
This list is meant to provide in a single location a summary of the publicly  
available microarrays targeting Plasmodium genomes (including <i>P. falciparum</i>, 
<i>P. berghei</i>, <i>P. yoelii</i>, and <i>P. vivax</i>). Also provided are 
details, including, for some of the arrays, an updated mapping to genes in PlasmoDB.
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem63"/>
    <a name="reannotation4"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                Fourth updated snapshot of the reannotation of the <i>P. falciparum</i> 3D7 genome <i class="date">15 July 2008</i>
                
            </h1>
            PlasmoDB has released the fourth snapshot of the reannotation of the<i> P. falciparum</i> 3D7 genome. The reannotation presented in PlasmoDB is a work in progress and intended to inform you about the new annotation and elicit any feedback that you may have.  Please add user comments to any genes for which you have evidence that supports or refutes the new annotation.  In the previous snapshot, many product names for re-annotated genes were accidentally reverted back to the original name. This new snapshot has the correct re-annotated product names.
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem64"/>
    <a name="sequencingApproval11June2008"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                Approval for sequencing additional <i>Plasmodium</i> isolates and species. <i class="date">11 June 2008</i>
                
            </h1>
            The proposal for sequencing of additional <i>Plasmodium</i> isolates and species 
     has been approved. 
     The whitepaper is available <a href="http://plasmodb.org/common/downloads/doc/PlasmodiumWhitePaperV5.pdf">here</a>.
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem65"/>
    <a name="reannotation3"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                Third updated snapshot of the reannotation of the <i>P. falciparum</i> 3D7 genome <i class="date">6 June 2008</i>
                
            </h1>
            PlasmoDB has released the third snapshot of the reannotation of the<i> P. falciparum</i> 3D7 genome. The reannotation presented in PlasmoDB is a work in progress and intended to inform you about the new annotation and elicit any feedback that you may have.  Please add user comments to any genes for which you have evidence that supports or refutes the new annotation.  In the previous snapshot, many product names for re-annotated genes were accidentally reverted back to the original name. This new snapshot has the correct re-annotated product names.
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem66"/>
    <a name="reannotation2"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                An updated snapshot of the reannotation of the <i>P. falciparum</i> 3D7 genome <i class="date">19 May 2008</i>
                
            </h1>
            <p>
The effort to systematically update the annotation for <i>P. falciparum</i> 3D7 genome started in October 2007 with a weeklong workshop co-organized by staff from the Wellcome Trust Sanger Institute (WTSI) and the ApiDB team. The Wellcome Trust and Burroughs Wellcome Fund-sponsored workshop was held in Hinxton, UK and was attended by nearly 30 Plasmodium researchers contributing expertise in various aspects of Plasmodium biology, plus more than 10 bioinformatics support staff from the Artemis, GeneDB and ApiDB teams. During the workshop 2700 genes were edited using Artemis and ACT software, mainly with the addition of new or alternative functional descriptions (e.g. product names, GO terms, EC numbers and miscellaneous textual comments).

 
<p>

After the workshop, intensive manual reannotation of the genome has continued at WTSI with the help of several members of the community. First, the structures of more than 700 gene models have been changed and new gene models have been added. In addition, curation staff at GeneDB have reviewed all User Comments submitted to PlasmoDB. Over the coming months, the emphasis will switch to a careful genome-wide review of functional annotation.

 
<p>
The annotation presented here in PlasmoDB is the second snapshot of a work in progress and intended to inform you about the new annotation and elicit any feedback that you may have. Please add a User Comment to any gene about which you have additional information. This information is forwarded to the GeneDB curators on a weekly basis and will be included in the next snapshot.  In order to expedite the annotation, the following are of particular value to the manual curators: 
<ul>
<li> New product names with the PubMed ID of the paper in which they were first cited, 

<li> Additional PubMed IDs of any papers from which the function of the gene/protein can be ascribed

<li> EC number

<li> Alternative/updated gene names and aliases.

<li> Evidence that a gene structure has been incorrectly called.  Please include a description of the evidence that you have used (e.g. full length cDNA) and the new sequence. 
</ul>
 
<p>
The ongoing reannotation is being systematically carried out by Ulrike B&#246;hme and Bart Barrell (Wellcome Trust Sanger Institute) with extensive help from several community volunteers, in particular Chris Newbold (Oxford, UK), Tom Templeton (Cornell, US), Eric Mar&#233;chal (Grenoble, France).  For the annotation, the Artemis software was redesigned by Tim Carver and changes are recorded within a Chado database (GeneDB) designed and maintained by Adrian Tivey and Chinmay Patel. 


<p>
Workshop participants included
<ul>
<li>Oliver Bilker (Imperial Coll, London)
<li>Serge Bonnefoy (Inst Pasteur, Paris)
<li>Pete Bull (Kenya Med Res Inst, Kilifi)
<li>Jane Carlton (New York Univ)
<li>Brendan Crabb (Walter & Eliza Hall Inst, Melbourne)
<li>Christian Doerig (Univ Glasgow)
<li>Malcolm Gardner (Seattle Biomedical Res Inst)
<li>Hagai Ginsburg (Hebrew Univ, Jerusalem)
<li>George Githinji (MOMS, Tanzania)
<li>Taco Kooij (Heidelberg Univ)
<li>Dominic Kwiatkowski (Oxford Univ & WTSI)
<li>Sue Kyes (Oxford Univ)
<li>Thomas Lavstsen (Copenhagen)
<li>Manuel Llinas (Princeton Univ)
<li>Eric Marechal (CNRS, Grenoble)
<li>Dan Milner (Harvard Univ)
<li>Fingani Mphande (Karolinska Inst)
<li>Dan Neafsey (Broad Inst)
<li>Chris Newbold (Oxford Univ)
<li>Stuart Ralph (Melbourne Univ)
<li>Gowthaman Ramasamy (Seattle Biomedical Res Inst)
<li>Robert Sinden (Imperial Coll, London)
<li>Worachart Sirawaraporn (Mahidol Univ, Bangkok)
<li>Dominique Soldati (Univ Geneva)
<li>Tim Stedman (MR4 / ATCC)
<li>Tom Templeton (Cornell Univ, New York)
<li>Akhil Vaidya (Drexel Univ, Philadelphia)
<li>Scott Westenberger (Scripps Inst, La Jolla)
<li>Jennifer Wortman (Univ Maryland, Baltimore)
</ul>
Sponsors and Facilitators included:
<ul>
<li>Andy Berry, Ulrike Boehme, Celine Carret, Tim Carver, Al Ivens, Arnab Pain, Adrian Tivey & Matt Berriman (GeneDB / Wellcome Trust Sanger Inst)
<li>Brian Brunk, Zhongqiang Chen, Mark Heiges, Lucia Peixoto, Dhanasekaran Shanmugam & David Roos (ApiDB / Univ Georgia & Univ Pennsylvania)
</ul>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem67"/>
    <a name="reannotation"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                Draft Reannotation of the <i>P. falciparum</i> genome is now available. <i class="date">1 February 2008</i>
                
            </h1>
            The effort to re-annotate the <i>falciparum</i> genome was started in October at a weeklong workshop co-organized by ApiDB and GeneDB staff.  The workshop was hosted at the Welcome Trust Sanger Institute in Hinxton, UK and was attended by nearly 30 <i>Plasmodium</i> researchers contributing expertise in various aspects of <i>Plasmodium</i> biology and more than 10 bioinformatics support staff from the GeneDB and ApiDB teams. More than 2,700 genes have been reviewed to date and most modified in some fashion; primarily refining exon boundaries, assigning product names, assigning GO functions and EC numbers and making comments / notes.  The annotation process is continuing in the attempt to review all genes at least twice (by different annotators).  The annotation presented here in PlasmoDB is thus a work in progress and intended to inform you, the community, about the new annotation and elicit any feedback that you may have.  Please add a user comment to any gene that you have additional information about.  This includes supporting data corroborating gene annotation as well as data that indicates the gene model, product name, functional assignments etc are incorrect.  These comments will be included in the annotation which we hope will be concluded in mid-march.
<p>
Workshop participants included
<ul>
<li>Oliver Bilker (Imperial Coll, London)
<li>Serge Bonnefoy (Inst Pasteur, Paris)
<li>Pete Bull (Kenya Med Res Inst, Kilifi)
<li>Jane Carlton (New York Univ)
<li>Brendan Crabb (Walter & Eliza Hall Inst, Melbourne)
<li>Christian Doerig (Univ Glasgow)
<li>Malcolm Gardner (Seattle Biomedical Res Inst)
<li>Hagai Ginsburg (Hebrew Univ, Jerusalem)
<li>George Githinji (MOMS, Tanzania)
<li>Taco Kooij (Heidelberg Univ)
<li>Dominic Kwiatkowski (Oxford Univ & WTSI)
<li>Sue Kyes (Oxford Univ)
<li>Thomas Lavstsen (Copenhagen)
<li>Manuel Llinas (Princeton Univ)
<li>Eric Marechal (CNRS, Grenoble)
<li>Dan Milner (Harvard Univ)
<li>Fingani Mphande (Karolinska Inst)
<li>Dan Neafsey (Broad Inst)
<li>Chris Newbold (Oxford Univ)
<li>Stuart Ralph (Melbourne Univ)
<li>Gowthaman Ramasamy (Seattle Biomedical Res Inst)
<li>Robert Sinden (Imperial Coll, London)
<li>Worachart Sirawaraporn (Mahidol Univ, Bangkok)
<li>Dominique Soldati (Univ Geneva)
<li>Tim Stedman (MR4 / ATCC)
<li>Tom Templeton (Cornell Univ, New York)
<li>Akhil Vaidya (Drexel Univ, Philadelphia)
<li>Scott Westenberger (Scripps Inst, La Jolla)
<li>Jennifer Wortman (Univ Maryland, Baltimore)
</ul>
Sponsors and Facilitators included:
<ul>
<li>Andy Berry, Ulrike Boehme, Celine Carret, Tim Carver, Al Ivens, Arnab Pain, Adrian Tivey & Matt Berriman (GeneDB / Wellcome Trust Sanger Inst)
<li>Brian Brunk, Zhongqiang Chen, Mark Heiges, Lucia Peixoto, Dhanasekaran Shanmugam & David Roos (ApiDB / Univ Georgia & Univ Pennsylvania)
</ul>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem68"/>
    <a name="release5.4"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 5.4 is released <i class="date">31 October 2007</i>
                
            </h1>
            The ApiDB team is pleased to announce the release of PlasmoDB_5.4.

<br><br><u>New data include:</u>
<ul>
<li> A slightly modified reference genome for <i>P. falciparum</i> (several gaps have been closed on chr10 & 11).
<li> P. berghei gametocyte proteomics data (Khan et al, Cell 121:675-87).
<li> Many additional <i>P. falciparum</i> SNPs (unpublished data from the Broad Inst).
<li> Additional ESTs (Watanabe et al, Nucl Acids Res 32:D334-8; Lu et al, BMC Genomics 8:255; Florent et al, Mol.Biochem.Parasitol. 135:143-8)
<li> Expression profiling data for antigenic and adherent variants of <i>P. falciparum</i> 3D7 (Mok et al, Molec Biochem Parasitol 151:184-92).
<li> User comments submitted prior to June 2007 have now been incorporated into the official annotation.
<li> Sequence assemblies for 11 <i>falciparum</i> strains from the Broad Institute have just been released and are available on the PlasmoDB download site.
</ul>

<u>New features include:</u>
<ul>
<li> Faster loading of Gene and Genome Browser pages. 
<li> Improved synteny views in the Genome Browser.
<li> Browser views of rodent malaria genomes colored to indicate chromosomes (Kooij et al, PLoS Pathog 1:e44). 
<li> Gene page links to various external data sources (including PlasmoMAP,  TDRtargets, UCSC <i>P. falciparum</i> genome browser, Ontology-based Pattern Identification and literature databases).
<li> More convenient access to help ... please click the "Ask us a Question" link on the left of every page, or the "Contact Us" at bottom to report problems or suggest improvements to the database.
</ul>

Note that a mechanism has now been established to ensure that User Comments on genes or sequences are incorporated into the official annotation in a timely manner.  In response to a banner posted on PlasmoDB and e-mail to malaria researchers, several hundred new User Comments were entered by the community in advance of a <i>P. falciparum</i> genome reannotation workshop jointly sponsored by the Wellcome Trust Sanger Inst and the PlasmoDB team during the week of Oct 21!
<br><br>
There is still time to enter additional User Comments while the updated annotation is being reviewed in preparation for deposition in GenBank and release on PlasmoDB and GeneDB in early 2008.  Please use the User Comments feature as a repository for valuable information on:
<ul>
<li> Naming, e.g. "Apical membrane antigen 1 is also known as AMA1, AMA-1, Pf83, Rhoptry Membrane Antigen, RMA1" or "this gene is now recognizable as XXX based on Pfam domain ###."
<li> Gene models, e.g. "Corrected cDNA sequence for gene XXX: &lt;sequence&gt;, based on RACE and primer extension for 5' UTR and polyA-tailed cDNA clone for 3' UTR" or "RT-PCR confirms annotated CDS sequence as correct."  Note that confirmation of existing annotation can be as valuable as corrections!
<li> Function, e.g. "Enzymatic activity for gene XXX has been confirmed" or "Protein XXX is predicted to target to the apicoplast, but an epitope-tagged construct shows cytoplasmic localization."  Be sure to provide name and/or PubMedID if available.
</ul>

For data that is not readily accommodated in this form, or datasets too extensive to enter as individual User Comments (expression profiling studies, systematic evaluation of gene models by RT-PCR, etc), please contact support@PlasmoDB.org.
<br><br>
Best regards,
<br>
Your ApiDB project team
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem69"/>
    <a name="Tutorials07October2007"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                Tutorials offered now for more platforms <i class="date">7 October 2007</i>
                
            </h1>
            Please check the PlasmoDB <a href="http://plasmodb.org/plasmo/showXmlDataContent.do?name=XmlQuestions.Tutorials">tutorials</a> offered now in three formats: QuickTime (.mov), Microsoft Windows (.wmv) and Flash Player (.flv). There is a link to these tutorials in PlasmoDB sidebar, in all pages.
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem70"/>
    <a name="Release 5.3"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 5.3 is released <i class="date">19 June 2007</i>
                
            </h1>
            PlasmoDB 5.3 provides the genome and annotation for the primate malarial parasite <i>Plasmodium knowlesi</i>, provided by the Wellcome Trust Sanger Institute Pathogen Sequencing Unit. This brings the number of Plasmodium genomes available for queries at PlasmoDB up to six. Also provided are expression data from recent papers using gene knockouts to study molecular mechanisms in human (<i>P. falciparum</i>) and rodent (<i>P. berghei</i>) malarial parasites. Queries have been developed for the new expression studies and a summary graphic (P.f.) or table (P.b.) added to the gene pages.
<p>
The PlasmoDB 5.3 web site also has significant improvements to the query and results pages. Many of the queries have been reorganized. For example, the Predicted Functional Interaction query now allows input of a list of genes and a clearer explanation of what to get from the query (e.g., find high confidence predicted interactions between members of the input gene list). The results from this and all queries can be sorted. It is also now possible to add columns of interest (e.g., protein features, GO annotation, even expression characteristics) and sort on them.
<p>
New data and queries in PlasmoDB 5.3:
<ul>
<li>Genome sequence and annotation for <i>Plasmodium knowlesi</i> from the Wellcome Trust Sanger Institute Pathogen Sequencing Unit.  <i>P. knowlesi</i> is included in many of the queries to identify genes, genomics sequences, ESTs, and ORFs by various features.
<li>Microarray studies have been incorporated from a series of 3 papers on gene silencing (Duraisingh et al. Cell 2005) and invasion pathways (Baum et al. PLoS Pathogens 2005; Stubbs et al. Science 2005) in <i>P. falciparum</i>.
<li>A microarray study in <i>P. berghei</i> from a paper on translational repression in gametocytes (Mair et al. Science 2006).
<li>A refined mapping of epitopes to <i>Plasmodium</i> provided by the <a href="http://www.immuneepitope.org/home.do">Immune Epitope Database and Analysis Resource</a>. A query is now provided to identify genes whose encoding protein has an epitope at different levels of confidence.
<li>Updated E.C. number and metabolic pathway assignments, Interpro protein domains and associated GO predictions, MR4 reagent links, homologies to entries in the NRDB protein database and links to structures in PDB.
<li>A new build of ortholog groups from Orthomcl for use in PlasmoDB. The list of genomes represented has increased from 55 to 87  and includes all the organisms covered by the different NIH/NIAID funded Bioinformatics Resource Centers.
</ul>
<p>
New features in PlasmoDB 5.3:
<ul>
<li>Most of the queries have been reorganized to make them more intuitive. Some include a new "Advanced Parameters" section. See for example "Identify Genes based on EST evidence."
<li>The columns displayed in query result pages are improved.  The result pages also offer options for sorting, adding new columns, and moving columns around.
<li>Downloading results is improved. A choice of formats is provided including tab-delimited, text, GFF, and FASTA to meet different needs.
  <ul>
  <li>Tab delimited provides an expanded list of annotations that can be directly exported as an Excel spreadsheet.
   <li>Text provides a detailed summary for each gene including SNPs, Y2H interactions, metabolic pathways, MR4 reagents, protein structures, notes, sequence, and many other choices.
   <li> GFF is a format recognized by many applications such as the genome browser, gbrowse.
  <li> FASTA also provides a common format used for BLAST and other applications. The FASTA option provides the functionality of the SRT (sequence retrieval tool) found in earlier versions of PlasmoDB.
  </ul>
</ul>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem71"/>
    <a name="pfgrc_course"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PFGRC offers course on Microarray Tech., April 2007 <i class="date">14 February 2007</i>
                
            </h1>
            February, 2007  The PFGRC announces that it will conduct a four day course titled, "Introduction to Microarray Technology", April 16-19, 2007.
<p>
The NIAID sponsored Pathogen Functional Genomics Resource Center at TIGR announces that it will conduct a four day course titled, "Introduction to Microarray Technology." The workshop will be held on the TIGR campus on April 16-19, 2007. The course will provide an overview of the two-dye spotted microarray platform. The course is designed for hands on participation and attendees will actively learn to make the most of their microarrays through a combination of wet lab and bioinformatics sessions.
<p>
The wet lab portion will be taught by PFGRC staff who are directly involved in microarray production. Participants will be guided through probe preparation and hybridization protocols. The training will also consist of detailed discussions on dye coupling, slide pre-hybridization, hybridization, and scanning techniques. Other topics for discussion will include the importance of assessing source material and probe quality, comparisons of various slide surfaces, and printing techniques.
<p>
The bioinformatics portion of the course covers an introduction to the stages of microarray data analysis. Major topics include data management, image analysis, normalization, and data mining. Fundamental data analysis concepts will be taught using a mix of lectures and interactive hands-on sessions. Participants will analyze the microarrays they prepared, using the open-source TM4 software suite while working closely with the software developers. CDs containing microarray datasets and the TM4 suite will be provided.
<p>
The course will be provided free of charge. Attendees will be responsible for their travel and lodging. This introductory course is one of several offered by the PFGRC on variety of functional genomics topics. Attendance will be limited to 14 participants.
<p>
Interested participants will find the preliminary course agenda <a href="http://pfgrc.tigr.org/seminars/schedule_IMT_042007.pdf">here</a>, and registration information <a href="http://pfgrc.tigr.org/seminars/registration_IMT_042007.pdf">here</a>.
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem72"/>
    <a name="workshop2"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                ApiDB Workshop, June 3-6, 2007 <i class="date">5 December 2006</i>
                
            </h1>
            The second annual <a href="http://apidb.org/workshop/2007/">ApiDB Workshop</a> will be held June 3-6, 2007
at the University of Georgia, Athens, GA, USA.
<p>The workshop will be a 3.5 day introduction to apicomplexan database resources that are part of the 
<a href="http://www.brc-central.org/cgi-bin/brc-central/brc_central.cgi">ApiDB Bioinformatics Resource Center</a> (
<a href="http://apidb.org">ApiDB.org</a>, 
<a href="http://plasmodb.org">PlasmoDB</a>, 
<a href="http://toxodb.org">ToxoDB</a>, and
<a href="http://cryptodb.org">CryptoDB</a> 
). 
<p>The application deadline is January 19, 2007 (<a href="http://apidb.org/workshop/2007/register.php">Application</a>).
Please <a href="http://apidb.org/workshop/2007/">click here</a> for further
information and to register.
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem73"/>
    <a name="5.2_release"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 5.2 is released <i class="date">17 November 2006</i>
                
            </h1>
            PlasmoDB 5.2 brings a new emphasis on genetic markers, in the form of four new datasets relating to <i>Plasmodium</i> polymorphisms.  This data is available in the Genome Browser, on the Gene page and in queries.  We continue to move key data from the PlasmoDB 4.4 site to the current site: this release re-introduces gametocyte expression data and expression profile similarity search. The Gene page now includes matches to Interpro protein domains and the Genome Browser offers tracks ESTs from dbEST.
<p>
The PlasmoDB 5.2 site also includes new features such as allowing you to keep your Query History for later use, an improved keyword search and a significantly upgraded Query & Tools page.
<p>

New data and queries in PlasmoDB 5.2:
<ul>
<li> Array data from <i>Kidgell et al. PLoS Pathogens 2006 A systematic map  of genetic variation in Plasmodium falciparum</i>.
<li> Polymorphism data based on resequencing and array analyses.
<ul>
    <li> SNPs generated by the NIH:  Based on CDS sequences from strains 3D7, 7G8, D10, Dd2,  HB3 (kindly provided in advance of publication by X. Su; reference:  J. Mu et al, Nature Genetics, in press)
    <li> SNPs generated by the Broad Institute:  Based on whole genome sequence for strains Dd2 and  HB3, and partial genome sequence for multiple strains (kindly  provided in advance of publication by D. Decaprio, S.K. Volkman &  D.F. Wirth; reference: S.K. Volkman et al, Nature Genetics, in press)
    <li> SNPs generated by the WTSI:  Based on whole genome sequence for strains Ghana1 and IT and a comparison with <i>P. reichenowi</i> (kindly provided in advance of publication by D.C. Jeffares & M.  Berriman; reference: D.C. Jeffares et al, Nature Genetics, in press).
    <li> Array CGH data on a variety of <i>Plasmodium</i> strains kindly provided by E. Winzeler.
</ul>
<li> A query to generate gene lists based on SNP characteristics such as non-synonymous / synonymous substitutions and SNP density.
<li> <i>Plasmodium</i> ESTs from the Oct. 2, 2006 release of dbEST.
<li> Interproscan analysis of <i>Plasmodium</i> proteins.
<li> PlasmoCyc pathways, available from gene pages
</ul>

New features in PlasmoDB 5.2:
<ul>
<li> Save past searches (Persistent Query History)
<li> Reorganized Query and Tools page
<li> Improved text search
<li> New tutorial movies
</ul>

Features and data moved from PlasmoDB 4.4 to 5.2:
<ul>
<li> Gametocyte data from <i>Young et al. Mol Biochem parasitol 2005.</i>
<li> Expression profile similarity for erythrocytic stage data from <i>P. falciparum</i> 3D7, HB3, and DD2 (<i>Llinas et al., Nucl. Acids. Res. 2006</i>;  <i>Bozdech et al PLos Biol 2003</i>). New to 5.2 is the inclusion of <i>P. berghei</i> data (<i>Hall et al. Science 2005</i>).          
</ul>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem74"/>
    <a name="5.1_release"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 5.1 is released <i class="date">16 August 2006</i>
                
            </h1>
            The PlasmoDB 5.1 release continues our effort to add new data and features while migrating PlasmoDB 4.4 functionality to the PlasmoDB 5 architecture.  This release includes new rodent malaria parasite data, improved mappings to external sites, improvements to the Genome Browser, more 4.4 functionality (we're almost done with the migration) and some important behind the scenes improvements.

Here are the details:
  <ul>
  <li> Rodent malaria parasite (RMP) data
     <ul>
     <li> For <i>P. berghei</i> and <i>P. chabaudi</i>
        <ul> 
        <li> Genome Browser tracks showing synteny to <i>P. falciparum</i>
        <li> TIGR Gene Index (TGI) tracks in the Genome Browser
        <li> Pfam domains mapped to predicted protein products
        </ul>
     <li> A Genome Browser track showing glass slide array oligos mapped to <i>P. yoelii</i>
     </ul>
  <li> Updated mappings from PlasmoDB genes to
        <ul>
        <li>The Protein Data Bank (PDB)
        <li>Genbank, RefSeq and Swissprot
        <li>Reagents from the MR4 Consortium
        </ul>
  <li> tRNA gene predictions for <i>P. vivax</i>, <i>P. berghei</i>, <i>P. chabaudi</i> and <i>P. yoelii</i>.
  <li> Migrated functionality PlasmoDB 4.4
     <ul>
     <li> Array element pages that describe the details of oligos and their expression.
     <li> A protein secondary structure query and track on the gene page
     <li> Links from PlasmoDB genes to predicted protein structure models
     <li> The "Add a Comment" facility (for commenting on genes and chromosomes/contigs).  
     <li> A newly designed User Login process.  (In upcoming releases logging in to the site will let you store your query history and set preferences.  For now, it just lets you Add a Comment).
     </ul>
  <li> Genome Browser features
     <ul>
     <li> Save your favorite track configurations and regions of interest.
     <li> Download the sequence of any feature in any track
     </ul>     
  <li> Improved citations on the Data Sources page
  <li> Behind the scenes, a new issue tracking system so we can better respond to your support issues.
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem75"/>
    <a name="5.0_release"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 5.0 is fully released (PlasmoDB 4.4 still available) <i class="date">25 May 2006</i>
                
            </h1>
            We are pleased to announce the first full release of the Plasmodium Genome Database, Version 5.0, now as www.plasmodb.org.  (Many of you have had a chance to explore the beta-release version as v5-0.plasmodb.org).  
<p>
PlasmoDB 5.0 implements significant 'under-the-hood' changes, making it much easier to generate future releases in short cycles.  In addition, these changes to ensure a consistent look for all <a href="http://www.apidb.org">ApiDB</a> projects, including PlasmoDB, <a href="http://www.cryptodb.org">CryptoDB</a>, and <a href="http://www.toxodb.org">ToxoDB</a> (supported by a Bioinformatics Resource Center contract from the US NIH / NIAID).
<p>

Because it will take some time to implement all of the PlasmoDB 4.4 functionality in the new architecture, <a href="http://v4-4.plasmodb.org">PlasmoDB 4.4</a> remains active, and the results of queries run under it may be exported to PlasmoDB 5.0 (see <a href="http://v4-4.plasmodb.org/plasmodb/servlet/sv?page=history">PlasmoDB 4.4 Query History</a>).
<p>

New Data in PlasmoDB 5.0 (relative to version 4.4) include:
<ul>
  <li>the complete genome sequence for P. vivax Salvador, along with a complete set of annotated gene models

  <li>the complete genome sequence for P. berghei, along with a complete set of annotated gene models

  <li>the complete genome sequence for P. chabaudi, along with a complete set of annotated gene models

  <li>TIGR Gene Indices for <i>P. vivax</i>, <i>P. falciparum</i>, and <i>P. yoelii</i>

  <li>protein-protein interaction data, based on a genome-wide yeast 2-hybrid analysis (LaCount et al. Nature 438:103; 2005)

  <li>new ortholog analyses, using the OrthoMCL algorithm to group genes from several Plasmodium species with other complete eukaryotic and prokaryotic genomes (Chen et al., Nucl. Acids Res. 34:D363; 2006)

  <li>Computational predictions for functional interactions in <i>P. falciparum</i> from the PlasmoMAP project are provided based on co-evolution and co-expression of genes (Date & Stoeckert Genome Res. 16:542; 2006). This data is now available as a PlasmoDB query.

  <li>Updated mappings for pathways and Enzyme Classifications (EC).
</ul>
<p>

New Features in PlasmoDB 5.0 Include:
<ul>
 <li>an interactive genome browser, based on the GBrowse module of the GMOD project
 <li>syntenic relationships between <i>P. falciparum</i>, <i>P. vivax</i>, and <i>P. yoelii</i> (Carlton et al., Nature 419:512; 2002).
</ul>
<p>

     Finally, we are pleased to report a new CD-ROM version of the Plasmodium Genome Database.  PlasmoCD provides users lacking reliable high-speed internet connections with full access to the complete genomes of <i>P. falciparum</i>, <i>P. vivax</i>, and <i>P. yoelii</i>, along with gene models, annotation, BLAST hits, ortholog results, expression data, and search functions for querying and integrating these results across species.
<p>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem76"/>
    <a name="berghei_and_chabaudi"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                P. berghei and P. chabaudi genomes added <i class="date">4 April 2006</i>
                
            </h1>
            The <i>P. berghei</i> and <i>P. chabaudi</i> genomes are now incorporated in PlasmoDB 5.0 Beta.  The <i>P. berghei</i> genome is accompanied by transcript and protein expression data.  We have also continued to make progress migrating functionality from PlasmoDB 4.4 to PlasmoDB 5.0 Beta
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem77"/>
    <a name="plasmoMap_included"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoMAP data included <i class="date">4 April 2006</i>
                
            </h1>
            The PlasmoMAP project provides computationally generated functional interactions between <i>P. falciparum</i> proteins. This data is now available as a PlasmoDB query.
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem78"/>
    <a name="5.0_announced"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 5.0 Beta announced <i class="date">1 February 2006</i>
                
            </h1>
            We are pleased to announce to the Plasmodium community Version 5.0 Beta of the Plasmodium Genome Database.  PlasmoDB 5.0 Beta had an early release at the Vivax Malaria Research conference at TIGR on December 9, 2005, and is now available for community use.  Please see <a href="http://plasmodb.org/plasmo/showXmlDataContent.do?name=XmlQuestions.News#initialRelease" class="blue">PlasmoDB 5.0 Beta released</a>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem79"/>
    <a name="initialRelease"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 5.0 Beta released <i class="date">9 December 2005</i>
                
            </h1>
            We are pleased to announce Version 5.0 Beta of the Plasmodium Genome Database. We have incorporated into PlasmoDB the new release of the <i>Plasmodium vivax</i> genome and annotation generated by Jane Carlton's group and the sequencing center at The Institute for Genome Research. We have also used this release to change the way we create the web site and underlying database that should make it easier for us to generate new PlasmoDB releases and be consistent with other ApiDB projects such as CryptoDB and ToxoDB. As a result, we have focused on providing access to <i>P. vivax</i> through the PlasmoDB 5.0 Beta site but have not completely reproduced all the functions in PlasmoDB 4.4. Therefore we have kept the PlasmoDB 4.4 site fully active for continued use. Furthermore we have provided a mechanism to take the results of a PlasmoDB 4.4 query and use it in PlasmoDB 5.0 Beta. 
<p>
PlasmoDB 5.0 Beta features:
<ul>
<li>A new interactive genome browser, based on the GBrowse module of the GMOD project, providing rapid visualization of the parasite genome and gene models, custom restriction site identification, open reading frame identification, and downloads in various formats.
<li>The complete genome sequence for P. vivax Salvador, along with a complete set of annotated gene models, and syntenic relationships between <i>P. falciparum, P. vivax</i>, and <i>P. yoelii</i> (from Jane Carlton, TIGR).
<li>Updated <i>P. falciparum</i> annotation, including curated literature citations; updated GO term curation from Andy Berry, Matt Berriman, and the Sanger curation effort; updated EC/pathway mappings from Hagai Ginsburg.
<li>TIGR Gene Indices for <i>P. vivax, P. falciparum, and P.yoelii</i>.
<li>Corrected expression profiles from glass-slide arrays for <i>P. falciparum</i> 3D7, HB3 & Dd2 (from Manuel Llinas, Zybnek Bozdech & Joe DeRisi).
<li>Updated structural data, including information from the structural genomics consortia.
<li>Protein-protein interaction data for <i>P. falciparum</i>, based on a genome-wide yeast 2-hybrid analysis (LaCount et al. Nature 438:103; 2005).
<li>New ortholog analyses, using the OrthoMCL algorithm to group genes from several <i>Plasmodium</i> species with other complete eukaryotic and prokaryotic genomes (Chen et al. Nucl Acids Res, in press).
</ul><p>
Finally, we are pleased to report a new CD-ROM version of the Plasmodium Genome Database.  PlasmoCD provides users lacking reliable high-speed internet connections with full access to the complete genomes of <i>P. falciparum, P. vivax</i>, and <i>P. yoelii</i>, along with gene models, annotation, BLAST hits, ortholog results, expression data, and search functions for querying and integrating these results across species. PlasmoCD 4.1 can be downloaded from the PlasmoDB web site, or requested by sending e-mail to &lt;help@PlasmoDB.org&gt;.
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem80"/>
    <a name="gbrowse"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                GBrowse added to PlasmoDB <i class="date">9 December 2005</i>
                
            </h1>
            GBrowse is an interactive generic genome browser developed as part of the GMOD project (<a href="http://www.gmod.org">www.gmod.org</a>). GBrowse is used in PlasmoDB to display gene models, EST alignments, and synteny relationships between Plasmodium species. With GBrowse, we now provide:
<ul><li>Custom restriction site identification
<li>Open reading frame identification
<li>Custom downloads in various formats:
<ul><li>FASTA
<li>GFF (2)
<li>GFF3
<li>BSML (XML)
<li>EMBL
<li>GenBank
<li>GAME
<li>GCG
</ul>
</ul>
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem81"/>
    <a name="gusAndWdk"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                PlasmoDB 5.0 Beta uses GUS 3.5 and the WDK <i class="date">9 December 2005</i>
                
            </h1>
            PlamoDB 5.0 Beta has been upgraded to  use the latest Genomics Unified Schema (GUS) database system and the new Web Development Kit (WDK). GUS is an open source project (<a href="http://www.gusdb.org">www.gusdb.org</a>) providing a functional genomics database schema covering sequence annotation, expression data, regulatory interactions, and proteomics using controlled vocabularies and ontologies. GUS also provides software to load data (plugins) and set up a web site (WDK). GUS has been adopted by several groups covering organism ranging from bacteria, protozoa, plants, and mammals.
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem82"/>
    <a name="orthoMCL"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                OrthoMCL orthologous groups have been updated. <i class="date">9 December 2005</i>
                
            </h1>
            OrthoMCL is an algorithm developed to identify orthologs and recent paralogs based on whole genome comparison of protein sequences (Li et al Genome Res. 13:2178; 2003). Previously, OrthoMCL groups were based on comparison of only 16 species in PlasmoDB 4.4. In the current release, OrthoMCL groups were generated with 61 species. These species include the 55 used to generate orthologs groups at the <a href="http://orthomcl.cbil.upenn.edu/cgi-bin/OrthoMclWeb.cgi">OrthoMCL DB site</a> plus <i>P. vivax, Leishmania major, T. brucei, T. cruzi, Apis mellifera</i> (honeybee) and <i>S. purpatus</i> (sea urchin).  The OrthoMCL DB site is described in Feng et al. Nucleic Acids Res. 2006 Database Issue.
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem83"/>
    <a name="y2h"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                Yeast Two-hybrid protein interaction data for <i>P. falciparum</i> <i class="date">9 December 2005</i>
                
            </h1>
            Yeast two-hybrid data for <i>P. falciparum</i> described in LaCount et al. Nature 438:103; 2005 has been provided by Doug LaCount (Purdue University) for inclusion into PlasmoDB. Interaction data can be seen on individual gene pages and used to search for interactors of a specified <i>P. falciparum</i> Gene ID.
          </article>
        </td>
      </tr>

    </table>
    
  



   
  

  
  
  

  
  
    <a name="newsItem84"/>
    <a name="ielRecalc"/>
    <table id="news">
      <tr><td colspan="2"><hr></td></tr>

      <tr class="rowLight">
        <td>
          <article>
            <h1>
              
                Intraerythocytic expression profiles have been recalculated <i class="date">15 September 2005</i>
                
            </h1>
            The expression profiles obtained from Joe DeRisi's group (UCSF) are processed by the PlasmoDB team for generating the images on the expression pages and for use in the expression profile queries. The original HB3 data were processed without background subtraction while the more recent 3D7 and DD2 data were processed with background subtraction. These have now all been processed consistently in the following manner.<p>
Raw data were loess normalized using the default loess implementation in the R package for microarray analysis (marray).  The non-background corrected foreground mean was used as input to the loess algorithm. Replicate arrays for the same time point were then averaged to give one value for each timepoint/gene. Data was then smoothed using a least squares fit to each data point and its two neighbors on each side.<p>
The profiles seen in PlasmoDB 4.4 have also been corrected. The correlation table that gives correlations between the different studies (the three DeRisi and the two Winzeler) has been recalculated based on the new profiles.  This new data is not yet displayed on 4.4 but will be on 5.0 Beta.
          </article>
        </td>
      </tr>

    </table>
    
  



<br>

<table width='100%'>
<tr><td>

</td><td align="right">
<a href="showXmlDataContent.do?name=XmlQuestions.NewsRss">
  <img border="0"  src="/plasmo.aurreco/images/feed-icon16x16-0ee254a56334189fd471afeec067186f.png"  alt="" />
<font size='-2' color='black'>RSS</font></a>
</td></tr>
</table>

<!-- main body end -->
