---
permalink: /glossary
tags: [general]
title: VEuPathDB glossary
---
<h1>VEuPathDB Glossary</h1>

<div class="static-content">

<p>Please check the <a href="https://www.genome.gov/genetics-glossary" target="_blank">NCBI glossary</a></p>

    <a name="3-Frame translation (forward)">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=3-Frame translation (forward)"><font color='black'><b>3-Frame translation (forward)</b></font></a> <br><br><a href="#Translation">Translation</a> of a nucleotide sequence in all three possible reading frames in one direction, usually "on the top <a href="#Strand">strand</a>" of DNA.</td></tr></table>
    

  
  

    <a name="3-Frame translation (reverse)">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=3-Frame translation (reverse)"><font color='black'><b>3-Frame translation (reverse)</b></font></a> <br><br><a href="#Translation">Translation</a> of a nucleotide sequence in all three possible reading frames in the reverse direction, usually "on the bottom <a href="#Strand">strand</a>" of DNA.</td></tr></table>
    

  
  

    <a name="AA sequence">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=AA sequence"><font color='black'><b>AA sequence</b></font></a> <br><br>Amino acid sequence.</td></tr></table>
    

  
  

    <a name="Affymetrix genotyped SNP probes">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Affymetrix genotyped SNP probes"><font color='black'><b>Affymetrix genotyped SNP probes</b></font></a> <br><br>Probes on Affymetrix <a href="#SNP">SNP</a> (single nucleotide polymorphism) arrays, which are used for SNP <a href="#Genotyping">genotyping</a>. See <a href="#Affymetrix microarray technology">Affymetrix microarray technology</a> and <a href="http://www.affymetrix.com">www.affymetrix.com</a></td></tr></table>
    

  
  

    <a name="Affymetrix microarray technology">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Affymetrix microarray technology"><font color='black'><b>Affymetrix microarray technology</b></font></a> <br><br><a href="#Microarray">Microarray</a> manufacturing technology developed by Affymetrix. Combines semiconductor fabrication techniques, solid phase chemistry, combinatorial chemistry, molecular biology, and robotics to generate a photolithographic manufacturing process in which oligonucleotides are synthesized directly on a chip. See <a href="http://www.affymetrix.com">www.affymetrix.com</a></td></tr></table>
    

  
  

    <a name="Affymetrix probes">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Affymetrix probes"><font color='black'><b>Affymetrix probes</b></font></a> <br><br>Probe on an Affymetrix <a href="#Microarray">microarray</a> designed to determine whether or not the complementary sequence of RNA or DNA is present in a sample. Generally 25 nucleotides in length (25-mers), their short length provides higher specificity than longer probes. See <a href="#Affymetrix microarray technology">Affymetrix microarray technology</a> and <a href="http://www.affymetrix.com">www.affymetrix.com</a></td></tr></table>
    

  
  

    <a name="Amitochondriate">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Amitochondriate"><font color='black'><b>Amitochondriate</b></font></a> <br><br>Eukaryotic organism that lacks a <a href="#Mitochondrion">mitochondrion</a>. Examples include Giardia and other parasites such as Trachipleistophora and Entamoeba. However, most of these organisms contain what appear to be mitochondrial remnants as well as mitochondrial genes in their nuclear genomes.</td></tr></table>
    

  
  

    <a name="Annotation">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Annotation"><font color='black'><b>Annotation</b></font></a> <br><br>Identified feature within a sequence, such as a known or predicted gene, domain, motif, post-translational modification, etc.</td></tr></table>
    

  
  

    <a name="Annotation density">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Annotation density"><font color='black'><b>Annotation density</b></font></a> <br><br>Level to which a nucleotide or protein sequence has been annotated. See <a href="#Annotation">Annotation</a>.</td></tr></table>
    

  
  

    <a name="Apicoplast">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Apicoplast"><font color='black'><b>Apicoplast</b></font></a> <br><br>Nonphotosynthetic plastid found in almost all protozoan parasites belonging to the phylum Apicomplexa that have been examined. The apicoplast is surrounded by four membranes, giving rise to the theory that its presence in the Apicomplexa is the result of a secondary endosymbiosis (acquired by the engulfment of an ancestral alga and retention of the algal plastid). Similar to other endosymbiotic organelles (mitochondria, chloroplasts), the apicoplast contains its own genome as well as proteins that are encoded in the nucleus and post-translationally imported. The apicoplast is a vital organelle to the parasite's long-term survival.</td></tr></table>
    

  
  

    <a name="ApiCyc">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=ApiCyc"><font color='black'><b>ApiCyc</b></font></a> <br><br>Database/utility on VEuPathDB used for searching and visualizing metabolic pathway information for organisms in VEuPathDB; derivative database generated by analyzing various genomes (for example from Plasmodium, Cryptosporidium, and Toxoplasma) with SRI International's pathway tools.</td></tr></table>
    

  
  

    <a name="ApiDots alignments">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=ApiDots alignments"><font color='black'><b>ApiDots alignments</b></font></a> <br><br>Consensus sequences found in the ApiDots database and generated by clustering and assembling Apicomplexan mRNA and <a href="#EST">EST</a> sequences. These consensus sequences were subjected to database searches against protein and protein domain sequences.</td></tr></table>
    

  
  

    <a name="Attribute">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Attribute"><font color='black'><b>Attribute</b></font></a> <br><br>Inherent characteristic or feature; in a database, a data item related to a database object. For example, attributes of genes can include features such as introns and untranslated regions (<a href="#UTR">UTRs</a>).</td></tr></table>
    

  
  

    <a name="BLAST">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=BLAST"><font color='black'><b>BLAST</b></font></a> <br><br>Basic local alignment search tool, a <a href="#Sequence similarity">sequence similarity</a> search tool used to quickly find local alignments between a <a href="#Query">query</a> sequence and sequences in nucleotide or protein databases. Different versions of this search tool are available to match the types of query sequence and database used. See <a href="#blastn">blastn</a>, <a href="#blastp">blastp</a>, <a href="#blastx">blastx</a>, <a href="#tblastn">tblastn</a>, and <a href="#tblastx">tblastx</a>.</td></tr></table>
    

  
  

    <a name="blastn">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=blastn"><font color='black'><b>blastn</b></font></a> <br><br>Version of the basic local alignment search tool (<a href="#BLAST">BLAST</a>) used to compare a nucleotide <a href="#Query">query</a> sequence against a nucleotide sequence database.</td></tr></table>
    

  
  

    <a name="blastp">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=blastp"><font color='black'><b>blastp</b></font></a> <br><br>Version of the basic local alignment search tool (<a href="#BLAST">BLAST</a>) used to compare a protein <a href="#Query">query</a> sequence against a protein sequence database.</td></tr></table>
    

  
  

    <a name="blastx">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=blastx"><font color='black'><b>blastx</b></font></a> <br><br>Version of the basic local alignment search tool (<a href="#BLAST">BLAST</a>) used to compare a nucleotide <a href="#Query">query</a> sequence translated in all reading frames against a protein sequence database.</td></tr></table>
    

  
  

    <a name="Boolean">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Boolean"><font color='black'><b>Boolean</b></font></a> <br><br>System of logical thought developed by George Boole (1815-1864). In Boolean searching, an "and" operator between two words or values (for example, "apple AND orange") generates a search for items in a database containing both of the words or values. Similarly, an "or" operator between two words or values (for example, "apple OR orange") generates a search for items containing either word.</td></tr></table>
    

  
  

    <a name="cDNA">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=cDNA"><font color='black'><b>cDNA</b></font></a> <br><br>Complementary DNA. DNA molecule synthesized by the enzyme reverse transcriptase using an mRNA as template.</td></tr></table>
    

  
  

    <a name="cDNA microarray">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=cDNA microarray"><font color='black'><b>cDNA microarray</b></font></a> <br><br>Collection of microscopic <a href="#cDNA">cDNA</a> spots commonly representing single genes and arrayed on a solid surface (commonly glass slides) by covalent attachment to chemically suitable matrices. Used for expression profiling, the monitoring of the <a href="#Expression level">expression levels</a> of hundreds or thousands of genes simultaneously.</td></tr></table>
    

  
  

    <a name="CDS">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=CDS"><font color='black'><b>CDS</b></font></a> <br><br>Coding sequence. Region of nucleotides that corresponds to the sequence of amino acids in a predicted protein and that includes start and stop codons. Unexpressed sequences (for example, the 5'-<a href="#UTR">UTR</a>, the 3'-UTR, and introns) are not included within a CDS. The CDS usually does not correspond to the actual mRNA sequence.</td></tr></table>
    

  
  

    <a name="Centromere">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Centromere"><font color='black'><b>Centromere</b></font></a> <br><br>Region of the <a href="#Chromosome">chromosome</a> or chromosomal structure essential for division and retention of the chromosome within the cell; point of a chromosome where the spindle fibers attach to pull the chromosome apart during cell division.</td></tr></table>
    

  
  

    <a name="Chromosome">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Chromosome"><font color='black'><b>Chromosome</b></font></a> <br><br>Macromolecule of DNA constituting the physical organization of DNA in a cell.</td></tr></table>
    

  
  

    <a name="Coil">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Coil"><font color='black'><b>Coil</b></font></a> <br><br>Three-dimensional spiral structure in protein macromolecules.</td></tr></table>
    

  
  

    <a name="Contig">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Contig"><font color='black'><b>Contig</b></font></a> <br><br>Contiguous genomic sequence assembled from overlapping primary sequences representing overlapping regions of a particular <a href="#Chromosome">chromosome</a>.</td></tr></table>
    

  
  

    <a name="CryptoCyc">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=CryptoCyc"><font color='black'><b>CryptoCyc</b></font></a> <br><br>Database/utility built by analyzing the Cryptosporidium genome with SRI International's pathway tools; used for searching and visualizing Cryptosporidium metabolic pathway information.</td></tr></table>
    

  
  

    <a name="Curated annotation">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Curated annotation"><font color='black'><b>Curated annotation</b></font></a> <br><br><a href="#Annotation">Annotation</a> made under the supervision of a curator as opposed to a purely computational prediction. Curated predictions often contain combinations of different types of evidence to support the annotation.</td></tr></table>
    

  
  

    <a name="Dalton">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Dalton"><font color='black'><b>Dalton</b></font></a> <br><br>Unit of mass abbreviated Da and used to express atomic and molecular masses.</td></tr></table>
    

  
  

    <a name="Deprecated gene">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Deprecated gene"><font color='black'><b>Deprecated gene</b></font></a> <br><br>Genes with little or no evidence (similarities / expression) that overlapped (or were subsumed by) larger genes for which there was evidence such as protein similarities, expression evidence from <a href="#EST">EST</a> alignments, SAGE or <a href="#Proteomics">proteomics</a> data were marked as deprecated. These genes will likely be removed in a subsequent release (and in GenBank) unless additional evidence is provided indicating they should be moved into the real gene category.</td></tr></table>
    

  
  

    <a name="DNA/GC content">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=DNA/GC content"><font color='black'><b>DNA/GC content</b></font></a> <br><br>Content of guanine (G) and cytosine (C) in a fragment of DNA or a genome. Because GC pairs are more thermostable compared to the AT pairs, it was commonly believed that GC content played a vital part in adaptation to high temperatures, a hypothesis that has been refuted. In the genome browser, the DNA/GC content track displays a GC content graph of the reference sequence at low magnifications and the DNA sequence itself at higher magnifications.</td></tr></table>
    

  
  

    <a name="EC numbers">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=EC numbers"><font color='black'><b>EC numbers</b></font></a> <br><br>Enzyme Commission numbers. EC numbers constitute the numerical classification scheme for enzymes based on the chemical reactions they catalyze. EC numbers do not refer to the enzymes, but to the reactions they catalyze.</td></tr></table>
    

  
  

    <a name="ePCR">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=ePCR"><font color='black'><b>ePCR</b></font></a> <br><br>Electronic PCR (polymerase chain reaction). Computational procedure used to check for uniqueness in spacing and number of primer binding sites within DNA sequences. Searches for subsequences that closely match a set of PCR primers and that have the correct order, orientation, and spacing to make a PCR product. Used to check the expected length of a PCR product, which can provide information regarding unexpected repetitive sequences.</td></tr></table>
    

  
  

    <a name="EST">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=EST"><font color='black'><b>EST</b></font></a> <br><br>Expressed sequence tag. Short (typically 100-500 base pairs) partial <a href="#cDNA">cDNA</a> produced by single-shot sequencing of a cloned mRNA (cDNA) and often used to identify gene transcripts.</td></tr></table>
    

  
  

    <a name="EST alignments">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=EST alignments"><font color='black'><b>EST alignments</b></font></a> <br><br>Alignments of expressed sequence tags (<a href="#EST">ESTs</a>) with a corresponding genomic region. For example, in ToxoDB you can visualize EST alignments by clicking on the "View this sequence in the genome browser" link and turning on the EST Alignments track. Useful for identifying intron boundaries.</td></tr></table>
    

  
  

    <a name="EST clusters">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=EST clusters"><font color='black'><b>EST clusters</b></font></a> <br><br>Groups of homologous, overlapping <a href="#EST">EST</a> sequences created to reduce redundancy of the EST database.</td></tr></table>
    

  
  

    <a name="Expression level">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Expression level"><font color='black'><b>Expression level</b></font></a> <br><br>Level at which an mRNA or protein is present in a sample. Value can be absolute or relative to other mRNA or protein species in the sample.</td></tr></table>
    

  
  

    <a name="Expression profile">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Expression profile"><font color='black'><b>Expression profile</b></font></a> <br><br>Pattern of expression of one or more genes or proteins over time or over a set of experimental conditions (for example, during development or treatment, or as a result of a genetic mutation such as a knockout).</td></tr></table>
    

  
  

    <a name="Expression profile correlation">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Expression profile correlation"><font color='black'><b>Expression profile correlation</b></font></a> <br><br>Method for correlation of gene <a href="#Expression profile">expression profiles</a> with gene ontology (<a href="#GO">GO</a>) <a href="#Annotation">annotations</a> developed for the purpose of identifying groups of genes, pathways, and processes reacting in concert to experimental perturbations.</td></tr></table>
    

  
  

    <a name="Expression timing">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Expression timing"><font color='black'><b>Expression timing</b></font></a> <br><br>Timing of gene expression during a developmental, metabolic, regulatory, or other biological process or response.</td></tr></table>
    

  
  

    <a name="Gametocyte">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Gametocyte"><font color='black'><b>Gametocyte</b></font></a> <br><br>Eukaryotic germ cell that divides by mitosis to generate other gametocytes or by meiosis to generate gametes. Male gametocytes are called spermatocytes, and female gametocytes are called oocytes. Term often used to describe gametocytes of Plasmodium.</td></tr></table>
    

  
  

    <a name="GBrowse">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=GBrowse"><font color='black'><b>GBrowse</b></font></a> <br><br>Interactive genome browser developed by the Generic Model Organism Database (GMOD) project (<a href="http://www.gmod.org">www.gmod.org</a>) that can be customized to show selected chromosomal features as well as display user-provided <a href="#Annotation">annotations</a>.</td></tr></table>
    

  
  

    <a name="GBrowse track">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=GBrowse track"><font color='black'><b>GBrowse track</b></font></a> <br><br>In the <a href="#GBrowse">GBrowse</a> viewer, a line of data that corresponds to a particular type of genomic information or feature and that is distinguished by a particular shape or color.</td></tr></table>
    

  
  

    <a name="GenBank protein record">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=GenBank protein record"><font color='black'><b>GenBank protein record</b></font></a> <br><br>Protein sequence file in the GenBank database generally derived by <a href="#Translation">translation</a> of a related nucleotide record.</td></tr></table>
    

  
  

    <a name="Gene">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Gene"><font color='black'><b>Gene</b></font></a> <br><br>Fundamental physical and functional unit of heredity. Ordered sequence of nucleotides located in a particular position on a particular <a href="#Chromosome">chromosome</a> that encodes a specific functional product, such as a protein or RNA molecule. A gene may have a number of parts, including the <a href="#Promoter">promoter</a> region, untranslated regions (5' and 3' <a href="#UTR">UTRs</a>), introns, and exons.</td></tr></table>
    

  
  

    <a name="GeneDB">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=GeneDB"><font color='black'><b>GeneDB</b></font></a> <br><br>Project developed by the Sanger Institute Pathogen Sequencing Unit (PSU) and aimed at developing and maintaining curated database resources for all projects handled by the PSU. The database is accessible at <a href="http://www.genedb.org">www.genedb.org</a></td></tr></table>
    

  
  

    <a name="Genetic markers">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Genetic markers"><font color='black'><b>Genetic markers</b></font></a> <br><br>Known DNA sequences that can be identified by a simple assay. Generally genetic variations caused by mutation or alterations in loci that can be observed, examples include restriction length polymorphisms (RFLPs), short tandem repeats (STRs), variable number tandem repeats (VNTRs), short DNA sequences surrounding single base-pair changes (single nucleotide polymorphisms, or <a href="#SNP">SNPs</a>), or longer <a href="#Microsatellite">microsatellite</a> sequences.</td></tr></table>
    

  
  

    <a name="Genomic context">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Genomic context"><font color='black'><b>Genomic context</b></font></a> <br><br>Location of a gene in the genome, which can influence the expression of the gene and functional interactions of the gene expression products. In this database, genes are depicted on individual gene pages with their surrounding genomic region and <a href="#Annotation">annotations</a>.</td></tr></table>
    

  
  

    <a name="Genotyped SNPs">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Genotyped SNPs"><font color='black'><b>Genotyped SNPs</b></font></a> <br><br>Single nucleotide polymorphisms (<a href="#SNP">SNPs</a>) identified during <a href="#Genotyping">genotyping</a> of individual organism strains. See <a href="#SNP genotyping">SNP genotyping</a>.</td></tr></table>
    

  
  

    <a name="Genotyping">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Genotyping"><font color='black'><b>Genotyping</b></font></a> <br><br>Process of determining the genotype of an individual with a biological assay using PCR, DNA sequencing, or hybridization to DNA <a href="#Microarray">microarrays</a> or beads. Provides a measurement of the genetic variation between members of a species.</td></tr></table>
    

  
  

    <a name="GenPept protein">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=GenPept protein"><font color='black'><b>GenPept protein</b></font></a> <br><br>Protein record from the GenPept database at the <a href="#NCBI">NCBI</a> GenBank, which contains inferred <a href="#Translation">translations</a> of <a href="#Protein-coding">protein-coding</a> sequences.</td></tr></table>
    

  
  

    <a name="GLEAN gene">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=GLEAN gene"><font color='black'><b>GLEAN gene</b></font></a> <br><br>Predicted gene sequence generated by GLEAN, an algorithm that integrates different sources of gene structure evidence (for example, gene model predictions, <a href="#EST">EST</a> and protein sequence alignments to the genome, and SAGE or peptide tags) to produce a consensus gene prediction in the absence of known genes.</td></tr></table>
    

  
  

    <a name="GO">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=GO"><font color='black'><b>GO</b></font></a> <br><br>Gene Ontology project. Collaborative project that has developed three structured, controlled vocabularies (ontologies) to describe gene products in terms of their associated biological processes, cellular components, and molecular functions in a species-independent manner. The use of a consistent vocabulary allows genes from different species to be compared based on their GO <a href="#Annotation">annotations</a>.</td></tr></table>
    

  
  

    <a name="GO component">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=GO component"><font color='black'><b>GO component</b></font></a> <br><br>Gene Ontology term used to describe a cellular component, or the location where a gene product may act, rather than physical features of proteins or RNAs. For example, membrane (<a href="#GO">GO</a>:0016020), extrinsic to membrane (GO:0019898), and integral to membrane (GO:0016021).</td></tr></table>
    

  
  

    <a name="GO function">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=GO function"><font color='black'><b>GO function</b></font></a> <br><br>Gene Ontology term used to describe the molecular function of a gene product, the jobs that it performs, or the "abilities" that it has (for example, transporting compounds, binding to things, holding things together, and changing one thing into another). This is different from the biological processes the gene product is involved in, which involve more than one activity.</td></tr></table>
    

  
  

    <a name="GO process">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=GO process"><font color='black'><b>GO process</b></font></a> <br><br>Gene Ontology term used to describe a biological process, a recognized series of events, or molecular functions associated with a gene product. A biological process is not equivalent to a pathway, though some <a href="#GO">GO</a> terms do describe pathways.</td></tr></table>
    

  
  

    <a name="GO term">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=GO term"><font color='black'><b>GO term</b></font></a> <br><br>Gene Ontology term. The building blocks of the Gene Ontology, each term is assigned to one of the three ontologies: molecular function, cellular component, or biological process. Each <a href="#GO">GO</a> term consists of a unique alphanumerical identifier, a common name, synonyms (if applicable), and a definition. When a term has multiple meanings depending on species, the GO uses a "sensu" tag to differentiate among them. For example, the enzyme fumarase has the GO term GO:0004333, fumarate hydratase activity (fumarase activity), catalysis of the reaction: (S)-malate = fumarate + H2O.</td></tr></table>
    

  
  

    <a name="GPI anchor">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=GPI anchor"><font color='black'><b>GPI anchor</b></font></a> <br><br>C-terminal post-translational modification of many eukaryotic proteins. The two fatty acids within the glycophosphatidylinositol (GPI) moiety anchor the protein to the outer leaflet of the plasma membrane. GPI-anchored proteins are believed to be involved in signal transduction and immune responses, as well as the pathobiology of many parasites.</td></tr></table>
    

  
  

    <a name="HMM">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=HMM"><font color='black'><b>HMM</b></font></a> <br><br>Hidden Markov model. Statistical model in which the system being modeled is assumed to be a Markov process with unknown parameters, and the challenge is to determine the hidden parameters from the observable parameters. The extracted model parameters can then be used to perform further analysis, for example for pattern recognition applications.</td></tr></table>
    

  
  

    <a name="Homolog">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Homolog"><font color='black'><b>Homolog</b></font></a> <br><br>Related by evolutionary descent, either between species (<a href="#Ortholog">ortholog</a>) or within a species (<a href="#Paralog">paralog</a>).</td></tr></table>
    

  
  

    <a name="Hydropathy">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Hydropathy"><font color='black'><b>Hydropathy</b></font></a> <br><br>Hydropathicity. Degree to which a peptide or protein is likely to be soluble in water. Protein hydropathy plots can be useful in predicting <a href="#Transmembrane domain">transmembrane domains</a>, potential antigenic sites, and regions that are likely to be exposed on the protein's surface.</td></tr></table>
    

  
  

    <a name="Intergenic region">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Intergenic region"><font color='black'><b>Intergenic region</b></font></a> <br><br>Stretch of DNA located between genes that may act to regulate gene expression.</td></tr></table>
    

  
  

    <a name="Intersect">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Intersect"><font color='black'><b>Intersect</b></font></a> <br><br>Similar to the <a href="#Boolean">Boolean</a> operator "AND", an action used in the <a href="#Query">query</a> history to find items that are common to two query result sets. For example, to search for items that appear in both result set X and result Y, type, "X INTERSECT Y".</td></tr></table>
    

  
  
    <a name="JBrowse">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=JBrowse"><font color='black'><b>JBrowse</b></font></a> <br><br>Interactive genome browser (<a href="https://jbrowse.org/">jbrowse.org</a>) developed by the Generic Model Organism Database (GMOD) project (<a href="http://www.gmod.org">www.gmod.org</a>) that can be customized to show selected chromosomal features as well as display user-provided <a href="#Annotation">annotations</a>.</td></tr></table>
      
    
      
      
    
    
    <a name="Join">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Join"><font color='black'><b>Join</b></font></a> <br><br>Similar to the <a href="#Boolean">Boolean</a> operator "UNION", an action used in the <a href="#Query">query</a> history to combine query sets. For example, to combine result sets X and Y, type, "X JOIN Y".</td></tr></table>
    

  
  

    <a name="KEGG map">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=KEGG map"><font color='black'><b>KEGG map</b></font></a> <br><br>Metabolic or regulatory interaction pathway generated by the Kyoto Encyclopedia of Genes and Genomes (KEGG) or by the use of their tools (<a href="http://www.genome.jp/kegg/">www.genome.jp/kegg</a>).</td></tr></table>
    

  
  

    <a name="Locus">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Locus"><font color='black'><b>Locus</b></font></a> <br><br>Position on a <a href="#Chromosome">chromosome</a> of a gene, feature (such as a <a href="#Telomere">telomere</a>), or other chromosomal marker; also, the DNA at that position. Use of this term is sometimes restricted to mean expressed DNA regions.</td></tr></table>
    

  
  

    <a name="Low complexity">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Low complexity"><font color='black'><b>Low complexity</b></font></a> <br><br>Pertaining to sequence regions that have an unusually repetitive nature (for example, a protein sequence of low complexity might look like PPTDPPPPKKDGGPPL, and a low-complexity nucleotide sequence might be AAATAAAAAAAAATAAAAAAATTA). Low-complexity regions can create problems in <a href="#Sequence similarity">sequence similarity</a> searching by causing artifactual hits. For this reason, filters are often used to remove low-complexity sequences. Low-complexity regions also contribute to antigenic variation in apicomplexan parasites.</td></tr></table>
    

  
  

    <a name="Mass spec data">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Mass spec data"><font color='black'><b>Mass spec data</b></font></a> <br><br>Mass spectrometry data. Mass spectrometry is an analytical technique used to measure the mass-to-charge ratios of small molecules in several applications, including identification of proteins or peptides. In our databases, the "Identify Genes by Mass Spec Evidence" <a href="#Query">query</a> is used to identify genes that have evidence of protein expression based on mass spec data.</td></tr></table>
    

  
  

    <a name="Metabolic pathways">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Metabolic pathways"><font color='black'><b>Metabolic pathways</b></font></a> <br><br>Series of chemical reactions occurring within a cell and often catalyzed by enzymes. In a pathway, a molecule is often changed or modified into another product, which can be stored by the cell, used as a metabolic product, or used to initiate another pathway.</td></tr></table>
    

  
  

    <a name="Microarray">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Microarray"><font color='black'><b>Microarray</b></font></a> <br><br>Microscopic array of biological molecules (for example, DNA or protein) used to determine the presence and/or amount (referred to as quantitation) of other biomolecules (other proteins, transcripts, etc.) in biological samples.</td></tr></table>
    

  
  

    <a name="Microsatellite">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Microsatellite"><font color='black'><b>Microsatellite</b></font></a> <br><br>Polymorphic <a href="#Locus">locus</a> in nuclear and organellar DNA that consists of repeating units of 1-4 base pairs in length. Mostly neutral and codominant, microsatellites are used as molecular markers and to study gene dosage (looking for duplications or deletions of a particular genetic region). Also known as simple sequence repeats (SSRs).</td></tr></table>
    

  
  

    <a name="Microsatellite map">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Microsatellite map"><font color='black'><b>Microsatellite map</b></font></a> <br><br>Map of <a href="#Microsatellite">microsatellite</a> locations and linkages on a genome.</td></tr></table>
    

  
  

    <a name="Mitochondrion">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Mitochondrion"><font color='black'><b>Mitochondrion</b></font></a> <br><br>Organelle responsible for respiration in a eukaryotic cell. Proteins required for mitochondrial function are encoded both in the nucleus and within the smaller mitochondrial genome.</td></tr></table>
    

  
  

    <a name="Motif search">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Motif search"><font color='black'><b>Motif search</b></font></a> <br><br>Tool used to identify and locate sequence patterns (motifs) in protein and nucleic acid sequences. In our databases, this flexible search can be based on the general characteristics of the pattern and not solely on specific sequences (for example, Cys-[9-11 amino acids]-Cys or Leu-Leu-[basic residue]-Val). This allows the user to <a href="#Query">query</a> using previously undescribed motifs.</td></tr></table>
    

  
  

    <a name="NCBI">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=NCBI"><font color='black'><b>NCBI</b></font></a> <br><br>National Center for Biotechnology Information. Established in 1988 as a national resource for molecular biology information, NCBI creates public databases, conducts research in computational biology, develops software tools for analyzing genome data, and disseminates biomedical information - all for the better understanding of molecular processes affecting human health and disease.</td></tr></table>
    

  
  

    <a name="ncRNA">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=ncRNA"><font color='black'><b>ncRNA</b></font></a> <br><br>Noncoding RNA. Any RNA that is not translated into a protein. Includes transfer RNA (<a href="#tRNA">tRNA</a>), ribosomal RNA (<a href="#rRNA">rRNA</a>), small RNAs such as snoRNAs, microRNAs, siRNAs and piRNAs, as well as long ncRNAs.</td></tr></table>
    

  
  

    <a name="Nonredundant protein DB (NRDB)">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Nonredundant protein DB (NRDB)"><font color='black'><b>Nonredundant protein DB (NRDB)</b></font></a> <br><br>Peptide sequence database containing all nonidentical protein sequences from the same species extracted from GenPept, <a href="#NCBI">NCBI</a> RefSeq, Swiss-Prot, and PRF databases. Used for <a href="#BLAST">BLAST</a> protein database searches because its smaller size results in shorter search times and more meaningful statistics.</td></tr></table>
    

  
  

    <a name="Nucmer">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Nucmer"><font color='black'><b>Nucmer</b></font></a> <br><br>NUCleotide MUMmer. Part of the MUMmer alignment package (<a href="www.tigr.org/software/mummer">www.tigr.org/software/mummer</a>), an alignment tool used for the rapid alignment of very large DNA and amino acid sequences.</td></tr></table>
    

  
  

    <a name="Oligo">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Oligo"><font color='black'><b>Oligo</b></font></a> <br><br>Oligonucleotide. Short sequence of DNA or RNA, typically of 20-70 nucleotides.</td></tr></table>
    

  
  

    <a name="Oligonucleotide microarray">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Oligonucleotide microarray"><font color='black'><b>Oligonucleotide microarray</b></font></a> <br><br>Collection of microscopic oligonucleotide spots arrayed on a solid surface by covalent attachment to chemically suitable matrices. Used for expression profiling, the monitoring of the <a href="#Expression level">expression levels</a> of hundreds or thousands of genes simultaneously. Probes for oligonucleotide <a href="#Microarray">microarrays</a> are designed to match parts of known or predicted mRNAs.</td></tr></table>
    

  
  

    <a name="Open Reading Frame">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Open Reading Frame"><font color='black'><b>Open Reading Frame</b></font></a> <br><br>See <a href="#ORF">ORF</a></td></tr></table>
    

  
  

    <a name="ORF">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=ORF"><font color='black'><b>ORF</b></font></a> <br><br>Amino acid sequences computed by translating the six frames of raw genomic sequence using the standard genetic code. We save the translated sequences having at least 50 amino acids. The sequences are not annotated nor human reviewed. ORF's do not necessarily begin with methionine residues, but they all terminate with stop codons.</td></tr></table>
    

  
  

    <a name="Ortholog">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Ortholog"><font color='black'><b>Ortholog</b></font></a> <br><br>Same gene in different organisms; having evolved from the same ancestral <a href="#Locus">locus</a>.</td></tr></table>
    

  
  

    <a name="Ortholog group">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Ortholog group"><font color='black'><b>Ortholog group</b></font></a> <br><br>Orthologous genes shared by group of organisms; the group of genes can also contain <a href="#Paralog">paralogs</a>.</td></tr></table>
    

  
  

    <a name="Orthology-based phylogenetic profile">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Orthology-based phylogenetic profile"><font color='black'><b>Orthology-based phylogenetic profile</b></font></a> <br><br>Tool used to find genes that are present or not present in a desired group of organisms on the tree of life (currently computed for 81 complete genomes). The user has control over the profile and over whether or not genes must be found in any particular group of organisms (for example, in Apicomplexa but not in mammals). Taxa can also be marked as indifferent (for example, it does not matter if the gene is also found in plants). <a href="#Ortholog">Ortholog</a> and <a href="#Paralog">paralog</a> relationships are determined using the <a href="#OrthoMCL">OrthoMCL</a> algorithm.</td></tr></table>
    

  
  

    <a name="OrthoMCL">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=OrthoMCL"><font color='black'><b>OrthoMCL</b></font></a> <br><br>Genome-scale algorithm for grouping orthologous protein sequences. It provides genes shared by two or more species/genomes and also genes representing species-specific gene expansion families. Therefore, it serves as a utility for automated eukaryotic genome <a href="#Annotation">annotation</a> and phylogenetic profiling. Available at <a href="http://orthomcl.org">orthomcl.org</a></td></tr></table>
    

  
  

    <a name="Paralog">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Paralog"><font color='black'><b>Paralog</b></font></a> <br><br>Related by gene duplication within a genome; originated by duplication and then diverged from the parent copy by mutation and selection or drift.</td></tr></table>
    

  
  

    <a name="PATS">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=PATS"><font color='black'><b>PATS</b></font></a> <br><br>Neural network analysis tool that identifies amino acid sequences within a <a href="#Query">query</a> sequence that are potentially targeted to the <a href="#Apicoplast">apicoplast</a> matrix of Plasmodium falciparum.</td></tr></table>
    

  
  

    <a name="PDB 3D structure">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=PDB 3D structure"><font color='black'><b>PDB 3D structure</b></font></a> <br><br>Three-dimensional macromolecular structure in the Protein Data Bank (PDB) (<a href="http://www.pdb.org">www.pdb.org</a>) obtained by one of three methods: X-ray crystallography (over 80%), solution nuclear magnetic resonance (NMR) (about 16%), or theoretical modeling (2%). A few structures were determined by other methods.</td></tr></table>
    

  
  

    <a name="Pearson correlation">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Pearson correlation"><font color='black'><b>Pearson correlation</b></font></a> <br><br>Pearson Product Moment Correlation, the most common measure of the correlation between two variables. Reflects the degree of linear relationship between two variables and ranges from +1 to -1, with a correlation of +1 indicating a perfect positive linear relationship between variables.</td></tr></table>
    

  
  

    <a name="Peptide mass fingerprinting">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Peptide mass fingerprinting"><font color='black'><b>Peptide mass fingerprinting</b></font></a> <br><br>Analytical technique for protein identification wherein an unknown protein of interest is cleaved into peptides by a protease such as trypsin, and the peptides resulting from this cleavage are analyzed using a mass spectrometric method such as MALDI-TOF or ESI-TOF. The masses derived for the peptides are then compared to a database containing known protein sequences or even to the genome. Computer programs theoretically cut the protein sequences in the database into peptides with the same protease (for example trypsin), and calculate the absolute masses of the peptides from each protein. They then compare the masses of the peptides of the unknown protein to the theoretical peptide masses of each protein encoded in the genome. The results are statistically analyzed to find the best match.</td></tr></table>
    

  
  

    <a name="Pfam domain">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Pfam domain"><font color='black'><b>Pfam domain</b></font></a> <br><br>Conserved protein region in the Pfam database (<a href="http://pfam.xfam.org">Pfam.org</a>), a collection of multiple sequence alignments and hidden Markov models covering many common protein families. The alignments may represent evolutionarily conserved structures that may shed light on protein function. Profile hidden Markov models (profile <a href="#HMM">HMMs</a>) built from the Pfam alignments can be useful for associating a new protein to a known protein family, even if the homology is weak. Unlike standard pairwise alignment methods (for example, <a href="#BLAST">BLAST</a> and FASTA), Pfam HMMs deal sensibly with multidomain proteins.</td></tr></table>
    

  
  

    <a name="Phylogeny">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Phylogeny"><font color='black'><b>Phylogeny</b></font></a> <br><br>Historical relationships among lineages of organisms or their parts, including their genes.</td></tr></table>
    

  
  

    <a name="PlasmoAP">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=PlasmoAP"><font color='black'><b>PlasmoAP</b></font></a> <br><br>Algorithm/tool that predicts the likelihood that a protein sequence is targeted to the <a href="#Apicoplast">apicoplast</a>. It provides the position of <a href="#Signal peptide">signal peptide</a> cleavage sites in amino acid sequences if targeting is predicted.</td></tr></table>
    

  
  

    <a name="PlasmoCyc">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=PlasmoCyc"><font color='black'><b>PlasmoCyc</b></font></a> <br><br>Database/utility built by analyzing the genomes of the Plasmodium species in VEuPathDB with SRI International's pathway tools; used for searching and visualizing Plasmodium metabolic pathway information.</td></tr></table>
    

  
  

    <a name="ProDom">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=ProDom"><font color='black'><b>ProDom</b></font></a> <br><br>Database of protein domain families generated from the global comparison of all available protein sequences (<a href="http://prodom.prabi.fr">prodom.prabi.fr</a>).</td></tr></table>
    

  
  

    <a name="Promoter">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Promoter"><font color='black'><b>Promoter</b></font></a> <br><br>Regulatory region of DNA located upstream (towards the 5' region) of a gene and providing a control point for regulated gene transcription.</td></tr></table>
    

  
  

    <a name="PROSITE motif">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=PROSITE motif"><font color='black'><b>PROSITE motif</b></font></a> <br><br>Protein sequence pattern or profile derived from multiple alignments of homologous sequences and stored in the PROSITE database (<a href="https://prosite.expasy.org/">prosite.expasy.org</a>), an annotated collection of motif descriptors dedicated to the identification of protein families and domains.</td></tr></table>
    

  
  

    <a name="Protein-coding">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Protein-coding"><font color='black'><b>Protein-coding</b></font></a> <br><br>Capable of encoding a protein sequence; generally refers to a sequence of DNA.</td></tr></table>
    

  
  

    <a name="Proteomics">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Proteomics"><font color='black'><b>Proteomics</b></font></a> <br><br>The large-scale study of proteins, particularly of the full set of proteins encoded by a genome.</td></tr></table>
    

  
  

    <a name="Pseudogene">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Pseudogene"><font color='black'><b>Pseudogene</b></font></a> <br><br>Defunct relatives of known genes that have lost their <a href="#Protein-coding">protein-coding</a> ability or are otherwise no longer expressed in the cell. Although they may have some gene-like features (such as <a href="#Promoter">promoters</a>, CpG islands, and splice sites), they are nonetheless considered nonfunctional due to their lack of protein-coding ability resulting from various genetic disablements (stop codons, frameshifts, or a lack of transcription) or their inability to function as an RNA (such as with <a href="#rRNA">rRNA</a> pseudogenes).</td></tr></table>
    

  
  

    <a name="PubCrawler">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=PubCrawler"><font color='black'><b>PubCrawler</b></font></a> <br><br>Free service that scans daily updates to the <a href="#NCBI">NCBI</a> Medline (PubMed) and GenBank databases and alerts users to any relevant updates. Available at <a href="http://pubcrawler.gen.tcd.ie">pubcrawler.gen.tcd.ie</a></td></tr></table>
    

  
  

    <a name="Query">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Query"><font color='black'><b>Query</b></font></a> <br><br>Sequence or term used in a database search. For example, the sequence submitted for a <a href="#BLAST">BLAST</a> search is the query sequence.</td></tr></table>
    

  
  

    <a name="RefSeq mRNA">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=RefSeq mRNA"><font color='black'><b>RefSeq mRNA</b></font></a> <br><br>Nonredundant mRNA sequence in the RefSeq database. RefSeq mRNA sequences with an NM_XXXXXX accession are curated sequences and are, therefore, considered more reliable than those with XM_XXXXXX accessions (predicted mRNA sequences).</td></tr></table>
    

  
  

    <a name="RefSeq noncoding RNA">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=RefSeq noncoding RNA"><font color='black'><b>RefSeq noncoding RNA</b></font></a> <br><br>Nonredundant noncoding RNA (<a href="#ncRNA">ncRNA</a>) sequence in the RefSeq database. RefSeq ncRNA sequences with an NR_XXXXXX accession are curated sequences and are, therefore, considered more reliable than those with XR_XXXXXX accessions (predicted ncRNA sequences).</td></tr></table>
    

  
  

    <a name="RefSeq protein">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=RefSeq protein"><font color='black'><b>RefSeq protein</b></font></a> <br><br>Nonredundant protein sequence in the RefSeq database. RefSeq protein sequences with NP_XXXXXX accessions are curated sequences and are, therefore, considered more reliable than those with XP_XXXXXX accessions (predicted protein sequences).</td></tr></table>
    

  
  

    <a name="RefSeq">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=RefSeqs"><font color='black'><b>RefSeqs</b></font></a> <br><br><a href="#NCBI">NCBI</a> reference sequences. A curated nonredundant collection of sequences representing genomes, transcripts, and proteins as annotated by NCBI (available at <a href="https://www.ncbi.nlm.nih.gov/refseq/">www.ncbi.nlm.nih.gov/refseq</a>). The <a href="#Annotation">annotation</a> in these records is often different from the original GenBank submission, which may not be updated every time new information is obtained.</td></tr></table>
    

  
  

    <a name="Repeat regions">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Repeat regions"><font color='black'><b>Repeat regions</b></font></a> <br><br>Sequences present in many identical or highly similar copies in the genome.</td></tr></table>
    

  
  

    <a name="RNA predictions">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=RNA predictions"><font color='black'><b>RNA predictions</b></font></a> <br><br>Predictions of genes that encode nonprotein-encoding RNA's such as <a href="#tRNA">tRNA</a>, snoRNA, <a href="#rRNA">rRNA</a>, etc.</td></tr></table>
    

  
  

    <a name="rRNA">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=rRNA"><font color='black'><b>rRNA</b></font></a> <br><br>Ribosomal RNA. Component of the ribosomes, which function in protein synthesis.</td></tr></table>
    

  
  

    <a name="SAGE tags">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=SAGE tags"><font color='black'><b>SAGE tags</b></font></a> <br><br>Serial analysis of gene expression (SAGE) tags. Short (14-nucleotide) sequences found within mRNA, the relative abundance of which indicates the level of expression of the mRNA containing that tag.</td></tr></table>
    

  
  

    <a name="Scaffolds">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Scaffolds"><font color='black'><b>Scaffolds</b></font></a> <br><br>In genomic mapping, a series of <a href="#Contig">contigs</a> that are in the right order and orientation, but not necessarily connected in one continuous stretch of sequence.</td></tr></table>
    

  
  

    <a name="Sequence similarity">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Sequence similarity"><font color='black'><b>Sequence similarity</b></font></a> <br><br>Degree of similarity between two or more protein or nucleotide sequences.</td></tr></table>
    

  
  

    <a name="Signal peptide">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Signal peptide"><font color='black'><b>Signal peptide</b></font></a> <br><br>Short (3-60 amino acids) peptide sequence that directs the co-translational import of a protein to certain organelles or for secretion.</td></tr></table>
    

  
  

    <a name="SignalP">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=SignalP"><font color='black'><b>SignalP</b></font></a> <br><br>Program that predicts the presence and location of <a href="#Signal peptide">signal peptide</a> cleavage sites in amino acid sequences from Gram-positive and -negative prokaryotes and eukaryotes. The method incorporates a prediction of cleavage sites and a signal peptide/nonsignal peptide prediction based on a combination of several artificial neural networks and hidden Markov models (<a href="#HMM">HMMs</a>).</td></tr></table>
    

  
  

    <a name="SNP density">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=SNP density"><font color='black'><b>SNP density</b></font></a> <br><br>Amount or number of single nucleotide polymorphisms (<a href="#SNP">SNPs</a>) in a region of the genome.</td></tr></table>
    

  
  

    <a name="SNP genotyping">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=SNP genotyping"><font color='black'><b>SNP genotyping</b></font></a> <br><br>Identifying and mapping single nucleotide polymorphisms (<a href="#SNP">SNPs</a>) in an effort to determine the genotype members of a species. SNPs usually consist of two alleles (where the rare allele frequency is less than 1%), are evolutionarily conserved, and are the most common type of genetic variation. See <a href="#Genotyping">Genotyping</a>.</td></tr></table>
    

  
  

    <a name="SNP">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=SNP"><font color='black'><b>SNP</b></font></a> <br><br>Single nucleotide polymorphism. Small genetic changes or variations that can occur within a DNA sequence, for example when a single nucleotide, such as an A replacing one of the other three nucleotide letters C, G, or T. Most SNPs are found outside of coding sequences, but SNPs found within a coding sequence are more likely to alter the biological function of a protein. SNPs may be synonymous (generating a conservative change not altering the amino acid sequence) or they can be nonsynonymous and change the amino acid that is encoded.</td></tr></table>
    

  
  

    <a name="snRNA">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=snRNA"><font color='black'><b>snRNA</b></font></a> <br><br>Small nuclear RNA. Class of small RNA molecules found within the nucleus, transcribed by RNA polymerase II or III, and involved in a variety of important processes such as RNA splicing (removal of introns from hnRNA), regulation of transcription factors (7SK RNA) or RNA polymerase II (B2 RNA), and maintaining <a href="#Telomere">telomeres</a>. They are always associated with specific proteins, and the complexes are referred to as small nuclear ribonucleoproteins (snRNP) or snurps. These elements are rich in uridine. A large group of snRNAs known as small nucleolar RNAs (snoRNAs) are small RNA molecules that play an essential role in RNA biogenesis and chemical modification of ribosomal RNAs (<a href="#rRNA">rRNAs</a>) and other RNA genes (<a href="#tRNA">tRNA</a> and snRNAs). They are located in the nucleus and the cajal bodies of eukaryotic cells (the major sites of RNA synthesis).</td></tr></table>
    

  
  

    <a name="Strand">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Strand"><font color='black'><b>Strand</b></font></a> <br><br>One half of the DNA helix; string or stretch of covalently linked nucleotides.</td></tr></table>
    

  
  

    <a name="Subtract">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Subtract"><font color='black'><b>Subtract</b></font></a> <br><br>Similar to the <a href="#Boolean">Boolean</a> operator "NOT", an action used in the <a href="#Query">query</a> history to remove items from one result set that occur in another result set. For example, to remove items that exist in set Y from those in set X, type, "X SUBTRACT Y".</td></tr></table>
    

  
  

    <a name="Syntenic">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Syntenic"><font color='black'><b>Syntenic</b></font></a> <br><br>Loci located on the same <a href="#Chromosome">chromosome</a> but not necessarily linked. For example, genes that are part of a syntenic group share a common chromosomal location. Also used to refer to conservation of gene order across species.</td></tr></table>
    

  
  

    <a name="tblastn">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=tblastn"><font color='black'><b>tblastn</b></font></a> <br><br>Version of the basic local alignment search tool (<a href="#BLAST">BLAST</a>) used to compare a protein <a href="#Query">query</a> sequence against a translated nucleotide sequence database.</td></tr></table>
    

  
  

    <a name="tblastx">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=tblastx"><font color='black'><b>tblastx</b></font></a> <br><br>Version of the basic local alignment search tool (<a href="#BLAST">BLAST</a>) used to compare the six-frame <a href="#Translation">translations</a> of a nucleotide <a href="#Query">query</a> sequence against the six-frame translations of a nucleotide sequence database.</td></tr></table>
    

  
  

    <a name="Telomere">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Telomere"><font color='black'><b>Telomere</b></font></a> <br><br>Nucleoprotein complexes that constitute the physical ends of linear eukaryotic <a href="#Chromosome">chromosomes</a> and that have important functions, primarily in the protection, replication, and stabilization of the chromosome ends. Telomeres often contain lengthy stretches of tandemly repeated simple DNA sequences composed of a G-rich <a href="#Strand">strand</a> and a C-rich strand (called terminal repeats). These terminal repeats are highly conserved. Sequences adjacent to the telomeric repeats are often highly polymorphic and rich in repetitive elements (termed subtelomeric repeats); in some cases, genes have been found in the proterminal regions of chromosomes.</td></tr></table>
    

  
  

    <a name="TIGR">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=TIGR"><font color='black'><b>TIGR</b></font></a> <br><br>The Institute for Genomic Research, a nonprofit center dedicated to deciphering and analyzing genomes.</td></tr></table>
    

  
  

    <a name="TigrScan gene">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=TigrScan gene"><font color='black'><b>TigrScan gene</b></font></a> <br><br>Gene model generated using TigrScan, a gene-finding tool based on the generalized hidden Markov model (<a href="#HMM">HMM</a>) framework, similar to GENSCAN and Genie. It is highly reconfigurable and includes software for retraining.</td></tr></table>
    

  
  

    <a name="TM domains">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=TM domains"><font color='black'><b>TM domains</b></font></a> <br><br>See <a href="#Transmembrane domain">transmembrane domain</a>.</td></tr></table>
    

  
  

    <a name="ToxoCyc">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=ToxoCyc"><font color='black'><b>ToxoCyc</b></font></a> <br><br>Database/utility built by analyzing the Toxoplasma gondii genome with SRI International's pathway tools; used for searching and visualizing Toxoplasma metabolic pathway information.</td></tr></table>
    

  
  

    <a name="Translation">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Translation"><font color='black'><b>Translation</b></font></a> <br><br>Synthesis of protein from an mRNA template.</td></tr></table>
    

  
  

    <a name="Transmembrane domain">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Transmembrane domain"><font color='black'><b>Transmembrane domain</b></font></a> <br><br>Three-dimensional protein structure that is thermodynamically stable in a membrane. This may be a single alpha helix, a stable complex of several transmembrane alpha helices, a transmembrane beta barrel, a beta-helix of gramicidin A, or any other structure. Transmembrane domains average 20 amino acid residues in length, though they may be much smaller or much longer.</td></tr></table>
    

  
  

    <a name="Transmembrane protein">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Transmembrane protein"><font color='black'><b>Transmembrane protein</b></font></a> <br><br>Protein that spans an entire biological membrane.</td></tr></table>
    

  
  

    <a name="tRNA">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=tRNA"><font color='black'><b>tRNA</b></font></a> <br><br>Transfer RNA. Small RNA chain (73-93 nucleotides) that transfers a specific amino acid to a growing polypeptide chain at the ribosomal site of protein synthesis during <a href="#Translation">translation</a>. A three-base region, the anticodon, pairs to the corresponding three-base codon region on the template mRNA. Each type of tRNA molecule can be attached to only one type of amino acid, but because the genetic code contains multiple codons that specify the same amino acid, tRNA molecules bearing different anticodons may also carry the same amino acid.</td></tr></table>
    

  
  

    <a name="TWINSCAN gene models">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=TWINSCAN gene models"><font color='black'><b>TWINSCAN gene models</b></font></a> <br><br>Gene models generated using TWINSCAN, a tool that integrates traditional probability models (such as those underlying GENSCAN and FGENESH) with information from alignments between two genomes. TWINSCAN is based on the idea that functional sequences show different patterns of evolutionary conservation than sequences under little selective pressure, such as the central regions of introns. TWINSCAN is designed for the analysis of high-throughput genomic sequences containing an unknown number of genes.</td></tr></table>
    

  
  

    <a name="UniGene">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=UniGene"><font color='black'><b>UniGene</b></font></a> <br><br>Project and database at <a href="#NCBI">NCBI</a> aimed at defining gene-oriented clusters of expressed sequence tags (<a href="#EST">ESTs</a>). Sets of ESTs are clustered based on strong sequence homology in an attempt to define a specific, nonredundant cluster for each transcript in a tissue or genome. Each UniGene cluster contains sequences that represent a unique gene in addition to information about the tissue types in which the gene has been expressed and map location.</td></tr></table>
    

  
  

    <a name="UTR">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=UTR"><font color='black'><b>UTR</b></font></a> <br><br>Untranslated region. Section of messenger RNA (mRNA) that either precedes (5' UTR) or follows (3' UTR) the coding region and is not itself translated. The UTR contains several regulatory regions, including the polyadenylation (polyA) site in the 3' UTR, sequences involved in the initiation of <a href="#Translation">translation</a> (in the 5' UTR), and binding regions for proteins and other regulatory molecules in both the 3' and 5' UTR.</td></tr></table>
    

  
  

    <a name="Wildcard character">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Wildcard character"><font color='black'><b>Wildcard character</b></font></a> <br><br>Character used to substitute for any other character(s) in a string.</td></tr></table>
    

  
  

    <a name="Xenolog">
    <table border="0" cellpadding="2" cellspacing="0" width="100%">
  
    <tr><td colspan="2"><hr></td></tr>
    <tr class="rowLight"><td>
      <a href="showXmlDataContent.do?name=XmlQuestions.News&amp;tag=Xenolog"><font color='black'><b>Xenolog</b></font></a> <br><br>Gene found in an unrelated species and that is related by gene transfer rather than common vertical descent.</td></tr></table>
    
</div>

