---
permalink: /infrastructure
tags: [general]
title: VEuPathDB infrastructure
---
<h1>VEuPathDB Infrastructure Document</h1>

<div class="static-content">

<p>This document provides a high-level overview of the software infrastructure utilized by the VEuPathDB BRC to load, 
  integrate and provide data to users. Please check <a target="_blank" href="https://eupathdb.org/eupathdb/app/search/dataset/AllDatasets/result">
  a list of all the data sets</a> loaded in our VEuPathDB sites utilizing this infrastructure.</p>

<a name="data"/>
<h3>Data Loading and Database Schema</h3>
<p>We use the <a target="_blank" href="https://github.com/EuPathDB/GusSchema"><b>G</b>enomics <b>U</b>nified <b>S</b>chema (GUS) database schema</a> and 
  data loading infrastructure and its framework available at <a target="_blank" href="https://github.com/EuPathDB/GusAppFramework">GusAppFramework</a>.  
  This includes not only a comprehensive database schema for integrating and representing genomic and functional (or post) 
  genomic data but also tools for loading said data into that system.  We have made some extensions to the schema and tools for 
  VEuPathDB specific purposes primarily to generate de-normalized views of the data for query optimization purposes.</p>
<p>Our data are all stored in Oracle12c databases.  Our software infrastructure also supports PostgreSQL but we have some Oracle 
  specific SQL constructs in our model that would need to be changed in order to run successfully in PostgreSQL.  </p>
<p>We load all data using an in house engineered workflow system called <a target="_blank" href="https://github.com/EuPathDB/ReFlow">
  ReFlow</a>.  Briefly, ReFlow is engineered to be an efficient graph-based workflow system. In it each step (node in the graph) has 
  the ability to be undone and subsequently rerun with updated data.  This was a significant requirement as it enables us to undo entire
  genomes when the annotation or underlying sequence changes.  This results in automated removal of all data dependent on that genome. 
  When the step is re-run with the new annotation, all dependent data are recomputed and reloaded automatically, thus greatly improving
  our ability to keep these complex databases up-to-date.</p>
<p>The ReFlow workflow system utilizes another piece of software developed at the University of Pennsylvania to schedule, manage and
  monitor running tasks called 
  <a target="_blank" href="http://www.google.com/url?q=https%3A%2F%2Fdocs.google.com%2Fdocument%2Fpub%3Fid%3D1BixZ5t2c0hnOZES-Rk2wG2loAQqclcmRj7AeKQjZMHA">DistribJob</a>.  
  DistribJob distributes tasks generated from a large input dataset such as a set of sequences to compute nodes in a cluster for analysis
  and retrieves and collates the results in an efficient manner. We automate the running of large compute tasks on compute clusters 
  located at the University of Pennsylvania and the University of Georgia.</p>
<details>
  <summary style="font-size:110%;font-weight:bold;margin-bottom:5px;cursor:pointer;"> >>> Github Repositories</summary>
  These are the repositories needed for the loading infrastructure:
  <ul>
    <li><a target="_blank" href="https://github.com/EuPathDB/install">install</a>: starter.</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/CBIL">CBIL</a>: scripts to generate GUS perl objects .</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/GusSchema">GusSchema</a>: Relational Database schema definition for Genomics data.</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/GusAppFramework">GusAppFramework</a>: GUS application framework.</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/ReFlow">ReFlow</a>: ReFlow (see explanation above).</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/DJob">DJob</a>: DistribJob (see explanation above).</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/ApiCommonWorkflow">ApiCommonWorkflow</a>: Supports the workflow graph for the Genomic sites.
        (Also: <a target="_blank" href="https://github.com/EuPathDB/OrthoMCLWorkflow">OrthoMCLWorkflow</a>, 
               <a target="_blank" href="https://github.com/EuPathDB/ClinEpiWorkflow">ClinEpiWorkflow</a>, 
               <a target="_blank" href="https://github.com/EuPathDB/MicrobiomeWorkflow">MicrobiomeWorkflow</a>.)</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/TuningManager">TuningManager</a>: Engine to generate denormalized tables in a database.</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/ApiCommonData">ApiCommonData</a>: Defines the workflow graph for the Genomic sites and contains SQL to generate denormalized tables.
        (Also: <a target="_blank" href="https://github.com/EuPathDB/OrthoMCLData">OrthoMCLData</a>,
               <a target="_blank" href="https://github.com/EuPathDB/ClinEpiData">ClinEpiData</a>.)</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/ExampleDatasets">ExampleDatasets</a>: Example of the Data Sets to be loaded in the database by the Workflows. Also needed at site build time for model injection.</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/ExamplePresenters">ExamplePresenters</a>: Example of the Data Set metadata to be loaded in denormalized tables. Also needed at site build time for model injection.</li>
  </ul>
</details>
     
<a name="wdk"/>
<h3>Web Presentation System and User Interfaces</h3>
<p>Our websites are based on code that we developed and have released to the community called the Strategies-WDK (Strategies Web 
  Development Kit) which enables the graphical strategies search system.  You can download the software and see documentation for this
  toolkit at <a target="_blank" href="https://github.com/EuPathDB/WDK">Strategies-WDK</a>.  This toolkit enables us to represent our 
  data as an XML model which is then turned into the web interfaces that are presented to users using these tools.</p>
<details>
  <summary style="font-size:110%;font-weight:bold;margin-bottom:5px;cursor:pointer;"> >>> Github Repositories</summary>
  These are the repositories needed to build a website:
  <ul>
    <li><a target="_blank" href="https://github.com/EuPathDB/install">install</a>: starter.</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/CBIL">CBIL</a>: scripts to generate GUS perl objects .</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/ExampleDatasets">ExampleDatasets</a>: Example of the Data Sets to be loaded in the database by the Workflows. Also needed at site build time for model injection.</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/ExamplePresenters">ExamplePresenters</a>: Example of the Data Set metadata to be loaded in denormalized tables. Also needed at site build time for model injection.</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/EbrcModelCommon">EbrcModelCommon</a>: xml files with basic data types definitions, and their structured searches, common to all sites (genomic, epidemiology, microbiome).</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/ApiCommonModel">ApiCommonModel</a>: xml files with basic data types definitions, and their structured searches, for genomic sites.
        (Also: <a target="_blank" href="https://github.com/EuPathDB/OrthoMCLModel">OrthoMCLModel</a>,
               <a target="_blank" href="https://github.com/EuPathDB/ClinEpiModel">ClinEpiModel</a>,
               <a target="_blank" href="https://github.com/EuPathDB/MicrobiomeModel">MicrobiomeModel</a>.)</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/FgpUtil">FgpUtil</a>: Basic utilities used by the WDK such as access to database, and SQL runs.</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/WDK">WDK</a>: The Web Development Kit. It reads the model (xml files) and loads into java structures. Engine for search runs, step and strategy creation, baskets and favorites and user workspace.</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/OAuth2Server">OAuth2Server</a>: authentication to one site used by all sites within VEuPathDB including the galaxy site.</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/WSF">WSF</a>: The WDK Services Framework: web service architecture to support searches that instead of accessing the database run plugins/processes.</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/EbrcWebSvcCommon">EbrcWebSvcCommon</a>: Plugins common to all sites (eg: Blast, Text search).</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/ApiCommonWebService">ApiCommonWebService</a>: Plugins specific to genomic sites.</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/OrthoMCLWebService">OrthoMCLWebService</a>: Plugins specific to OrthoMCL.</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/WDKClient">WDKClient</a>: a React-based frontend for WDK (js, jsx, css).</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/WDKWebsite">WDKWebsite</a>: legacy frontend (html, jsp, css) being replaced.</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/EbrcWebsiteCommon">EbrcWebsiteCommon</a>: frontend pages (html, jsp, js, css) that add or modify WDKClient, common to all sites.</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/ApiCommonWebsite">ApiCommonWebsite</a>: frontend pages (html, jsp, js, css) that add or modify WDKClient, common to the genomic sites.
        (Also: <a target="_blank" href="https://github.com/EuPathDB/OrthoMCLWebsite">OrthoMCLWebsite</a>,
               <a target="_blank" href="https://github.com/EuPathDB/ClinEpiWebsite">ClinEpiWebsite</a>,
               <a target="_blank" href="https://github.com/EuPathDB/MicrobiomeWebsite">MicrobiomeWebsite</a>.)</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/JBrowse">JBrowse</a>: modern genome browser, replaces GBrowse.</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/EuPathDBIrods">EuPathDBIrods</a>: Irods storage for the user workspace, which allows users to export data sets from the VEuPathDB Galaxy site and run analyses on them.</li>
    <li><a target="_blank" href="https://github.com/EuPathDB/EuPathGalaxy">EuPathGalaxy</a>: code that runs in the VEuPathDB Galaxy, which includes the VEuPathDB written tools and workflows.</li>
  </ul>
</details>
    
  
<a name="hardware"/>
<h3>System Hardware and Third Party Software</h3>
<p>VEuPathDB maintains redundant database and content web servers at the University of Pennsylvania and the University of Georgia to 
  minimize interruptions for our users during maintenance periods. Additionally VEuPathDB compute and data loading servers are located at
  the University of Pennsylvania.</p> 
<p>Server configurations are coordinated and deployed through Puppet automation software (<a href="http://puppetlabs.com/" target="_blank">http://puppetlabs.com/</a>). 
  Custom infrastructure software is versioned and deployed through standard RPM/YUM mechanisms. When appropriate, software builds are 
  automated using Jenkins Continuous Integration Server (<a href="http://jenkins-ci.org/" target="_blank">http://jenkins-ci.org/</a>)</p>
<p>System infrastructure statistics (CPU load, I/O, etc) are gathered with collectd (<a href="http://collectd.org/" target="_blank">http://collectd.org/</a>) 
  and in-house applications and feed to Graphite (<a href="http://graphite.wikidot.com/" target="_blank">http://graphite.wikidot.com/</a>) 
  for human review. Nagios (<a href="http://www.nagios.org/" target="_blank">http://www.nagios.org/</a>) provides notifications of system
  degradations.</p>
<p>Both Universities also maintain large compute clusters that are heavily utilized by VEuPathDB in order to analyze and load incoming
  data in a timely fashion. The linked document below describes our actual hardware and includes a list of third-party software required
  in order to analyze, load and present data via our websites.</p>
<div id="cirbulletlist">
<ul><li><a href="/documents/EuPathDB_Hardware_3P_Software.pdf">Hardware and Third Party Software</a></ul>
</div>

<a name="community"/>
<h3>Community Interactions and Data Submission Policies</h3>
<p>VEuPathDB serves a global scientific community that demands direct active support and community involvement.   VEuPathDB outreach 
  activities include:</p>
<div id="cirbulletlist">
<ul>
<li>Organizing and running hands on training workshops and webinars (<a href="http://tinyurl.com/7qjxumg">Google Map</a>).
<li>Developing educational material in the form of <a href="http://workshop.eupathdb.org/current/index.php?page=schedule">exercises</a> and <a href="http://www.youtube.com/user/EuPathDB/">online tutorials</a>.
<li>Responding to support emails for users who contact us directly by clicking the "Contact Us" links in the header or footer of any VEuPathDB webpage (average response time is 48 hours).
<li>Holding semi-annual meetings with our <a href="/a/app/community/embedded/help/general/index.html#swg">Scientific Working Group</a> to discuss high-level goals and deliverables for the project.
<li>Holding bimonthly conference calls with our <a href="/a/app/community/EuPathDB/about#advisors">Scientific Advisory Groups</a> (these are organism specific, one per component site with TriTrypDB separated into three) to learn about new datasets, set priorities and share with the community current activities.
<li>Attending national and international meetings with active participation in the form of posters, presentations or help desks.
<li>Authoring <a target="_blank" href="https://eupathdb.org/eupathdb/app/community/veupathPubs">peer reviewed manuscripts</a>.
<li>Maintaining active social media presence in the form of a <a target="_blank" href="https://www.facebook.com/pages/EuPathDB/133123003429972">FaceBook page</a> and <a href="https://twitter.com/eupathdb">Twitter feed</a>.
<li>Providing a clear <a href="/EuPathDB_datasubm_SOP.pdf">data handling and release policy</a> to investigators to encourage submission of data prepublication.
</ul>
</div>

</div>
