---
permalink: /methods
tags: [general]
title: VEuPathDB methods
---
  These are the methods we used to generate/analyze the data in our queries.<br/>
  
            
              <table border="0" cellpadding="2" cellspacing="0" width="100%">
                
  
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Method</b></td>
                    <td>WU-BLASTX (Genomic sequence vs. NRDB)</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Input</b></td>
                    <td>Genomic sequence and NCBI non-redundant protein database</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Output</b></td>
                    <td>BLASTX alignments</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Details</b></td>
                    <td>WU-BLASTX is used to identify statistically significant DNA-protein sequence similarities found between DNA sequence queries and protein sequence libraries</td>
                  </tr>
                
  
                
                
                  
                  <tr>
                    <td valign="top" width="10%"><b>Tools</b></td>
                    <td>
                      <table border="0" cellpadding="2" cellspacing="0" width="100%">
                        
                        
                          
                            <tr>
                              <td valign="top" width="10%"><i>Tool</i></td>
                              <td>BLASTX</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Version</i></td>
                              <td>2.0 [06-Apr-2005]</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Parameters</i></td>
                              <td>-cpus=2 -topcomboN=1 W=4 T=18 V=100 B=1000000 -hspmax=1000000 -gi E=1e-3 -wordmask=seg+xnu -hspsepQmax=4000 -span1</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>URL</i></td>
                              <td>http://blast.wustl.edu</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>PubMed IDs</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Credits</i></td>
                              <td>Warren R. Gish, Washington University, Saint Louis, Missouri</td>
                            </tr>
                          
                        
                      </table>
                    </td>
                  </tr>
                
                <tr><td colspan="2">&nbsp;</td></tr>
              </table>
              
            
              <table border="0" cellpadding="2" cellspacing="0" width="100%">
                
                  <tr><td colspan="2"><hr></td></tr>
                
  
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Method</b></td>
                    <td>HMMPfam</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Input</b></td>
                    <td>fasta file of protein sequences and PFAM database</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Output</b></td>
                    <td>file containing the score and E-value indicating confidence that a query 
                                      sequence contains one or more domains belonging to a domain family</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Details</b></td>
                    <td>Searches HMMs from the PFAM database for significantly similar sequences in the input protein sequence.</td>
                  </tr>
                
  
                
                
                  
                  <tr>
                    <td valign="top" width="10%"><b>Tools</b></td>
                    <td>
                      <table border="0" cellpadding="2" cellspacing="0" width="100%">
                        
                        
                          
                            <tr>
                              <td valign="top" width="10%"><i>Tool</i></td>
                              <td>HMMPfam</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Version</i></td>
                              <td>2.3.2</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Parameters</i></td>
                              <td>--A 0 --acc --cut_ga</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>URL</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>PubMed IDs</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Credits</i></td>
                              <td>R. Durbin, S. Eddy, A. Krogh, and G. Mitchison,
                                                  Biological sequence analysis: probabilistic models of proteins and nucleic acids,
                                                  Cambridge University Press, 1998.
                                                  http://hmmer.wustl.edu/</td>
                            </tr>
                          
                        
                      </table>
                    </td>
                  </tr>
                
                <tr><td colspan="2">&nbsp;</td></tr>
              </table>
              
            
              <table border="0" cellpadding="2" cellspacing="0" width="100%">
                
                  <tr><td colspan="2"><hr></td></tr>
                
  
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Method</b></td>
                    <td>splign</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Input</b></td>
                    <td>output of megablast analysis of a fasta formatted transcript file with a genomic sequence database formatted using formatdb</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Output</b></td>
                    <td>a tab-delimited text with every line representing a separate segment on the query sequence.</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Details</b></td>
                    <td>Splign uses transcript to genomic sequence BLAST hit output to generate information about exon-intron boundaries, splice-junctions, potential frameshifts, and alternative models when there is more than one possibility</td>
                  </tr>
                
  
                
                
                  
                  <tr>
                    <td valign="top" width="10%"><b>Tools</b></td>
                    <td>
                      <table border="0" cellpadding="2" cellspacing="0" width="100%">
                        
                        
                          
                            <tr>
                              <td valign="top" width="10%"><i>Tool</i></td>
                              <td>splign</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Version</i></td>
                              <td>07/19/06</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Parameters</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>URL</i></td>
                              <td>http://www.ncbi.nlm.nih.gov/sutils/splign/</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>PubMed IDs</i></td>
                              <td>16381840</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Credits</i></td>
                              <td>Yu.Kapustin, A.Souvorov, T.Tatusova. 
                             Splign - a Hybrid Approach To Spliced Alignments.
                             RECOMB 2004 - Currents in Computational Molecular Biology. p.741.</td>
                            </tr>
                          
                        
                      </table>
                    </td>
                  </tr>
                
                <tr><td colspan="2">&nbsp;</td></tr>
              </table>
              
            
              <table border="0" cellpadding="2" cellspacing="0" width="100%">
                
                  <tr><td colspan="2"><hr></td></tr>
                
  
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Method</b></td>
                    <td>WU-BLASTP (Protein sequences vs. NRDB)</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Input</b></td>
                    <td>Protein sequences and NRDB</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Output</b></td>
                    <td>BLASTP alignments</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Details</b></td>
                    <td>WU-BLASTP is used to identify statistically significant protein sequence similarities found between protein sequence queries and protein sequence libraries</td>
                  </tr>
                
  
                
                
                  
                  <tr>
                    <td valign="top" width="10%"><b>Tools</b></td>
                    <td>
                      <table border="0" cellpadding="2" cellspacing="0" width="100%">
                        
                        
                          
                            <tr>
                              <td valign="top" width="10%"><i>Tool</i></td>
                              <td>BLASTP</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Version</i></td>
                              <td>2.0 [06-Apr-2005]</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Parameters</i></td>
                              <td>-cpus=2 -topcomboN=1 V=100 B=20 -hspmax=1000000 -gi E=1e-3 -wordmask=seg -hspsepQmax=4000 -span1</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>URL</i></td>
                              <td>http://blast.wustl.edu</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>PubMed IDs</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Credits</i></td>
                              <td>Warren R. Gish, Washington University, Saint Louis, Missouri</td>
                            </tr>
                          
                        
                      </table>
                    </td>
                  </tr>
                
                <tr><td colspan="2">&nbsp;</td></tr>
              </table>
              
            
              <table border="0" cellpadding="2" cellspacing="0" width="100%">
                
                  <tr><td colspan="2"><hr></td></tr>
                
  
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Method</b></td>
                    <td>Predict transmembrane domains</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Input</b></td>
                    <td>Protein sequences</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Output</b></td>
                    <td>Predicted transmembrane domain topology</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Details</b></td>
                    <td>TMHMM is used to predict transmembrane domain presence and topology</td>
                  </tr>
                
  
                
                
                  
                  <tr>
                    <td valign="top" width="10%"><b>Tools</b></td>
                    <td>
                      <table border="0" cellpadding="2" cellspacing="0" width="100%">
                        
                        
                          
                            <tr>
                              <td valign="top" width="10%"><i>Tool</i></td>
                              <td>tmhmm</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Version</i></td>
                              <td>2.0a</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Parameters</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>URL</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>PubMed IDs</i></td>
                              <td>11152613</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Credits</i></td>
                              <td>Anders Krogh, Bjorn Larsson, Gunnar von Heijne, and Erik L.L. Sonnhammer</td>
                            </tr>
                          
                        
                      </table>
                    </td>
                  </tr>
                
                <tr><td colspan="2">&nbsp;</td></tr>
              </table>
              
            
              <table border="0" cellpadding="2" cellspacing="0" width="100%">
                
                  <tr><td colspan="2"><hr></td></tr>
                
  
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Method</b></td>
                    <td>SignalP</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Input</b></td>
                    <td>Protein sequences</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Output</b></td>
                    <td>Signal peptide predictions</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Details</b></td>
                    <td>SignalP is used to identify signal peptides and their likely cleavage sites.</td>
                  </tr>
                
  
                
                
                  
                  <tr>
                    <td valign="top" width="10%"><b>Tools</b></td>
                    <td>
                      <table border="0" cellpadding="2" cellspacing="0" width="100%">
                        
                        
                          
                            <tr>
                              <td valign="top" width="10%"><i>Tool</i></td>
                              <td>SignalP</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Version</i></td>
                              <td>3.0</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Parameters</i></td>
                              <td>-t euk -f short -m nn+hmm -q -trunc 70</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>URL</i></td>
                              <td>http://www.cbs.dtu.dk/services/SignalP/</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>PubMed IDs</i></td>
                              <td>15223320</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Credits</i></td>
                              <td>Bendtsen JD, Nielsen H, von Heijne G, Brunak S.</td>
                            </tr>
                          
                        
                      </table>
                    </td>
                  </tr>
                
                <tr><td colspan="2">&nbsp;</td></tr>
              </table>
              
            
              <table border="0" cellpadding="2" cellspacing="0" width="100%">
                
                  <tr><td colspan="2"><hr></td></tr>
                
  
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Method</b></td>
                    <td>WU-BLASTN (Oligo sequences vs. Genomic or transcript sequences)</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Input</b></td>
                    <td>Oligo sequences and Genomic or transcript sequences</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Output</b></td>
                    <td>BLASTN alignments</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Details</b></td>
                    <td>WU-BLASTN is used to identify statistically significant DNA sequence similarities found between DNA sequence queries and DNA sequence libraries</td>
                  </tr>
                
  
                
                
                  
                  <tr>
                    <td valign="top" width="10%"><b>Tools</b></td>
                    <td>
                      <table border="0" cellpadding="2" cellspacing="0" width="100%">
                        
                        
                          
                            <tr>
                              <td valign="top" width="10%"><i>Tool</i></td>
                              <td>BLASTN</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Version</i></td>
                              <td>2.0 [06-Apr-2005]</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Parameters</i></td>
                              <td>-nogaps E=0.05</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>URL</i></td>
                              <td>http://blast.wustl.edu</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>PubMed IDs</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Credits</i></td>
                              <td>Warren R. Gish, Washington University, Saint Louis, Missouri</td>
                            </tr>
                          
                        
                      </table>
                    </td>
                  </tr>
                
                <tr><td colspan="2">&nbsp;</td></tr>
              </table>
              
            
              <table border="0" cellpadding="2" cellspacing="0" width="100%">
                
                  <tr><td colspan="2"><hr></td></tr>
                
  
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Method</b></td>
                    <td>ExportPred</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Input</b></td>
                    <td>fasta file of protein sequences</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Output</b></td>
                    <td>file containing export sequences and scores</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Details</b></td>
                    <td>Program that predicts the exported proteins of Plasmodium.</td>
                  </tr>
                
  
                
                
                  
                  <tr>
                    <td valign="top" width="10%"><b>Tools</b></td>
                    <td>
                      <table border="0" cellpadding="2" cellspacing="0" width="100%">
                        
                        
                          
                            <tr>
                              <td valign="top" width="10%"><i>Tool</i></td>
                              <td>ExportPred</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Version</i></td>
                              <td>1.0.1</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Parameters</i></td>
                              <td>default</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>URL</i></td>
                              <td>http://bioinf.wehi.edu.au/exportpred/</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>PubMed IDs</i></td>
                              <td>16507167</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Credits</i></td>
                              <td></td>
                            </tr>
                          
                        
                      </table>
                    </td>
                  </tr>
                
                <tr><td colspan="2">&nbsp;</td></tr>
              </table>
              
            
              <table border="0" cellpadding="2" cellspacing="0" width="100%">
                
                  <tr><td colspan="2"><hr></td></tr>
                
  
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Method</b></td>
                    <td>TRF</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Input</b></td>
                    <td>fasta file of DNA sequences</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Output</b></td>
                    <td>a repeat table file and an alignment file</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Details</b></td>
                    <td>The Tandem Repeats Finder program locates and displays tandem repeats in DNA sequences</td>
                  </tr>
                
  
                
                
                  
                  <tr>
                    <td valign="top" width="10%"><b>Tools</b></td>
                    <td>
                      <table border="0" cellpadding="2" cellspacing="0" width="100%">
                        
                        
                          
                            <tr>
                              <td valign="top" width="10%"><i>Tool</i></td>
                              <td>TRF</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Version</i></td>
                              <td>3.21</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Parameters</i></td>
                              <td>2 7 7 80 20 50 500</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>URL</i></td>
                              <td>http://tandem.bu.edu/trf/trf.html</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>PubMed IDs</i></td>
                              <td>9862982</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Credits</i></td>
                              <td></td>
                            </tr>
                          
                        
                      </table>
                    </td>
                  </tr>
                
                <tr><td colspan="2">&nbsp;</td></tr>
              </table>
              
            
              <table border="0" cellpadding="2" cellspacing="0" width="100%">
                
                  <tr><td colspan="2"><hr></td></tr>
                
  
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Method</b></td>
                    <td>Filter Low Complexity Protein Sequences</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Input</b></td>
                    <td>Protein sequences</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Output</b></td>
                    <td>Protein sequences containing regions of low complexity are masked with the character 'x'</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Details</b></td>
                    <td>seg was used to mask low complexity regions.</td>
                  </tr>
                
  
                
                
                  
                  <tr>
                    <td valign="top" width="10%"><b>Tools</b></td>
                    <td>
                      <table border="0" cellpadding="2" cellspacing="0" width="100%">
                        
                        
                          
                            <tr>
                              <td valign="top" width="10%"><i>Tool</i></td>
                              <td>seg</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Version</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Parameters</i></td>
                              <td>-x</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>URL</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>PubMed IDs</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Credits</i></td>
                              <td>Wootton and Federhen</td>
                            </tr>
                          
                        
                      </table>
                    </td>
                  </tr>
                
                <tr><td colspan="2">&nbsp;</td></tr>
              </table>
              
            
              <table border="0" cellpadding="2" cellspacing="0" width="100%">
                
                  <tr><td colspan="2"><hr></td></tr>
                
  
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Method</b></td>
                    <td>Filter Low Complexity Genomic Sequences</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Input</b></td>
                    <td>Genomic sequences</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Output</b></td>
                    <td>Genomic sequences containing regions of low complexity are masked with the character 'N'</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Details</b></td>
                    <td>dust was used to mask low complexity regions.</td>
                  </tr>
                
  
                
                
                  
                  <tr>
                    <td valign="top" width="10%"><b>Tools</b></td>
                    <td>
                      <table border="0" cellpadding="2" cellspacing="0" width="100%">
                        
                        
                          
                            <tr>
                              <td valign="top" width="10%"><i>Tool</i></td>
                              <td>dust</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Version</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Parameters</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>URL</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>PubMed IDs</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Credits</i></td>
                              <td>Tatusov and Lipman</td>
                            </tr>
                          
                        
                      </table>
                    </td>
                  </tr>
                
                <tr><td colspan="2">&nbsp;</td></tr>
              </table>
              
            
              <table border="0" cellpadding="2" cellspacing="0" width="100%">
                
                  <tr><td colspan="2"><hr></td></tr>
                
  
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Method</b></td>
                    <td>psipred</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Input</b></td>
                    <td>fasta file of protein sequences and a filtered and formatted protein database</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Output</b></td>
                    <td>file containing a residue by residue prediction of helix, strand, xand coil accompanied by
                                     confidence level</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Details</b></td>
                    <td>Psipred predicts secondary structure using a neural network analysis of PSI-BLAST output</td>
                  </tr>
                
  
                
                
                  
                  <tr>
                    <td valign="top" width="10%"><b>Tools</b></td>
                    <td>
                      <table border="0" cellpadding="2" cellspacing="0" width="100%">
                        
                        
                          
                            <tr>
                              <td valign="top" width="10%"><i>Tool</i></td>
                              <td>psipred</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Version</i></td>
                              <td>2.4</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Parameters</i></td>
                              <td>default</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>URL</i></td>
                              <td>http://bioinf.cs.ucl.ac.uk/psipred/</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>PubMed IDs</i></td>
                              <td>10493868</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Credits</i></td>
                              <td>Jones DT.(1999)Protein secondary structure prediction based on position-specific scoring matrices.J.Mol.Biol.292:195-202.</td>
                            </tr>
                          
                        
                      </table>
                    </td>
                  </tr>
                
                <tr><td colspan="2">&nbsp;</td></tr>
              </table>
              
            
              <table border="0" cellpadding="2" cellspacing="0" width="100%">
                
                  <tr><td colspan="2"><hr></td></tr>
                
  
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Method</b></td>
                    <td>tRNAScan-SE</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Input</b></td>
                    <td>Transcript or genomic sequence file in fasta format</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Output</b></td>
                    <td>tab delimited file with the type of tRNA, anticodon, score, and locations on the sequences analyzed</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Details</b></td>
                    <td>tRNAscan-SE identifies transfer RNA genes in genomic DNA or RNA sequences.</td>
                  </tr>
                
  
                
                
                  
                  <tr>
                    <td valign="top" width="10%"><b>Tools</b></td>
                    <td>
                      <table border="0" cellpadding="2" cellspacing="0" width="100%">
                        
                        
                          
                            <tr>
                              <td valign="top" width="10%"><i>Tool</i></td>
                              <td>tRNAScan-SE</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Version</i></td>
                              <td>1.4</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Parameters</i></td>
                              <td>trainingOption=C</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>URL</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>PubMed IDs</i></td>
                              <td>9023104</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Credits</i></td>
                              <td></td>
                            </tr>
                          
                        
                      </table>
                    </td>
                  </tr>
                
                <tr><td colspan="2">&nbsp;</td></tr>
              </table>
              
            
              <table border="0" cellpadding="2" cellspacing="0" width="100%">
                
                  <tr><td colspan="2"><hr></td></tr>
                
  
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Method</b></td>
                    <td>InterProScan</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Input</b></td>
                    <td>fasta file of protein sequences and InterPro databases</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Output</b></td>
                    <td>file containing the domain matched, description of the interpro entry, GO description of the Interpro entry and E-value of the match</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Details</b></td>
                    <td>InterProScan scans protein sequences against the protein signatures of the InterPro member databases including PROSITE, PRINTS, xPfam, ProDom,SMART, TIGRFAMs, PIR, SUPERFAMILY.</td>
                  </tr>
                
  
                
                
                  
                  <tr>
                    <td valign="top" width="10%"><b>Tools</b></td>
                    <td>
                      <table border="0" cellpadding="2" cellspacing="0" width="100%">
                        
                        
                          
                            <tr>
                              <td valign="top" width="10%"><i>Tool</i></td>
                              <td>InterProScan</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Version</i></td>
                              <td>most recent</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Parameters</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>URL</i></td>
                              <td>http://www.ebi.ac.uk/InterProScan/</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>PubMed IDs</i></td>
                              <td>11590104x</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Credits</i></td>
                              <td>Zdobnov E.M. and Apweiler R. ,
                                                  InterProScan - an integration platform for the signature-recognition methods in InterPro,
                                                  Bioinformatics, 2001, 17(9): 847-8</td>
                            </tr>
                          
                        
                      </table>
                    </td>
                  </tr>
                
                <tr><td colspan="2">&nbsp;</td></tr>
              </table>
              
            
              <table border="0" cellpadding="2" cellspacing="0" width="100%">
                
                  <tr><td colspan="2"><hr></td></tr>
                
  
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Method</b></td>
                    <td>ORF Generation</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Input</b></td>
                    <td>Genomic DNA or EST sequences</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Output</b></td>
                    <td>gff file containing the 6 reading frame, 50 or more amino acid translations of the input nucleic acid</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Details</b></td>
                    <td>The translations in all 6 reading frames do not necessarily begin with MET but always end at a stop codon. ORF names are in the form template-frame-start-end, e.g. AAEL01000396-5-5847-4366</td>
                  </tr>
                
  
                
                
                  
                  <tr>
                    <td valign="top" width="10%"><b>Tools</b></td>
                    <td>
                      <table border="0" cellpadding="2" cellspacing="0" width="100%">
                        
                        
                          
                            <tr>
                              <td valign="top" width="10%"><i>Tool</i></td>
                              <td>orfFinder</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Version</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Parameters</i></td>
                              <td>--minPepLength 50</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>URL</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>PubMed IDs</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Credits</i></td>
                              <td></td>
                            </tr>
                          
                        
                      </table>
                    </td>
                  </tr>
                
                <tr><td colspan="2">&nbsp;</td></tr>
              </table>
              
            
              <table border="0" cellpadding="2" cellspacing="0" width="100%">
                
                  <tr><td colspan="2"><hr></td></tr>
                
  
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Method</b></td>
                    <td>CAP4 EST Assembly</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Input</b></td>
                    <td>Clusters of EST sequences</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Output</b></td>
                    <td>assembled consensus sequence and quality values</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Details</b></td>
                    <td>CAP4 is used to generate the EST assemblies. CAP4 aligns and assembles clustered sequence reads and produces consensus sequences</td>
                  </tr>
                
  
                
                
                  
                  <tr>
                    <td valign="top" width="10%"><b>Tools</b></td>
                    <td>
                      <table border="0" cellpadding="2" cellspacing="0" width="100%">
                        
                        
                          
                            <tr>
                              <td valign="top" width="10%"><i>Tool</i></td>
                              <td>CAP4</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Version</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Parameters</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>URL</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>PubMed IDs</i></td>
                              <td>10508846</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Credits</i></td>
                              <td>Huang, X. and Madan, A.
                                  (1999) CAP3: A DNA sequence assembly program. 
                                   Genome Res., 9, 868-877.</td>
                            </tr>
                          
                        
                      </table>
                    </td>
                  </tr>
                
                <tr><td colspan="2">&nbsp;</td></tr>
              </table>
              
            
              <table border="0" cellpadding="2" cellspacing="0" width="100%">
                
                  <tr><td colspan="2"><hr></td></tr>
                
  
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Method</b></td>
                    <td>BLAT EST Alignments</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Input</b></td>
                    <td>Fasta file of EST sequences blocked using repeatMasker with RepBase and genome sequences in nib format</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Output</b></td>
                    <td>Text file of alignments in .psl format</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Details</b></td>
                    <td>ESTs are clustered based on BLAT alignments. BLAT is a tool to rapidly find mRNA/DNA alignments that can be run using the GfClient program to query an indexed DNA database on a GfServer</td>
                  </tr>
                
  
                
                
                  
                  <tr>
                    <td valign="top" width="10%"><b>Tools</b></td>
                    <td>
                      <table border="0" cellpadding="2" cellspacing="0" width="100%">
                        
                        
                          
                            <tr>
                              <td valign="top" width="10%"><i>Tool</i></td>
                              <td>BLAT</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Version</i></td>
                              <td>31</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Parameters</i></td>
                              <td>-minIntron 500,-minScore 30,-minIdentity 90</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>URL</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>PubMed IDs</i></td>
                              <td>11932250</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Credits</i></td>
                              <td>Kent W.J.,
                              BLAT - The BLAST-Like Alignment Tool,
                              Genome Research, 2002,  12(4), 656-664</td>
                            </tr>
                          
                        
                      </table>
                    </td>
                  </tr>
                
                <tr><td colspan="2">&nbsp;</td></tr>
              </table>
              
            
              <table border="0" cellpadding="2" cellspacing="0" width="100%">
                
                  <tr><td colspan="2"><hr></td></tr>
                
  
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Method</b></td>
                    <td>MercatorAndMavid</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Input</b></td>
                    <td>genomes, exon coordinates, and a phylogenetic newick tree</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Output</b></td>
                    <td>file of pairwise synteny alignments with defined spans</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Details</b></td>
                    <td>Mercator and Mavid are used for comparative genome analysis. Mercator uses genomes and exon coordinates to create exon translations for protein BLAT alignments and outputs orthology maps. The orthology maps are used as a guide for Mavid which also uses a phylogenetic newick tree to generate gene alignments</td>
                  </tr>
                
  
                
                
                  
                  <tr>
                    <td valign="top" width="10%"><b>Tools</b></td>
                    <td>
                      <table border="0" cellpadding="2" cellspacing="0" width="100%">
                        
                        
                          
                            <tr>
                              <td valign="top" width="10%"><i>Tool</i></td>
                              <td>MercatorAndMavid</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Version</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Parameters</i></td>
                              <td>-minIntron 500,-minScore 30,-minIdentity 90</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>URL</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>PubMed IDs</i></td>
                              <td>17993677</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Credits</i></td>
                              <td>Dewey CN.
                                  Aligning multiple whole genomes with Mercator and MAVID. 
                                  Methods in Molecular Biology. 395:221-36, 2007.</td>
                            </tr>
                          
                        
                      </table>
                    </td>
                  </tr>
                
                <tr><td colspan="2">&nbsp;</td></tr>
              </table>
              
            
              <table border="0" cellpadding="2" cellspacing="0" width="100%">
                
                  <tr><td colspan="2"><hr></td></tr>
                
  
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Method</b></td>
                    <td>RepeatMasker</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Input</b></td>
                    <td>Fasta file of nucleic acid sequences and RepBase</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Output</b></td>
                    <td>Fasta file of masked nucleic acid sequences</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Details</b></td>
                    <td>RepeatMasker uses cross-match and a RepBase repeat library to screen DNA sequences for interspersed repeats and low complexity DNA sequences and outputs a modified version of the query sequence in which all the repeats have been masked.</td>
                  </tr>
                
  
                
                
                  
                  <tr>
                    <td valign="top" width="10%"><b>Tools</b></td>
                    <td>
                      <table border="0" cellpadding="2" cellspacing="0" width="100%">
                        
                        
                          
                            <tr>
                              <td valign="top" width="10%"><i>Tool</i></td>
                              <td>RepeatMasker</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Version</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Parameters</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>URL</i></td>
                              <td>http://www.repeatmasker.org</td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>PubMed IDs</i></td>
                              <td></td>
                            </tr>
                          
                            <tr>
                              <td valign="top" width="10%"><i>Credits</i></td>
                              <td>Smit, AFA, Hubley, R & Green, P. 
                                  RepeatMasker Open-3.0.
                                  1996-2004</td>
                            </tr>
                          
                        
                      </table>
                    </td>
                  </tr>
                
                <tr><td colspan="2">&nbsp;</td></tr>
              </table>
              
            
              <table border="0" cellpadding="2" cellspacing="0" width="100%">
                
                  <tr><td colspan="2"><hr></td></tr>
                
  
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Method</b></td>
                    <td>Protein pI (isoelectric point) calculation</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Input</b></td>
                    <td>Protein sequences</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Output</b></td>
                    <td>pI (isoelectric point) values</td>
                  </tr>
                
                  <tr class="rowLight">
                    <td valign="top" width="10%"><b>Details</b></td>
                    <td>The pKa values used to calculate pI are those used in the EMBOSS package; the pI is calculated to only two decimal points.</td>
                  </tr>
                
  
                
                
                  
                  <tr>
                    <td valign="top" width="10%"><b>Tools</b></td>
                    <td>
                      <table border="0" cellpadding="2" cellspacing="0" width="100%">
                        
                        
                      </table>
                    </td>
                  </tr>
                
                <tr><td colspan="2">&nbsp;</td></tr>
              </table>
              
            
