---
layout: post
title: AmoebaDB 49 Released
category: [AmoebaDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h3>We are pleased to announce the release of {{project}} 49.</h3>

<p>The URL <a href="https://{{project}}.org">https://{{project}}.org</a> directs you to the newest version of our interface.  The site contains all previous data plus some new tools and functions in an updated and streamlined interface that empasizes easy access to help information.   Navigate to <a href="https://legacy.{{project}}.org">https://legacy.{{project}}.org</a> for the previous version of the interface.  The legacy interface will remain active for at least one release.</p>

<section>
  <h3>Release 49 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on November 12, 2020 at 10am EDT for our Release 49 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 49.  
    <a href="https://attendee.gotowebinar.com/register/5693189155695816719" target="_blank">Register here</a>  
  </div>
</section>



<section>
  <h3>New data in {{project}} 49</h3>
  <div style="margin-left: 3em;">
    <h4>EPI Pipeline</h4>
    {{project}} 49 represents the first release that employs the EBI pipeline for analyzing genomes and RNA-Seq data. (<a href="/a/app/static-content/methods.html">{{project}} Analysis Methods</a>) All genomes were run through the EBI pipeline, updating all analyses such as InterproScan and repeat identification.  While much of the EBI pipeline is the same as our previous in-house pipeline, these differences are noted:
    <ul>
      <li>Expression values calculated from RNA-Seq data are now presented as TPM (Transcripts Per Kilobase Million) instead of FPKM (Fragments Per Kilobase Million) or RPKM (Reads Per Kilobase Million). <a href="https://rna-seqblog.com/rpkm-fpkm-and-tpm-clearly-explained/" target="_blank">Learn more</a></li>
      <li>Signal peptide predictions are now made with <a href="http://www.cbs.dtu.dk/services/SignalP/" target="_blank">SignalP-5.0</a>, updated from <a href="http://www.cbs.dtu.dk/services/SignalP-3.0/" target="_blank">SignalP-3.0</a>.</li>
    </ul>
    <h4>Genome sequence and annotation:</h4>
    <ul>
      <li><a href="/a/app/record/dataset/DS_e57644bc33"><i>Naegleria fowleri</i> strain ATCC 30894</a> &ndash; The genome of <i>N. fowleri</i> ATCC 30894 was sequenced using Oxford Nanopore Technology (ONT).  <a href="https://pubmed.ncbi.nlm.nih.gov/31690847/" target="_blank">Liechti et al. 2019.</a></li>
      <li><a href="/a/app/record/dataset/DS_f94d0c2db0"><i>Naegleria gruberi</i> strain NEG-M</a> &ndash; Genomic DNA from an axenic culture of <i>Naegleria gruberi</i> strain NEG-M (ATCC 30224) was sequenced and annotated using the JGI pipeline combining EST, homology, and ab initio data. <a href="http://www.ncbi.nlm.nih.gov/pubmed/20211133" target="_blank">Fritz-Laylin et al. 2010.</a></li>
    </ul>

    <h4>Unannotated genome sequence</h4>
    <ul>
      <li><a href="/a/app/record/dataset/DS_2af8c73d86"><i>Naegleria lovaniensis</i> strain ATCC 30569</a> &ndash; <i>Naegleria lovaniensis</i> is the closest non-pathogenic relative to <i>Naegleria fowleri</i>. The genome was sequenced using long read sequencing technology. <a href="http://www.ncbi.nlm.nih.gov/pubmed/30185166" target="_blank">Liechti et al. 2018.</a>
      </li>
    </ul> 

    <h4>Whole genome sequencing of isolates for SNP analysis</h4>
    <ul>
      <li><a href="</a/app/record/dataset/DS_98b6bcc3d8">Genome sequencing of clinical strains of <i>Entamoeba histolytica</i></a> &ndash; Genome sequencing of <i>E. histolytica</i> strains from 3 different clinical populations: asymptomatic, diarrhea/dysenteric strains and liver abscess strains. The data were used to call single nucleotide polymorphisms.</li>
    </ul>
  </div>   
</section>

<section>
  <h3>New features in {{project}} 49</h3>
  <div style="margin-left: 3em;">
   <ul>
    <li>When revisiting a search page, your previous parameter values are automatically loaded.  This is especially useful for the BLAST search which now retains the input sequence.  </li>
    <li>The cytoscape drawing on metabolic pathway record pages now contains:
      <ul>
      <li>a new search function for finding nodes within the drawing.  After running a search, the drawing centers on the nodes which match the search term and highlights them with a bold green border.</li>
      <li>a "Reset Layout" button that will clear all node paintings as well as reset the node positions and zoom levels.</li>
      <li>a checkbox for toggling your mouse function between zooming the drawing or scrolling the page.</li>
      </ul>
    </li>
    <li>Data set and Organism record pages now contain direct links to our Downloads section where the option is available to download genome-scale sequence and annotation files.  </li>
   </ul>
  </div>
</section>

<section>
  <aside>
   <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary.  Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits. When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it. When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>
