---
layout: post
title: ToxoDB 41 Released
category: [EuPathDB,VEuPathDB,ToxoDB]
---

<p><strong>We are pleased to announce the release of ToxoDB 41</strong></p>
      
      <section>
      <h3><b>New data in ToxoDB 41</b></h3>
      <div style="margin-left: 3em;">
      <h4>Transcriptomics - Microarray</h4>
          <ul>
             <li>
               <a href="http://toxodb.org/toxo/app/record/dataset/DS_1bdad09720">PMRT1 KO, WT and KOCM expression profiles</a> &mdash; 
               Expression profiles of arginine methyltransferase 1 (PMRT1) knockout, complement and wild type <i>T. gondii</i> RH-hxgprt parasites determined by Affymetrix microarray (Toxo Gene chip) 
               <a href="http://www.ncbi.nlm.nih.gov/pubmed/26838719" target="_blank">El Bissati et al.</a>
               <br>
            </li>
          </ul>   
      <h4>Whole genome sequencing of isolate for SNP analysis</h4>
        <ul>
          <li>
            <a href="http://toxodb.org/toxo/app/record/dataset/DS_e19cdf0a8c">Toxoplasma gondii strain CZ clone H3 aligned genome sequence</a> &mdash; 
            Genomic DNA sequencing from Toxoplasma gondii strain CZ clone H3.  Data were used to call SNPs and 
            determine CNV. Data were provided by the Heil Lab, University of Zurich.
          </li>
        </ul>
      </div>
      </section>
      
      <section>
      <h3><b>New features in ToxoDB 41</b></h3>
      <h4>Introducing Clustal Omega </h4><br>
      <div style="margin-left: 3em;">
      EuPathDB sites are now using Clustal Omega for gene page protein alignments in the 'Orthologs and Paralogs within EuPathDB' table and the 
      'Retrieve multiple sequence alignment or multi-FASTA' tool in the 'Orthology and synteny' section of gene pages. The 'Isolate Alignments in this Gene Region' tool 
      still uses CulstalW.  
       </div> 
      <h4>New Gene page tools </h4>
         <ul>
           <li>Interpro  &mdash; <br>
              The 'Protein features and properties' section now includes an option to submit the protein sequence for 
              sequence analysis and classification via <a href="https://www.ebi.ac.uk/interpro/" target="_blank">InterPro</a>. 
          </li>
          <li>GPI anchor prediction  &mdash; <br>
              The 'Protein features and properties' section now includes an option to submit the protein sequence for 
              GPI modification site prediction via the <a href="http://mendel.imp.ac.at/sat/gpi/gpi_server.html" target="_blank">GPI Lipid Anchor Project</a>. 
          </li>
         </ul>
      </section>
      
      <section>
         <aside>
           <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary.
           Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits.
           When search parameters change with a new release, we invalidate (Ã˜) the search and ask you to rerun it.
           When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
         </aside>
      </section>
