---
layout: post
title: ToxoDB 51 Released
category: [ToxoDB,VEuPathDB]
---
{% assign project = page.category[0] %}


<h4>We are pleased to announce the release of {{project}} 51.</h4>  

<section>
  <h3>Release 51 Webinar</h3>
  <div style="margin-left: 3em;">
    Join us on April 1, 2021 at 10am EDT for our Release 51 Highlights webinar where we will demonstrate and discuss new data and features in VEuPathDB 51.  
    <a href="https://attendee.gotowebinar.com/register/8137280361730606607" target="_blank">Register here</a>  
  </div>
</section>


<section>
  <h3>New data in {{project}} 51</h3>
  <div style="margin-left: 3em;">
    
    <h4>Transcriptomics</h4>
    <ul>
      <li>
        <a href="/a/app/record/dataset/TODO"><i>Toxoplasma gondii </i> Parasite transcriptome during infection in mouse brain.  </a> &ndash; Transcriptome of <i>Toxoplasma gondii</i> from injected mouse neurons. <a href="http://www.ncbi.nlm.nih.gov/pubmed/32878927" target="_blank">Merritt et al. 2020</a>. 
      </li>
    </ul>
    <br>
    <h4>Proteomics</h4>
    <ul>
      <li>
        <a href="/a/app/record/dataset/TODO">Secretome of T gondii tachyzoites (RH) isolated from infected mice and the proteome of extracellular vesicles (exosomes and ectosomes)</a> &ndash; Proteomic analysis of the secretome of <i>Toxoplasma gondii</i> tachyzoites (RH) and the proteomic profile of the vesicles released. For the secretome, parasites were isolated from infected mice, purified, and secretions were collected. The complete secretion was analyzed by MS generating the secretome. In addition, a proteomic characterization was achieved in exosomes, ectosomes and the supernatant obtained by differential centrifugation of the secretory products.  <a href="http://www.ncbi.nlm.nih.gov/pubmed/31434009" target="_blank"> Ramírez-Flores et al. 2019 </a>.
      </li>
    </ul>
    <br>
  </div>
</section>

<section>
  <h3>New features in {{project}} 51</h3>
  <div style="margin-left: 3em;">
   <ul>
    <li><a href="https://orthomcl.org/orthomcl/app/" target="_blank">OrthoMCL DB</a> now shares web architecture with other VEuPathDB sites. The updated and streamlined interface emphasizes easy access to tools and help information.  </li>
      <ul>
        <li>OrthoMCL now includes orthology information for <a href="https://orthomcl.org/orthomcl/app/release-summary" target="_blank">564 species</a>, including all of the VEuPathDB organisms with annotated genomes. </li>
        <li>The redesigned home page includes a new site-wide search for easily finding groups, proteins or documents. A categorized list of Searches appears on the left of the home page and the center section is devoted to tutorials for getting you started with OrthoMCL features.</li>
        <li>Group and protein record pages are redesigned and include a contents section for navigating to the data you are interested in. </li>
        <li>For more information see the OrthoMCL DB <a href="https://orthomcl.org/orthomcl/app/static-content/OrthoMCL/about.html" target="_blank">About</a> page, the <a href="https://orthomcl.org/orthomcl/app/static-content/OrthoMCL/faq.html" target="_blank">FAQ</a> page or complete this <a href="{{'/documents/OrthoMCL_tutorial1.pdf' | absolute_url}}" target="_blank">tutorial</a>. </li>
      </ul>
    <li>Community annotation via <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006790" target="_blank">Apollo</a> is now available in {{project}}. Apollo is an open source software enabling users to inspect, refine and add gene models to the current genome annotations. Community annotation offers a value-added perspective to  curation, especially for genomes without actively funded curation efforts. To contribute structural gene model changes, users inspect data as JBrowse tracks, create new or modified gene models, and save them in Apollo. To access Apollo, navigate to the Tools menu in the {{project}} header.  To learn more, see this <a href="https://youtu.be/RMXZDBMh8Fo" target="_blank">webinar recording describing manual gene annotation in Apollo</a>.  </li>
    </ul>
  </div>
</section>

<section>
  <aside>
   <br><br>Note: Our bimonthly database releases incorporate new data and correct errors in old data when necessary.  Changes in annotation and new experimental data may slightly alter your search results by increasing or decreasing the number of hits. When search parameters change with a new release, we invalidate (Ø) the search and ask you to rerun it. When IDs are updated or removed, we map the old IDs to the new ones, remove the old IDs from your Basket, and leave your Favorites page alone.
  </aside>
</section>

